{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Antibody Specification"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Create Training dataset"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Step 1**  \n",
    "Get a list containing PMCID and PMID from ```pmcids-pmids.txt```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('resources/pmcids-pmids.txt', 'r') as file:\n",
    "    lines = file.readlines()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Each lines seperate between PMID and PMCID"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "list_of_pmids_and_pmcids = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "for line in lines:\n",
    "    sep_line = line.split('\\t')\n",
    "    pmid = sep_line[0]\n",
    "    pmcid = sep_line[1].replace('\\n', '')\n",
    "    \n",
    "    list_of_pmids_and_pmcids.append({ 'pmid': pmid, 'pmcid': pmcid })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'pmcid': 'PMC2222243', 'pmid': '18036257'},\n",
       " {'pmcid': 'PMC2936283', 'pmid': '20723247'},\n",
       " {'pmcid': 'PMC3137399', 'pmid': '21654676'},\n",
       " {'pmcid': 'PMC3247256', 'pmid': '22216273'},\n",
       " {'pmcid': 'PMC3565217', 'pmid': '23390418'},\n",
       " {'pmcid': 'PMC3585132', 'pmid': '23468629'},\n",
       " {'pmcid': 'PMC3723660', 'pmid': '23936125'},\n",
       " {'pmcid': 'PMC3794087', 'pmid': '24124410'},\n",
       " {'pmcid': 'PMC3821019', 'pmid': '24251095'},\n",
       " {'pmcid': 'PMC3827131', 'pmid': '24265795'},\n",
       " {'pmcid': 'PMC3869677', 'pmid': '24381807'},\n",
       " {'pmcid': 'PMC3877098', 'pmid': '24391855'},\n",
       " {'pmcid': 'PMC3943208', 'pmid': '24634647'},\n",
       " {'pmcid': 'PMC3956895', 'pmid': '24637809'},\n",
       " {'pmcid': 'PMC3981180', 'pmid': '23714779'},\n",
       " {'pmcid': 'PMC4023601', 'pmid': '24735464'},\n",
       " {'pmcid': 'PMC4059454', 'pmid': '24906209'},\n",
       " {'pmcid': 'PMC4097366', 'pmid': '25075304'},\n",
       " {'pmcid': 'PMC4114201', 'pmid': '25120436'},\n",
       " {'pmcid': 'PMC4124705', 'pmid': '25147503'},\n",
       " {'pmcid': 'PMC4141641', 'pmid': '25177484'},\n",
       " {'pmcid': 'PMC4142709', 'pmid': '25202271'},\n",
       " {'pmcid': 'PMC4156431', 'pmid': '25192442'},\n",
       " {'pmcid': 'PMC4157608', 'pmid': '25249943'},\n",
       " {'pmcid': 'PMC4162472', 'pmid': '25309320'},\n",
       " {'pmcid': 'PMC4172589', 'pmid': '25247590'},\n",
       " {'pmcid': 'PMC4176425', 'pmid': '25285204'},\n",
       " {'pmcid': 'PMC4184377', 'pmid': '25309736'},\n",
       " {'pmcid': 'PMC4199601', 'pmid': '25330147'},\n",
       " {'pmcid': 'PMC4201086', 'pmid': '25368549'},\n",
       " {'pmcid': 'PMC4207903', 'pmid': '25332063'},\n",
       " {'pmcid': 'PMC4214784', 'pmid': '25165136'},\n",
       " {'pmcid': 'PMC4216124', 'pmid': '25360677'},\n",
       " {'pmcid': 'PMC4224379', 'pmid': '25380249'},\n",
       " {'pmcid': 'PMC4226736', 'pmid': '25365995'},\n",
       " {'pmcid': 'PMC4227046', 'pmid': '25313866'},\n",
       " {'pmcid': 'PMC4240249', 'pmid': '25469233'},\n",
       " {'pmcid': 'PMC4247200', 'pmid': '25416145'},\n",
       " {'pmcid': 'PMC4256450', 'pmid': '25474204'},\n",
       " {'pmcid': 'PMC4262395', 'pmid': '25493560'},\n",
       " {'pmcid': 'PMC4264638', 'pmid': '25520781'},\n",
       " {'pmcid': 'PMC4265792', 'pmid': '24782245'},\n",
       " {'pmcid': 'PMC4270231', 'pmid': '25194025'},\n",
       " {'pmcid': 'PMC4278744', 'pmid': '25546454'},\n",
       " {'pmcid': 'PMC4302597', 'pmid': '25552352'},\n",
       " {'pmcid': 'PMC4304311', 'pmid': '25653835'},\n",
       " {'pmcid': 'PMC4313697', 'pmid': '25698922'},\n",
       " {'pmcid': 'PMC4321040', 'pmid': '24700109'},\n",
       " {'pmcid': 'PMC4321392', 'pmid': '25722947'},\n",
       " {'pmcid': 'PMC4322648', 'pmid': '25644616'},\n",
       " {'pmcid': 'PMC4328392', 'pmid': '25420775'},\n",
       " {'pmcid': 'PMC4335040', 'pmid': '25695429'},\n",
       " {'pmcid': 'PMC4350939', 'pmid': '25742620'},\n",
       " {'pmcid': 'PMC4354442', 'pmid': '25400217'},\n",
       " {'pmcid': 'PMC4355975', 'pmid': '25880931'},\n",
       " {'pmcid': 'PMC4357301', 'pmid': '25814946'},\n",
       " {'pmcid': 'PMC4359021', 'pmid': '25476605'},\n",
       " {'pmcid': 'PMC4365039', 'pmid': '25785851'},\n",
       " {'pmcid': 'PMC4368520', 'pmid': '25793288'},\n",
       " {'pmcid': 'PMC4374371', 'pmid': '25887698'},\n",
       " {'pmcid': 'PMC4380931', 'pmid': '25814555'},\n",
       " {'pmcid': 'PMC4392586', 'pmid': '25914629'},\n",
       " {'pmcid': 'PMC4393184', 'pmid': '25677543'},\n",
       " {'pmcid': 'PMC4405263', 'pmid': '25898113'},\n",
       " {'pmcid': 'PMC4406474', 'pmid': '25902315'},\n",
       " {'pmcid': 'PMC4407839', 'pmid': '25380328'},\n",
       " {'pmcid': 'PMC4412406', 'pmid': '25919390'},\n",
       " {'pmcid': 'PMC4414138', 'pmid': '25650666'},\n",
       " {'pmcid': 'PMC4414452', 'pmid': '25923671'},\n",
       " {'pmcid': 'PMC4418305', 'pmid': '25556858'},\n",
       " {'pmcid': 'PMC4425615', 'pmid': '25381005'},\n",
       " {'pmcid': 'PMC4431865', 'pmid': '25973881'},\n",
       " {'pmcid': 'PMC4458151', 'pmid': '25688551'},\n",
       " {'pmcid': 'PMC4467669', 'pmid': '26077244'},\n",
       " {'pmcid': 'PMC4467681', 'pmid': '26047946'},\n",
       " {'pmcid': 'PMC4468165', 'pmid': '26075743'},\n",
       " {'pmcid': 'PMC4472396', 'pmid': '26084360'},\n",
       " {'pmcid': 'PMC4472752', 'pmid': '26086440'},\n",
       " {'pmcid': 'PMC4477153', 'pmid': '26157433'},\n",
       " {'pmcid': 'PMC4479795', 'pmid': '26161083'},\n",
       " {'pmcid': 'PMC4482680', 'pmid': '26114750'},\n",
       " {'pmcid': 'PMC4488723', 'pmid': '26110582'},\n",
       " {'pmcid': 'PMC4489868', 'pmid': '26135580'},\n",
       " {'pmcid': 'PMC4499192', 'pmid': '26162823'},\n",
       " {'pmcid': 'PMC4501753', 'pmid': '26172113'},\n",
       " {'pmcid': 'PMC4502912', 'pmid': '26174015'},\n",
       " {'pmcid': 'PMC4503569', 'pmid': '26176237'},\n",
       " {'pmcid': 'PMC4505884', 'pmid': '26186005'},\n",
       " {'pmcid': 'PMC4514672', 'pmid': '26207633'},\n",
       " {'pmcid': 'PMC4521930', 'pmid': '26230935'},\n",
       " {'pmcid': 'PMC4523177', 'pmid': '26237312'},\n",
       " {'pmcid': 'PMC4526663', 'pmid': '26244549'},\n",
       " {'pmcid': 'PMC4529706', 'pmid': '26253104'},\n",
       " {'pmcid': 'PMC4530481', 'pmid': '26259688'},\n",
       " {'pmcid': 'PMC4539228', 'pmid': '26280348'},\n",
       " {'pmcid': 'PMC4544385', 'pmid': '26339475'},\n",
       " {'pmcid': 'PMC4550358', 'pmid': '26308623'},\n",
       " {'pmcid': 'PMC4557910', 'pmid': '26331617'},\n",
       " {'pmcid': 'PMC4559018', 'pmid': '26357588'},\n",
       " {'pmcid': 'PMC4559019', 'pmid': '26357589'},\n",
       " {'pmcid': 'PMC4561338', 'pmid': '26441552'},\n",
       " {'pmcid': 'PMC4566282', 'pmid': '26401264'},\n",
       " {'pmcid': 'PMC4569565', 'pmid': '26366556'},\n",
       " {'pmcid': 'PMC4578941', 'pmid': '26393351'},\n",
       " {'pmcid': 'PMC4581247', 'pmid': '26258776'},\n",
       " {'pmcid': 'PMC4582632', 'pmid': '26408173'},\n",
       " {'pmcid': 'PMC4583291', 'pmid': '26406232'},\n",
       " {'pmcid': 'PMC4605787', 'pmid': '26465754'},\n",
       " {'pmcid': 'PMC4611065', 'pmid': '26539084'},\n",
       " {'pmcid': 'PMC4621747', 'pmid': '26503721'},\n",
       " {'pmcid': 'PMC4622696', 'pmid': '25517306'},\n",
       " {'pmcid': 'PMC4626034', 'pmid': '26512722'},\n",
       " {'pmcid': 'PMC4631488', 'pmid': '26529324'},\n",
       " {'pmcid': 'PMC4634965', 'pmid': '26539718'},\n",
       " {'pmcid': 'PMC4643007', 'pmid': '26561856'},\n",
       " {'pmcid': 'PMC4643987', 'pmid': '26566032'},\n",
       " {'pmcid': 'PMC4644300', 'pmid': '26566974'},\n",
       " {'pmcid': 'PMC4644319', 'pmid': '26568816'},\n",
       " {'pmcid': 'PMC4647582', 'pmid': '26572741'},\n",
       " {'pmcid': 'PMC4648211', 'pmid': '26594330'},\n",
       " {'pmcid': 'PMC4650558', 'pmid': '25880092'},\n",
       " {'pmcid': 'PMC4660684', 'pmid': '26608466'},\n",
       " {'pmcid': 'PMC4665759', 'pmid': '26620712'},\n",
       " {'pmcid': 'PMC4667627', 'pmid': '26665000'},\n",
       " {'pmcid': 'PMC4667876', 'pmid': '26630286'},\n",
       " {'pmcid': 'PMC4669805', 'pmid': '25675298'},\n",
       " {'pmcid': 'PMC4673192', 'pmid': '26087190'},\n",
       " {'pmcid': 'PMC4681077', 'pmid': '26674138'},\n",
       " {'pmcid': 'PMC4681705', 'pmid': '26659064'},\n",
       " {'pmcid': 'PMC4684178', 'pmid': '26599696'},\n",
       " {'pmcid': 'PMC4688937', 'pmid': '26702299'},\n",
       " {'pmcid': 'PMC4692435', 'pmid': '26709919'},\n",
       " {'pmcid': 'PMC4709145', 'pmid': '26751788'},\n",
       " {'pmcid': 'PMC4709451', 'pmid': '26834559'},\n",
       " {'pmcid': 'PMC4711940', 'pmid': '26760782'},\n",
       " {'pmcid': 'PMC4712188', 'pmid': '26626482'},\n",
       " {'pmcid': 'PMC4714636', 'pmid': '26807334'},\n",
       " {'pmcid': 'PMC4715492', 'pmid': '26375402'},\n",
       " {'pmcid': 'PMC4737253', 'pmid': '25975775'},\n",
       " {'pmcid': 'PMC4737260', 'pmid': '25845831'},\n",
       " {'pmcid': 'PMC4737294', 'pmid': '26660233'},\n",
       " {'pmcid': 'PMC4737730', 'pmid': '26806606'},\n",
       " {'pmcid': 'PMC4742982', 'pmid': '26837579'},\n",
       " {'pmcid': 'PMC4754497', 'pmid': '26925305'},\n",
       " {'pmcid': 'PMC4754697', 'pmid': '26909026'},\n",
       " {'pmcid': 'PMC4758405', 'pmid': '26053631'},\n",
       " {'pmcid': 'PMC4762545', 'pmid': '26901525'},\n",
       " {'pmcid': 'PMC4765774', 'pmid': '26900850'},\n",
       " {'pmcid': 'PMC4767815', 'pmid': '26914463'},\n",
       " {'pmcid': 'PMC4768776', 'pmid': '26498202'},\n",
       " {'pmcid': 'PMC4769598', 'pmid': '27014645'},\n",
       " {'pmcid': 'PMC4791394', 'pmid': '27013971'},\n",
       " {'pmcid': 'PMC4797683', 'pmid': '27047346'},\n",
       " {'pmcid': 'PMC4801847', 'pmid': '27047336'},\n",
       " {'pmcid': 'PMC4806294', 'pmid': '27034849'},\n",
       " {'pmcid': 'PMC4829250', 'pmid': '27071013'},\n",
       " {'pmcid': 'PMC4831197', 'pmid': '27075038'},\n",
       " {'pmcid': 'PMC4834942', 'pmid': '27110440'},\n",
       " {'pmcid': 'PMC4835572', 'pmid': '26880068'},\n",
       " {'pmcid': 'PMC4839613', 'pmid': '27101288'},\n",
       " {'pmcid': 'PMC4839714', 'pmid': '27100625'},\n",
       " {'pmcid': 'PMC4844616', 'pmid': '27199660'},\n",
       " {'pmcid': 'PMC4844952', 'pmid': '27112667'},\n",
       " {'pmcid': 'PMC4847956', 'pmid': '27065364'},\n",
       " {'pmcid': 'PMC4848316', 'pmid': '27199657'},\n",
       " {'pmcid': 'PMC4848777', 'pmid': '27125505'},\n",
       " {'pmcid': 'PMC4850861', 'pmid': '27158443'},\n",
       " {'pmcid': 'PMC4858671', 'pmid': '27150735'},\n",
       " {'pmcid': 'PMC4860965', 'pmid': '27462437'},\n",
       " {'pmcid': 'PMC4862212', 'pmid': '27160145'},\n",
       " {'pmcid': 'PMC4866687', 'pmid': '27177152'},\n",
       " {'pmcid': 'PMC4869021', 'pmid': '27185464'},\n",
       " {'pmcid': 'PMC4873194', 'pmid': '27195791'},\n",
       " {'pmcid': 'PMC4874359', 'pmid': '27069253'},\n",
       " {'pmcid': 'PMC4880021', 'pmid': '26551059'},\n",
       " {'pmcid': 'PMC4881167', 'pmid': '27230275'},\n",
       " {'pmcid': 'PMC4884737', 'pmid': '27303266'},\n",
       " {'pmcid': 'PMC4887420', 'pmid': '26619807'},\n",
       " {'pmcid': 'PMC4892606', 'pmid': '27257960'},\n",
       " {'pmcid': 'PMC4893163', 'pmid': '27161524'},\n",
       " {'pmcid': 'PMC4893164', 'pmid': '27133465'},\n",
       " {'pmcid': 'PMC4893942', 'pmid': '27335633'},\n",
       " {'pmcid': 'PMC4893944', 'pmid': '27335634'},\n",
       " {'pmcid': 'PMC4893971', 'pmid': '27335637'},\n",
       " {'pmcid': 'PMC4893991', 'pmid': '27335636'},\n",
       " {'pmcid': 'PMC4899448', 'pmid': '27375437'},\n",
       " {'pmcid': 'PMC4899679', 'pmid': '27294196'},\n",
       " {'pmcid': 'PMC4900556', 'pmid': '27280719'},\n",
       " {'pmcid': 'PMC4912163', 'pmid': '27315061'},\n",
       " {'pmcid': 'PMC4913281', 'pmid': '27321892'},\n",
       " {'pmcid': 'PMC4913955', 'pmid': '27322388'},\n",
       " {'pmcid': 'PMC4919337', 'pmid': '27445711'},\n",
       " {'pmcid': 'PMC4919854', 'pmid': '27342975'},\n",
       " {'pmcid': 'PMC4919869', 'pmid': '27338237'},\n",
       " {'pmcid': 'PMC4920669', 'pmid': '27163546'},\n",
       " {'pmcid': 'PMC4925239', 'pmid': '27214567'},\n",
       " {'pmcid': 'PMC4929735', 'pmid': '27364523'},\n",
       " {'pmcid': 'PMC4935463', 'pmid': '27253063'},\n",
       " {'pmcid': 'PMC4942897', 'pmid': '27405506'},\n",
       " {'pmcid': 'PMC4943231', 'pmid': '27468278'},\n",
       " {'pmcid': 'PMC4943598', 'pmid': '27410964'},\n",
       " {'pmcid': 'PMC4945633', 'pmid': '27471450'},\n",
       " {'pmcid': 'PMC4947983', 'pmid': '27482535'},\n",
       " {'pmcid': 'PMC4949683', 'pmid': '26779909'},\n",
       " {'pmcid': 'PMC4951194', 'pmid': '27434666'},\n",
       " {'pmcid': 'PMC4951196', 'pmid': '27336722'},\n",
       " {'pmcid': 'PMC4951197', 'pmid': '27431763'},\n",
       " {'pmcid': 'PMC4951200', 'pmid': '27336723'},\n",
       " {'pmcid': 'PMC4956235', 'pmid': '27442134'},\n",
       " {'pmcid': 'PMC4956249', 'pmid': '27442505'},\n",
       " {'pmcid': 'PMC4956664', 'pmid': '27499734'},\n",
       " {'pmcid': 'PMC4959878', 'pmid': '27383270'},\n",
       " {'pmcid': 'PMC4960424', 'pmid': '27507971'},\n",
       " {'pmcid': 'PMC4962981', 'pmid': '27462707'},\n",
       " {'pmcid': 'PMC4966956', 'pmid': '27472061'},\n",
       " {'pmcid': 'PMC4969039', 'pmid': '27478985'},\n",
       " {'pmcid': 'PMC4970949', 'pmid': '26972791'},\n",
       " {'pmcid': 'PMC4972537', 'pmid': '27383052'},\n",
       " {'pmcid': 'PMC4972548', 'pmid': '27333032'},\n",
       " {'pmcid': 'PMC4972567', 'pmid': '27372014'},\n",
       " {'pmcid': 'PMC4972960', 'pmid': '27488131'},\n",
       " {'pmcid': 'PMC4973988', 'pmid': '27489949'},\n",
       " {'pmcid': 'PMC4975574', 'pmid': '27434667'},\n",
       " {'pmcid': 'PMC4977155', 'pmid': '27501441'},\n",
       " {'pmcid': 'PMC4977824', 'pmid': '27506200'},\n",
       " {'pmcid': 'PMC4978478', 'pmid': '27504635'},\n",
       " {'pmcid': 'PMC4982087', 'pmid': '26587966'},\n",
       " {'pmcid': 'PMC4987138', 'pmid': '27501334'},\n",
       " {'pmcid': 'PMC4987413', 'pmid': '27570824'},\n",
       " {'pmcid': 'PMC4990423', 'pmid': '27537197'},\n",
       " {'pmcid': 'PMC4991008', 'pmid': '27538435'},\n",
       " {'pmcid': 'PMC4993585', 'pmid': '27472897'},\n",
       " {'pmcid': 'PMC4993588', 'pmid': '27494275'},\n",
       " {'pmcid': 'PMC4995096', 'pmid': '27549338'},\n",
       " {'pmcid': 'PMC4996649', 'pmid': '27474796'},\n",
       " {'pmcid': 'PMC4996650', 'pmid': '27557444'},\n",
       " {'pmcid': 'PMC4996653', 'pmid': '27549340'},\n",
       " {'pmcid': 'PMC4999312', 'pmid': '27504807'},\n",
       " {'pmcid': 'PMC4999423', 'pmid': '27574484'},\n",
       " {'pmcid': 'PMC5000811', 'pmid': '27570829'},\n",
       " {'pmcid': 'PMC5002269', 'pmid': '27523608'},\n",
       " {'pmcid': 'PMC5004264', 'pmid': '27576487'},\n",
       " {'pmcid': 'PMC5005036', 'pmid': '27525485'},\n",
       " {'pmcid': 'PMC5006588', 'pmid': '27342872'},\n",
       " {'pmcid': 'PMC5006644', 'pmid': '27381477'},\n",
       " {'pmcid': 'PMC5006695', 'pmid': '27453322'},\n",
       " {'pmcid': 'PMC5006801', 'pmid': '27423190'},\n",
       " {'pmcid': 'PMC5008145', 'pmid': '25963823'},\n",
       " {'pmcid': 'PMC5011436', 'pmid': '27474797'},\n",
       " {'pmcid': 'PMC5011687', 'pmid': '27622213'},\n",
       " {'pmcid': 'PMC5012858', 'pmid': '27525480'},\n",
       " {'pmcid': 'PMC5012864', 'pmid': '27472900'},\n",
       " {'pmcid': 'PMC5014547', 'pmid': '27552050'},\n",
       " {'pmcid': 'PMC5014548', 'pmid': '27525482'},\n",
       " {'pmcid': 'PMC5015242', 'pmid': '27609221'},\n",
       " {'pmcid': 'PMC5016092', 'pmid': '27554484'},\n",
       " {'pmcid': 'PMC5017579', 'pmid': '27611432'},\n",
       " {'pmcid': 'PMC5017655', 'pmid': '27611690'},\n",
       " {'pmcid': 'PMC5018252', 'pmid': '27610576'},\n",
       " {'pmcid': 'PMC5019149', 'pmid': '26470600'},\n",
       " {'pmcid': 'PMC5019378', 'pmid': '27617441'},\n",
       " {'pmcid': 'PMC5021170', 'pmid': '27017927'},\n",
       " {'pmcid': 'PMC5021521', 'pmid': '27623148'},\n",
       " {'pmcid': 'PMC5023747', 'pmid': '27240603'},\n",
       " {'pmcid': 'PMC5025287', 'pmid': '27542193'},\n",
       " {'pmcid': 'PMC5025782', 'pmid': '27582081'},\n",
       " {'pmcid': 'PMC5026468', 'pmid': '27637097'},\n",
       " {'pmcid': 'PMC5026472', 'pmid': '27635635'},\n",
       " {'pmcid': 'PMC5026473', 'pmid': '27552055'},\n",
       " {'pmcid': 'PMC5027985', 'pmid': '27638600'},\n",
       " {'pmcid': 'PMC5028188', 'pmid': '27642710'},\n",
       " {'pmcid': 'PMC5028190', 'pmid': '27642785'},\n",
       " {'pmcid': 'PMC5030084', 'pmid': '27644592'},\n",
       " {'pmcid': 'PMC5032171', 'pmid': '27653972'},\n",
       " {'pmcid': 'PMC5032890', 'pmid': '27679811'},\n",
       " {'pmcid': 'PMC5033604', 'pmid': '27657167'},\n",
       " {'pmcid': 'PMC5033606', 'pmid': '27657169'},\n",
       " {'pmcid': 'PMC5035145', 'pmid': '27564575'},\n",
       " {'pmcid': 'PMC5036882', 'pmid': '27669412'},\n",
       " {'pmcid': 'PMC5036963', 'pmid': '27669410'},\n",
       " {'pmcid': 'PMC5036964', 'pmid': '27669409'},\n",
       " {'pmcid': 'PMC5036965', 'pmid': '27668431'},\n",
       " {'pmcid': 'PMC5037319', 'pmid': '27641504'},\n",
       " {'pmcid': 'PMC5039111', 'pmid': '27662090'},\n",
       " {'pmcid': 'PMC5039579', 'pmid': '27618111'},\n",
       " {'pmcid': 'PMC5040589', 'pmid': '27543448'},\n",
       " {'pmcid': 'PMC5040590', 'pmid': '27623146'},\n",
       " {'pmcid': 'PMC5045296', 'pmid': '27690357'},\n",
       " {'pmcid': 'PMC5047746', 'pmid': '27692066'},\n",
       " {'pmcid': 'PMC5047748', 'pmid': '27692068'},\n",
       " {'pmcid': 'PMC5049482', 'pmid': '26179777'},\n",
       " {'pmcid': 'PMC5049656', 'pmid': '26346430'},\n",
       " {'pmcid': 'PMC5049759', 'pmid': '27568223'},\n",
       " {'pmcid': 'PMC5049993', 'pmid': '27569656'},\n",
       " {'pmcid': 'PMC5050000', 'pmid': '27649637'},\n",
       " {'pmcid': 'PMC5050025', 'pmid': '27700985'},\n",
       " {'pmcid': 'PMC5052454', 'pmid': '27641957'},\n",
       " {'pmcid': 'PMC5053472', 'pmid': '27711204'},\n",
       " {'pmcid': 'PMC5054943', 'pmid': '27418273'},\n",
       " {'pmcid': 'PMC5055049', 'pmid': '27693358'},\n",
       " {'pmcid': 'PMC5055083', 'pmid': '27693354'},\n",
       " {'pmcid': 'PMC5055122', 'pmid': '27716510'},\n",
       " {'pmcid': 'PMC5055363', 'pmid': '27716814'},\n",
       " {'pmcid': 'PMC5055395', 'pmid': '27612382'},\n",
       " {'pmcid': 'PMC5058508', 'pmid': '27727322'},\n",
       " {'pmcid': 'PMC5059137', 'pmid': '27725085'},\n",
       " {'pmcid': 'PMC5059141', 'pmid': '27725083'},\n",
       " {'pmcid': 'PMC5059142', 'pmid': '27725089'},\n",
       " {'pmcid': 'PMC5063101', 'pmid': '26502195'},\n",
       " {'pmcid': 'PMC5064349', 'pmid': '27781151'},\n",
       " {'pmcid': 'PMC5065316', 'pmid': '27630123'},\n",
       " {'pmcid': 'PMC5066264', 'pmid': '27785463'},\n",
       " {'pmcid': 'PMC5067114', 'pmid': '27644593'},\n",
       " {'pmcid': 'PMC5067305', 'pmid': '27803646'},\n",
       " {'pmcid': 'PMC5069639', 'pmid': '26878258'},\n",
       " {'pmcid': 'PMC5070948', 'pmid': '27760002'},\n",
       " {'pmcid': 'PMC5073248', 'pmid': '27800545'},\n",
       " {'pmcid': 'PMC5074802', 'pmid': '27669144'},\n",
       " {'pmcid': 'PMC5075284', 'pmid': '27745970'},\n",
       " {'pmcid': 'PMC5078599', 'pmid': '27658737'},\n",
       " {'pmcid': 'PMC5078604', 'pmid': '27633463'},\n",
       " {'pmcid': 'PMC5080701', 'pmid': '27782840'},\n",
       " {'pmcid': 'PMC5083949', 'pmid': '27822497'},\n",
       " {'pmcid': 'PMC5083950', 'pmid': '27822498'},\n",
       " {'pmcid': 'PMC5084647', 'pmid': '27810916'},\n",
       " {'pmcid': 'PMC5084649', 'pmid': '27810915'},\n",
       " {'pmcid': 'PMC5088776', 'pmid': '27822503'},\n",
       " {'pmcid': 'PMC5089553', 'pmid': '27802335'},\n",
       " {'pmcid': 'PMC5092050', 'pmid': '27759562'},\n",
       " {'pmcid': 'PMC5092053', 'pmid': '27805901'},\n",
       " {'pmcid': 'PMC5092539', 'pmid': '27693023'},\n",
       " {'pmcid': 'PMC5093152', 'pmid': '27844053'},\n",
       " {'pmcid': 'PMC5094851', 'pmid': '27779093'},\n",
       " {'pmcid': 'PMC5096884', 'pmid': '27692063'},\n",
       " {'pmcid': 'PMC5098910', 'pmid': '27718356'},\n",
       " {'pmcid': 'PMC5099603', 'pmid': '27844057'},\n",
       " {'pmcid': 'PMC5099604', 'pmid': '27844058'},\n",
       " {'pmcid': 'PMC5100090', 'pmid': '27826359'},\n",
       " {'pmcid': 'PMC5100999', 'pmid': '27823583'},\n",
       " {'pmcid': 'PMC5101189', 'pmid': '27881301'},\n",
       " {'pmcid': 'PMC5101915', 'pmid': '27666823'},\n",
       " {'pmcid': 'PMC5102578', 'pmid': '27751231'},\n",
       " {'pmcid': 'PMC5103429', 'pmid': '27829437'},\n",
       " {'pmcid': 'PMC5106211', 'pmid': '27787197'},\n",
       " {'pmcid': 'PMC5108592', 'pmid': '27743477'},\n",
       " {'pmcid': 'PMC5111885', 'pmid': '27849519'},\n",
       " {'pmcid': 'PMC5113791', 'pmid': '27072916'},\n",
       " {'pmcid': 'PMC5114014', 'pmid': '27692069'},\n",
       " {'pmcid': 'PMC5115829', 'pmid': '27861562'},\n",
       " {'pmcid': 'PMC5116651', 'pmid': '27957526'},\n",
       " {'pmcid': 'PMC5117855', 'pmid': '27863209'},\n",
       " {'pmcid': 'PMC5119885', 'pmid': '27874831'},\n",
       " {'pmcid': 'PMC5119888', 'pmid': '27871361'},\n",
       " {'pmcid': 'PMC5119890', 'pmid': '27874828'},\n",
       " {'pmcid': 'PMC5119991', 'pmid': '27863242'},\n",
       " {'pmcid': 'PMC5120881', 'pmid': '27782877'},\n",
       " {'pmcid': 'PMC5122457', 'pmid': '27845622'},\n",
       " {'pmcid': 'PMC5123863', 'pmid': '27719761'},\n",
       " {'pmcid': 'PMC5124690', 'pmid': '27965913'},\n",
       " {'pmcid': 'PMC5124765', 'pmid': '27965624'},\n",
       " {'pmcid': 'PMC5125753', 'pmid': '27791980'},\n",
       " {'pmcid': 'PMC5127639', 'pmid': '27894419'},\n",
       " {'pmcid': 'PMC5127640', 'pmid': '27874829'},\n",
       " {'pmcid': 'PMC5127641', 'pmid': '27805565'},\n",
       " {'pmcid': 'PMC5127645', 'pmid': '27845624'},\n",
       " {'pmcid': 'PMC5130271', 'pmid': '27902765'},\n",
       " {'pmcid': 'PMC5130293', 'pmid': '27901468'},\n",
       " {'pmcid': 'PMC5130294', 'pmid': '27805569'},\n",
       " {'pmcid': 'PMC5132343', 'pmid': '27805568'},\n",
       " {'pmcid': 'PMC5133762', 'pmid': '27912755'},\n",
       " {'pmcid': 'PMC5135102', 'pmid': '27911942'},\n",
       " {'pmcid': 'PMC5136615', 'pmid': '27957530'},\n",
       " {'pmcid': 'PMC5138035', 'pmid': '27849155'},\n",
       " {'pmcid': 'PMC5140266', 'pmid': '27919319'},\n",
       " {'pmcid': 'PMC5140268', 'pmid': '27919317'},\n",
       " {'pmcid': 'PMC5140269', 'pmid': '27919320'},\n",
       " {'pmcid': 'PMC5142322', 'pmid': '27927242'},\n",
       " {'pmcid': 'PMC5142333', 'pmid': '27927196'},\n",
       " {'pmcid': 'PMC5142810', 'pmid': '27879205'},\n",
       " {'pmcid': 'PMC5144555', 'pmid': '27957531'},\n",
       " {'pmcid': 'PMC5144556', 'pmid': '27957532'},\n",
       " {'pmcid': 'PMC5147936', 'pmid': '27936042'},\n",
       " {'pmcid': 'PMC5148600', 'pmid': '27935478'},\n",
       " {'pmcid': 'PMC5148611', 'pmid': '27901466'},\n",
       " {'pmcid': 'PMC5154758', 'pmid': '27874832'},\n",
       " {'pmcid': 'PMC5154760', 'pmid': '27960075'},\n",
       " {'pmcid': 'PMC5155501', 'pmid': '27990272'},\n",
       " {'pmcid': 'PMC5156526', 'pmid': '27855781'},\n",
       " {'pmcid': 'PMC5156537', 'pmid': '27798237'},\n",
       " {'pmcid': 'PMC5157111', 'pmid': '27974617'},\n",
       " {'pmcid': 'PMC5158136', 'pmid': '27879206'},\n",
       " {'pmcid': 'PMC5159840', 'pmid': '27982047'},\n",
       " {'pmcid': 'PMC5159958', 'pmid': '27978830'},\n",
       " {'pmcid': 'PMC5161439', 'pmid': '27913155'},\n",
       " {'pmcid': 'PMC5161750', 'pmid': '27916276'},\n",
       " {'pmcid': 'PMC5161751', 'pmid': '27984725'},\n",
       " {'pmcid': 'PMC5167349', 'pmid': '27840077'},\n",
       " {'pmcid': 'PMC5167519', 'pmid': '27906129'},\n",
       " {'pmcid': 'PMC5167520', 'pmid': '27935475'},\n",
       " {'pmcid': 'PMC5168645', 'pmid': '28032116'},\n",
       " {'pmcid': 'PMC5171881', 'pmid': '28045048'},\n",
       " {'pmcid': 'PMC5172425', 'pmid': '28008422'},\n",
       " {'pmcid': 'PMC5173320', 'pmid': '27914199'},\n",
       " {'pmcid': 'PMC5173323', 'pmid': '27855783'},\n",
       " {'pmcid': 'PMC5173324', 'pmid': '27995898'},\n",
       " {'pmcid': 'PMC5176354', 'pmid': '27897970'},\n",
       " {'pmcid': 'PMC5179108', 'pmid': '28005958'},\n",
       " {'pmcid': 'PMC5179193', 'pmid': '27938665'},\n",
       " {'pmcid': 'PMC5179194', 'pmid': '27922451'},\n",
       " {'pmcid': 'PMC5187389', 'pmid': '28032120'},\n",
       " {'pmcid': 'PMC5191574', 'pmid': '27005435'},\n",
       " {'pmcid': 'PMC5197407', 'pmid': '28058271'},\n",
       " {'pmcid': 'PMC5199195', 'pmid': '27995895'},\n",
       " {'pmcid': 'PMC5200942', 'pmid': '28105318'},\n",
       " {'pmcid': 'PMC5201415', 'pmid': '27938664'},\n",
       " {'pmcid': 'PMC5201416', 'pmid': '28035902'},\n",
       " {'pmcid': 'PMC5206578', 'pmid': '28096800'},\n",
       " {'pmcid': 'PMC5207768', 'pmid': '27938667'},\n",
       " {'pmcid': 'PMC5207769', 'pmid': '27897969'},\n",
       " {'pmcid': 'PMC5207771', 'pmid': '27935477'},\n",
       " {'pmcid': 'PMC5207774', 'pmid': '28045370'},\n",
       " {'pmcid': 'PMC5214169', 'pmid': '27995894'},\n",
       " {'pmcid': 'PMC5214542', 'pmid': '27996939'},\n",
       " {'pmcid': 'PMC5214885', 'pmid': '28051768'},\n",
       " {'pmcid': 'PMC5215100', 'pmid': '27966429'},\n",
       " {'pmcid': 'PMC5215418', 'pmid': '28052116'},\n",
       " {'pmcid': 'PMC5216032', 'pmid': '28111539'},\n",
       " {'pmcid': 'PMC5216448', 'pmid': '27511739'},\n",
       " {'pmcid': 'PMC5218530', 'pmid': '28005008'},\n",
       " {'pmcid': 'PMC5222728', 'pmid': '27616678'},\n",
       " {'pmcid': 'PMC5224920', 'pmid': '27935476'},\n",
       " {'pmcid': 'PMC5224921', 'pmid': '28071588'},\n",
       " {'pmcid': 'PMC5224922', 'pmid': '28012275'},\n",
       " {'pmcid': 'PMC5231406', 'pmid': '28012274'},\n",
       " {'pmcid': 'PMC5233812', 'pmid': '27993630'},\n",
       " {'pmcid': 'PMC5233819', 'pmid': '28024734'},\n",
       " {'pmcid': 'PMC5234438', 'pmid': '28097054'},\n",
       " {'pmcid': 'PMC5236062', 'pmid': '28041852'},\n",
       " {'pmcid': 'PMC5236063', 'pmid': '27916275'},\n",
       " {'pmcid': 'PMC5241115', 'pmid': '28094001'},\n",
       " {'pmcid': 'PMC5241118', 'pmid': '28095293'},\n",
       " {'pmcid': 'PMC5241941', 'pmid': '28101533'},\n",
       " {'pmcid': 'PMC5242495', 'pmid': '28099489'},\n",
       " {'pmcid': 'PMC5245816', 'pmid': '28103316'},\n",
       " {'pmcid': 'PMC5245961', 'pmid': '27906128'},\n",
       " {'pmcid': 'PMC5245965', 'pmid': '28035899'},\n",
       " {'pmcid': 'PMC5245967', 'pmid': '28009255'},\n",
       " {'pmcid': 'PMC5245968', 'pmid': '28100394'},\n",
       " {'pmcid': 'PMC5245970', 'pmid': '28100392'},\n",
       " {'pmcid': 'PMC5249261', 'pmid': '28063257'},\n",
       " {'pmcid': 'PMC5267430', 'pmid': '28122547'},\n",
       " {'pmcid': 'PMC5270589', 'pmid': '28163906'},\n",
       " {'pmcid': 'PMC5271606', 'pmid': '28092268'},\n",
       " {'pmcid': 'PMC5279869', 'pmid': '28017544'},\n",
       " {'pmcid': 'PMC5279941', 'pmid': '28134616'},\n",
       " {'pmcid': 'PMC5279945', 'pmid': '28132667'},\n",
       " {'pmcid': 'PMC5283828', 'pmid': '28139198'},\n",
       " {'pmcid': 'PMC5283830', 'pmid': '28072389'},\n",
       " {'pmcid': 'PMC5283831', 'pmid': '28137359'},\n",
       " {'pmcid': 'PMC5283832', 'pmid': '28029095'},\n",
       " {'pmcid': 'PMC5283834', 'pmid': '28139197'},\n",
       " {'pmcid': 'PMC5289939', 'pmid': '28065575'},\n",
       " {'pmcid': 'PMC5290455', 'pmid': '28197545'},\n",
       " {'pmcid': 'PMC5292596', 'pmid': '28197549'},\n",
       " {'pmcid': 'PMC5293492', 'pmid': '28001126'},\n",
       " {'pmcid': 'PMC5295816', 'pmid': '28168949'},\n",
       " {'pmcid': 'PMC5296252', 'pmid': '28132839'},\n",
       " {'pmcid': 'PMC5296320', 'pmid': '28228748'},\n",
       " {'pmcid': 'PMC5298878', 'pmid': '28098558'},\n",
       " {'pmcid': 'PMC5299403', 'pmid': '28181483'},\n",
       " {'pmcid': 'PMC5300053', 'pmid': '28171747'},\n",
       " {'pmcid': 'PMC5300705', 'pmid': '28079521'},\n",
       " {'pmcid': 'PMC5301079', 'pmid': '28197553'},\n",
       " {'pmcid': 'PMC5302883', 'pmid': '28186874'},\n",
       " {'pmcid': 'PMC5302884', 'pmid': '28182867'},\n",
       " {'pmcid': 'PMC5304403', 'pmid': '28193224'},\n",
       " {'pmcid': 'PMC5306440', 'pmid': '28261056'},\n",
       " {'pmcid': 'PMC5308697', 'pmid': '27542204'},\n",
       " {'pmcid': 'PMC5308896', 'pmid': '28193319'},\n",
       " {'pmcid': 'PMC5308898', 'pmid': '28195039'},\n",
       " {'pmcid': 'PMC5309732', 'pmid': '28275709'},\n",
       " {'pmcid': 'PMC5315295', 'pmid': '28212395'},\n",
       " {'pmcid': 'PMC5315463', 'pmid': '28211791'},\n",
       " {'pmcid': 'PMC5316413', 'pmid': '28065601'},\n",
       " {'pmcid': 'PMC5316414', 'pmid': '28190769'},\n",
       " {'pmcid': 'PMC5316415', 'pmid': '28212753'},\n",
       " {'pmcid': 'PMC5318544', 'pmid': '28275710'},\n",
       " {'pmcid': 'PMC5318796', 'pmid': '28337100'},\n",
       " {'pmcid': 'PMC5319836', 'pmid': '28220755'},\n",
       " {'pmcid': 'PMC5319840', 'pmid': '28219480'},\n",
       " {'pmcid': 'PMC5319997', 'pmid': '28275336'},\n",
       " {'pmcid': 'PMC5322270', 'pmid': '28280459'},\n",
       " {'pmcid': 'PMC5323041', 'pmid': '28231043'},\n",
       " {'pmcid': 'PMC5323046', 'pmid': '28195531'},\n",
       " {'pmcid': 'PMC5324679', 'pmid': '27223051'},\n",
       " {'pmcid': 'PMC5325621', 'pmid': '28169830'},\n",
       " {'pmcid': 'PMC5329619', 'pmid': '28275713'},\n",
       " {'pmcid': 'PMC5330803', 'pmid': '28231464'},\n",
       " {'pmcid': 'PMC5330804', 'pmid': '28231470'},\n",
       " {'pmcid': 'PMC5331070', 'pmid': '28298886'},\n",
       " {'pmcid': 'PMC5333494', 'pmid': '28303090'},\n",
       " {'pmcid': 'PMC5333792', 'pmid': '28159551'},\n",
       " {'pmcid': 'PMC5337618', 'pmid': '28245920'},\n",
       " {'pmcid': 'PMC5337692', 'pmid': '28321181'},\n",
       " {'pmcid': 'PMC5338691', 'pmid': '28231465'},\n",
       " {'pmcid': 'PMC5338724', 'pmid': '28289693'},\n",
       " {'pmcid': 'PMC5338800', 'pmid': '28264063'},\n",
       " {'pmcid': 'PMC5338918', 'pmid': '28263180'},\n",
       " {'pmcid': 'PMC5338919', 'pmid': '28263709'},\n",
       " {'pmcid': 'PMC5338920', 'pmid': '28263708'},\n",
       " {'pmcid': 'PMC5338925', 'pmid': '28221134'},\n",
       " {'pmcid': 'PMC5340527', 'pmid': '28264193'},\n",
       " {'pmcid': 'PMC5340530', 'pmid': '28244869'},\n",
       " {'pmcid': 'PMC5342204', 'pmid': '28273100'},\n",
       " {'pmcid': 'PMC5342823', 'pmid': '28092264'},\n",
       " {'pmcid': 'PMC5342825', 'pmid': '28211790'},\n",
       " {'pmcid': 'PMC5344683', 'pmid': '28190770'},\n",
       " {'pmcid': 'PMC5344684', 'pmid': '28257702'},\n",
       " {'pmcid': 'PMC5345931', 'pmid': '28092266'},\n",
       " {'pmcid': 'PMC5346524', 'pmid': '28293477'},\n",
       " {'pmcid': 'PMC5346530', 'pmid': '28293483'},\n",
       " {'pmcid': 'PMC5350092', 'pmid': '28360836'},\n",
       " {'pmcid': 'PMC5350102', 'pmid': '28360833'},\n",
       " {'pmcid': 'PMC5350129', 'pmid': '28361042'},\n",
       " {'pmcid': 'PMC5352222', 'pmid': '28296634'},\n",
       " {'pmcid': 'PMC5352711', 'pmid': '28361052'},\n",
       " {'pmcid': 'PMC5354407', 'pmid': '28301608'},\n",
       " {'pmcid': 'PMC5354430', 'pmid': '28301587'},\n",
       " {'pmcid': 'PMC5355896', 'pmid': '28321439'},\n",
       " {'pmcid': 'PMC5356222', 'pmid': '28317020'},\n",
       " {'pmcid': 'PMC5356223', 'pmid': '28317021'},\n",
       " {'pmcid': 'PMC5357040', 'pmid': '28357033'},\n",
       " {'pmcid': 'PMC5358112', 'pmid': '28219945'},\n",
       " {'pmcid': 'PMC5358974', 'pmid': '28257000'},\n",
       " {'pmcid': 'PMC5360444', 'pmid': '28322188'},\n",
       " {'pmcid': 'PMC5360445', 'pmid': '28323621'},\n",
       " {'pmcid': 'PMC5360446', 'pmid': '28323616'},\n",
       " {'pmcid': 'PMC5360447', 'pmid': '28322190'},\n",
       " {'pmcid': 'PMC5360449', 'pmid': '28229859'},\n",
       " {'pmcid': 'PMC5360571', 'pmid': '28254561'},\n",
       " {'pmcid': 'PMC5361677', 'pmid': '28192277'},\n",
       " {'pmcid': 'PMC5362263', 'pmid': '28262096'},\n",
       " {'pmcid': 'PMC5362264', 'pmid': '28240595'},\n",
       " {'pmcid': 'PMC5362729', 'pmid': '28386219'},\n",
       " {'pmcid': 'PMC5364369', 'pmid': '28306503'},\n",
       " {'pmcid': 'PMC5364370', 'pmid': '28286024'},\n",
       " {'pmcid': 'PMC5364375', 'pmid': '28306505'},\n",
       " {'pmcid': 'PMC5364458', 'pmid': '28327612'},\n",
       " {'pmcid': 'PMC5366752', 'pmid': '28396628'},\n",
       " {'pmcid': 'PMC5367084', 'pmid': '28374012'},\n",
       " {'pmcid': 'PMC5367895', 'pmid': '28186488'},\n",
       " {'pmcid': 'PMC5368712', 'pmid': '28377822'},\n",
       " {'pmcid': 'PMC5368998', 'pmid': '28347335'},\n",
       " {'pmcid': 'PMC5370185', 'pmid': '28290985'},\n",
       " {'pmcid': 'PMC5370187', 'pmid': '28347402'},\n",
       " {'pmcid': 'PMC5370188', 'pmid': '28226240'},\n",
       " {'pmcid': 'PMC5370190', 'pmid': '28244368'},\n",
       " {'pmcid': 'PMC5372674', 'pmid': '28335493'},\n",
       " {'pmcid': 'PMC5373823', 'pmid': '28145867'},\n",
       " {'pmcid': 'PMC5374161', 'pmid': '28408870'},\n",
       " {'pmcid': 'PMC5375644', 'pmid': '28304275'},\n",
       " {'pmcid': 'PMC5376188', 'pmid': '28362576'},\n",
       " {'pmcid': 'PMC5376383', 'pmid': '28285822'},\n",
       " {'pmcid': 'PMC5377515', 'pmid': '28372543'},\n",
       " {'pmcid': 'PMC5378058', 'pmid': '28396882'},\n",
       " {'pmcid': 'PMC5380436', 'pmid': '28372636'},\n",
       " {'pmcid': 'PMC5382255', 'pmid': '28367951'},\n",
       " {'pmcid': 'PMC5385340', 'pmid': '28443003'},\n",
       " {'pmcid': 'PMC5386587', 'pmid': '28300838'},\n",
       " {'pmcid': 'PMC5387669', 'pmid': '28344080'},\n",
       " {'pmcid': 'PMC5387670', 'pmid': '28344082'},\n",
       " {'pmcid': 'PMC5388530', 'pmid': '28397686'},\n",
       " {'pmcid': 'PMC5388532', 'pmid': '28287395'},\n",
       " {'pmcid': 'PMC5388536', 'pmid': '28395733'},\n",
       " {'pmcid': 'PMC5388538', 'pmid': '28266911'},\n",
       " {'pmcid': 'PMC5388539', 'pmid': '28395732'},\n",
       " {'pmcid': 'PMC5388668', 'pmid': '28413822'},\n",
       " {'pmcid': 'PMC5388670', 'pmid': '28413824'},\n",
       " {'pmcid': 'PMC5388891', 'pmid': '28388414'},\n",
       " {'pmcid': 'PMC5389883', 'pmid': '28399410'},\n",
       " {'pmcid': 'PMC5391095', 'pmid': '28406961'},\n",
       " {'pmcid': 'PMC5391203', 'pmid': '28355132'},\n",
       " {'pmcid': 'PMC5391206', 'pmid': '28332979'},\n",
       " {'pmcid': 'PMC5391208', 'pmid': '28327289'},\n",
       " {'pmcid': 'PMC5391209', 'pmid': '28406396'},\n",
       " {'pmcid': 'PMC5391210', 'pmid': '28350299'},\n",
       " {'pmcid': 'PMC5392495', 'pmid': '28407483'},\n",
       " {'pmcid': 'PMC5392496', 'pmid': '28399403'},\n",
       " {'pmcid': 'PMC5392497', 'pmid': '28366282'},\n",
       " {'pmcid': 'PMC5394940', 'pmid': '28451635'},\n",
       " {'pmcid': 'PMC5395213', 'pmid': '28419135'},\n",
       " {'pmcid': 'PMC5395295', 'pmid': '28418331'},\n",
       " {'pmcid': 'PMC5396130', 'pmid': '28451637'},\n",
       " {'pmcid': 'PMC5396301', 'pmid': '27806429'},\n",
       " {'pmcid': 'PMC5399753', 'pmid': '28451639'},\n",
       " {'pmcid': 'PMC5400476', 'pmid': '28300646'},\n",
       " {'pmcid': 'PMC5400502', 'pmid': '28430576'},\n",
       " {'pmcid': 'PMC5400506', 'pmid': '28318489'},\n",
       " {'pmcid': 'PMC5400507', 'pmid': '28346142'},\n",
       " {'pmcid': 'PMC5400511', 'pmid': '28355134'},\n",
       " {'pmcid': 'PMC5401562', 'pmid': '28431535'},\n",
       " {'pmcid': 'PMC5402947', 'pmid': '28437454'},\n",
       " {'pmcid': 'PMC5404835', 'pmid': '28366620'},\n",
       " {'pmcid': 'PMC5404848', 'pmid': '28343940'},\n",
       " {'pmcid': 'PMC5404915', 'pmid': '28315523'},\n",
       " {'pmcid': 'PMC5404919', 'pmid': '28440221'},\n",
       " {'pmcid': 'PMC5404924', 'pmid': '28440222'},\n",
       " {'pmcid': 'PMC5404928', 'pmid': '28323615'},\n",
       " {'pmcid': 'PMC5405110', 'pmid': '28392072'},\n",
       " {'pmcid': 'PMC5405111', 'pmid': '28416140'},\n",
       " {'pmcid': 'PMC5405198', 'pmid': '28411140'},\n",
       " {'pmcid': 'PMC5406028', 'pmid': '28445500'},\n",
       " {'pmcid': 'PMC5406205', 'pmid': '28323614'},\n",
       " {'pmcid': 'PMC5406386', 'pmid': '28431247'},\n",
       " {'pmcid': 'PMC5407663', 'pmid': '28326944'},\n",
       " {'pmcid': 'PMC5407854', 'pmid': '28362263'},\n",
       " {'pmcid': 'PMC5407856', 'pmid': '28383283'},\n",
       " {'pmcid': 'PMC5407858', 'pmid': '28315522'},\n",
       " {'pmcid': 'PMC5407860', 'pmid': '28332981'},\n",
       " {'pmcid': 'PMC5409825', 'pmid': '28362262'},\n",
       " {'pmcid': 'PMC5409983', 'pmid': '28462393'},\n",
       " {'pmcid': 'PMC5410565', 'pmid': '28507510'},\n",
       " {'pmcid': 'PMC5410580', 'pmid': '28507511'},\n",
       " {'pmcid': 'PMC5411165', 'pmid': '28466071'},\n",
       " {'pmcid': 'PMC5413348', 'pmid': '28432788'},\n",
       " {'pmcid': 'PMC5413352', 'pmid': '28463115'},\n",
       " {'pmcid': 'PMC5413836', 'pmid': '28213922'},\n",
       " {'pmcid': 'PMC5416693', 'pmid': '28375211'},\n",
       " {'pmcid': 'PMC5417601', 'pmid': '28472177'},\n",
       " {'pmcid': 'PMC5417851', 'pmid': '28414273'},\n",
       " {'pmcid': 'PMC5418202', 'pmid': '28426965'},\n",
       " {'pmcid': 'PMC5418924', 'pmid': '28529477'},\n",
       " {'pmcid': 'PMC5419567', 'pmid': '28475602'},\n",
       " {'pmcid': 'PMC5420127', 'pmid': '28476120'},\n",
       " {'pmcid': 'PMC5422071', 'pmid': '28387198'},\n",
       " {'pmcid': 'PMC5422073', 'pmid': '28414270'},\n",
       " {'pmcid': 'PMC5422074', 'pmid': '28440746'},\n",
       " {'pmcid': 'PMC5423769', 'pmid': '28463110'},\n",
       " {'pmcid': 'PMC5425251', 'pmid': '28457793'},\n",
       " {'pmcid': 'PMC5425253', 'pmid': '28489528'},\n",
       " {'pmcid': 'PMC5425254', 'pmid': '28425914'},\n",
       " {'pmcid': 'PMC5425255', 'pmid': '28457792'},\n",
       " {'pmcid': 'PMC5425256', 'pmid': '28486133'},\n",
       " {'pmcid': 'PMC5425402', 'pmid': '28416184'},\n",
       " {'pmcid': 'PMC5426642', 'pmid': '28306514'},\n",
       " {'pmcid': 'PMC5426901', 'pmid': '28406400'},\n",
       " {'pmcid': 'PMC5428558', 'pmid': '28377582'},\n",
       " {'pmcid': 'PMC5429090', 'pmid': '28498099'},\n",
       " {'pmcid': 'PMC5429091', 'pmid': '28498098'},\n",
       " {'pmcid': 'PMC5429092', 'pmid': '28447936'},\n",
       " {'pmcid': 'PMC5429093', 'pmid': '28425913'},\n",
       " {'pmcid': 'PMC5429095', 'pmid': '28395144'},\n",
       " {'pmcid': 'PMC5430609', 'pmid': '28506231'},\n",
       " {'pmcid': 'PMC5430619', 'pmid': '28432353'},\n",
       " {'pmcid': 'PMC5432422', 'pmid': '27862183'},\n",
       " {'pmcid': 'PMC5432432', 'pmid': '28494243'},\n",
       " {'pmcid': 'PMC5432758', 'pmid': '28512649'},\n",
       " {'pmcid': 'PMC5433843', 'pmid': '28463114'},\n",
       " {'pmcid': 'PMC5433852', 'pmid': '28510528'},\n",
       " {'pmcid': 'PMC5434184', 'pmid': '28523221'},\n",
       " {'pmcid': 'PMC5435110', 'pmid': '28292428'},\n",
       " {'pmcid': 'PMC5435961', 'pmid': '28295329'},\n",
       " {'pmcid': 'PMC5437309', 'pmid': '28497795'},\n",
       " {'pmcid': 'PMC5437413', 'pmid': '28534042'},\n",
       " {'pmcid': 'PMC5438246', 'pmid': '28525316'},\n",
       " {'pmcid': 'PMC5438252', 'pmid': '28489002'},\n",
       " {'pmcid': 'PMC5438253', 'pmid': '28467300'},\n",
       " {'pmcid': 'PMC5438254', 'pmid': '28525314'},\n",
       " {'pmcid': 'PMC5438556', 'pmid': '28526034'},\n",
       " {'pmcid': 'PMC5440600', 'pmid': '28434946'},\n",
       " {'pmcid': 'PMC5441873', 'pmid': '28406399'},\n",
       " {'pmcid': 'PMC5443664', 'pmid': '28537220'},\n",
       " {'pmcid': 'PMC5444901', 'pmid': '28509666'},\n",
       " {'pmcid': 'PMC5445107', 'pmid': '28603497'},\n",
       " {'pmcid': 'PMC5446238', 'pmid': '28481198'},\n",
       " {'pmcid': 'PMC5446240', 'pmid': '28485712'},\n",
       " {'pmcid': 'PMC5446316', 'pmid': '28677529'},\n",
       " {'pmcid': 'PMC5446411', 'pmid': '28525743'},\n",
       " {'pmcid': 'PMC5449181', 'pmid': '28555567'},\n",
       " {'pmcid': 'PMC5449184', 'pmid': '28463109'},\n",
       " {'pmcid': 'PMC5449187', 'pmid': '28504640'},\n",
       " {'pmcid': 'PMC5451210', 'pmid': '28463112'},\n",
       " {'pmcid': 'PMC5451211', 'pmid': '28477406'},\n",
       " {'pmcid': 'PMC5451212', 'pmid': '28463111'},\n",
       " {'pmcid': 'PMC5451213', 'pmid': '28425915'},\n",
       " {'pmcid': 'PMC5453633', 'pmid': '28570653'},\n",
       " {'pmcid': 'PMC5454405', 'pmid': '28580417'},\n",
       " {'pmcid': 'PMC5456066', 'pmid': '28574997'},\n",
       " {'pmcid': 'PMC5457137', 'pmid': '28498105'},\n",
       " {'pmcid': 'PMC5457479', 'pmid': '28502659'},\n",
       " {'pmcid': 'PMC5457487', 'pmid': '28575671'},\n",
       " {'pmcid': 'PMC5457488', 'pmid': '28575672'},\n",
       " {'pmcid': 'PMC5459755', 'pmid': '28575657'},\n",
       " {'pmcid': 'PMC5459756', 'pmid': '28343983'},\n",
       " {'pmcid': 'PMC5459927', 'pmid': '28444351'},\n",
       " {'pmcid': 'PMC5460794', 'pmid': '28324005'},\n",
       " {'pmcid': 'PMC5460807', 'pmid': '28324051'},\n",
       " {'pmcid': 'PMC5460924', 'pmid': '28324024'},\n",
       " {'pmcid': 'PMC5460926', 'pmid': '28323954'},\n",
       " {'pmcid': 'PMC5460930', 'pmid': '28368470'},\n",
       " {'pmcid': 'PMC5460932', 'pmid': '28368537'},\n",
       " {'pmcid': 'PMC5460942', 'pmid': '28324012'},\n",
       " {'pmcid': 'PMC5461110', 'pmid': '28583252'},\n",
       " {'pmcid': 'PMC5461557', 'pmid': '28593193'},\n",
       " {'pmcid': 'PMC5462532', 'pmid': '28552575'},\n",
       " {'pmcid': 'PMC5462538', 'pmid': '28504928'},\n",
       " {'pmcid': 'PMC5462539', 'pmid': '28589878'},\n",
       " {'pmcid': 'PMC5462544', 'pmid': '28500756'},\n",
       " {'pmcid': 'PMC5464769', 'pmid': '28548639'},\n",
       " {'pmcid': 'PMC5464770', 'pmid': '28594322'},\n",
       " {'pmcid': 'PMC5464773', 'pmid': '28594323'},\n",
       " {'pmcid': 'PMC5465064', 'pmid': '28596586'},\n",
       " {'pmcid': 'PMC5466419', 'pmid': '28598327'},\n",
       " {'pmcid': 'PMC5467397', 'pmid': '28612045'},\n",
       " {'pmcid': 'PMC5468090', 'pmid': '28500754'},\n",
       " {'pmcid': 'PMC5469615', 'pmid': '28561735'},\n",
       " {'pmcid': 'PMC5472790', 'pmid': '28604739'},\n",
       " {'pmcid': 'PMC5473940', 'pmid': '28622508'},\n",
       " {'pmcid': 'PMC5473941', 'pmid': '28602352'},\n",
       " {'pmcid': 'PMC5474440', 'pmid': '28117277'},\n",
       " {'pmcid': 'PMC5474502', 'pmid': '28159573'},\n",
       " {'pmcid': 'PMC5474510', 'pmid': '28163043'},\n",
       " {'pmcid': 'PMC5474518', 'pmid': '28161398'},\n",
       " {'pmcid': 'PMC5475347', 'pmid': '28552668'},\n",
       " {'pmcid': 'PMC5478095', 'pmid': '28632765'},\n",
       " {'pmcid': 'PMC5478185', 'pmid': '28499923'},\n",
       " {'pmcid': 'PMC5478201', 'pmid': '28583742'},\n",
       " {'pmcid': 'PMC5478209', 'pmid': '28566250'},\n",
       " {'pmcid': 'PMC5478214', 'pmid': '28483582'},\n",
       " {'pmcid': 'PMC5478727', 'pmid': '28680400'},\n",
       " {'pmcid': 'PMC5479563', 'pmid': '28636640'},\n",
       " {'pmcid': 'PMC5479815', 'pmid': '28638111'},\n",
       " {'pmcid': 'PMC5480393', 'pmid': '28622512'},\n",
       " {'pmcid': 'PMC5480394', 'pmid': '28622511'},\n",
       " {'pmcid': 'PMC5481184', 'pmid': '28639940'},\n",
       " {'pmcid': 'PMC5481186', 'pmid': '28633017'},\n",
       " {'pmcid': 'PMC5481623', 'pmid': '28239027'},\n",
       " {'pmcid': 'PMC5482329', 'pmid': '28695215'},\n",
       " {'pmcid': 'PMC5482343', 'pmid': '28690832'},\n",
       " {'pmcid': 'PMC5482572', 'pmid': '28621664'},\n",
       " {'pmcid': 'PMC5483234', 'pmid': '28636956'},\n",
       " {'pmcid': 'PMC5483448', 'pmid': '28694773'},\n",
       " {'pmcid': 'PMC5484315', 'pmid': '27726164'},\n",
       " {'pmcid': 'PMC5484615', 'pmid': '28650315'},\n",
       " {'pmcid': 'PMC5484618', 'pmid': '28577314'},\n",
       " {'pmcid': 'PMC5484619', 'pmid': '28594325'},\n",
       " {'pmcid': 'PMC5484620', 'pmid': '28650797'},\n",
       " {'pmcid': 'PMC5484719', 'pmid': '28169915'},\n",
       " {'pmcid': 'PMC5484994', 'pmid': '28158906'},\n",
       " {'pmcid': 'PMC5486433', 'pmid': '28412414'},\n",
       " {'pmcid': 'PMC5486491', 'pmid': '28604220'},\n",
       " {'pmcid': 'PMC5487210', 'pmid': '28654420'},\n",
       " {'pmcid': 'PMC5487214', 'pmid': '28653623'},\n",
       " {'pmcid': 'PMC5487217', 'pmid': '28653617'},\n",
       " {'pmcid': 'PMC5487291', 'pmid': '28504671'},\n",
       " {'pmcid': 'PMC5487475', 'pmid': '28701929'},\n",
       " {'pmcid': 'PMC5490065', 'pmid': '28546313'},\n",
       " {'pmcid': 'PMC5492891', 'pmid': '28662698'},\n",
       " {'pmcid': 'PMC5493185', 'pmid': '28624615'},\n",
       " {'pmcid': 'PMC5493435', 'pmid': '28665270'},\n",
       " {'pmcid': 'PMC5493436', 'pmid': '28561736'},\n",
       " {'pmcid': 'PMC5493437', 'pmid': '28598330'},\n",
       " {'pmcid': 'PMC5495569', 'pmid': '28671550'},\n",
       " {'pmcid': 'PMC5495570', 'pmid': '28574338'},\n",
       " {'pmcid': 'PMC5495571', 'pmid': '28556776'},\n",
       " {'pmcid': 'PMC5495572', 'pmid': '28644127'},\n",
       " {'pmcid': 'PMC5495865', 'pmid': '28725178'},\n",
       " {'pmcid': 'PMC5495965', 'pmid': '28725181'},\n",
       " {'pmcid': 'PMC5496738', 'pmid': '28574339'},\n",
       " {'pmcid': 'PMC5496739', 'pmid': '28673385'},\n",
       " {'pmcid': 'PMC5497248', 'pmid': '28725215'},\n",
       " {'pmcid': 'PMC5498133', 'pmid': '28623666'},\n",
       " {'pmcid': 'PMC5499943', 'pmid': '28621662'},\n",
       " {'pmcid': 'PMC5500312', 'pmid': '28683270'},\n",
       " {'pmcid': 'PMC5501007', 'pmid': '28683826'},\n",
       " {'pmcid': 'PMC5501724', 'pmid': '28602823'},\n",
       " {'pmcid': 'PMC5501726', 'pmid': '28683288'},\n",
       " {'pmcid': 'PMC5501728', 'pmid': '28648608'},\n",
       " {'pmcid': 'PMC5503510', 'pmid': '28665272'},\n",
       " {'pmcid': 'PMC5503511', 'pmid': '28692422'},\n",
       " {'pmcid': 'PMC5503512', 'pmid': '28656888'},\n",
       " {'pmcid': 'PMC5505209', 'pmid': '28498971'},\n",
       " {'pmcid': 'PMC5505217', 'pmid': '28472300'},\n",
       " {'pmcid': 'PMC5505698', 'pmid': '28639938'},\n",
       " {'pmcid': 'PMC5505699', 'pmid': '28693721'},\n",
       " {'pmcid': 'PMC5505703', 'pmid': '28693719'},\n",
       " {'pmcid': 'PMC5505866', 'pmid': '28366589'},\n",
       " {'pmcid': 'PMC5507289', 'pmid': '28700706'},\n",
       " {'pmcid': 'PMC5508517', 'pmid': '28472656'},\n",
       " {'pmcid': 'PMC5509544', 'pmid': '28689641'},\n",
       " {'pmcid': 'PMC5509656', 'pmid': '28706261'},\n",
       " {'pmcid': 'PMC5509794', 'pmid': '28769928'},\n",
       " {'pmcid': 'PMC5511011', 'pmid': '28632130'},\n",
       " {'pmcid': 'PMC5511112', 'pmid': '28669600'},\n",
       " {'pmcid': 'PMC5513697', 'pmid': '28617242'},\n",
       " {'pmcid': 'PMC5514127', 'pmid': '28717241'},\n",
       " {'pmcid': 'PMC5514405', 'pmid': '28625517'},\n",
       " {'pmcid': 'PMC5515576', 'pmid': '28695822'},\n",
       " {'pmcid': 'PMC5515578', 'pmid': '28650316'},\n",
       " {'pmcid': 'PMC5515579', 'pmid': '28691901'},\n",
       " {'pmcid': 'PMC5515580', 'pmid': '28613156'},\n",
       " {'pmcid': 'PMC5515582', 'pmid': '28716183'},\n",
       " {'pmcid': 'PMC5516609', 'pmid': '28729941'},\n",
       " {'pmcid': 'PMC5517148', 'pmid': '28722651'},\n",
       " {'pmcid': 'PMC5517149', 'pmid': '28617241'},\n",
       " {'pmcid': 'PMC5518148', 'pmid': '28724384'},\n",
       " {'pmcid': 'PMC5518208', 'pmid': '28398653'},\n",
       " {'pmcid': 'PMC5518774', 'pmid': '28752052'},\n",
       " {'pmcid': 'PMC5519170', 'pmid': '28727763'},\n",
       " {'pmcid': 'PMC5519492', 'pmid': '28669544'},\n",
       " {'pmcid': 'PMC5519504', 'pmid': '28709802'},\n",
       " {'pmcid': 'PMC5522448', 'pmid': '28733667'},\n",
       " {'pmcid': 'PMC5522529', 'pmid': '28625738'},\n",
       " {'pmcid': 'PMC5522884', 'pmid': '28790894'},\n",
       " {'pmcid': 'PMC5524708', 'pmid': '28740261'},\n",
       " {'pmcid': 'PMC5526654', 'pmid': '28785727'},\n",
       " {'pmcid': 'PMC5526663', 'pmid': '28742497'},\n",
       " {'pmcid': 'PMC5526790', 'pmid': '28648781'},\n",
       " {'pmcid': 'PMC5526834', 'pmid': '28648780'},\n",
       " {'pmcid': 'PMC5526835', 'pmid': '28712572'},\n",
       " {'pmcid': 'PMC5528256', 'pmid': '28746404'},\n",
       " {'pmcid': 'PMC5529105', 'pmid': '28628005'},\n",
       " {'pmcid': 'PMC5529108', 'pmid': '28746029'},\n",
       " {'pmcid': 'PMC5531831', 'pmid': '28699891'},\n",
       " {'pmcid': 'PMC5531833', 'pmid': '28508746'},\n",
       " {'pmcid': 'PMC5536273', 'pmid': '28759630'},\n",
       " {'pmcid': 'PMC5536945', 'pmid': '28755662'},\n",
       " {'pmcid': 'PMC5536946', 'pmid': '28722652'},\n",
       " {'pmcid': 'PMC5538823', 'pmid': '28760201'},\n",
       " {'pmcid': 'PMC5540466', 'pmid': '28731406'},\n",
       " {'pmcid': 'PMC5542008', 'pmid': '28782055'},\n",
       " {'pmcid': 'PMC5542010', 'pmid': '28782056'},\n",
       " {'pmcid': 'PMC5542431', 'pmid': '28771492'},\n",
       " {'pmcid': 'PMC5542770', 'pmid': '28504929'},\n",
       " {'pmcid': 'PMC5543419', 'pmid': '28815219'},\n",
       " {'pmcid': 'PMC5544428', 'pmid': '28682239'},\n",
       " {'pmcid': 'PMC5545183', 'pmid': '28669519'},\n",
       " {'pmcid': 'PMC5548361', 'pmid': '28795134'},\n",
       " {'pmcid': 'PMC5548486', 'pmid': '28784204'},\n",
       " {'pmcid': 'PMC5548921', 'pmid': '28790351'},\n",
       " {'pmcid': 'PMC5548955', 'pmid': '28772121'},\n",
       " {'pmcid': 'PMC5550275', 'pmid': '28708061'},\n",
       " {'pmcid': 'PMC5550278', 'pmid': '28792888'},\n",
       " {'pmcid': 'PMC5550279', 'pmid': '28737488'},\n",
       " {'pmcid': 'PMC5550726', 'pmid': '28848395'},\n",
       " {'pmcid': 'PMC5551547', 'pmid': '28575322'},\n",
       " {'pmcid': 'PMC5551553', 'pmid': '28586424'},\n",
       " {'pmcid': 'PMC5552246', 'pmid': '28733045'},\n",
       " {'pmcid': 'PMC5552275', 'pmid': '28650317'},\n",
       " {'pmcid': 'PMC5552276', 'pmid': '28722653'},\n",
       " {'pmcid': 'PMC5553934', 'pmid': '28799896'},\n",
       " {'pmcid': 'PMC5554379', 'pmid': '28848384'},\n",
       " {'pmcid': 'PMC5554785', 'pmid': '28787591'},\n",
       " {'pmcid': 'PMC5555718', 'pmid': '28806173'},\n",
       " {'pmcid': 'PMC5555721', 'pmid': '28665271'},\n",
       " {'pmcid': 'PMC5557598', 'pmid': '28809157'},\n",
       " {'pmcid': 'PMC5559767', 'pmid': '28729440'},\n",
       " {'pmcid': 'PMC5559903', 'pmid': '28819639'},\n",
       " {'pmcid': 'PMC5562169', 'pmid': '28825052'},\n",
       " {'pmcid': 'PMC5562298', 'pmid': '28828399'},\n",
       " {'pmcid': 'PMC5562442', 'pmid': '28718767'},\n",
       " {'pmcid': 'PMC5562443', 'pmid': '28820723'},\n",
       " {'pmcid': 'PMC5562444', 'pmid': '28820388'},\n",
       " {'pmcid': 'PMC5562708', 'pmid': '28821740'},\n",
       " {'pmcid': 'PMC5562808', 'pmid': '28821715'},\n",
       " {'pmcid': 'PMC5563003', 'pmid': '28821269'},\n",
       " {'pmcid': 'PMC5563072', 'pmid': '28781174'},\n",
       " {'pmcid': 'PMC5563073', 'pmid': '28781168'},\n",
       " {'pmcid': 'PMC5563080', 'pmid': '28781123'},\n",
       " {'pmcid': 'PMC5563082', 'pmid': '28817804'},\n",
       " {'pmcid': 'PMC5563843', 'pmid': '28828403'},\n",
       " {'pmcid': 'PMC5565316', 'pmid': '28825895'},\n",
       " {'pmcid': 'PMC5565368', 'pmid': '28825401'},\n",
       " {'pmcid': 'PMC5568271', 'pmid': '28832631'},\n",
       " {'pmcid': 'PMC5569379', 'pmid': '28856239'},\n",
       " {'pmcid': 'PMC5569380', 'pmid': '28856240'},\n",
       " {'pmcid': 'PMC5569963', 'pmid': '28531297'},\n",
       " {'pmcid': 'PMC5570191', 'pmid': '28806172'},\n",
       " {'pmcid': 'PMC5573386', 'pmid': '28842709'},\n",
       " {'pmcid': 'PMC5574703', 'pmid': '28849761'},\n",
       " {'pmcid': 'PMC5574929', 'pmid': '28890686'},\n",
       " {'pmcid': 'PMC5575348', 'pmid': '28852138'},\n",
       " {'pmcid': 'PMC5575469', 'pmid': '28485896'},\n",
       " {'pmcid': 'PMC5575502', 'pmid': '28500787'},\n",
       " {'pmcid': 'PMC5575520', 'pmid': '28488766'},\n",
       " {'pmcid': 'PMC5576486', 'pmid': '28849762'},\n",
       " {'pmcid': 'PMC5576920', 'pmid': '28826487'},\n",
       " {'pmcid': 'PMC5576921', 'pmid': '28742022'},\n",
       " {'pmcid': 'PMC5576923', 'pmid': '28749339'},\n",
       " {'pmcid': 'PMC5577914', 'pmid': '28816653'},\n",
       " {'pmcid': 'PMC5578739', 'pmid': '28762944'},\n",
       " {'pmcid': 'PMC5579201', 'pmid': '28597098'},\n",
       " {'pmcid': 'PMC5579261', 'pmid': '28860598'},\n",
       " {'pmcid': 'PMC5583157', 'pmid': '28912690'},\n",
       " {'pmcid': 'PMC5583349', 'pmid': '28871134'},\n",
       " {'pmcid': 'PMC5584673', 'pmid': '28858619'},\n",
       " {'pmcid': 'PMC5585289', 'pmid': '28299422'},\n",
       " {'pmcid': 'PMC5585320', 'pmid': '28874824'},\n",
       " {'pmcid': 'PMC5585371', 'pmid': '28874679'},\n",
       " {'pmcid': 'PMC5587084', 'pmid': '28826471'},\n",
       " {'pmcid': 'PMC5587665', 'pmid': '28878401'},\n",
       " {'pmcid': 'PMC5589211', 'pmid': '28841418'},\n",
       " {'pmcid': 'PMC5589213', 'pmid': '28880918'},\n",
       " {'pmcid': 'PMC5589416', 'pmid': '28880149'},\n",
       " {'pmcid': 'PMC5589418', 'pmid': '28826492'},\n",
       " {'pmcid': 'PMC5590806', 'pmid': '28826472'},\n",
       " {'pmcid': 'PMC5590985', 'pmid': '28886159'},\n",
       " {'pmcid': 'PMC5591021', 'pmid': '28886342'},\n",
       " {'pmcid': 'PMC5592244', 'pmid': '28689657'},\n",
       " {'pmcid': 'PMC5594205', 'pmid': '28757209'},\n",
       " {'pmcid': 'PMC5594246', 'pmid': '28886337'},\n",
       " {'pmcid': 'PMC5594616', 'pmid': '28893254'},\n",
       " {'pmcid': 'PMC5595433', 'pmid': '28895534'},\n",
       " {'pmcid': 'PMC5595435', 'pmid': '28767039'},\n",
       " {'pmcid': 'PMC5595858', 'pmid': '28900205'},\n",
       " {'pmcid': 'PMC5597334', 'pmid': '28841136'},\n",
       " {'pmcid': 'PMC5598544', 'pmid': '28898696'},\n",
       " {'pmcid': 'PMC5598545', 'pmid': '28870739'},\n",
       " {'pmcid': 'PMC5599231', 'pmid': '28826503'},\n",
       " {'pmcid': 'PMC5599232', 'pmid': '28853393'},\n",
       " {'pmcid': 'PMC5599234', 'pmid': '28826505'},\n",
       " {'pmcid': 'PMC5599237', 'pmid': '28703706'},\n",
       " {'pmcid': 'PMC5599238', 'pmid': '28832322'},\n",
       " {'pmcid': 'PMC5601234', 'pmid': '28657115'},\n",
       " {'pmcid': 'PMC5601282', 'pmid': '28452394'},\n",
       " {'pmcid': 'PMC5601995', 'pmid': '28829038'},\n",
       " {'pmcid': 'PMC5601996', 'pmid': '28869038'},\n",
       " {'pmcid': 'PMC5602299', 'pmid': '28920576'},\n",
       " {'pmcid': 'PMC5603509', 'pmid': '28924206'},\n",
       " {'pmcid': 'PMC5605197', 'pmid': '28925357'},\n",
       " {'pmcid': 'PMC5605198', 'pmid': '28926338'},\n",
       " {'pmcid': 'PMC5605570', 'pmid': '28966570'},\n",
       " {'pmcid': 'PMC5605756', 'pmid': '28955722'},\n",
       " {'pmcid': 'PMC5606845', 'pmid': '28927503'},\n",
       " {'pmcid': 'PMC5607351', 'pmid': '28886383'},\n",
       " {'pmcid': 'PMC5607556', 'pmid': '28948087'},\n",
       " {'pmcid': 'PMC5607558', 'pmid': '28948089'},\n",
       " {'pmcid': 'PMC5608509', 'pmid': '28933694'},\n",
       " {'pmcid': 'PMC5608512', 'pmid': '28933695'},\n",
       " {'pmcid': 'PMC5608715', 'pmid': '28935884'},\n",
       " {'pmcid': 'PMC5610174', 'pmid': '28916263'},\n",
       " {'pmcid': 'PMC5610175', 'pmid': '28938124'},\n",
       " {'pmcid': 'PMC5610190', 'pmid': '28938116'},\n",
       " {'pmcid': 'PMC5611444', 'pmid': '28979192'},\n",
       " {'pmcid': 'PMC5612718', 'pmid': '28945785'},\n",
       " {'pmcid': 'PMC5613126', 'pmid': '28983232'},\n",
       " {'pmcid': 'PMC5614429', 'pmid': '28949993'},\n",
       " {'pmcid': 'PMC5614555', 'pmid': '28950008'},\n",
       " {'pmcid': 'PMC5614561', 'pmid': '28891792'},\n",
       " {'pmcid': 'PMC5614563', 'pmid': '28841135'},\n",
       " {'pmcid': 'PMC5614564', 'pmid': '28949916'},\n",
       " {'pmcid': 'PMC5616192', 'pmid': '28966972'},\n",
       " {'pmcid': 'PMC5617080', 'pmid': '28966974'},\n",
       " {'pmcid': 'PMC5617086', 'pmid': '28979176'},\n",
       " {'pmcid': 'PMC5617207', 'pmid': '28966976'},\n",
       " {'pmcid': 'PMC5617629', 'pmid': '28950935'},\n",
       " {'pmcid': 'PMC5617631', 'pmid': '28901286'},\n",
       " {'pmcid': 'PMC5618286', 'pmid': '28805664'},\n",
       " {'pmcid': 'PMC5619949', 'pmid': '28884683'},\n",
       " {'pmcid': 'PMC5619950', 'pmid': '28901288'},\n",
       " {'pmcid': 'PMC5621834', 'pmid': '28914604'},\n",
       " {'pmcid': 'PMC5623156', 'pmid': '28892093'},\n",
       " {'pmcid': 'PMC5624514', 'pmid': '28957659'},\n",
       " {'pmcid': 'PMC5624781', 'pmid': '28968219'},\n",
       " {'pmcid': 'PMC5624785', 'pmid': '28875935'},\n",
       " {'pmcid': 'PMC5625638', 'pmid': '28969637'},\n",
       " {'pmcid': 'PMC5625722', 'pmid': '28969641'},\n",
       " {'pmcid': 'PMC5626803', 'pmid': '28957681'},\n",
       " {'pmcid': 'PMC5628699', 'pmid': '29033541'},\n",
       " {'pmcid': 'PMC5628951', 'pmid': '28918952'},\n",
       " {'pmcid': 'PMC5629024', 'pmid': '28895840'},\n",
       " {'pmcid': 'PMC5630258', 'pmid': '28875930'},\n",
       " {'pmcid': 'PMC5630260', 'pmid': '28952437'},\n",
       " {'pmcid': 'PMC5632562', 'pmid': '28918903'},\n",
       " {'pmcid': 'PMC5632719', 'pmid': '28420716'},\n",
       " {'pmcid': 'PMC5633342', 'pmid': '28990925'},\n",
       " {'pmcid': 'PMC5634782', 'pmid': '28949295'},\n",
       " {'pmcid': 'PMC5634783', 'pmid': '28945193'},\n",
       " {'pmcid': 'PMC5634784', 'pmid': '28994390'},\n",
       " {'pmcid': 'PMC5634787', 'pmid': '28994389'},\n",
       " {'pmcid': 'PMC5637121', 'pmid': '28847810'},\n",
       " {'pmcid': 'PMC5637169', 'pmid': '29017027'},\n",
       " {'pmcid': 'PMC5638581', 'pmid': '28814508'},\n",
       " {'pmcid': 'PMC5639419', 'pmid': '29034319'},\n",
       " {'pmcid': 'PMC5640428', 'pmid': '29028184'},\n",
       " {'pmcid': 'PMC5640506', 'pmid': '28943312'},\n",
       " {'pmcid': 'PMC5640507', 'pmid': '28985505'},\n",
       " {'pmcid': 'PMC5640514', 'pmid': '28966089'},\n",
       " {'pmcid': 'PMC5643091', 'pmid': '28945191'},\n",
       " {'pmcid': 'PMC5643094', 'pmid': '29035202'},\n",
       " {'pmcid': 'PMC5643475', 'pmid': '29075178'},\n",
       " {'pmcid': 'PMC5644667', 'pmid': '29024643'},\n",
       " {'pmcid': 'PMC5644720', 'pmid': '29024642'},\n",
       " {'pmcid': 'PMC5644957', 'pmid': '28925355'},\n",
       " {'pmcid': 'PMC5645400', 'pmid': '29057315'},\n",
       " {'pmcid': 'PMC5645705', 'pmid': '29062910'},\n",
       " {'pmcid': 'PMC5645936', 'pmid': '29041904'},\n",
       " {'pmcid': 'PMC5646851', 'pmid': '29045486'},\n",
       " {'pmcid': 'PMC5648527', 'pmid': '29022879'},\n",
       " {'pmcid': 'PMC5648528', 'pmid': '28984573'},\n",
       " ...]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list_of_pmids_and_pmcids"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Step 2**  \n",
    "find the snippets from nxml file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from xml.etree import ElementTree\n",
    "from tqdm import trange\n",
    "import pprint\n",
    "import re\n",
    "from nltk.tokenize import sent_tokenize\n",
    "import nltk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package punkt to /home/ploy/nltk_data...\n",
      "[nltk_data]   Package punkt is already up-to-date!\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "nltk.download('punkt')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Find only ```<p>``` and then extract the sentences we want (find the regex pattern):\n",
    "  \n",
    "All others tags in ```<p>``` I convert them back to string and remove the xml tag out. \n",
    "- (S|s)pecific\n",
    "- (B|b)ackground staining\n",
    "- (C|c)ross( |-)reactiv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "def remove_xml_tags(text):\n",
    "    clean = re.compile('<.*?>')\n",
    "    return re.sub(clean, '', text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_snippets(text):\n",
    "    \"\"\"\n",
    "    extract snippets from each paragraph\n",
    "    \"\"\"\n",
    "    snippets = []\n",
    "    define_words = ['(S|s)pecific', '((B|b)ackground staining)', '(C|c)ross( |-)reactiv']\n",
    "    # split sentences from text\n",
    "    split_texts = sent_tokenize(text)\n",
    "    for word in define_words:\n",
    "        snippet = []\n",
    "        # find snippet which contains define_words\n",
    "        for s_index in range(len(split_texts)):\n",
    "            word_contain = re.findall(r\"([^.]*?%s[^.]*\\.)\" % word, split_texts[s_index])\n",
    "            if len(word_contain) != 0:\n",
    "                snip = ''\n",
    "                if s_index - 1 >= 0:\n",
    "                    snip = snip + split_texts[s_index-1] + '\\n'\n",
    "                snip = snip + split_texts[s_index] + '\\n'\n",
    "                if s_index + 1 < len(split_texts):\n",
    "                    snip = snip + split_texts[s_index+1] + '\\n'\n",
    "                    \n",
    "                snippet.append(snip)\n",
    "        if len(snippet) != 0:\n",
    "            snippets.append(set(snippet))\n",
    "    if len(snippets) != 0:\n",
    "        return snippets\n",
    "    return None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "snippet_list = []\n",
    "\n",
    "def find_paragraph(node):\n",
    "    \"\"\"\n",
    "    find snippets in <p>\n",
    "    \"\"\"\n",
    "    global snippet_list\n",
    "    if node.tag == 'p':\n",
    "        # convert all contents in <p> to string\n",
    "        xml_str = ElementTree.tostring(node).decode('utf-8')\n",
    "        text = remove_xml_tags(xml_str)\n",
    "\n",
    "        if node.text is not None:\n",
    "            snippets = extract_snippets(text)\n",
    "            if snippets is not None:\n",
    "                snippet_list.append(snippets)\n",
    "    for child in node:\n",
    "        find_paragraph(child)\n",
    "    \n",
    "    return snippet_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_snippets(tree):\n",
    "    \"\"\"\n",
    "    get snippets from each file\n",
    "    \"\"\"\n",
    "    global snippet_list\n",
    "    snippets = []\n",
    "    node = tree.find('./body')\n",
    "\n",
    "    for elem in node:\n",
    "        snippet = find_paragraph(elem)\n",
    "        snippets.extend(snippet)\n",
    "        snippet_list = []\n",
    "        \n",
    "    if snippets is not None and len(snippet) != 0:\n",
    "        return snippets\n",
    "    return None"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Resources Papers path"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "resources_path = 'resources/papers_4chunnan/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "def clean_snippet(snip):\n",
    "    snip = snip.replace('\\n', ' ')\n",
    "    return snip[:-1]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "```outputs``` will contains the dict of outputs that we will save in ```.tsv``` file later."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "outputs = []"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To parse the file, pass an open file handle to parse()  \n",
    "It will read the data, parse the XML, and return an ElementTree object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "reading and finding snippets in file: 100%|██████████| 2223/2223 [13:44<00:00,  2.70it/s]\n"
     ]
    }
   ],
   "source": [
    "for index in trange(len(list_of_pmids_and_pmcids), desc='reading and finding snippets in file'):\n",
    "    with open(resources_path + list_of_pmids_and_pmcids[index]['pmcid'] + '.nxml', 'rt') as file:\n",
    "        tree = ElementTree.parse(file)\n",
    "        snippets = get_snippets(tree)\n",
    "        if snippets is not None and len(snippets) != 0:\n",
    "            for snips in snippets:\n",
    "                for each_snip in snips:\n",
    "                    for turple in each_snip:\n",
    "                        outputs.append(\n",
    "                            { \n",
    "                              'pmid': list_of_pmids_and_pmcids[index]['pmid'], \n",
    "                              'pmcid': list_of_pmids_and_pmcids[index]['pmcid'], \n",
    "                              'snippet': clean_snippet(turple)\n",
    "                            }\n",
    "                        )"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'The family of AUF1 proteins appears to be able to confer either stability or instability to target mRNAs, an effect being cell type and AUF1 isoform-dependent [20]. It has been previously shown that AUF1 expression is up-regulated by E2 in uterus [21], [22] and thus, can regulate the half-life of specific mRNAs [23]. Besides, it has been demonstrated that AUF1 binds sGC &#945;2 mRNA in brain, which in turn decreases its half-life [24].'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'Previously we have shown that E2 effects on sGC subunits occur specifically through ER [5]. To investigate which kind of ER is involved in such effect, anterior pituitary cell cultures were incubated with membrane-impermeable E2 compounds: bovine serum albumin-conjugated E2 (E2-BSA) and estrogen dendrimer conjugate (EDC) [25] using the free steroid (E2) as control.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'One of the first steps starting upon union of ligand to extranuclear ER is the activation of phosphoinositide 3-kinase (PI3K) [26]. To further confirm that E2 is acting through non-nuclear ER, cells were co-incubated with EDC and 50 &#181;M LY294002 (LY), a specific inhibitor of PI3K. Results show that LY had no effect by itself on sGC subunits expression.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'We first demonstrated, by PCR using specific primers, that HuR (data not shown) and all exon 2- and exon 7-containing AUF1 species (p45, p42 and p40) were constitutively expressed in pituitary ( Figure 7A ). Then we investigate whether HuR and AUF1 species are involved in E2-sGC mRNA regulation in anterior pituitary.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'Specific primers for both subunits of sGC were designed from published sequences [6] with Oligo Perfect designer software (Invitrogen) and are detailed in  Table 1 . The amplified products spanned from nucleotide position base 1971&#8211;2054 in the C-terminal region of sGC &#945;1 (GenBank accession number NM_017090); from 714&#8211;823 in the N-terminal region of sGC &#946;1 (M22562); from 741&#8211;955 in the N-terminal region of HuR (NM_001108848); from 649&#8211;674 in the middle region of p45AUF1 (AB_046615), from 503&#8211;574 in the A cassette (N-terminal region) of p45AUF1 and from 1077&#8211;1179 in the B cassette (C-terminal region) of p45AUF1.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Functional characterisation of CAX proteins (mainly from plants and fungi) has shown that these H+ coupled exchangers all transport Ca2+, with some being highly specific for Ca2+, whilst others mediate the transport of a broad range of additional divalent cations or, in some cases, transport additional monovalent cations [11]&#8211;[13]. Their primary role in plants and fungi is to enable tolerance to high extracellular Ca2+ concentrations, by internal sequestration of Ca2+ into acidic organelles when cytosolic levels rise [14].'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'In addition to developing a yeast heterologous expression system for the functional characterisation of apicomplexan CAXs, the genetically amenable P. berghei and T. gondii parasites were used with tagging and knock-out strategies to define expression and essentiality of their respective CAXs. The data demonstrate that under the control of their respective endogenous promoters only the expression of the tagged P. berghei CAX, PbCAX, could be established, and this, only in sexual stages of development (predominantly female specific). Furthermore, genetic disruption of pbcax, while having little or no effect on blood stage growth, was lethal during ookinete development.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'This is due to the fertilisation of functional &#916;map2 female gametes by male &#916;pbcax parasites. These data suggest that PbCAX activity, while not specific, is predominantly important to female gametes.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': \"The integration was confirmed by RT-PCR (Figure S9D) but no signal was detected by immunofluorescence (data not shown), supporting the hypothesis that the protein is expressed at a very low level during this stage of the parasite's life cycle. Furthermore, transiently expressed Ty-tagged TgCAX shows no colocalization with HSP70 or with co-transfected SOD2-GFP (Figure 8C) that label specifically the mitochondrion of T. gondii [26], [27].\"},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Plaque assay experiments were performed and revealed no obvious defect in the lytic cycle of the &#916;TgCAX strain compared with wild-type parasites after 7 days (Figure 9A). To confirm this observation, intracellular growth and egress were assessed specifically. In the case of intracellular growth, the distributions of parasites per vacuole were essentially identical for both the &#916;TgCAX and wild-type strains (Figure 9B) and parasite egress induced by the Ca2+ ionophore A23187 was also similar between strains (Figure 9C).'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Nearly all cax genes characterised to date from bacteria, fungi and plants encode H+-coupled Ca2+ transporters [11], [12], [28]. While some CAX transporters are highly specific for Ca2+, such as yeast Vcx1, others have a broader ion specificity and can mediate the transport of different divalent cations [13], [15] or can transport both divalent (e.g. Ca2+) and monovalent (e.g.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Furthermore, plasmodial CAXs are single copy genes with no close paralogues and CAXs are not found in higher animals. At present there are no potent and specific CAX inhibitors, KB-R7943 (a first-generation inhibitor of Na2+/Ca2+ exchangers) being the best known [7]. Our study has identified a new transmission blocking target, which is required for ookinete development by protecting the parasite from environmental Ca2+, and provides the experimental tools necessary to aid development of this therapeutic strategy.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'For ookinete conversion assays, blood was taken by cardiac puncture from P. berghei-infected mice on day 4 post-infection into heparinised syringes, mixed with ookinete culture medium (RPMI1640 culture medium containing 25 mM HEPES, 25% (v/v) fetal bovine serum, 10 mM sodium bicarbonate, 50 &#181;M xanthurenic acid, pH 7.6) and cultured at 19&#176;C for a further 21&#8211;24 h before assessment of conversion. For direct immunolabelling (to aid identification), cultured cells were pelleted for 2 min at 800&#215; g and then labelled for 10 min on ice in 50 &#181;l of ookinete medium containing Hoechst 33342 and Cy3-conjugated mouse monoclonal antibody specific for P28 [21].'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'For immunofluorescence assays (IFA), parasite-infected HFF cells were fixed with 4% (v/v) paraformaldehyde/0.05% (v/v) glutaraldehyde (PFA/GA) in phosphate buffered saline (PBS) and processed as described previously [60]. For generation of specific antibodies against TgCAX, the peptide NH2-GAPSRQLHLGLLSEGW-COOH was used to immunise two guinea pigs (Eurogentec) according to their standard protocol.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': ' PbCAX-GFP expression. (A) Expression of PbCAX-GFP in live parasites at specific P. berghei life cycle stages is shown. Where appropriate, parasites are immunostained for the female gamete/zygote/ookinete marker P28 (red) and counterstained with the nuclear marker Hoechst 33342 (blue).'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'SINE-R11, 14, and 19 have been isolated by colony blot hybridization using the LTR element as probe [7]. SINE-R.C2 has been found in the third intron of the C2 gene on the short arm of human chromosome 6, which was a human-specific element [8]. Within the Xq21.3 block, two SINE-R retroposons (HS307 and HS408) were identified [9].'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'The SVA families are evolutionarily young and hominoid-specific retroelements. They have ability to influence a genomic locus in which they reside and cause various human diseases as insertional mutagens.'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': \"SVA elements could also drive transcription of functional genes. In the 5' upstream region of TBPL2, the SVA element serves as a promoter, while the SVA-D element in the WDR66 gene promotes the transcription of a human-specific transcript variant [19]. Here, we analyzed structure variants of functional genes mediated by SVA subfamilies and examined their expression patterns in various human tissues.\"},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'Moloney-Murine-Leukemia-Virus reverse transcriptase with an annealing temperature of 42&#8451; was used for the RT reaction with RNase inhibitor (Promega, Madison, WI, USA). To develop the specific primers for individual alternative transcripts, primer pairs were designed with the aid of Primer3 (http://frodo.wi.mit.edu/) (Table 1). In each run, 1 &#181;L of cDNA was used as template for amplification per reaction.'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'This integration event of SVA elements into the exon or intron causes the promotion of transcriptional variants of functional genes. In WDR66 (NM_144668), the SVA-F element, inserted in intron 19, could produce human-specific transcripts that spliced to the last three exons [19]. Those transcripts, derived from SVA elements, could have important biological function in humans, therefore deserving further investigation in various tissues of hominoid primates.'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'The transcripts of the LTR-derived promoter were widely distributed in various tissues, whereas the transcripts of the original promoter were found in stomach tissues, compared to those of various human tissues [25]. A human-specific solitary LTR element (L47334) was previously shown to have enhancer activity in Tera-1 human testicular carcinoma cells [26]. The varying genetic structure of the alternative promoter or enhancer results in different functional effects [27].'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'In the case of leptin receptor isoform 219.1, SVA sequences form the C-terminal coding exon. Using the SVA-specific probe, Southern blot analysis in hominoids and Old World monkeys was performed, indicating that it was found to be a human-specific SVA element [29]. Furthermore, new exons are generated from intronic retrotransposons or the elements inserts into the exon, leading to a new transcript [30].'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'hKIAA1199 is reportedly expressed in a wide range of normal human tissues including brain [12]. Our previous study showed that hKIAA1199 is essential for endogenous HA degradation in human skin fibroblasts, and that cells transfected with hKIAA1199 cDNA degrade HA through specific binding with HA [6]. We have also demonstrated that hKIAA1199 is expressed by dermal fibroblasts in normal skin and over-expressed by synovial fibroblasts and tissues from arthritic joints [6].'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'Rat monoclonal antibody against hKIAA1199 was previously developed using a peptide of CA762RYSPHQDADPLKPRE777, which corresponds to amino acid residues Ala762 to Glu777 of hKIAA1199 (GenBank Accession No.&#160;NM_018689) [6]. Since mKiaa1199 has the same amino acid sequence in the molecule (GenBank Accession No.&#160;AB_103331), this antibody specifically recognized both hKIAA1199 and mKiaa1199. Antibodies against clathrin heavy chain (CHC), &#945;-adaptin, caveolin-1 and GAPDH were purchased from Santa Cruz Biotechnology.'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'Cell homogenate supernatants were separated by electrophoresis on NuPAGE 4&#8211;12% Bis&#8211;Tris gels (Invitrogen) and proteins were transferred onto polyvinylidene difluoride membranes. The membranes were reacted with the antibodies specific to KIAA1199, CHC, &#945;-adaptin, caveolin-1 or GAPDH. Then, they were incubated with horseradish peroxidase-conjugated secondary antibodies: donkey anti-rat IgG antibody for KIAA1199 (Jackson ImmunoResearch); goat anti-rabbit IgG antibody for caveolin-1 and GAPDH (DAKO); rabbit anti-mouse IgG antibody for &#945;-adaptin (DAKO); and rabbit anti-goat IgG antibody for CHC (DAKO).'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'Meanwhile, no HA-degrading activity was detected in cell lysates from mKiaa1199/HEK293 cells (data not shown). All these data on HA specific binding and depolymerization of mKiaa1199 match our previous findings with hKIAA1199 cells [6]. Interestingly, however, the peak size of minimum degradates depolymerized from FA-HA H1 by mKiaa1199/HEK293 cells (about 4.1&#160;kDa, corresponding to ten disaccharide units) differed slightly from the peak size obtained by hKIAA1199/HEK293 cells under the same condition (about 3.3&#160;kDa, corresponding to eight disaccharide units) (Fig.'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'The current study provides the first evidence that like hKIAA1199, mKiaa1199 has the capability of binding specifically to HA, leading to HA depolymerization. The HA depolymerization by mKiaa1199 was almost identical to that by hKIAA1199, although slight differences were found in the elution profiles and peak sizes of the minimum degradates of HA.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'In a previous study, using the same approach, YFP-Hermes protein was found to localise to a vegetal field of particles, typical of germ plasm. All co-injected Cy5-nanos1 co-localised in particles with the YFP-Hermes fusion and subsequent staining of the oocytes with Hermes-specific antiserum revealed that it localised with endogenous Hermes. Furthermore, the Hermes RNP particles were interspersed with particles containing the endogenous germ plasm protein Xpat [4].'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'We also tested the distribution of GFP-Rbm24b and 42b. The panels in Figure 2D and E respectively show that these fusion proteins localised generally to the germ plasm region, but less specifically than the Xvelo proteins. The two Rbm proteins were also observed to be very strongly and uniformly localised very close to the oocyte surface (not shown).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Since there is always a small degree of reassociation of the VENUS moieties, regardless of the specific interactions of candidate partners, all comparisons were made using constant gain settings during confocal microscopy. In Figure 3A-E we see the strong signals produced by Hermes interacting with itself, as well as with some of the other proteins studied here.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'The specificity of the XveloSV antibody was confirmed by depleting XveloSV RNA and using the antibody to see if the XveloSV protein signal was reduced. The reduction was very marked, and is dealt with in a later section.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Hatched boxes represent amino acid residues present in both XveloFL and XveloSV proteins. Clear and grey boxes represent amino acid residues specific to XveloFL and XveloSV proteins respectively. The position of a putative Dynein light chain 8 (DLC-8) interacting motif, identified using the ELM software (elm.eu.org) database is also shown.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'For XveloFL and SV we made isoform-specific oligos. Since XveloSV contains sequences present in XveloFL except for one internal exon we designed a specific XveloSV oligo spanning the splice junction (AS XveloSV). Complimentary oligonucleotides were also designed to the unique sequence in XveloFL (AS XveloFL) and to a region common to the two RNAs (AS XveloFL/SV).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Of course this will only deplete proteins if they are unstable. For XveloFL and SV we made isoform-specific oligos. Since XveloSV contains sequences present in XveloFL except for one internal exon we designed a specific XveloSV oligo spanning the splice junction (AS XveloSV).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'We used the Xvelo isoform-specific antisera described in the last section to assess protein depletion of Xvelo variants by the antisense oligos, using confocal microscopy (Figure 8). XveloFL protein localised to germ plasm showed no detectable diminution with either the specific oligo (AS-XveloFL) or that recognising the two Xvelo variants (AS XveloFL/SV) (not shown).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'XveloFL protein localised to germ plasm showed no detectable diminution with either the specific oligo (AS-XveloFL) or that recognising the two Xvelo variants (AS XveloFL/SV) (not shown). However, while XveloSV protein was unaffected by the oligo specific to XveloFL, its abundance in germ plasm islands was reduced following injection of the specific AS XveloSV oligo, and somewhat less by the oligo recognising both types of transcript (AS XveloFL/SV; Figure 8). The use of two entirely different oligos is important in eliminating off target effects.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'We used the Xvelo isoform-specific antisera described in the last section to assess protein depletion of Xvelo variants by the antisense oligos, using confocal microscopy (Figure 8). XveloFL protein localised to germ plasm showed no detectable diminution with either the specific oligo (AS-XveloFL) or that recognising the two Xvelo variants (AS XveloFL/SV) (not shown). However, while XveloSV protein was unaffected by the oligo specific to XveloFL, its abundance in germ plasm islands was reduced following injection of the specific AS XveloSV oligo, and somewhat less by the oligo recognising both types of transcript (AS XveloFL/SV; Figure 8).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Furthermore, we found that the consequence of XveloSV depletion applied to mitochondria in germ plasm islands, as well as to nanos1 RNA localisation (Figure 9). Specifically, mitochondrial aggregates were seen to be significantly smaller, with numerous mitochondria no longer tightly localised within them. Thus XeloSV appears to have a general role in compacting germ plasm islands.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'As expected from its ineffectiveness at protein depletion, AS XveloFL had little phenotypic effect, consistent with the AS XveloSV phenotype being a specific consequence of XveloSV RNA depletion, rather than simply oligo injection (Figure 8). Furthermore the common AS XveloFL/SV oligo produced a phenotype similar to that seen with AS XveloSV, although it was slightly attenuated, consistent with its somewhat less efficient depletion of protein.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Furthermore the common AS XveloFL/SV oligo produced a phenotype similar to that seen with AS XveloSV, although it was slightly attenuated, consistent with its somewhat less efficient depletion of protein. The fact that two independent oligos have the same specific effect suggests that an off target effect is not involved. A general effect of toxicity is unlikely since combinations of oligos at three times the concentration of AS XveloSV alone, do not affect germ plasm island size, unless they contain AS XveloSV (see below).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'A small reduction in the total amount of these proteins might have a significant effect on their combined spare Hermes-binding capacity. A control for the specificity of the observed germ plasm changes was provided by AS XveloFL, which when combined with AS Rbm24 and 42 had no effect on island size.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Here we report a number of candidate proteins that interact with Hermes and provide evidence, in some cases, that these interactions actually occur in germ plasm, or in other instances that they could do so if the proteins are naturally expressed in oocytes. The fact that these interactions occur in yeast, in the presence only of fragments of the encoding mRNAs, shows that the protein interactions detected are independent of specific interactions with oocyte RNAs.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Both GFP-Xvelo fusions enter germ plasm in large oocytes and both (as well as YFP-Hermes) enter the Balbiani bodies of previtellogenic oocytes (data not shown). Staining of oocytes with Xvelo isoform-specific antisera demonstrates that both proteins are present in the RNP particles of germ plasm in large oocytes and fertilized eggs, where they co-localise with Cy5-nanos1 RNA in a distribution identical to their respective GFP fusions. However, only the larger variant, XveloFL, appeared to be naturally present in the earlier mitochondrial cloud.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'On the other hand it proved possible to reduce XveloSV considerably, resulting in the consistent break-up of the germ plasm islands into smaller structures. This phenotype was produced both by an XveloSV-specific anti-sense oligo and one to the region common to both Xvelo transcripts. This suggests that Xvelo is important for the coherence of germ plasm islands.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'In contrast, the ineffective XveloFL depletion has no effect on island size in combination with antisense Rbm24b and 42b. These experiments are not strong evidence as yet, but do provide a preliminary indication that these two RNA binding proteins have a role in germ plasm structure, specifically via Hermes recruitment.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Products were gel purified and directly sequenced with the primer 5&#8242;-GAGTTTGAGCAGATGTTA-3&#8242;. To confirm the specificity of interactions, the Hermes ORF was amplified using the primers 5&#8242;-AAAGGTACCCCATGAGCGGCATCAAGTCAG-3&#8242; and 5&#8242;-AAAGGTACCTTAACAAAACTGCCGAGACTT-3&#8242;, digested with KpnI and ligated into the corresponding site within the activation domain vector pVP16 to generate pVP16-Hermes. Selected inserts from positive colonies (Table 1) were amplified and ligated into the bait plasmid pBTM116 and co-transformed with pVP16-Hermes into the yeast strain L40, then grown on selective media as described above.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Positive clones were selected on medium lacking leucine, tryptophan and histidine, in the presence of 15 mM 3-aminotriazole and checked with a filter assay for &#946;-galactosidase activity. The inserts from 13 positive prey plasmids were then amplified using VP16 vector-specific primers 5&#8242;-GAGTTTGAGCAGATGTTA-3&#8242; and 5&#8242;-TGTAAAACGACGGCCAGT-3&#8242;, which flank the NotI insertion site. Products were gel purified and directly sequenced with the primer 5&#8242;-GAGTTTGAGCAGATGTTA-3&#8242;.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'peGFPXveloFL&#916;2 was prepared using the primers 5&#8242;AAAAAGCTTATGAACACGACGGCACCTCCCCCAG-3&#8242; and 5&#8242;-AAAGTCGACTTAAGAAGATCCTTCATCCCTAGAT-3&#8242; to amplify the sequence encoding amino acid residues 1&#8211;451 of the long isoform of Xvelo1. cDNA sequences encoding amino acid residues 122&#8211;779 of the long isoform of Xvelo1 were amplified using the primers 5&#8242;-AAAAAGCTTGATCCACGGCAACAAGAATGTGCTT-3&#8242; and 5&#8242;-AAAGTCGACTTACAGTTTGCCATTCAGCTGCTTC-3&#8242; to construct peGFPXveloFL&#916;3. peGFP-XveloSV&#916;1 contained the c-terminal region of the short Xvelo isoform (XveloSV), specifically amino acid residues 205&#8211;269, this was prepared using the primers 5&#8242;-AAAAAGCTTGAAAAGCTCAAAAGAAAACCCCTGG-3&#8242; and 5&#8242;-AAAGTCGACCTACTTCTTCTGGATCAGTTCTAGT-3&#8242;, using peGFP-XveloSV as a template. All inserts were digested with HindIII/HincII and ligated into the corresponding sites in peGFPJC-N. For in vitro RNA synthesis all mutant constructs were linearised with XhoI and transcribed with SP6 RNA polymerase, using the mMESSAGE mMACHINE kit (Ambion).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Polyclonal antibodies against XveloFL (long isoform) and XveloSV (short isoform) were raised in goats using the Eurogentec anti-peptide 28-day Speedy programme (Eurogentec, Seraing, Belgium). The antigens were synthetic peptides specific to XveloFL (H2N-QTREPRYETSKQGKQC-CONH2 and H2N-CKKNPKKSQSLSEPED-CONH2) and XveloSV (H2N-CSSLLKENKLELIQKK-COOH and H2N-CKIQPKSKSKISLQKR-CONH2). For the affinity purification of Xvelo-specific antibodies, sequences corresponding to nucleotides 864&#8211;1759 of the XveloFL cDNA and nucleotides 653&#8211;850 of the XveloSV cDNA [16], which contain the synthetic peptide sequences used for immunisation, were amplified by PCR and cloned into the bacterial expression vector pET14b to prepare pET14b-VFL&#916;1 and pET14b-VSV&#916;1 respectively.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'The antigens were synthetic peptides specific to XveloFL (H2N-QTREPRYETSKQGKQC-CONH2 and H2N-CKKNPKKSQSLSEPED-CONH2) and XveloSV (H2N-CSSLLKENKLELIQKK-COOH and H2N-CKIQPKSKSKISLQKR-CONH2). For the affinity purification of Xvelo-specific antibodies, sequences corresponding to nucleotides 864&#8211;1759 of the XveloFL cDNA and nucleotides 653&#8211;850 of the XveloSV cDNA [16], which contain the synthetic peptide sequences used for immunisation, were amplified by PCR and cloned into the bacterial expression vector pET14b to prepare pET14b-VFL&#916;1 and pET14b-VSV&#916;1 respectively. Fusion proteins were expressed in E.coli strain BL21(DE3), following the addition of 0.1 M IPTG.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'XveloFL, 5&#8242;-A*T*C*C*TTAAGTCGGC*A*C*A*C-3; XveloSV, 5&#8242;-A*G*C*T*TTTCCTCCTC*T*T*T*C-3&#8242; (specificity achieved by nine nucleotides flanking either side of the splice junction in Xvelo1 RNA [16]; XveloFL/SV, 5&#8242;-C*T*T*G*TTGCCGTGGA*T*C*A*G-3&#8242; (this targets both the long and short forms of Xvelo RNA); Rbm24b, 5&#8242;-G*T*G*T*TGCAGCAGTA*A*G*T*G-3&#8242;; Rbm42b, 5&#8242;-A*A*G*C*GCGAGCAAGG*A*T*G*T-3&#8242;, where &#8216;*&#8217; indicates phosphorothioate linkages (Invitrogen). Successful depletion of specific RNAs was assayed by semi-quantitative RT-PCR, essentially as described by Hudson et al. [44].'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Stage VI oocytes were injected with 11 ng of each oligo, alone or in combination. XveloFL, 5&#8242;-A*T*C*C*TTAAGTCGGC*A*C*A*C-3; XveloSV, 5&#8242;-A*G*C*T*TTTCCTCCTC*T*T*T*C-3&#8242; (specificity achieved by nine nucleotides flanking either side of the splice junction in Xvelo1 RNA [16]; XveloFL/SV, 5&#8242;-C*T*T*G*TTGCCGTGGA*T*C*A*G-3&#8242; (this targets both the long and short forms of Xvelo RNA); Rbm24b, 5&#8242;-G*T*G*T*TGCAGCAGTA*A*G*T*G-3&#8242;; Rbm42b, 5&#8242;-A*A*G*C*GCGAGCAAGG*A*T*G*T-3&#8242;, where &#8216;*&#8217; indicates phosphorothioate linkages (Invitrogen). Successful depletion of specific RNAs was assayed by semi-quantitative RT-PCR, essentially as described by Hudson et al.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'In these experiments oocytes were incubated in OCM medium within intact follicles (except for experiments where microtubules were stained with antibodies, where they were defolliculated with collagenase [45]). Endogenous XveloFL and XveloSV protein localisation experiments were performed on a fixed oocyte series (Stages I&#8211;VI) using isoform-specific, affinity purified goat polyclonals and a Rabbit Anti-Goat Dylight 594 (Jackson ImmunoResearch Laboratories, Inc) secondary antibody. Antibody staining was described by Machado et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'The specific statistical analysis used is noted below for each experimental approach. The Design Analysis Core at WFSM provided advice to the investigators, and the Design and Analysis Core performed select analyses.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Well-characterized antibodies against cell- and synapse-specific markers were used for single- or double-labeling studies. A complete list of antibodies, immunogens, manufacturers, host species, dilutions used, and references are provided in Table 1.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'To identify specific motor pools in the L3/L4 segments of the spinal cord for ultrastructural level, we initially used retrograde labeling with colloidal gold-labeled cholera toxin subunit B (CTB; List Biological Laboratories, Denver, CO); however, gold particles could not be definitely identified in MNs. Therefore, we identified the TA and soleus motor pools by retrograde labeling with alexa-fluor CTB (Invitrogen; Fig.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Dotted line outlines the VH boundary, and the box indicates location of thin section. X-Y coordinates on the electron microscope were used to map the adjacent sections and located specific MNs. MNs, motoneurons; TA, tibialis anterior; VH, ventral horn.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': \"Specific muscles were dissected from the mice used for ultrastructural analysis of the spinal cord described above and placed in fixative overnight at 4&#176;C. Muscles were cut at 600 &#956;m on a tissue chopper, then reacted for cholinesterase to reveal NJMs using Karnovsky's method with the following changes: sodium cacodylate buffer was used instead of PBS and the tissue was incubated for 1 h or until the endplates were visible under the dissecting microscope (Karnovsky, 1964). Specimens were then embedded in Araldite 502 using a Lynx processor.\"},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': '2004; Hampton and Amende 2010). Briefly, we used a motor-driven treadmill with a transparent treadmill belt (DigiGait Imaging System, Mouse Specifics, Inc., Quincy, MA). A high-speed digital video camera was positioned below the transparent belt to focus on the ventral view of subjects walking on the belt.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Additionally the toxicity of mutant SOD1 involves other cell types besides MNs and therefore is at least partly noncell (MN) autonomous. For example, cell-specific deletion of mutant SOD1 in genetically altered mice has implicated microglia and astrocytes as contributors to the progression but not the onset of disease (Clement et al. 2003; Boill&#233;e et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'A cardinal feature of most developmental and adult onset neurodegenerative diseases, including motor neuron diseases such as ALS, is the death of specific population of neurons. Largely as a result of the progress made in elucidating the cellular and molecular mechanisms underlying neuronal death during development (Oppenheim et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Denervation of NMJs by fast-fatigable (FF) MNs that innervate specific types of muscle fibers &#8211; myosin heavy chain (MyHC) Type IIB &#8211; in SOD1 fALS mice begins as early as P25 (Gould et al. 2006 (disease onset at P90-100), followed later by loss of NMJ innervation of Type IIa muscle fibers by fast-fatigue resistant (FR) MNs and lastly denervation by slow (S) MNs that innervate Type I/Ia muscle fibers (Pun et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'For example, fast-fatigable muscle fibers, the first ones denervated in the SOD1 mouse model, are innervated by the most phasic subtype of MNs. Hyperexcitability and evidence of ER stress is detected specifically in MNs innervating fast-fatigable muscles prior to initial denervation further suggesting that these events serve as stressors that continue until the neuron reaches a threshold that jeopardizes its survival (Saxena and Caroni 2011). Nonetheless, many of the events associated with cellular stress often represent a protective response by the cell and therefore do not precipitate neuronal dysfunction, but rather prevent or delay the process (reviewed in Robinson et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': \"While selected neuronal populations are affected in neurodegenerative diseases such as ALS, Alzheimer's and Parkinson's diseases, age is a common feature in all neurodegenerative diseases. Results from numerous studies suggest that there are common features across disease-specific populations including aggregation of misfolded proteins, altered proteasome activity and stress responses including ER stress, increased autophagy and mitochondrial changes noted above. Furthermore, patterns of resistance and susceptibility in NMJs in ALS mice are also observed in normally aging mouse muscles (Valdez et al.\"},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': '2007; Conforti et al. 2007; Gould and Oppenheim 2007) and (2) the specific molecular mechanisms mediating axon/synapse loss in ALS are largely unknown (Saxena and Caroni 2007). Mitochondrial morphological and functional changes are likely involved in disease pathology; however, alterations in synaptic input, axonal transport, ER stress, protein aggregates are also MN intracellular events that are associated with pathology.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Pathological events are well characterized in the ALS mouse models, but review of the literature fails to identify a specific initiating event that precipitates disease pathology. There is now a growing consensus in the field that the axon and synapses are the first cellular sites of degeneration, but there is still controversy over (1) whether axon and synapse loss is initiated autonomously at those sites or by pathology elsewhere (Bettini et al.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'The family of AUF1 proteins appears to be able to confer either stability or instability to target mRNAs, an effect being cell type and AUF1 isoform-dependent [20]. It has been previously shown that AUF1 expression is up-regulated by E2 in uterus [21], [22] and thus, can regulate the half-life of specific mRNAs [23]. Besides, it has been demonstrated that AUF1 binds sGC &#945;2 mRNA in brain, which in turn decreases its half-life [24].'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'Previously we have shown that E2 effects on sGC subunits occur specifically through ER [5]. To investigate which kind of ER is involved in such effect, anterior pituitary cell cultures were incubated with membrane-impermeable E2 compounds: bovine serum albumin-conjugated E2 (E2-BSA) and estrogen dendrimer conjugate (EDC) [25] using the free steroid (E2) as control.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'One of the first steps starting upon union of ligand to extranuclear ER is the activation of phosphoinositide 3-kinase (PI3K) [26]. To further confirm that E2 is acting through non-nuclear ER, cells were co-incubated with EDC and 50 &#181;M LY294002 (LY), a specific inhibitor of PI3K. Results show that LY had no effect by itself on sGC subunits expression.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'We first demonstrated, by PCR using specific primers, that HuR (data not shown) and all exon 2- and exon 7-containing AUF1 species (p45, p42 and p40) were constitutively expressed in pituitary ( Figure 7A ). Then we investigate whether HuR and AUF1 species are involved in E2-sGC mRNA regulation in anterior pituitary.'},\n",
       " {'pmcid': 'PMC3247256',\n",
       "  'pmid': '22216273',\n",
       "  'snippet': 'Specific primers for both subunits of sGC were designed from published sequences [6] with Oligo Perfect designer software (Invitrogen) and are detailed in  Table 1 . The amplified products spanned from nucleotide position base 1971&#8211;2054 in the C-terminal region of sGC &#945;1 (GenBank accession number NM_017090); from 714&#8211;823 in the N-terminal region of sGC &#946;1 (M22562); from 741&#8211;955 in the N-terminal region of HuR (NM_001108848); from 649&#8211;674 in the middle region of p45AUF1 (AB_046615), from 503&#8211;574 in the A cassette (N-terminal region) of p45AUF1 and from 1077&#8211;1179 in the B cassette (C-terminal region) of p45AUF1.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Functional characterisation of CAX proteins (mainly from plants and fungi) has shown that these H+ coupled exchangers all transport Ca2+, with some being highly specific for Ca2+, whilst others mediate the transport of a broad range of additional divalent cations or, in some cases, transport additional monovalent cations [11]&#8211;[13]. Their primary role in plants and fungi is to enable tolerance to high extracellular Ca2+ concentrations, by internal sequestration of Ca2+ into acidic organelles when cytosolic levels rise [14].'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'In addition to developing a yeast heterologous expression system for the functional characterisation of apicomplexan CAXs, the genetically amenable P. berghei and T. gondii parasites were used with tagging and knock-out strategies to define expression and essentiality of their respective CAXs. The data demonstrate that under the control of their respective endogenous promoters only the expression of the tagged P. berghei CAX, PbCAX, could be established, and this, only in sexual stages of development (predominantly female specific). Furthermore, genetic disruption of pbcax, while having little or no effect on blood stage growth, was lethal during ookinete development.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'This is due to the fertilisation of functional &#916;map2 female gametes by male &#916;pbcax parasites. These data suggest that PbCAX activity, while not specific, is predominantly important to female gametes.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': \"The integration was confirmed by RT-PCR (Figure S9D) but no signal was detected by immunofluorescence (data not shown), supporting the hypothesis that the protein is expressed at a very low level during this stage of the parasite's life cycle. Furthermore, transiently expressed Ty-tagged TgCAX shows no colocalization with HSP70 or with co-transfected SOD2-GFP (Figure 8C) that label specifically the mitochondrion of T. gondii [26], [27].\"},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Plaque assay experiments were performed and revealed no obvious defect in the lytic cycle of the &#916;TgCAX strain compared with wild-type parasites after 7 days (Figure 9A). To confirm this observation, intracellular growth and egress were assessed specifically. In the case of intracellular growth, the distributions of parasites per vacuole were essentially identical for both the &#916;TgCAX and wild-type strains (Figure 9B) and parasite egress induced by the Ca2+ ionophore A23187 was also similar between strains (Figure 9C).'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Nearly all cax genes characterised to date from bacteria, fungi and plants encode H+-coupled Ca2+ transporters [11], [12], [28]. While some CAX transporters are highly specific for Ca2+, such as yeast Vcx1, others have a broader ion specificity and can mediate the transport of different divalent cations [13], [15] or can transport both divalent (e.g. Ca2+) and monovalent (e.g.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'Furthermore, plasmodial CAXs are single copy genes with no close paralogues and CAXs are not found in higher animals. At present there are no potent and specific CAX inhibitors, KB-R7943 (a first-generation inhibitor of Na2+/Ca2+ exchangers) being the best known [7]. Our study has identified a new transmission blocking target, which is required for ookinete development by protecting the parasite from environmental Ca2+, and provides the experimental tools necessary to aid development of this therapeutic strategy.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'For ookinete conversion assays, blood was taken by cardiac puncture from P. berghei-infected mice on day 4 post-infection into heparinised syringes, mixed with ookinete culture medium (RPMI1640 culture medium containing 25 mM HEPES, 25% (v/v) fetal bovine serum, 10 mM sodium bicarbonate, 50 &#181;M xanthurenic acid, pH 7.6) and cultured at 19&#176;C for a further 21&#8211;24 h before assessment of conversion. For direct immunolabelling (to aid identification), cultured cells were pelleted for 2 min at 800&#215; g and then labelled for 10 min on ice in 50 &#181;l of ookinete medium containing Hoechst 33342 and Cy3-conjugated mouse monoclonal antibody specific for P28 [21].'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': 'For immunofluorescence assays (IFA), parasite-infected HFF cells were fixed with 4% (v/v) paraformaldehyde/0.05% (v/v) glutaraldehyde (PFA/GA) in phosphate buffered saline (PBS) and processed as described previously [60]. For generation of specific antibodies against TgCAX, the peptide NH2-GAPSRQLHLGLLSEGW-COOH was used to immunise two guinea pigs (Eurogentec) according to their standard protocol.'},\n",
       " {'pmcid': 'PMC3585132',\n",
       "  'pmid': '23468629',\n",
       "  'snippet': ' PbCAX-GFP expression. (A) Expression of PbCAX-GFP in live parasites at specific P. berghei life cycle stages is shown. Where appropriate, parasites are immunostained for the female gamete/zygote/ookinete marker P28 (red) and counterstained with the nuclear marker Hoechst 33342 (blue).'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'SINE-R11, 14, and 19 have been isolated by colony blot hybridization using the LTR element as probe [7]. SINE-R.C2 has been found in the third intron of the C2 gene on the short arm of human chromosome 6, which was a human-specific element [8]. Within the Xq21.3 block, two SINE-R retroposons (HS307 and HS408) were identified [9].'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'The SVA families are evolutionarily young and hominoid-specific retroelements. They have ability to influence a genomic locus in which they reside and cause various human diseases as insertional mutagens.'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': \"SVA elements could also drive transcription of functional genes. In the 5' upstream region of TBPL2, the SVA element serves as a promoter, while the SVA-D element in the WDR66 gene promotes the transcription of a human-specific transcript variant [19]. Here, we analyzed structure variants of functional genes mediated by SVA subfamilies and examined their expression patterns in various human tissues.\"},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'Moloney-Murine-Leukemia-Virus reverse transcriptase with an annealing temperature of 42&#8451; was used for the RT reaction with RNase inhibitor (Promega, Madison, WI, USA). To develop the specific primers for individual alternative transcripts, primer pairs were designed with the aid of Primer3 (http://frodo.wi.mit.edu/) (Table 1). In each run, 1 &#181;L of cDNA was used as template for amplification per reaction.'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'This integration event of SVA elements into the exon or intron causes the promotion of transcriptional variants of functional genes. In WDR66 (NM_144668), the SVA-F element, inserted in intron 19, could produce human-specific transcripts that spliced to the last three exons [19]. Those transcripts, derived from SVA elements, could have important biological function in humans, therefore deserving further investigation in various tissues of hominoid primates.'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'The transcripts of the LTR-derived promoter were widely distributed in various tissues, whereas the transcripts of the original promoter were found in stomach tissues, compared to those of various human tissues [25]. A human-specific solitary LTR element (L47334) was previously shown to have enhancer activity in Tera-1 human testicular carcinoma cells [26]. The varying genetic structure of the alternative promoter or enhancer results in different functional effects [27].'},\n",
       " {'pmcid': 'PMC3794087',\n",
       "  'pmid': '24124410',\n",
       "  'snippet': 'In the case of leptin receptor isoform 219.1, SVA sequences form the C-terminal coding exon. Using the SVA-specific probe, Southern blot analysis in hominoids and Old World monkeys was performed, indicating that it was found to be a human-specific SVA element [29]. Furthermore, new exons are generated from intronic retrotransposons or the elements inserts into the exon, leading to a new transcript [30].'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'hKIAA1199 is reportedly expressed in a wide range of normal human tissues including brain [12]. Our previous study showed that hKIAA1199 is essential for endogenous HA degradation in human skin fibroblasts, and that cells transfected with hKIAA1199 cDNA degrade HA through specific binding with HA [6]. We have also demonstrated that hKIAA1199 is expressed by dermal fibroblasts in normal skin and over-expressed by synovial fibroblasts and tissues from arthritic joints [6].'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'Rat monoclonal antibody against hKIAA1199 was previously developed using a peptide of CA762RYSPHQDADPLKPRE777, which corresponds to amino acid residues Ala762 to Glu777 of hKIAA1199 (GenBank Accession No.&#160;NM_018689) [6]. Since mKiaa1199 has the same amino acid sequence in the molecule (GenBank Accession No.&#160;AB_103331), this antibody specifically recognized both hKIAA1199 and mKiaa1199. Antibodies against clathrin heavy chain (CHC), &#945;-adaptin, caveolin-1 and GAPDH were purchased from Santa Cruz Biotechnology.'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'Cell homogenate supernatants were separated by electrophoresis on NuPAGE 4&#8211;12% Bis&#8211;Tris gels (Invitrogen) and proteins were transferred onto polyvinylidene difluoride membranes. The membranes were reacted with the antibodies specific to KIAA1199, CHC, &#945;-adaptin, caveolin-1 or GAPDH. Then, they were incubated with horseradish peroxidase-conjugated secondary antibodies: donkey anti-rat IgG antibody for KIAA1199 (Jackson ImmunoResearch); goat anti-rabbit IgG antibody for caveolin-1 and GAPDH (DAKO); rabbit anti-mouse IgG antibody for &#945;-adaptin (DAKO); and rabbit anti-goat IgG antibody for CHC (DAKO).'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'Meanwhile, no HA-degrading activity was detected in cell lysates from mKiaa1199/HEK293 cells (data not shown). All these data on HA specific binding and depolymerization of mKiaa1199 match our previous findings with hKIAA1199 cells [6]. Interestingly, however, the peak size of minimum degradates depolymerized from FA-HA H1 by mKiaa1199/HEK293 cells (about 4.1&#160;kDa, corresponding to ten disaccharide units) differed slightly from the peak size obtained by hKIAA1199/HEK293 cells under the same condition (about 3.3&#160;kDa, corresponding to eight disaccharide units) (Fig.'},\n",
       " {'pmcid': 'PMC3821019',\n",
       "  'pmid': '24251095',\n",
       "  'snippet': 'The current study provides the first evidence that like hKIAA1199, mKiaa1199 has the capability of binding specifically to HA, leading to HA depolymerization. The HA depolymerization by mKiaa1199 was almost identical to that by hKIAA1199, although slight differences were found in the elution profiles and peak sizes of the minimum degradates of HA.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'In a previous study, using the same approach, YFP-Hermes protein was found to localise to a vegetal field of particles, typical of germ plasm. All co-injected Cy5-nanos1 co-localised in particles with the YFP-Hermes fusion and subsequent staining of the oocytes with Hermes-specific antiserum revealed that it localised with endogenous Hermes. Furthermore, the Hermes RNP particles were interspersed with particles containing the endogenous germ plasm protein Xpat [4].'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'We also tested the distribution of GFP-Rbm24b and 42b. The panels in Figure 2D and E respectively show that these fusion proteins localised generally to the germ plasm region, but less specifically than the Xvelo proteins. The two Rbm proteins were also observed to be very strongly and uniformly localised very close to the oocyte surface (not shown).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Since there is always a small degree of reassociation of the VENUS moieties, regardless of the specific interactions of candidate partners, all comparisons were made using constant gain settings during confocal microscopy. In Figure 3A-E we see the strong signals produced by Hermes interacting with itself, as well as with some of the other proteins studied here.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'The specificity of the XveloSV antibody was confirmed by depleting XveloSV RNA and using the antibody to see if the XveloSV protein signal was reduced. The reduction was very marked, and is dealt with in a later section.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Hatched boxes represent amino acid residues present in both XveloFL and XveloSV proteins. Clear and grey boxes represent amino acid residues specific to XveloFL and XveloSV proteins respectively. The position of a putative Dynein light chain 8 (DLC-8) interacting motif, identified using the ELM software (elm.eu.org) database is also shown.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'For XveloFL and SV we made isoform-specific oligos. Since XveloSV contains sequences present in XveloFL except for one internal exon we designed a specific XveloSV oligo spanning the splice junction (AS XveloSV). Complimentary oligonucleotides were also designed to the unique sequence in XveloFL (AS XveloFL) and to a region common to the two RNAs (AS XveloFL/SV).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Of course this will only deplete proteins if they are unstable. For XveloFL and SV we made isoform-specific oligos. Since XveloSV contains sequences present in XveloFL except for one internal exon we designed a specific XveloSV oligo spanning the splice junction (AS XveloSV).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'We used the Xvelo isoform-specific antisera described in the last section to assess protein depletion of Xvelo variants by the antisense oligos, using confocal microscopy (Figure 8). XveloFL protein localised to germ plasm showed no detectable diminution with either the specific oligo (AS-XveloFL) or that recognising the two Xvelo variants (AS XveloFL/SV) (not shown).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'XveloFL protein localised to germ plasm showed no detectable diminution with either the specific oligo (AS-XveloFL) or that recognising the two Xvelo variants (AS XveloFL/SV) (not shown). However, while XveloSV protein was unaffected by the oligo specific to XveloFL, its abundance in germ plasm islands was reduced following injection of the specific AS XveloSV oligo, and somewhat less by the oligo recognising both types of transcript (AS XveloFL/SV; Figure 8). The use of two entirely different oligos is important in eliminating off target effects.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'We used the Xvelo isoform-specific antisera described in the last section to assess protein depletion of Xvelo variants by the antisense oligos, using confocal microscopy (Figure 8). XveloFL protein localised to germ plasm showed no detectable diminution with either the specific oligo (AS-XveloFL) or that recognising the two Xvelo variants (AS XveloFL/SV) (not shown). However, while XveloSV protein was unaffected by the oligo specific to XveloFL, its abundance in germ plasm islands was reduced following injection of the specific AS XveloSV oligo, and somewhat less by the oligo recognising both types of transcript (AS XveloFL/SV; Figure 8).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Furthermore, we found that the consequence of XveloSV depletion applied to mitochondria in germ plasm islands, as well as to nanos1 RNA localisation (Figure 9). Specifically, mitochondrial aggregates were seen to be significantly smaller, with numerous mitochondria no longer tightly localised within them. Thus XeloSV appears to have a general role in compacting germ plasm islands.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'As expected from its ineffectiveness at protein depletion, AS XveloFL had little phenotypic effect, consistent with the AS XveloSV phenotype being a specific consequence of XveloSV RNA depletion, rather than simply oligo injection (Figure 8). Furthermore the common AS XveloFL/SV oligo produced a phenotype similar to that seen with AS XveloSV, although it was slightly attenuated, consistent with its somewhat less efficient depletion of protein.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Furthermore the common AS XveloFL/SV oligo produced a phenotype similar to that seen with AS XveloSV, although it was slightly attenuated, consistent with its somewhat less efficient depletion of protein. The fact that two independent oligos have the same specific effect suggests that an off target effect is not involved. A general effect of toxicity is unlikely since combinations of oligos at three times the concentration of AS XveloSV alone, do not affect germ plasm island size, unless they contain AS XveloSV (see below).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'A small reduction in the total amount of these proteins might have a significant effect on their combined spare Hermes-binding capacity. A control for the specificity of the observed germ plasm changes was provided by AS XveloFL, which when combined with AS Rbm24 and 42 had no effect on island size.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Here we report a number of candidate proteins that interact with Hermes and provide evidence, in some cases, that these interactions actually occur in germ plasm, or in other instances that they could do so if the proteins are naturally expressed in oocytes. The fact that these interactions occur in yeast, in the presence only of fragments of the encoding mRNAs, shows that the protein interactions detected are independent of specific interactions with oocyte RNAs.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Both GFP-Xvelo fusions enter germ plasm in large oocytes and both (as well as YFP-Hermes) enter the Balbiani bodies of previtellogenic oocytes (data not shown). Staining of oocytes with Xvelo isoform-specific antisera demonstrates that both proteins are present in the RNP particles of germ plasm in large oocytes and fertilized eggs, where they co-localise with Cy5-nanos1 RNA in a distribution identical to their respective GFP fusions. However, only the larger variant, XveloFL, appeared to be naturally present in the earlier mitochondrial cloud.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'On the other hand it proved possible to reduce XveloSV considerably, resulting in the consistent break-up of the germ plasm islands into smaller structures. This phenotype was produced both by an XveloSV-specific anti-sense oligo and one to the region common to both Xvelo transcripts. This suggests that Xvelo is important for the coherence of germ plasm islands.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'In contrast, the ineffective XveloFL depletion has no effect on island size in combination with antisense Rbm24b and 42b. These experiments are not strong evidence as yet, but do provide a preliminary indication that these two RNA binding proteins have a role in germ plasm structure, specifically via Hermes recruitment.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Products were gel purified and directly sequenced with the primer 5&#8242;-GAGTTTGAGCAGATGTTA-3&#8242;. To confirm the specificity of interactions, the Hermes ORF was amplified using the primers 5&#8242;-AAAGGTACCCCATGAGCGGCATCAAGTCAG-3&#8242; and 5&#8242;-AAAGGTACCTTAACAAAACTGCCGAGACTT-3&#8242;, digested with KpnI and ligated into the corresponding site within the activation domain vector pVP16 to generate pVP16-Hermes. Selected inserts from positive colonies (Table 1) were amplified and ligated into the bait plasmid pBTM116 and co-transformed with pVP16-Hermes into the yeast strain L40, then grown on selective media as described above.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Positive clones were selected on medium lacking leucine, tryptophan and histidine, in the presence of 15 mM 3-aminotriazole and checked with a filter assay for &#946;-galactosidase activity. The inserts from 13 positive prey plasmids were then amplified using VP16 vector-specific primers 5&#8242;-GAGTTTGAGCAGATGTTA-3&#8242; and 5&#8242;-TGTAAAACGACGGCCAGT-3&#8242;, which flank the NotI insertion site. Products were gel purified and directly sequenced with the primer 5&#8242;-GAGTTTGAGCAGATGTTA-3&#8242;.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'peGFPXveloFL&#916;2 was prepared using the primers 5&#8242;AAAAAGCTTATGAACACGACGGCACCTCCCCCAG-3&#8242; and 5&#8242;-AAAGTCGACTTAAGAAGATCCTTCATCCCTAGAT-3&#8242; to amplify the sequence encoding amino acid residues 1&#8211;451 of the long isoform of Xvelo1. cDNA sequences encoding amino acid residues 122&#8211;779 of the long isoform of Xvelo1 were amplified using the primers 5&#8242;-AAAAAGCTTGATCCACGGCAACAAGAATGTGCTT-3&#8242; and 5&#8242;-AAAGTCGACTTACAGTTTGCCATTCAGCTGCTTC-3&#8242; to construct peGFPXveloFL&#916;3. peGFP-XveloSV&#916;1 contained the c-terminal region of the short Xvelo isoform (XveloSV), specifically amino acid residues 205&#8211;269, this was prepared using the primers 5&#8242;-AAAAAGCTTGAAAAGCTCAAAAGAAAACCCCTGG-3&#8242; and 5&#8242;-AAAGTCGACCTACTTCTTCTGGATCAGTTCTAGT-3&#8242;, using peGFP-XveloSV as a template. All inserts were digested with HindIII/HincII and ligated into the corresponding sites in peGFPJC-N. For in vitro RNA synthesis all mutant constructs were linearised with XhoI and transcribed with SP6 RNA polymerase, using the mMESSAGE mMACHINE kit (Ambion).'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Polyclonal antibodies against XveloFL (long isoform) and XveloSV (short isoform) were raised in goats using the Eurogentec anti-peptide 28-day Speedy programme (Eurogentec, Seraing, Belgium). The antigens were synthetic peptides specific to XveloFL (H2N-QTREPRYETSKQGKQC-CONH2 and H2N-CKKNPKKSQSLSEPED-CONH2) and XveloSV (H2N-CSSLLKENKLELIQKK-COOH and H2N-CKIQPKSKSKISLQKR-CONH2). For the affinity purification of Xvelo-specific antibodies, sequences corresponding to nucleotides 864&#8211;1759 of the XveloFL cDNA and nucleotides 653&#8211;850 of the XveloSV cDNA [16], which contain the synthetic peptide sequences used for immunisation, were amplified by PCR and cloned into the bacterial expression vector pET14b to prepare pET14b-VFL&#916;1 and pET14b-VSV&#916;1 respectively.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'The antigens were synthetic peptides specific to XveloFL (H2N-QTREPRYETSKQGKQC-CONH2 and H2N-CKKNPKKSQSLSEPED-CONH2) and XveloSV (H2N-CSSLLKENKLELIQKK-COOH and H2N-CKIQPKSKSKISLQKR-CONH2). For the affinity purification of Xvelo-specific antibodies, sequences corresponding to nucleotides 864&#8211;1759 of the XveloFL cDNA and nucleotides 653&#8211;850 of the XveloSV cDNA [16], which contain the synthetic peptide sequences used for immunisation, were amplified by PCR and cloned into the bacterial expression vector pET14b to prepare pET14b-VFL&#916;1 and pET14b-VSV&#916;1 respectively. Fusion proteins were expressed in E.coli strain BL21(DE3), following the addition of 0.1 M IPTG.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'XveloFL, 5&#8242;-A*T*C*C*TTAAGTCGGC*A*C*A*C-3; XveloSV, 5&#8242;-A*G*C*T*TTTCCTCCTC*T*T*T*C-3&#8242; (specificity achieved by nine nucleotides flanking either side of the splice junction in Xvelo1 RNA [16]; XveloFL/SV, 5&#8242;-C*T*T*G*TTGCCGTGGA*T*C*A*G-3&#8242; (this targets both the long and short forms of Xvelo RNA); Rbm24b, 5&#8242;-G*T*G*T*TGCAGCAGTA*A*G*T*G-3&#8242;; Rbm42b, 5&#8242;-A*A*G*C*GCGAGCAAGG*A*T*G*T-3&#8242;, where &#8216;*&#8217; indicates phosphorothioate linkages (Invitrogen). Successful depletion of specific RNAs was assayed by semi-quantitative RT-PCR, essentially as described by Hudson et al. [44].'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'Stage VI oocytes were injected with 11 ng of each oligo, alone or in combination. XveloFL, 5&#8242;-A*T*C*C*TTAAGTCGGC*A*C*A*C-3; XveloSV, 5&#8242;-A*G*C*T*TTTCCTCCTC*T*T*T*C-3&#8242; (specificity achieved by nine nucleotides flanking either side of the splice junction in Xvelo1 RNA [16]; XveloFL/SV, 5&#8242;-C*T*T*G*TTGCCGTGGA*T*C*A*G-3&#8242; (this targets both the long and short forms of Xvelo RNA); Rbm24b, 5&#8242;-G*T*G*T*TGCAGCAGTA*A*G*T*G-3&#8242;; Rbm42b, 5&#8242;-A*A*G*C*GCGAGCAAGG*A*T*G*T-3&#8242;, where &#8216;*&#8217; indicates phosphorothioate linkages (Invitrogen). Successful depletion of specific RNAs was assayed by semi-quantitative RT-PCR, essentially as described by Hudson et al.'},\n",
       " {'pmcid': 'PMC3827131',\n",
       "  'pmid': '24265795',\n",
       "  'snippet': 'In these experiments oocytes were incubated in OCM medium within intact follicles (except for experiments where microtubules were stained with antibodies, where they were defolliculated with collagenase [45]). Endogenous XveloFL and XveloSV protein localisation experiments were performed on a fixed oocyte series (Stages I&#8211;VI) using isoform-specific, affinity purified goat polyclonals and a Rabbit Anti-Goat Dylight 594 (Jackson ImmunoResearch Laboratories, Inc) secondary antibody. Antibody staining was described by Machado et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'The specific statistical analysis used is noted below for each experimental approach. The Design Analysis Core at WFSM provided advice to the investigators, and the Design and Analysis Core performed select analyses.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Well-characterized antibodies against cell- and synapse-specific markers were used for single- or double-labeling studies. A complete list of antibodies, immunogens, manufacturers, host species, dilutions used, and references are provided in Table 1.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'To identify specific motor pools in the L3/L4 segments of the spinal cord for ultrastructural level, we initially used retrograde labeling with colloidal gold-labeled cholera toxin subunit B (CTB; List Biological Laboratories, Denver, CO); however, gold particles could not be definitely identified in MNs. Therefore, we identified the TA and soleus motor pools by retrograde labeling with alexa-fluor CTB (Invitrogen; Fig.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Dotted line outlines the VH boundary, and the box indicates location of thin section. X-Y coordinates on the electron microscope were used to map the adjacent sections and located specific MNs. MNs, motoneurons; TA, tibialis anterior; VH, ventral horn.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': \"Specific muscles were dissected from the mice used for ultrastructural analysis of the spinal cord described above and placed in fixative overnight at 4&#176;C. Muscles were cut at 600 &#956;m on a tissue chopper, then reacted for cholinesterase to reveal NJMs using Karnovsky's method with the following changes: sodium cacodylate buffer was used instead of PBS and the tissue was incubated for 1 h or until the endplates were visible under the dissecting microscope (Karnovsky, 1964). Specimens were then embedded in Araldite 502 using a Lynx processor.\"},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': '2004; Hampton and Amende 2010). Briefly, we used a motor-driven treadmill with a transparent treadmill belt (DigiGait Imaging System, Mouse Specifics, Inc., Quincy, MA). A high-speed digital video camera was positioned below the transparent belt to focus on the ventral view of subjects walking on the belt.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Additionally the toxicity of mutant SOD1 involves other cell types besides MNs and therefore is at least partly noncell (MN) autonomous. For example, cell-specific deletion of mutant SOD1 in genetically altered mice has implicated microglia and astrocytes as contributors to the progression but not the onset of disease (Clement et al. 2003; Boill&#233;e et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'A cardinal feature of most developmental and adult onset neurodegenerative diseases, including motor neuron diseases such as ALS, is the death of specific population of neurons. Largely as a result of the progress made in elucidating the cellular and molecular mechanisms underlying neuronal death during development (Oppenheim et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Denervation of NMJs by fast-fatigable (FF) MNs that innervate specific types of muscle fibers &#8211; myosin heavy chain (MyHC) Type IIB &#8211; in SOD1 fALS mice begins as early as P25 (Gould et al. 2006 (disease onset at P90-100), followed later by loss of NMJ innervation of Type IIa muscle fibers by fast-fatigue resistant (FR) MNs and lastly denervation by slow (S) MNs that innervate Type I/Ia muscle fibers (Pun et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'For example, fast-fatigable muscle fibers, the first ones denervated in the SOD1 mouse model, are innervated by the most phasic subtype of MNs. Hyperexcitability and evidence of ER stress is detected specifically in MNs innervating fast-fatigable muscles prior to initial denervation further suggesting that these events serve as stressors that continue until the neuron reaches a threshold that jeopardizes its survival (Saxena and Caroni 2011). Nonetheless, many of the events associated with cellular stress often represent a protective response by the cell and therefore do not precipitate neuronal dysfunction, but rather prevent or delay the process (reviewed in Robinson et al.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': \"While selected neuronal populations are affected in neurodegenerative diseases such as ALS, Alzheimer's and Parkinson's diseases, age is a common feature in all neurodegenerative diseases. Results from numerous studies suggest that there are common features across disease-specific populations including aggregation of misfolded proteins, altered proteasome activity and stress responses including ER stress, increased autophagy and mitochondrial changes noted above. Furthermore, patterns of resistance and susceptibility in NMJs in ALS mice are also observed in normally aging mouse muscles (Valdez et al.\"},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': '2007; Conforti et al. 2007; Gould and Oppenheim 2007) and (2) the specific molecular mechanisms mediating axon/synapse loss in ALS are largely unknown (Saxena and Caroni 2007). Mitochondrial morphological and functional changes are likely involved in disease pathology; however, alterations in synaptic input, axonal transport, ER stress, protein aggregates are also MN intracellular events that are associated with pathology.'},\n",
       " {'pmcid': 'PMC3869677',\n",
       "  'pmid': '24381807',\n",
       "  'snippet': 'Pathological events are well characterized in the ALS mouse models, but review of the literature fails to identify a specific initiating event that precipitates disease pathology. There is now a growing consensus in the field that the axon and synapses are the first cellular sites of degeneration, but there is still controversy over (1) whether axon and synapse loss is initiated autonomously at those sites or by pathology elsewhere (Bettini et al.'},\n",
       " {'pmcid': 'PMC4114201',\n",
       "  'pmid': '25120436',\n",
       "  'snippet': 'Their brainstems were taken out and post fixed in 4% PFA for 2 h. After being washed in PBS (3 &#215; 5 min), the brainstems were covered in 4% agarose and cut into 40 &#956;m thick slices. These slices were washed 3 &#215; 10 min in PBS and incubated in AB media [0.1 M phosphate buffer (PB; KH2PO4: 50 mM, Na2HPO4: 150 mM], 150 mM NaCl, 3 mM Triton-X, 1% bovine serum albumin; 1% normal goat serum) containing unlabeled Fab fragments of goat anti-mouse IgG (Jackson ImmunoResearch) overnight to enhance the specificity of the primary antibody in mouse tissue (in the case of the GlyT2 labeling, the Fab fragments were left out). After this blocking step the slices were kept in the primary antibody solution for 2 days at 4&#176;C. The primary antibodies were a purified mouse monoclonal IgG2a against GAD67 (Millipore; MAB5406; RRID: AB_2278725) and a guinea pig polyclonal antibody raised against a rat GlyT2 C-terminal region (Millipore; AB1773; RRID: AB_90953).'},\n",
       " {'pmcid': 'PMC4114201',\n",
       "  'pmid': '25120436',\n",
       "  'snippet': 'After this blocking step the slices were kept in the primary antibody solution for 2 days at 4&#176;C. The primary antibodies were a purified mouse monoclonal IgG2a against GAD67 (Millipore; MAB5406; RRID: AB_2278725) and a guinea pig polyclonal antibody raised against a rat GlyT2 C-terminal region (Millipore; AB1773; RRID: AB_90953). Specificity of both antibodies has been shown previously (Ito et al., 2007; Dufour et al., 2010). Following the incubation in the primary antibody solution, the slices were washed 3 &#215; 10 min in PBS and incubated in secondary antibody solution for 2 h. The secondary antibodies were acquired from Invitrogen (goat anti-mouse IgG conjugated with Alexa 568; A11031; RRID: AB_144696 and goat anti-guinea pig IgG conjugated with Alexa 568; A11075; RRID: AB_141954).'},\n",
       " {'pmcid': 'PMC4114201',\n",
       "  'pmid': '25120436',\n",
       "  'snippet': '(2005), as well as immunohistochemical and tracing studies from other investigators in different rodent species (MNTB, LNTB, MVPO/VNTB: Peyret et al., 1987; Helfert et al., 1989; SPN: Helfert et al., 1989; Saint Marie and Baker, 1990). Thus, the GFP expression in this mouse line is specific and reliably shows the (sub)populations of glycinergic cells in the auditory brainstem nuclei despite some variability in the level of GFP-expression between different nuclei and individual animals.'},\n",
       " {'pmcid': 'PMC4114201',\n",
       "  'pmid': '25120436',\n",
       "  'snippet': 'Lastly, Warr and Beck (1996) looked specifically at efferents of the VNTB and suggested that a projection from VNTB to MNTB &#8220;&#8230; does not appear to exist, at least in the rat, for VNTB neurons situated some distance away from the MNTB&#8230;&#8221; Our results are inconsistent with those findings from the rat, but support the findings of Kuwabara et al. (1991) in two other rodent species, the gerbil and the mouse, hinting at possible species-related differences in the intrinsic projection patterns within the SOC.'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'To quantify the number of c-Fos positive and &#946;-gal positive cells, the red (c-Fos) and green (&#946;-gal) color channels in each image were first filtered with a stringent threshold (mean +2 standard deviations of background pixel intensity level), and each color channel was then converted to a binary, black/white (BW) image using ImageBWconvertGUI- a custom-made program running in the 2013a Matlab Computing Environment with the Image Processing Toolbox (The MathWorks, Natick, MA; program available on Github: https://github.com/neuropil/ImageBWconvert/). This resulted in binary BW images representing c-Fos and &#946;-gal staining &gt;2 standard deviations of background staining for each animal. Then, the number of c-Fos positive and &#946;-gal- positive cell bodies was quantified using the cell counter plugin in ImageJ (version 1.47, Bethesda, MD).'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'However, this staining did not resemble traditional c-Fos protein nuclear staining (Figure 2, middle column, blue arrow) and was not the result of non-specific secondary antibody binding as this staining was not present when the primary antibody was omitted (data not shown). Moreover, usually high background staining, defined as non-specific immunofluorescence that was not localized to cell nuclei, was found within the nTS of MSG-stimulated FTL animals (Figure 2, middle column), but was not present in the nTS of MSG-stimulated WT control mice (tissue was processed concurrently with that of FTL mice, using the same antibodies and reagents; see Figure S38). Lastly, Fos-LI was found in the cellular processes of some c-Fos positive cells in the nTS of MSG-stimulated animals (Figure 2, middle column, inset).'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'First, basal punctate Fos-LI was seen in both unstimulated groups (Unstim and No Food/Water) as well as in the MSG 5 h Post group (Figure 2, middle column, red arrow). However, this staining did not resemble traditional c-Fos protein nuclear staining (Figure 2, middle column, blue arrow) and was not the result of non-specific secondary antibody binding as this staining was not present when the primary antibody was omitted (data not shown). Moreover, usually high background staining, defined as non-specific immunofluorescence that was not localized to cell nuclei, was found within the nTS of MSG-stimulated FTL animals (Figure 2, middle column), but was not present in the nTS of MSG-stimulated WT control mice (tissue was processed concurrently with that of FTL mice, using the same antibodies and reagents; see Figure S38).'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'However, this staining did not resemble traditional c-Fos protein nuclear staining (Figure 2, middle column, blue arrow) and was not the result of non-specific secondary antibody binding as this staining was not present when the primary antibody was omitted (data not shown). Moreover, usually high background staining, defined as non-specific immunofluorescence that was not localized to cell nuclei, was found within the nTS of MSG-stimulated FTL animals (Figure 2, middle column), but was not present in the nTS of MSG-stimulated WT control mice (tissue was processed concurrently with that of FTL mice, using the same antibodies and reagents; see Figure S38). Lastly, Fos-LI was found in the cellular processes of some c-Fos positive cells in the nTS of MSG-stimulated animals (Figure 2, middle column, inset).'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'First, other researchers report anatomical congruence between staining by Xgal reaction and &#946;-gal antibody [1], [3]. Second, the &#946;-gal antibody used in the current study was thoroughly tested for specificity in our lab with the appropriate antibody controls, including a lack of staining in LacZ negative animals, and an absence of staining when the primary antibody was omitted. Thus, the reason for this disparity is unknown.'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'However, unusual punctate, basal c-Fos protein staining was found in all unstimulated groups as well as the MSG 5 h Post group (Figure 2), and high c-Fos background label (i.e. non-specific immunofluorescence) was present in tissue sections from MSG- stimulated FTL mice (but was not present in WT control mice using the same reagents; Figure S38). Furthermore, in the MSG group, Fos-LI was found in the cellular processes of some c-Fos positive cells, in addition to stained cell bodies (Figure 2 inset).'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'Images converted to greyscale colors for clarity. Background staining is defined as &#8216;non-specific immunofluorescence that was not localized to cell nuclei.&#8217;'},\n",
       " {'pmcid': 'PMC4156431',\n",
       "  'pmid': '25192442',\n",
       "  'snippet': 'Images converted to greyscale colors for clarity. Background staining is defined as &#8216;non-specific immunofluorescence that was not localized to cell nuclei.&#8217;'},\n",
       " {'pmcid': 'PMC4157608',\n",
       "  'pmid': '25249943',\n",
       "  'snippet': 'The use of thymidine analogs has long been the most common tool for the study of cell proliferation, particularly within the nervous system (Gratzner, 1982; Taupin, 2007; Altman, 2011). More recently, the administration of two thymidine analogs discriminated by way of specific primary antibodies, along with experimental modification of the temporal windows between analogs administration (Vega and Peterson, 2005; Llorens-Mart&#237;n and Trejo, 2011), has advanced the discrimination of proliferating cell subpopulations in the brains of amniotes (Vega and Peterson, 2005; Llorens-Mart&#237;n et al., 2010), anamniotes (Grandel et al., 2006; Alunni et al., 2010) and invertebrates (Sullivan et al., 2007). To characterize the distribution and cellular composition of brain proliferation zones in adult G. omaroum, two thymidine analogs, 5&#8242; chlorodeoxiuridine (CldU; Sigma, #C-6891) and 5&#8242; iododeoxiuridine (IdU; MP Biomedicals, #100357) were administered according to two protocols (Vega and Peterson, 2005; Alunni et al., 2010).'},\n",
       " {'pmcid': 'PMC4157608',\n",
       "  'pmid': '25249943',\n",
       "  'snippet': 'The mouse monoclonal anti-BrdU antibody, Clone B44, was first obtained by immunization of mice with IdU (Gratzner, 1982). The author proved that this antibody binds with high specificity with BrdU and IdU, does not cross-react with thymidine, and is useful to detect BrdU incorporated into the DNA of individual cells due to DNA replication. The rat anti-BrdU antibody, Clone BU1/75 (ICR1) binds with BrdU and CldU but not with IdU (Bakker et al., 1991; Aten et al., 1992).'},\n",
       " {'pmcid': 'PMC4157608',\n",
       "  'pmid': '25249943',\n",
       "  'snippet': 'On the other hand, the rat anti-BrdU antibody, but not the mouse anti-BrdU monoclonal antibody, detects CldU in tissue samples from animals exposed to this thymidine analog (Vega and Peterson, 2005). Cross reactivity of primary antibodies with the thymidine analogs was tested by incubation of sections of CldU only and IdU only treated animals with anti-IdU or anti-CldU antibodies, respectively (followed by incubation in the corresponding secondary antibodies; Supplementary Figure 1).'},\n",
       " {'pmcid': 'PMC4157608',\n",
       "  'pmid': '25249943',\n",
       "  'snippet': \"Some proliferation zones were extensive (such as those of the Cb and hypothalamus) whereas others were discrete (as in TeO, TS, and ELL). The resemblance between the distribution of adult G. omarorum proliferating cells and larva's distribution (Iribarne and Castell&#243;, 2014) indicates the persistence of a specific pattern of cell proliferation throughout postnatal life in the brain of G. omarorum. As occurs in other teleosts, this might subserve the continuous brain growth that accompanies indefinite body growth, along with the maintenance of the particular brain morphology (Zupanc and Horschke, 1995; Ekstr&#246;m et al., 2001; Candal et al., 2005b; Zupanc et al., 2005; Zupanc, 2006).\"},\n",
       " {'pmcid': 'PMC4157608',\n",
       "  'pmid': '25249943',\n",
       "  'snippet': 'The implementation of the sequential administration of two thymidine analogs along with their specific and simultaneous immunohistochemical demonstration, by means of already tested antibodies and visualization by confocal microscopy, allowed us to reveal the heterogeneous cellular composition of adult G. omarorum brain proliferation zones. According to Zupanc and Horschke (1995), once a thymidine analog is administrated, it is available for its incorporation into DNA for 2&#8211;4 h. In addition, Zupanc and Zupanc (1992) estimated the duration of ventricular proliferating cells cycle in 2 days, but also demonstrated that 10&#8211;30% of ventricular cells undergo mitosis within 12 h. Thus, the protocols we employed were appropriated to label both actively cycling cells at the moment of IdU and CldU administration, in addition to cells that progressed through successive S-phases, thus revealing their proliferative history (Vega and Peterson, 2005; Llorens-Mart&#237;n and Trejo, 2011).'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'At present de- and remyelination are almost wholly characterized by histology, and remyelination is only definitively distinguished at the electron microscopic (EM) level. These assays routinely use oligodendrocyte antibodies that label antigens on myelin processes as well as cell bodies making the ability to distinguish individual cells from background staining, and thus quantification, difficult. For animal models of demyelination such as EAE these assays are time-consuming and prone to high statistical variance and subjectivity given the spatial, temporal, and inter-animal variability of the disease.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'This technique allows for analyzing protein expression on the cellular level by suspending cells from dissociated tissue in a buffered solution and analyzing single cells with an optical detection apparatus [10]. Individual cells are excited by laser light and spectral emission is detected using specific filters and a series of photomultiplier tubes (PMTs). Inherent spectral characteristics as well as proteins labeled with fluorescent dyes or antibodies can be detected and analyzed.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Female C57BL/6 and SJL/J mice were purchased from Harlan Laboratories (Bethesda, MD). The mice were housed under specific pathogen-free conditions in the Northwestern University Center for Comparative Medicine barrier facility (Chicago, IL). All protocols (2010-0357) were approved by the Northwestern University Institutional Animal Care and Use Committee in accordance with Federal Animal Welfare Regulations.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Sorted cells were stored at &#8722;80&#176;C. After thawing on ice, total RNA was isolated using &#181;MACS mRNA Isolation Kit (Miltenyi Biotec) and cDNA was synthesized on column with &#181;MACS One-step cDNA Synthesis Kit (Miltenyi Biotec). Gene expression was analyzed by qRT-PCR using specific primers (Qiagen, Valencia, CA, Table S2), RT2 SYBR Green FAST Mastermix (Qiagen), and an iQ5 real-time PCR detection system (Biorad, Hercules, CA). Results were standardized to a housekeeping gene and normalized to gene expression in the A2B5+PDGFR&#945;+ sorted early OPC population.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Thousands of cells per second are flowed in a stream of buffer through the flow cytometer apparatus. Cells are individually interrogated by laser light, and emitted light is directed through a series of filters to photomultiplier tubes for specific fluorochromes. For analysis cells are identified by their forward and side scatter of light.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Because of distinct proteins changes that occur during the maturation of the oligodendrocyte, antibodies can be utilized to characterize cells throughout the oligodendrocyte lineage. Thousands of cells per second are streamed through the flow cytometer, excited by laser light, and emitted fluorescence is directed through filters to photomultiplier tubes specific for each fluorochrome. For subsequent analysis, single cells are distinguished from small debris using forward (FSC) and side (SSC) scatter of light.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'We began by optimizing technical parameters specifically for working with CNS tissues, examining total cell recovery from the mouse brain and antigen integrity following multiple tissue-dissociating enzyme and lipid debris removal techniques. All tested enzymes returned more viable cells than manual dissociation alone ( Fig.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': '2D ). Recovery of stage-specific oligodendroglial populations was next compared by purification technique ( Fig. 2E&#8211;G ).'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': '3 ). Antibodies and the optimal concentrations that demonstrated clear positive staining and/or selectively defined a specific stage in oligodendrocyte development were chosen for validation experiments and application in mouse models of demyelinating disease ( Table 1 ). Primary antibodies were assigned to specific fluorochromes for conjugation to allow for assembly of antibody staining cocktails for surveying oligodendroglial cells throughout the lineage in a single sample.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Antibodies and the optimal concentrations that demonstrated clear positive staining and/or selectively defined a specific stage in oligodendrocyte development were chosen for validation experiments and application in mouse models of demyelinating disease ( Table 1 ). Primary antibodies were assigned to specific fluorochromes for conjugation to allow for assembly of antibody staining cocktails for surveying oligodendroglial cells throughout the lineage in a single sample.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'We first validated OLC-specific antibodies using highly purified cultures of OPCs and differentiated oligodendrocytes. Primary OPCs were isolated from postnatal mouse brains, selected for by O4+ immunopanning, and cultured in the presence of the recombinant OPC mitogen platelet-derived growth factor (PDGF).'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Representative sorting flow plots are shown. mRNA was purified and transcription levels of stage-specific oligodendroglial genes Olig2, Pdgfra, Cspg4, Plp1, Mbp, and Mog were quantified by RT-PCR. Data were normalized to a housekeeping gene and are reported relative to the A2B5+PDGFR&#945;+ sorted population.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'To further test the efficacy of the flow cytometric method for characterizing cell populations, we used a similar panel of OLC-specific antibodies to FACS sort oligodendroglial populations from adult mouse brains ( Fig. 4D ).'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Mbp, Plp1, and Mog were all significantly greater in late OPCs or mature oligodendrocytes compared to early OPCs. The genes Rbfox3 and Gfap, encoding for neuron-specific nuclear protein NeuN and astrocyte-specific GFAP respectively, were undetectable in all sorted populations.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'To prevent masking of OLC changes by subtler changes in the brain, we specifically analyzed OLCs in the spinal cord. Furthermore to prevent skewing due to immune cell infiltration, a hematopoietic marker and an exclusion gate were incorporated to specifically characterize CNS resident cells.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Similar to MS, EAE bears a strong inflammatory component characterized by mass pathogenic immune cell infiltration into the spinal cord and corresponding motor deficits [17]. To prevent masking of OLC changes by subtler changes in the brain, we specifically analyzed OLCs in the spinal cord. Furthermore to prevent skewing due to immune cell infiltration, a hematopoietic marker and an exclusion gate were incorporated to specifically characterize CNS resident cells.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Relapsing-remitting EAE was induced in SJL/J mice by priming with the PLP139&#8211;151 peptide, and mice were scored daily for clinical disease. At specific disease time points: onset, peak, remission, and relapse, a cohort of mice was sacrificed for OLC analysis in the spinal cord by flow cytometry ( Fig. 7D ).'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': '7A ). We further expanded the staining panel to incorporate labeling a sixth antigen additionally demonstrating the flexibility of the method in investigating specific populations ( Fig. 7B ).'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'The trepidation in utilizing flow cytometry likely stems from four major obstacles encountered when working with CNS tissue: complex cytoarchitecture, high lipid content, oligodendroglial antigen preservation, and oligodendroglial antibody specificity and format. In this work we have optimized a protocol to overcome each of these barriers allowing for effective analysis of OLCs.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'It was also necessary to identify a purification technique that spared OLC antigens located on the cell bodies. Unlike other enzymes that either degraded OLC antigens or affected viability of specific populations, we found that Accutase dissociation allowed for recovery of a high number of viable cells and most efficiently preserved oligodendroglial antigens. A linear 40% Percoll spin most effectively isolated both OPC and oligodendrocyte cell bodies expressing the oligodendroglial antigens of interest.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'We thus directly conjugated each antibody to a unique fluorochrome enabling the detection of eight different proteins expressed throughout the lineage in a single CNS. The ease of the antibody conjugation step allows for panels to be easily modified to incorporate different oligodendroglial or other cell markers, and the variety of available conjugating fluorochromes allows for tailoring to specific flow cytometers with different spectral configurations.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Oligodendroglial antibodies for flow cytometry should ideally meet several requirements: 1) bind antigens restricted to OLCs specifically in a narrowly defined stage in oligodendrocyte development, 2) bind epitopes on the extracellular surface of the cell body, and 3) be monoclonal antibodies of different classes and/or host species. Through titration experiments we were able to identify a number of oligodendroglial antibodies commonly used for IHC and ICC that labeled a population detectable by flow cytometry.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'To restrict labeling to CNS-resident cells, we incorporated anti-CD45 antibody to exclude hematopoietic cells from analysis. To further definitively describe stage-specific OLCs and exclude prospective binding to non-oligodendroglial cells, we defined all populations of interest by double labeling. We were able to confirm the success of this strategy in identifying precise windows in OLC development both in vitro using defined OPC and oligodendrocyte cultures, and in vivo through RNA analysis of the populations and OLC characterization during developmental myelination.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'A single researcher can objectively characterize OLC populations in individual brains and make comparisons between groups of mice or brain regions in a few days. Additionally antibody cross-reactivity with antigens in humans has enabled us to preliminarily characterize oligodendroglial populations in human tissue samples. Most importantly we demonstrated the broad utility of the method during developmental myelination and in two distinct models of demyelination.'},\n",
       " {'pmcid': 'PMC4172589',\n",
       "  'pmid': '25247590',\n",
       "  'snippet': 'Single cells isolated and purified from adult mouse CNS were sorted by FACS: A2B5+PDGFR&#945;+ early OPCs, NG2+O4+ intermediate OPCs, GALC+MOG+ mature oligodendrocytes. mRNA was purified and transcription levels of the stage-specific oligodendroglial genes listed were quantified by RT-PCR.'},\n",
       " {'pmcid': 'PMC4199601',\n",
       "  'pmid': '25330147',\n",
       "  'snippet': 'Among these gangliosides, 9-O-acetyl GD3 is well known for its role during the development of nerves and dorsal root ganglia (DRG) [2], [3]. The GD3 moiety is the direct precursor of 9-O-acetyl GD3 and is acetylated by the GD3-specific 9-O-acetyltransferase. Once 9-O-acetyl GD3 is incorporated in the lipid portion of the plasma membrane, this molecule is involved in cell division, motility, extension and death during the development and regeneration of the peripheral nervous system (PNS) and central nervous system (CNS) [4].'},\n",
       " {'pmcid': 'PMC4199601',\n",
       "  'pmid': '25330147',\n",
       "  'snippet': 'Immunoinhibition using antibodies specific to components of the 9-O-acetyl GD3 pathway alters several neuronal processes, including cell migration, both in vitro and in vivo [8], [9]. However, it is not clear whether animals exhibit a normal phenotype or altered neural function in the absence of 9-O-acetyl GD3 ganglioside.'},\n",
       " {'pmcid': 'PMC4199601',\n",
       "  'pmid': '25330147',\n",
       "  'snippet': 'We found that the peak of ganglioside expression is on day 7 after nerve crush lesioning, and its expression then gradually reduces to basal levels by day 10 after lesioning [10]. We examined selected parameters of WD, including axonal breakdown, macrophage invasion and cell proliferation, specifically including Schwann cell proliferation, at the distal stump. All of these processes acted in synchrony for Schwann cells to form the bands of B&#252;ngner, allowing axons to regenerate [29].'},\n",
       " {'pmcid': 'PMC4247200',\n",
       "  'pmid': '25416145',\n",
       "  'snippet': 'Edaravone (3-methyl-1-phenyl-2-pyrazolin-5-one), a free radical scavenger that is currently used in the treatment of acute ischemic stroke as a neuroprotective reagent, has been shown to significantly reduce the infarct size, improve neurological scores, and decrease ROS generation [6]. More specifically, it can counteract toxicity from activated microglia [7]. Neuroinflammation in middle cerebral artery occlusion (MCAO) may be attenuated by Edaravone which acts through suppression of expression of proinflammatory cytokines in activated microglia [8].'},\n",
       " {'pmcid': 'PMC4247200',\n",
       "  'pmid': '25416145',\n",
       "  'snippet': 'Coronal sections of 7&#160;&#956;m thickness were cut on a microtome (Model: 2165; Leica, Bensheim, Germany).The sections were rinsed with phosphate-buffered saline (PBS). For blocking of non-specific binding proteins, tissue sections were incubated in 5% normal goat serum diluted in PBS for 1&#160;h at room temperature (22&#8211;24&#176;C). After discarding the serum, the sections were incubated in a humidified chamber with primary polyclonal antibody iNOS (mouse monoclonal IgG 1:100) (BD Pharmingen, San Jose, CA USA; Cat.'},\n",
       " {'pmcid': 'PMC4247200',\n",
       "  'pmid': '25416145',\n",
       "  'snippet': 'All images were captured using a confocal microscope. The isotopic control confirmed the specificity of all primary antibodies used (data not shown).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'In the vertebrate retina, rods and cones serve as photoreceptor neurons that deliver light-evoked signals to dendrites of bipolar cells in the outer plexiform layer (Wassle, 2004; Li and DeVries, 2006). Each of &#8764;10 types of bipolar cells carries a different parameter of visual information and synapses with the dendrites of specific types of ganglion cell at a unique depth of the inner plexiform layer, forming a stratic structure (Famiglietti and Kolb, 1976; Wu et al., 2000). Modulated by amacrine cells ramifying horizontally within specific strata or vertically over several strata (MacNeil and Masland, 1998; MacNeil et al., 1999), complex visual signals become progressively more selectively tuned as they proceed through a series of synaptic sublamina (Sanes and Zipursky, 2010).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Each of &#8764;10 types of bipolar cells carries a different parameter of visual information and synapses with the dendrites of specific types of ganglion cell at a unique depth of the inner plexiform layer, forming a stratic structure (Famiglietti and Kolb, 1976; Wu et al., 2000). Modulated by amacrine cells ramifying horizontally within specific strata or vertically over several strata (MacNeil and Masland, 1998; MacNeil et al., 1999), complex visual signals become progressively more selectively tuned as they proceed through a series of synaptic sublamina (Sanes and Zipursky, 2010). The diverse morphological and biochemical features of amacrine cells indicate their multifaceted functions (MacNeil and Masland, 1998; MacNeil et al., 1999; Roska and Werblin, 2001; Roska et al., 2006).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Starting with the compound eye, visual signals are transmitted through a two-dimensional array of modular longitudinal structures called cartridges in the lamina and columns in the other three neuropils, which retains the spatial organization of the visual field captured by the ommatidia. In the medulla, each column is subdivided into 10 strata (M1&#8211;M10) with R7, R8, and the monopolar L1&#8211;L5 neurons of the lamina specifically terminating in one or a few of the distal (relative to brain) M1&#8211;M6 strata (Fischbach and Dittrich, 1989; Meinertzhagen and Sorra, 2001; Morante and Desplan, 2004; Takemura et al., 2008). R7 and R8 neurons terminate in M6 and M3, respectively, making the medulla the first stage of color-vision processing and simultaneously the second stage of motion-detection processing through L1&#8211;L5 neurons.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'As the largest neuropil in the Drosophila brain, the medulla contains more than 30 morphologically distinct types of neuron, having branches that are extensive yet limited to the region (Fischbach and Dittrich, 1989; Morante and Desplan, 2008; Raghu and Borst, 2011; Raghu et al., ,). Few of their arborizations display layer specificity (e.g., Dm1 and Dm6 in M1; Pm1 and Pm2 in M9), leaving open the question of whether each of the 10 medulla strata functions as a distinct unit, with its own layer-specific LNs for visual information processing, in a manner similar to mammalian amacrine cells in the inner plexiform strata.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Here, by morphologically aligning and warping 233 single LNs into a common M6 volume model defined by R7 terminals, we demonstrate that the M6 stratum contains at least two morphologically distinct families of M6-specific LNs (M6-LNs), the directional and nondirectional LNs. Each family contains a diverse population of neurons, which differ in their size, shape, location, and coverage of the retinotopic field.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Table1 lists the primary antibodies used in this study, with their sources and specificity tests on Drosophila tissue. Anti-Dscs large (Developmental Studies Hybridoma Bank; catalog No.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'S5545, RRID:AB_477522). Staining specificity in the Drosophila brain has been demonstrated by immunoreactivity of anti-5HT antibody in the tryptophan hydroxylase promoter-driven neurons (Huser et al., 2012) that are antitryptophan hydroxylase immunopositive (Park et al., 2006). Tryptophan hydroxylase is the key enzyme for 5HT biosynthesis.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'A2052, RRID:AB_477652). Staining specificity in the Drosophila brain has been demonstrated in the lamina C2 neurons by colocalization of immunopositive signals from the anti-GABA antibody with those from antibodies against glutamic acid decarboxylase, the key enzyme for GABA biosynthesis (Kolodziejczyk et al., 2008) and Drosophila vesicular GABA transporter (Enell et al., 2007).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'This antiserum was affinity purified and found to recognize a single band of DVGLUT of the expected molecular weight (2.3 kD) in Western blots (Daniels et al., 2004). Its specificity was further confirmed by the finding of a reduced immunopositive signal in a hypomorphic dvglut allele expressing low levels of DVGLUT protein (Daniels et al., 2006).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'The GMR-GAL4 expressing pattern was manually segmented in Amira 4.1.2 and compiled into the common 3D framework of our representative M6 volume model, as described above. Global and local registrations were performed as described previously (Chiang et al., 2011), by applying the automated global affine registration and the optic-lobe-specific local registration. The registration precision was estimated to be 3.9&#8201;&#177;&#8201;0.4 &#956;m after global registration and 1.1&#8201;&#177;&#8201;0.2 &#956;m after local registration (Chiang et al., 2011).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Here we describe our procedure for the analysis of spatial relationships between R7/R8 terminals and M6-LNs in the medulla. First, we generated a volume model of the M6 stratum, defined by the array of terminals of specific kinds of columnar neurons. We then generated an internal retinotopic map, by intersecting (displaying the overlaps of) the M6 volume model and the terminals of photoreceptor neurons.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'The medulla has previously been subdivided into 10 neuropilar strata, based on neurite arborizations of selected input neurons at a specific stratum (Fischbach and Dittrich, 1989). To identify the boundary of each specific medulla stratum, we used the following drivers as references: Rh3-LexA, which labels both short- and long-form R7 neurons terminating at the middle and at the lower boundary of M6, respectively (Fischbach and Dittrich, 1989); Rh5-GAL4, which labels R8 neurons terminating at M3; c202a-GAL4, which labels L1 neurons terminating at M1 and M5; and 21D-GAL4, which labels L2 neurons terminating at M2 (Fig.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': '1A). These landmark references allowed us to identify reliably specific distal medulla strata directly from brains immunostained with the postsynaptic density marker anti-DLG. M7, also referred to as the serpentine layer, is the stratum with the weakest intensity of anti-DLG immunostaining, separating the distal medulla (M1&#8211;M6) from the proximal medulla (M8&#8211;M10).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'The medulla has previously been subdivided into 10 neuropilar strata, based on neurite arborizations of selected input neurons at a specific stratum (Fischbach and Dittrich, 1989). To identify the boundary of each specific medulla stratum, we used the following drivers as references: Rh3-LexA, which labels both short- and long-form R7 neurons terminating at the middle and at the lower boundary of M6, respectively (Fischbach and Dittrich, 1989); Rh5-GAL4, which labels R8 neurons terminating at M3; c202a-GAL4, which labels L1 neurons terminating at M1 and M5; and 21D-GAL4, which labels L2 neurons terminating at M2 (Fig. 1A).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'A long-form R7 (arrowhead) defines the proximal boundary of M6. B: Identification of specific strata in the distal medulla (M1&#8211;M6) based on differences in DLG-immunostaining intensity (left), using specific columnar neurons as references (right). C: Segmentation of the M6 stratum (magenta) and medulla (yellow) from the DLG-immunostained optic lobe (gray).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'A specific driver of M6-LNs. A: Expression pattern of 30A-GAL4 (magenta).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Variability and domain specificity of nondirectional M6-LNs. A: M6-LN1.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': '11C,D). Brainbow labeling of multiple VT12760-GAL4 neurons with different colors in the same brain confirmed that M6-LNs exhibit individual variability limited within the same domain and domain specificity between different clusters (Fig. 11E&#8211;H).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': '10, Table12; n&#8201;=&#8201;33). In addition, individual M6-LNs in the same cluster covered a similar number of retinotopic columns, with extensive areal overlapping between their axonal arbors, but made different choices for which specific columns to innervate (Fig. 11A,B).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'D: Merged skeletal models of three M6-LN12s. LNs in the same cluster show domain-specific distribution, grouped cell bodies, and overlapping and complementary innervations of the M6 retinotopic columns. Arrowhead indicates the cell body.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Variability and domain specificity of nondirectional M6-LNs in VT12760-GAL4. A: Variability of nondirectional M6-LN11.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'One possible explanation for this discrepancy is that Rh3-LexA and Rh6-GAL4 drivers report the neuron type but not the opsin type. Alternatively, there might be two subsets of DRA R8s with different layer specificity, i.e., Rh3-positive R8s terminating at M3 and Rh6-positive R8s terminating at M6.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': \"It will be interesting to determine whether directional M6-LNs pointing in the opposite direction exist for visual information integration and reiteration, similar to those in the protocerebral bridge, and how the directionality of information flow in M6 affects the fly's polarotaxis and phototaxis behavior. A more comprehensive anatomical mapping of M6-LNs and other layer-specific LNs will help us to understand why the fly has a better phototactic response to UV than to visible light (Hu and Stark, 1977).\"},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'However, some ventral M6-LNs derived from the FlyCircuit database have only one example (Table2), and their neurotransmitters are unclear. Further studies of specific Gal4 drivers expressed in the ventral M6-LNs are necessary to determine whether specifically GABAergic M6-LNs cover the entire M6 retinotopic field.'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'By aligning 72 single neurons derived from different brains to the common M6 volume model and by Brainbow labeling multiple neurons in the same brain, we showed that nondirectional M6-LNs are highly variable in their morphology and spatial distribution but could be grouped into distinct clusters, each cluster covering a specific M6 domain (Figs. 5, 6, 10).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': '8, 9), it is tempting to speculate that such a domain-specific inhibitory local field might be useful in preventing signal saturation from a bright spot, without compromising sensitivity to the surrounding visual field. Alternatively, such domain-specific inhibition in the visual field could serve as lateral inhibition in sharpening the input signal, similarly to the role of medium-field amacrine cells in the vertebrate retina (Werblin, 2011).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Because many, if not all, of these nondirectional M6-LNs are anti-GABA immunopositive (Figs. 8, 9), it is tempting to speculate that such a domain-specific inhibitory local field might be useful in preventing signal saturation from a bright spot, without compromising sensitivity to the surrounding visual field. Alternatively, such domain-specific inhibition in the visual field could serve as lateral inhibition in sharpening the input signal, similarly to the role of medium-field amacrine cells in the vertebrate retina (Werblin, 2011).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'For Drosophila, three classes of medulla neurons have been previously described, 1) medulla intrinsic neurons (Mi) that connect the distal with the proximal medulla neuropil, 2) distal medulla amacrine cells (Dm) that arborize exclusively in the distal medulla, and 3) proximal medulla amacrine cells (Pm) that branch exclusively in the proximal medulla (Fischbach and Dittrich, 1989). At the single-cell level, each of these types innervates specific medulla layers and exhibits diverse sizes of field coverage within each layer (Fischbach and Dittrich, 1989; Bausenwein et al., 1992; Bausenwein and Fischbach, 1992; Morante and Desplan, 2008).'},\n",
       " {'pmcid': 'PMC4265792',\n",
       "  'pmid': '24782245',\n",
       "  'snippet': 'Intriguingly, except for a few long-form R7 terminals, most R7 terminals locate close to, but separate from, M6-LN fibers. Specific contact with long-form, but not short-form, R7s is consistent with the results of the previous Golgi study showing two potential R7 subtypes terminating at different depths in the medulla (Fischbach and Dittrich, 1989). This observation is of potential interest because it implies subdivision of R7s into two morphologically and functionally distinct types.'},\n",
       " {'pmcid': 'PMC4278744',\n",
       "  'pmid': '25546454',\n",
       "  'snippet': 'C: Nissl staining of coronal sections of the brain at 7 d post-stroke. The left rostral and caudal forelimb motor cortex show specific injuries. The mean injury volume was 5.13&#177;1.23 mm3 (n&#8202;=&#8202;6).'},\n",
       " {'pmcid': 'PMC4278744',\n",
       "  'pmid': '25546454',\n",
       "  'snippet': 'In other words, we might have extracted solutions containing membrane-enriched fractions of both cell membranes, and also dendrite shafts. As we can specifically analyze the KCC2-positive area in the cell membrane by immunohistochemical analysis, we determined that this approach was more sensitive than western blot analysis.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'Using intrinsic chloride-dependent Vmem shift from depolarizing to hyperpolarizing in developing neurons (Ehrlich et&#160;al. 1999), cortical shape and connectivity were controlled with a Vmem-specific manner using Ivm. The data reveal that changes in gross cortical anatomy can be recapitulated at the cellular level, and suggest Vmem as a useful tool to establish in vitro models for both normal and diseased cortex.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'Ivm can change the Vmem specifically by binding to Glycine Receptor Chloride Channel alpha-1 subunit. That makes Ivm a specifically targeted and a convenient tool to change the Vmem in cells expressing the channel (Sharmeen et&#160;al. 2010; Blackiston et&#160;al.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'Ivm is known to change the cellular chloride content. Ivm can change the Vmem specifically by binding to Glycine Receptor Chloride Channel alpha-1 subunit. That makes Ivm a specifically targeted and a convenient tool to change the Vmem in cells expressing the channel (Sharmeen et&#160;al.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'The total number of projections emerging from single neurons (arrows) was counted manually in the fluorescent (beta III tubulin/AlexaFluor488) images taken from control (Fig.2A) or Ivm-treated 3w cocultures (Fig.2B). Depolarization of neural Vmem was verified by ratiometric measurement of Vmem-specific dye di-8-ANEPPS in control (no Ivm) and in Ivm-treated 3w cocultures. Accuracy of di-8-ANEPPS dye was verified by whole-cell patch clamp measurements as shown in the methods section.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'The relative Vmem of cells was measured using di-8-ANEPPS dye (Fig.5A). Immature neurons were characterized using specific markers vimentin and GFAP (Fig.5B). Phase-contrast images were collected of the same cells before- and after Ivm exposure at given time points for quantification.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'Ivm causes membrane hyperpolarization in 1&#160;day in vitro cultures of E18 rat primary cortical cells. (A) Cell membrane potential was detected colorimetrically using a specific dye Di-8-ANEPPS. **P&#160;&#8804;&#160;0.01.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'In another study, we could rescue the downstream effects of Ivm depolarization such as mitochondrial transmembrane potential changes by blocking the gap junctions. This demonstrates that the Vmem itself, not specific ion flux, controls the downstream effects (Ozkucur et&#160;al., in review).'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': \"2013). It has been also shown that Alzheimer's disease affects specific and connected neural networks via synaptic terminals affecting brain connectivity well before neuronal loss and compartmental atrophy (D'Amelio and Rossini 2012). Moreover, an abnormal connectivity and impaired cognition was found in Hyperekplexia and in Stiff Person Syndrome.\"},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': 'However, the underlying mechanisms of these anatomical abnormalities in diseased cortices cannot be gauged at the cellular level via these currently available diagnostic tools. Understanding the cell physiological basis of brain diseases is crucial to specifically characterize the nature of a given disorder. This is needed in order to establish realistic models of neurological diseases in the laboratory, which can then be employed for diagnostic and therapeutic purposes.'},\n",
       " {'pmcid': 'PMC4321392',\n",
       "  'pmid': '25722947',\n",
       "  'snippet': '2011). In the monkey brain, cell density in distinct regions can predict the laminar-specificity of connectivity between two regions (Barbas et&#160;al. 2005).'},\n",
       " {'pmcid': 'PMC4328392',\n",
       "  'pmid': '25420775',\n",
       "  'snippet': 'We sought to revisit this question by using recombinant adeno-associated viruses (rAAVs) for anterograde tracing. Specifically, we expressed enhanced green fluorescent protein (EGFP) fused to synaptophysin to augment visualization of presynaptic terminals (Nakata et al., 1998; Li and Murthy, 2001; Oh et al., 2014). We thus aimed to target distinct raphe nuclei with small injections to establish the precise termination patterns of ascending projections in the olfactory bulb, specifically with respect to the distinct layers.'},\n",
       " {'pmcid': 'PMC4328392',\n",
       "  'pmid': '25420775',\n",
       "  'snippet': 'Specifically, we expressed enhanced green fluorescent protein (EGFP) fused to synaptophysin to augment visualization of presynaptic terminals (Nakata et al., 1998; Li and Murthy, 2001; Oh et al., 2014). We thus aimed to target distinct raphe nuclei with small injections to establish the precise termination patterns of ascending projections in the olfactory bulb, specifically with respect to the distinct layers.'},\n",
       " {'pmcid': 'PMC4328392',\n",
       "  'pmid': '25420775',\n",
       "  'snippet': 'Furthermore, combining data from the two injection protocols revealed that the relative amount of labeled fibers in the GCL and GL corresponds tightly to the ratio of infected neurons in the DRN and MRN neurons (r&#8201;=&#8201;0.84, P&#8201;=&#8201;0.017, n&#8201;=&#8201;7 animals; Fig. 3F), indicating that the termination pattern of afferents from the raphe is nucleus-specific, and only the MRN projects to the GL (Fig. 3G).'},\n",
       " {'pmcid': 'PMC4328392',\n",
       "  'pmid': '25420775',\n",
       "  'snippet': 'The nucleus-specific innervation pattern we observe here is reminiscent of largely nonoverlapping projection patterns from the raphe to other brain areas (Steinbusch, 1981; Jacobs and Azmitia, 1992). In addition, the GL and the GCL likely implement distinct functions within the OB, for example gating incoming signals in the GL (Petzold et al., 2009), or transforming signals on different timescales (Fukunaga et al., 2012, 2014; Linster and Fontanini, 2014).'},\n",
       " {'pmcid': 'PMC4328392',\n",
       "  'pmid': '25420775',\n",
       "  'snippet': 'In addition, the GL and the GCL likely implement distinct functions within the OB, for example gating incoming signals in the GL (Petzold et al., 2009), or transforming signals on different timescales (Fukunaga et al., 2012, 2014; Linster and Fontanini, 2014). Whether the distinct projection patterns by the raphe translate to modulation of specific combination of areas or circuits within a region, and as a result distinct sets of functions, remains to be investigated. Our results indicate that at least in the OB, addressing layer selective neuromodulation may be possible by selective manipulation of distinct raphe nuclei.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'Four and a half LIM protein 1 (FHL1) is a protein that is highly expressed in skeletal muscle [43] and significant recent interest has focused on FHL1 and its role in the maintenance of healthy muscle since the discovery of FHL1 mutations as the cause of human muscle disease [44&#8211;47]. We have previously reported wild type FHL1 promotes myoblast fusion in vitro and skeletal muscle-specific FHL1-transgenic mice exhibit enhanced muscle growth (hypertrophy) via activation of calcineurin/NFAT signaling [48]. Therefore, we investigate here whether FHL1 could enhance myoblast fusion in diseased muscle.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'FRG1-transgenic mice [2] were obtained from Professor Rossella Tupler (University of Massachuetts Medical School, Worcester, MA) and maintained by breeding to C57BL6/J mice. Skeletal muscle-specific FHL1-transgenic mice were generated as previously described [48] using the human skeletal actin (HSA) promoter and were maintained by breeding to FVB/N wild type mice. FRG1 mice over-expressing FHL1 were generated by breeding male FRG1 mice with female FHL1-transgenic mice to generate wild type, FRG1 and FRG1/FHL1 littermates for analysis.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'Several pIRESneo3-HA-FRG1 and pIRESneo3-HA clones were selected and amplified for further analysis. To confirm overexpression of HA-FRG1, cell lysates were prepared using Tris Saline pH 7.4 and 1% Triton X-100 and immunoblotted with HA (Covance, MMS-101R, 1:5000) and FRG1 (Abcam, 55024, 1:500)-specific antibodies. Lysates from C2C12 myoblasts transiently expressing HA-FRG1 (pCGN-FRG1) were used as a positive control.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'For Western blot analyses, Triton X-100&#8211;soluble lysates were prepared as described [52]. Protein concentration was measured using a DC protein assay reagent (Bio-Rad Laboratories), and 25&#956;g lysates were separated by SDS-PAGE and immunoblotted with antibodies specific for myogenin [48] (Santa Cruz Biotechnology Cat# sc-12732, RRID:AB_627980, 1:500); MHC [53] (Developmental Studies Hybridoma Bank Cat# MF 20, RRID:AB_2147781, 1/500) and &#946;-tubulin [54] (Life Technologies Cat# 480011, RRID:AB_10375603, 1:5000). Ponceau red staining of membranes was used to confirm equal protein loading.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'The tibialis anterior, quadriceps, triceps and trapezius muscles were dissected, minced, and homogenized on ice for 5 &#215; 30 s (Tissue Rupter, Qiagen) in 5 times the weight/volume of 1% NP-40, Tris-HCl buffer, pH 8, then extracted for 1 h at 4&#176;C. Lysates were centrifuged at 1,000g for 5 minutes. Protein concentration was determined on the soluble fraction using a DC protein assay kit (Biorad), and 10&#956;g of soluble protein was separated by SDS-PAGE and immunoblotted with antibodies specific for HA [48] (Covance Research Products Inc Cat# MMS-101R-500, RRID:AB_10063630, 1:5000), FRG1 (Abcam Cat# ab55024, RRID:AB_941653, 1:500), FHL1 [56] (Abcam Cat# ab23937, RRID:AB_732361, 1/1000), Eid3 [57] (Santa Cruz Biotechnology Cat# sc-167738, 1/500) and &#946;-tubulin. Band signal intensities were determined as described above.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'Optimum muscle length (Lo) was determined from maximum isometric twitch force (Pt), and maximum isometric tetanic force (Po) was recorded from the plateau of a complete frequency-force relationship. Specific force (sPo) was determined by normalizing Po to muscle cross-sectional area (CSA) and expressed in kN/m2. The protocol for assessment of muscle fatigability was assessed using a standard fatigue protocol.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'We selectively examined the effect of FRG1 overexpression on myogenesis by generating several C2C12 myoblast cell lines stably over-expressing HA-tagged FRG1. Immunoblot analysis using a HA-specific antibody confirmed expression of HA-FRG1 in two independent clones (Clones 13 and 16), which were selected for further analysis (Fig. 1A).'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '1B). Myoblast fusion was evaluated by co-staining cultures for a myotube-specific marker myosin heavy chain (MHC), and ToPro 3-iodide to detect nuclei (as previously) [52]. Control myoblasts fused to form long, multinucleated MHC-positive myotubes that contained an average of 7 or more nuclei after 96 hours differentiation (Fig.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '(A) Immunoblot analysis of FRG1 expression in C2C12 myoblasts expressing HA-vector or HA-FRG1. HA-tagged FRG1 was detected using a HA-specific antibody. Clones HA-FRG1 13 and HA-FRG1 16 were selected for further analysis.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'We have previously reported that the LIM-only protein, FHL1, promotes myoblast fusion in vitro and skeletal muscle hypertrophy in vivo by enhancing NFAT transcriptional activity [48]. To determine if FHL1 expression can reduce muscle wasting in FRG1 mice, we crossed our skeletal muscle-specific FHL1-transgenic mice [48] with the dystrophic FRG1 mouse model [2]. A previous study has demonstrated that the effect of FRG1 overexpression on muscle is dose-dependent [2].'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '2B). QRT-PCR analysis using human-specific primers (to distinguish transgene-derived human transcripts from endogenous murine transcripts) confirmed a &#8764;5&#8211;9-fold increase in FRG1 mRNA across various muscles in FRG1 and FRG1/FHL1 mice (Fig. 2C).'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '2A-B). The expression of transgenic HA-tagged FHL1 in FRG1/FHL1 mouse muscles was detected by immunoblotting with a HA-specific antibody. Increased FHL1 protein expression was further confirmed using a FHL1 antibody, which detects both the endogenous and transgene-derived FHL1, and revealed a modest 4&#8211;5 fold increase in FHL1 expression in FRG1/FHL1 muscle (Fig.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '(A) Immunoblot analysis of protein expression in the tibialis anterior, quadriceps, triceps and trapezius muscles from wild type, FRG1 and FRG1/FHL1 mice. HA-tagged FHL1 was detected using a HA-specific antibody; &#946;-tubulin immunoblotting and ponceau red staining of membranes were used as loading controls. (B) Relative FRG1 and FHL1 expression levels in the tibialis anterior, quadriceps, triceps and trapezius muscles from wild type, FRG1 and FRG1/FHL1 mice.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '4A and 4D, middle panels). We next examined specific features of the dystrophic phenotype by examining the quadriceps and triceps muscles, tissues representing high and intermediate degrees of muscle disease in the FRG1 mouse respectively [2]. These muscles, particularly the quadriceps, have been used in several previous studies to characterize the dystrophic phenotype of the FRG1 mouse including the examination of satellite cell and myoblast function [33].'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'Analysis of several parameters was undertaken to compare muscle function in wild type, FRG1 and FRG1/FHL1 mice including, maximum force, specific (normalized) force, frequency force relationship and resistance to fatigue (n &#8805; 5 mice per genotype). For all parameters measured, a marked decrease in muscle function in FRG1 mice was observed compared to wild type, however no improvement was observed in FRG1/FHL1 mice (S2 Fig.).'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '(A) Transverse muscle sections from the triceps of wild type, FRG1 and FRG1/FHL1 mice (aged 6- and 12-weeks) were co-stained with a satellite cell specific marker (pax7) and DAPI to detect nuclei. Arrows indicate pax7+ satellite cells.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'It was recently reported that the histone methyltransferase Suv4&#8211;20h1 is a gene-specific repressor required for myogenic differentiation [35]. FRG1 over-expression interferes with the repressive action of Suv4&#8211;20h1 leading to aberrant upregulation of the inhibitor, EP300 interacting inhibitor of differentiation 3 (Eid3), resulting in myoblast differentiation defects [35].'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'FRG1 and FHL1 are under control of the human skeletal muscle actin (HSA) promoter, which should only be active in mature muscle fibers or myoblasts post induction of differentiation [79,80]. In our study, qRT-PCR analysis using human-specific primers (to distinguish from endogenous murine transcripts) confirmed that both transgene-derived FRG1 and FHL1 mRNA were not induced in myoblast cultures until 48 hours differentiation consistent with the reported activation of the HSA promoter at this time point. We observed a 2&#8211;3-fold increase in FRG1 mRNA in all differentiating myoblast cultures from FRG1 and FRG1/FHL1 mice (Fig.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': 'Maximum force (A), specific (normalized) force (B), frequency force relationship (C), and resistance to fatigue expressed both as a percentage of initial force (D), and as raw force (E) of TA muscles from 8-week old wild type, FRG1 and FRG1/FHL1 mice, measured in situ. Data represent the mean &#177; SEM from n &#8805; 5 mice per genotype; ns not significant; *p&lt;0.05; ***p&lt;0.0005 determined by two-tailed Student&#8217;s T-test.'},\n",
       " {'pmcid': 'PMC4335040',\n",
       "  'pmid': '25695429',\n",
       "  'snippet': '(A) Transverse muscle sections from the quadriceps of FRG1 and FRG1/FHL1 mice (aged 6- and 12-weeks) co-stained with a satellite cell specific marker (pax7) and DAPI to detect nuclei. Arrows indicate pax7+ satellite cells.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'The principal way to tackling these questions has been to compare brain size and structure across multiple species with divergent ecologies (Felsenstein, 1985; Harvey and Pagel, 1998; Strausfeld, 2012). Studies of vertebrate brain evolution have benefited from large interspecific datasets (Stephan et al., 1981; Iwaniuk et al., 2004) to test hypotheses about developmental constraints (Barton and Harvey, 2000; Finlay and Darlington, 1995) and adaptive hypotheses concerning, for example, sensory adaptations (Barton et al., 1995) or social ecology (Dunbar, 1992). The assumptions and methods used in some of these studies have, however, been critiqued, along with calls for more experimental manipulation to support the results of comparative studies (Healy and Rowe, 2007).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': \"This work has identified potential adaptations to species-specific ecologies (Heinze and Reppert 2012; Heinze et al., 2013; El Jundi et al., 2009b; O'Donnell and Molina 2011; Streinzer et al., 2013) and environmentally induced plasticity (Withers et al., 1993; Snell-Rood et al., 2009; Ott and Rogers, 2010; Heinze et al., 2013). With some exceptions (e.g., Kondoh et al., 2003; Farris and Roberts, 2005; O'Donnell et al., 2011; Streinzer et al., 2013), these studies focused on specific questions concerning intraspecific variation, and as such any interspecific comparisons have considered distantly related species often with radically different ecologies.\"},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': \"However, comparative analyses in insects have so far largely focused on particular neural pathways or traits in a single or small number of model species (Brandt et al., 2005; Kurylas et al., 2008; El Jundi et al., 2009a,b; Kvello et al., 2009; Dreyer et al., 2010; Heinze and Reppert, 2012; Heinze et al., 2013). This work has identified potential adaptations to species-specific ecologies (Heinze and Reppert 2012; Heinze et al., 2013; El Jundi et al., 2009b; O'Donnell and Molina 2011; Streinzer et al., 2013) and environmentally induced plasticity (Withers et al., 1993; Snell-Rood et al., 2009; Ott and Rogers, 2010; Heinze et al., 2013). With some exceptions (e.g., Kondoh et al., 2003; Farris and Roberts, 2005; O'Donnell et al., 2011; Streinzer et al., 2013), these studies focused on specific questions concerning intraspecific variation, and as such any interspecific comparisons have considered distantly related species often with radically different ecologies.\"},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'However, like moths, butterflies respond to floral and host-plant related odors (Andersson and Dobson, 2003; &#212;mura and Honda, 2009; Carlsson et al., 2011,2013), and chemical communication does play an important role in mating behavior (Scott, 1973; Andersson et al., 2007; Costanzo and Monteiro, 2007). The continued importance of olfaction in butterflies is further indicated by the similar numbers of olfactory receptor genes identified in the butterfly and moth genomes sequenced to date (Xia et al., 2004; Zhan et al., 2011; Dasmahapatra et al., 2012), and by the occurrence of sex-specific expression patterns of these genes in both groups (Nakagawa et al., 2005; Wanner et al., 2007; Briscoe et al., 2013). Conversely, some moths, including the crepuscular M. sexta, have greater visual acuity than others (Theobald et al., 2010) and may favor visual cues over olfactory cues when foraging (Raguso and Willis, 2005; Goyret et al., 2007).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Chemical communication is therefore of heightened importance for ithomiines, both at close range and over larger spatial scales. Indeed, this pattern is predicted in M&#252;llerian mimicry complexes: where visual communication cues may be unreliable, chemical communication can reduce male&#8211;male or interspecific pursuits (Brower et al., 1963; Scott, 1973).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Finally, Ithomiinae are the sister-clade to Danainae, the subfamily to which D. plexippus belongs (Freitas and Brown, 2004). The phylogenetic position of ithomiines therefore offers an improved comparison with D. plexippus to identify traits shared by diurnal species, or specific to either subfamily.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'In this article we test two hypotheses: 1) that the derived mating behavior, sex-specific motivation to locate PA sources, and sex-specific chemical communication mediated by hind-wing hair-pencils result in sexual dimorphism in the antennal lobes of G. zavaleta that is not observed in other butterflies; and 2) that the generally enhanced role of olfaction in G. zavaleta is supported by an investment in sensory neuropil intermediate between moths and D. plexippus. In addition, by describing the general layout of the brain we lay the foundations for future comparative studies across a greater number of species.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'The 3C11 antibody was raised against a bacterially expressed fusion protein generated by adding a glutathione S-transferase (GST)-tag to a cDNA comprising most of the 5&#8242; open reading frame 1 of the Drosophila melanogaster synapsin gene (Syn, CG3985). The binding specificity of this antibody has been characterized in D. melanogaster by Klagges et al. (1996) and the epitope was later narrowed down to within LFGGMEVCGL in the C domain (Hofbauer et al., 2009).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Normal goat serum (NGS; New England BioLabs, Hitchin, Hertfordshire, UK) and antibodies were diluted in 0.1 M phosphate-buffered saline (PBS; pH 7.4) containing 1% DMSO and 0.005% NaN3 (PBSd). Nonspecific antibody binding was blocked by preincubation in 5% NGS (PBSd-NGS) for 2 hours at RT. Antibody 3C11 was then applied at a 1:30 dilution in PBSd-NGS for 3.5 days at 4&#176;C under agitation.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'These two approaches to calculating relative size make different assumptions about the underlying interspecific allometric relationships and their functional interpretation (Montgomery, 2013). Using percentage volumes assumes a 1:1 allometric relationship, which is rarely the case, and does not account for the nonindependence of data.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'For Heliothis the AL was measured without the CFN, the data presented are &#8220;corrected&#8221; to estimated total AL volume using the ratio of total glomerular volume/total AL volume from G. zavaleta. This correction does not affect the results of interspecific analyses.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Heinze and Reppert (2012) noted a number of unusual features in the optic lobes of D. plexippus that may reflect adaptations to their general ecology or perhaps specifically to their migratory behavior. First, they observed an inner rim of intense synapsin staining in the lamina.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': '4H&#8211;L). The presence of the strap in both species suggests that it is not specifically linked to long-distance navigation. Rather, it may be a butterfly synapomorphy, since moths reportedly lack the strap (Heinze and Reppert, 2012).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Again, in terms of percentage volume, the antennal lobes and the mushroom body of G. zavaleta are much closer to the moths than D. plexippus (Table3). This pattern is borne out in a multivariate principal component (PC) analysis measuring relative size as deviation from interspecific allometric relationships with total brain size (Fig. 7).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Based on their size, location, and the fibrous appearance of their internal structure, we tentatively label these four glomeruli as a macroglomerular complex analogous to the MGCs present in a wide variety of moths (Bretschneider, 1924; Rospars, 1983; Berg et al., 2002; Huetteroth and Schachtner, 2005; Masante-Roca et al., 2005; Skiri et al., 2005; El Jundi et al., 2009b; Kazawa et al., 2009; Varela et al., 2009). Through an interspecific analysis within Lepidoptera, we further show the adaptation to greater olfactory dependence extends to the major sensory neuropils, with the relative size of the visual and olfactory neuropil of G. zavaleta being intermediate between night-flying moths and the diurnal monarch butterfly. In the following discussion we explore the selection pressures that may have favored these adaptations, the potential relationship of the G. zavaleta MGC to the MGC in moths and other insects, and the strength of constraints that may limit the effect of selection on brain composition.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'As the estimated average lifespan across ithomiines is &#8764;10 weeks and potentially up to 25 weeks (Drummond, 1976), enhancing foraging efficiency with olfactory adaptations may be more efficient than learning the location of transient resources. It is conceivable that selection for increased foraging efficiency targeted the antennal lobe in a sex-specific manner. Although less common, macroglomeruli associated with nonsexual pheromones are found in some taxa (Galizia and R&#246;ssler, 2010; Hansson and Stensmyr, 2011), most notably in leaf cutter ants (Atta vollenweideri, Atta sexdens) where the single macroglomerulus processes trail-pheromones (Kleineidam et al., 2005), and in Drosophila sechellia, where large glomeruli are involved in specialization to a single food source (Dekker et al., 2006).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Whether structurally homologous glomeruli have had an unbroken functional history in pheromone detection, and whether this function was always associated with male-specific enlargement are separate questions, as enlargements could occur in parallel in structurally homologous glomeruli. Given the phylogenetic position of the butterflies examined thus far (Freitas and Brown, 2004; Wahlberg, 2006; Regier et al., 2013) it seems most likely that the male-specific enlargement of particular glomeruli into an MGC is a secondarily derived trait of G. zavaleta among butterflies, and an example of convergence at the level of the Lepidoptera. Whether the putative MGC in G. zavaleta is homologous to the MGCs in moths at the level of \"structural correspondence\" is less clear.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': \"The number of glomerular units that comprise the MGC in moths varies between species, partly but not wholly dependent on the number of chemical components in the species' pheromone blend (Hansson et al., 1991; Todd et al., 1995; Berg et al., 1998). Structural homology may therefore be limited to a core set of glomeruli, or a single glomerulus, and the associated neural circuitry, around which the MGC evolves in a taxa-specific manner.\"},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Thus, at any given phylogenetic level, one may ask whether MGCs are homologous as structures, that is, linked through unbroken phylogenetic presence to a set of glomeruli in the last common ancestor. Whether structurally homologous glomeruli have had an unbroken functional history in pheromone detection, and whether this function was always associated with male-specific enlargement are separate questions, as enlargements could occur in parallel in structurally homologous glomeruli. Given the phylogenetic position of the butterflies examined thus far (Freitas and Brown, 2004; Wahlberg, 2006; Regier et al., 2013) it seems most likely that the male-specific enlargement of particular glomeruli into an MGC is a secondarily derived trait of G. zavaleta among butterflies, and an example of convergence at the level of the Lepidoptera.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'That male-specific MGCs can evolve rapidly and independently even at the family and genus level is witnessed by the report that an MGC has evolved independently multiple times in the two Drosophilid genera, Drosophila and Scaptomyza (Kondoh et al., 2003). Among Hawaiian species, it is the same two (quite uncontentiously homologous) glomeruli, DA1 and DL3, that have been independently enlarged in males in multiple lineages, a case of parallelism of enlargement of homologous structures.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'Among Hawaiian species, it is the same two (quite uncontentiously homologous) glomeruli, DA1 and DL3, that have been independently enlarged in males in multiple lineages, a case of parallelism of enlargement of homologous structures. But within D. melanogaster specifically, enlargement affects DA1 and VA1v, rather than DL3 (Kondoh et al., 2003). Therefore, even within one genus, structurally nonhomologous glomeruli can show convergent male-specific enlargement.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'But within D. melanogaster specifically, enlargement affects DA1 and VA1v, rather than DL3 (Kondoh et al., 2003). Therefore, even within one genus, structurally nonhomologous glomeruli can show convergent male-specific enlargement. Similarly, five species of bombycid moths share a core set of two sexually dimorphic macroglomeruli, the toroid and cumulus, to which two additional glomeruli have been co-opted into the MGC in Rondotia and Bombyx (Namiki et al., 2014).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'One might define homologous glomeruli according to the orthology of their associated olfactory receptors (ORs), on the basis that the gene-regulatory link between OR gene expression and glomeruli formation might couple gains and losses in OR genes to concomitant gains and losses of glomeruli (see below). However, in Drosophila OR expression is not required for glomerulus-specific axonal targeting, which is instead controlled by separate mechanisms (Dobritsa et al., 2003; Hummel et al., 2003; Hong and Luo, 2014). It therefore seems conceivable that OR gene expression can be switched out for expression of a nonorthologous OR in a structurally homologous glomerulus.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'The observed interspecific variation in antennal lobe size contrasts with the relative constancy of glomerular number. Across Lepidoptera the total number of glomeruli found in each antennal lobe is between 60 and 70 (Boeckh and Boeckh, 1979; Rospars, 1983; Berg et al., 2002; Huetteroth and Schachtner, 2005; Masante-Roca et al., 2005; Skiri et al., 2005; Kazawa et al., 2009; Carlsson et al., 2013) with the exception of the oligophagous moth genus Cydia, which has only &#8764;50 (Varela et al., 2009; Couton et al., 2009; Trona et al., 2010).'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'One of the most debated themes in comparative vertebrate neurobiology is the extent to which genetic and developmental constraints limit the scope of adaptive evolution of brain structure, enforcing a concerted, as opposed to a mosaic, pattern of brain evolution (Finlay and Darlington, 1995; Barton and Harvey, 2000). Our analysis suggests that invertebrate brains can readily respond to changing selection pressures, with the gain and loss of distinct structures, or subcompartmentalization of existing neuropil, occurring in a lineage-specific manner, often without any clear correlative change in other structures. Within Lepidoptera, species vary markedly not only in the relative volume of different neuropils, as discussed above, but also in the presence or absence of some traits such as the optic glomeruli, the AOTu strap, and the zonation of the mushroom body calyx.'},\n",
       " {'pmcid': 'PMC4354442',\n",
       "  'pmid': '25400217',\n",
       "  'snippet': 'In this study we present multiple lines of evidence that ecology-specific behavioral selection pressures shape the composition of invertebrate brains. In the butterfly G. zavaleta, heightened reliance on olfaction apparently drove the independent evolution of a group of specialized olfactory glomeruli into a prominent MGC with a putative function analogous to the pheromone-sensing MGC in moths.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'In the adult brain, reelin is expressed primarily in cortical and hippocampal GABAergic neurons as well as in cerebellar granule cells [6-10]. In the mature brain, it is involved in a variety of functions, including the ccontrol of neurotransmitter release [11,12], maturation and stabilization of dendritic spines [13-15], specification of the molecular identity of the distal dendritic compartment of pyramidal neurons [16], regulation of glutamate receptor homeostasis [17], and in the regulation of different aspects of adult hippocampal neurogenesis [18-25]. However, reelin expression is not restricted to the central nervous system (CNS), and has also been observed in the enteric nervous system [26], blood serum, liver, pituitary pars intermedia, and adrenal chromaffin cells [27], as well as in platelets [28], and lymphatic vessels [29,30].'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'However, as far as we know there are no reports on the possible expression of reelin in the main component of the blood-brain barrier, the brain endothelial cells. The expression of reelin in this particular type of endothelial cells, and in specific subcellular compartments within them, could be related to possible roles for this protein in the regulation of brain microvasculature (i.e., as it has been shown for lymphatic vessels), and/or mediating an interconnection between brain and peripheral tissues (i.e., being transported by transcytosis). In order to assess these possibilities we undertook an electron microscopy study of reelin immunolabeling in brain endothelial cells.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'On the other hand, reelin-labeled capillaries were not observed in areas lacking diffuse immunostaining, such as deeper layers of the cortex, and the CA1 area of the hippocampus (Figures&#160;1C-D, G-H). The fact that reelin immunostaining was only observed in some (but not all) capillaries within areas presenting strong diffuse labeling argues in favor of its specificity. In addition, this also suggests that a possible secretion and/or transport of reelin by endothelial cells may take place primarily in areas of heavy diffuse reelin staining (i.e., brain areas where reelin tends to accumulate in the extracellular matrix) [see reference 32].Figure 1 Reelin labeling at the light microscope.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'B-C) Detail images of the areas indicated with dashed-line boxes in A. Note that reelin labeling is specifically located in the rough endoplasmic reticulum (outlined black arrows). m: myelinated process; n: nucleus; p: neuronal process.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'B-C) Detail images of the areas indicated with dashed-line boxes in A. Note that reelin labeling is specifically located in the rough endoplasmic reticulum (outlined black arrows). m: myelinated process; n: nucleus; p: neuronal process.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'Taking together our findings and these previous reports, it is possible that the crossing of the blood-brain barrier might be accomplished via a receptor-mediated mechanism that includes ApoER2. In addition, it has been shown that the adaptor protein DAB1 also tends to cluster within specific membrane domains (i.e., in detergent-resistant membrane fractions such as lipid rafts and caveolar vesicles) [39,40], that reelin induces the clustering of its own receptor Apolipoprotein ER2 and other proteins [41,42], and that the sorting of these proteins to different membrane domains yields differential endocytosis of reelin [43]. We have recently shown that deficits in reelin expression result in changes in membrane protein clustering in peripheral lymphocytes [44], and that the addition of recombinant reelin to synaptosomes increase protein expression and alters membrane protein clustering [45,46].'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'The presence of reelin in caveolar vesicles of brain endothelial cells suggests a possible role for this protein in the regulation of the crosstalk between brain and immune system, and in the perturbation of this crosstalk in several psychiatric disorders. Although the specific consequences of reelin downregulation in psychiatric disorders are still unknown, and it should be also taken into account that reelin deficits could be associated with brain cell malfunction independent of its association with the blood-brain barrier.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'Reelin immunolabeling is specifically present in caveolar vesicles within endothelial cells located in brain areas that present strong reelin labeling in the extracellular matrix. In addition, electron microscopy images evidence that reelin-labeled caveolar vesicles can be observed from their formation to the opening of their membrane towards the capillary lumen.'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'Free-floating sections were thoroughly rinsed in 0.01 M phosphate buffered saline (PBS) pH 7.4, and subsequently treated with 0.3% (v/v) H2O2 in PBS for 30 min to block endogenous peroxidase activity. They were then pre-incubated for 30 min in a blocking solution containing 0.3% Triton X-100, 5% normal horse serum (NHS; Vector Laboratories, Burlingame, CA, USA), and 1% bovine serum albumin (BSA) dissolved in PBS in order to block any non-specific antibody binding. The sections were then incubated with a mouse monoclonal anti-reelin antibody (Millipore clone G-10 Cat# MAB5364, RRID:AB_2179313; diluted 1:2000, 48 hr, at 4&#176;C).'},\n",
       " {'pmcid': 'PMC4374371',\n",
       "  'pmid': '25887698',\n",
       "  'snippet': 'Endogenous peroxidase was blocked by immersing the sections in a solution of 5% hydrogen peroxide in PBS for 30 minutes, which was followed by 5 rinses (5 min each) in PBS. Afterwards, to block non-specific binding sites in the sections, a blocking solution of 10% NHS prepared in PBS containing 0.1% Triton X-100 was applied for one hour. The sections were then incubated for 72 hours at 4&#176;C with a mouse monoclonal anti-reelin antibody (Millipore clone G-10 Cat# MAB5364, RRID:AB_2179313clone) diluted 1:1000 in PBS containing 5% NHS.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'Sensitive cell lines are arbitrarily defined based on their IC50s (the most widely used metric for evaluation of drug response; log10[IC50]&#160;&lt;&#160;&#8722;6.5). Cell line-specific model calibration for two selected cell lines C32 and WM115. Left: R2, Q2, and MSPE for C32 and WM115 models built with increasing numbers of PLSR components.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': ' Cell line-specific model calibration for two selected cell lines C32 and WM115. Left: R2, Q2, and MSPE for C32 and WM115 models built with increasing numbers of PLSR components.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'Phospho-p38(Thr180/Tyr182) levels also fell within 1&#160;h of exposure to vemurafenib, PLX4720, or AZ628; KINOMEscan binding data for these drugs (Vin et&#160;al, 2013) suggest that they have off-target activity on regulators of p38 such as ZAK kinases. We observed that co-treatment of LOXIMVI cells with SB202190, a specific inhibitor of p38 kinase, diminished the effectiveness of vemurafenib and AZ628 (Supplementary Fig S1C&#8211;E). Thus, off-target effects of RAF inhibitors on the p38 pathways are likely to be undesirable from a therapeutic perspective.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'To identify adaptive changes that are predictive of drug response, we calculated the &#8220;variable importance in the projection&#8221; (VIP) for each of ten PLSR models. VIP scores report the sum (over all model dimensions) of each variable x (an RPPA assay at a specific time point), weighted by the change in response (cell viability) explained by the same variable x (see Materials and Methods for mathematical details) (Wold, 1994; Janes et&#160;al, 2008). We generated a VIP score for each three-way cell line/signal/time point combination.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'Variability in the magnitude, direction, and timing of signaling changes in response to RAF/MEK inhibition  PLSR-derived variable importance in the projection (VIP) scores predicting viability for each of the ten studied cell lines. VIP scores are shown for each cell line-specific model, each signal and measurement time point. The direction of the bars (left or right) shows whether the changes in signal correlated negatively or positively with relative viability.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': ' PLSR-derived variable importance in the projection (VIP) scores predicting viability for each of the ten studied cell lines. VIP scores are shown for each cell line-specific model, each signal and measurement time point. The direction of the bars (left or right) shows whether the changes in signal correlated negatively or positively with relative viability.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'In melanoma cells, JNK-IN-8 caused dose-dependent inhibition of c-Jun S63/S73 phosphorylation, a modification required for transcriptional activity, but did not measurably alter the levels of pERK, pAKT(Ser473), pSTAT3(Tyr705), the p38/MK2 substrate pHSP27(Ser82) or the nuclear translocation of NF-&#954;B (Supplementary Fig S2C&#8211;I). These modification states report on potential off-target activities of JNK-IN-8, and the absence of significant changes suggests that the drug acted in a JNK-specific manner in melanoma cells at the doses we used. Exposure of cells to JNK-IN-8 alone reduced p-cJun(Ser73) to background levels but induced little if any apoptosis (Fig3B).'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'To determine whether the up-regulation of c-Jun impacted vemurafenib-mediated cell killing, we co-treated cells with JNK-IN-8, a kinase inhibitor that is specific for JNK relative to other MAP family kinases (Zhang et&#160;al, 2012). In melanoma cells, JNK-IN-8 caused dose-dependent inhibition of c-Jun S63/S73 phosphorylation, a modification required for transcriptional activity, but did not measurably alter the levels of pERK, pAKT(Ser473), pSTAT3(Tyr705), the p38/MK2 substrate pHSP27(Ser82) or the nuclear translocation of NF-&#954;B (Supplementary Fig S2C&#8211;I).'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'Mean values of four biological replicates are shown. Protein levels represent log2 fold change of each signal (at a specific dose and time) relative to a DMSO-treated control.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'Mean values of four biological replicates are shown. Protein levels represent log2 fold change of each signal (at a specific dose and time) relative to a DMSO-treated control.'},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': \"pS6(Ser235/236) levels 24&#160;h after treatment were the best single predictor of drug-induced cell killing in our data (&#961;&#160;=&#160;0.62, P&#160;=&#160;2&#160;&#215;&#160;10&#8722;6) (Fig6F), reflecting the role of S6 kinases in integrating the activities of multiple signaling pathways. In the specific case of PLX4720, p-cJun(Ser73) levels 24&#160;h after drug exposure were also a good predictor of cell killing in 8/10 lines (&#961;&#160;=&#160;0.76, P&#160;=&#160;0.03) (Fig6G). We observed a significant difference in the partial correlation coefficient between viability and pS6 inhibition (controlling for pERK levels; Spearman's &#961;partial&#160;= 0.58;&#160;P&#160;=&#160;1.7&#160;&#215;&#160;10&#8722;4) and the partial correlation between viability and pERK inhibition (controlling for pS6 levels; &#961;partial&#160;= 0.34;&#160;P&#160;=&#160;0.04).\"},\n",
       " {'pmcid': 'PMC4380931',\n",
       "  'pmid': '25814555',\n",
       "  'snippet': 'Model fitness was calculated using R2, Q2, and mean squared prediction error (MSPE) which were calculated as previously described (Gaudet et&#160;al, 2005). For the assessment of relative variable importance in each PLSR model, the information content of each variable (representing a signal measurement at a specific time point) was assessed by its variable importance in the projection (VIP) (Wold, 1994; Janes et&#160;al, 2008):  where K is the total number of signaling variables (K&#160;=&#160;21&#160;&#215;5&#160;=&#160;105), wnk is the weight of the kth variable for the nth PLSR component, N is the total number of PLSR components, and SSn is the sum of squares explained by the nth PLSR component.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The ability of the transcription unit to produce a product is blocked by a transcriptional stop signal. The stop signal can be removed by the action of a site-specific recombinase, and the fraction of cells in which the stop signal is removed&#8212;and thus the density of immunolabeling&#8212;can be varied by adjusting the level of recombinase produced (Struhl and Basler, 1993). Briefly, UAS reporter constructs carrying Flag, VK5, and HA epitope tags (pJFRC206-5xUAS-IVS-myr::smGFP-FLAG in VK00005 and pJFRC200-10xUAS-IVS-myr::smGFP-HA in attP18, Viswanathan et al., submitted) downstream of an FRT-flanked stop signal were excised by limited expression of FLP recombinase activity.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Flies were dissected anywhere from eclosion to 2 weeks to achieve the desired density of labeled cells. Tissue was subsequently labeled with epitope tag-specific antibodies. With one exception, exclusively female brains were analyzed in this study.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The specificity of nc82 against BRP protein has been demonstrated by: 1) the expression pattern of GFP-tagged bruchpilot driven under tissue-specific drivers, which matches nc82 signals in wing discs and tracheal cells, and is also targeted to the active zone of larval NMJ boutons (Wagh et al., 2006); 2) western blots of adult head extracts using nc82 (Wagh et al., 2006); and 3) the loss of immune-expression in brp mutant neuromuscular junctions and rescue by expression of BRP in brp mutants (Kittel et al., 2006). nc82 has been widely used to label synaptic sites in Drosophila, based largely on the pattern of labeling demonstrated at neuromuscular junctions and fly photoreceptor synapses (Hamanaka and Meinertzhagen, 2010), but reports of its specificity are mostly not complete for synapses of the CNS.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The specificity of nc82 against BRP protein has been demonstrated by: 1) the expression pattern of GFP-tagged bruchpilot driven under tissue-specific drivers, which matches nc82 signals in wing discs and tracheal cells, and is also targeted to the active zone of larval NMJ boutons (Wagh et al., 2006); 2) western blots of adult head extracts using nc82 (Wagh et al., 2006); and 3) the loss of immune-expression in brp mutant neuromuscular junctions and rescue by expression of BRP in brp mutants (Kittel et al., 2006). nc82 has been widely used to label synaptic sites in Drosophila, based largely on the pattern of labeling demonstrated at neuromuscular junctions and fly photoreceptor synapses (Hamanaka and Meinertzhagen, 2010), but reports of its specificity are mostly not complete for synapses of the CNS. In particular, nc82 labels the platform of the T-bar ribbon, and not only has nc82 not been shown to label the platforms at CNS synapses, but not all synapses in the CNS have such organelles (Butcher et al., 2012).'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The specificities of the three epitope-tagged antibodies, rat anti-FLAG, rabbit anti-hemagglutinin (anti-HA), and mouse anti-V5, are validated by the internal controls of the flip-out approach in that: 1) expression patterns differ by GAL4 line and 2) the extent of labeling varies from no label to dense label even though the GAL4 drivers are reasonably broad.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The subdomains (abbreviated sd in the figure) are included in the neuron\\'s name to define the subregion/s of the neuropils in which the cell type arborizes. For example, there are several distinct volumes in the EB, so the volume that is specific to a given cell type is included in the name. The third component of a cell\\'s name is the predominant morphology of its arbors, abbreviated as either \"s\" for spines, or \"b\" for boutons.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Finally, this naming scheme has the added advantage that additional detail can be incorporated as necessary or as it becomes available. For example, during the course of this study it became evident that neurons arborize in only a subset of the glomeruli of the PB, so the designation \"Gx-y\" was incorporated to identify the specific glomeruli targeted by the cell type, where \"x\" refers to the number of the medial glomerulus and \"y\" refers to the lateral glomerulus.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Eighteen glomeruli have been reliably counted in multiple GAL4 lines that label diverse cell types in the PB. Furthermore, this is not a sex-specific morphological phenomenon, since PBs from both male and female brains contain 18 glomeruli (Fig. 4).'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'D: Schematic representation of PB:FB&#8467;1 circuitry of the PBG2&#8211;9.s-FB&#8467;1.b-NO3P.b neuron. The glomeruli in which this neuron arborizes (G2&#8211;G9), the specific FB teeth they target and the lines connecting them, are similarly colored to more easily follow the projection pattern for this cell type. The numbers beneath each tooth of the FB refer to the glomeruli from which they receive input, and \"i\" and \"c\" refer to ipsilateral and contralateral.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': '7D), extends the entire depth of the EB, populating the anterior, medial, and posterior shells (AMP). (More specifically, this cell has spiny arbors in all three shells&#8212;anterior, medial, and posterior&#8212;of an EB wedge, and bouton-type terminals in an undefined region surrounding the dorsal gall, which we call the Dorsal gall-surround.) A previously uncharacterized R1-like neuron, EBR1.b-cre.s, also occupies the anterior shell (not shown).'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'We named this region the nubbin (nub) because it is a projection that appears stunted and undeveloped. The nubbin is evident in nc82-labeled brains and, as with other brain regions and subdomains, is defined as a distinct volume in the EB by the arbors of a neuron, the BU.s-EBnub.b cell, which specifically targets this region of the EB (Fig. 7E; [R41G11]).'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The ventralmost nodulus, NO3, is more square than NO2 and occupies the largest volume of the three. NO2 and NO3 have distinct subcompartments that are revealed by distinguishable texture, contrast, and intensity in nc82-labeled samples and, more definitively, by arborization domains of cells that specifically target these subregions. NO1 may also have subcompartments (see below).'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Second, individual cell types project to different subsets of glomeruli (for example, G1&#8211;G8, G2&#8211;G9, or exclusively G9 in the case of the PBG9.b-EB.P.s-ga-t.b cell, described below). Consequently, while the overall projection pattern is preserved in these unique cell types, the precise correspondence between a given glomerulus and a specific geographic location in the EB differs.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'G5L and G3R each have just one labeled cell, but G3R fills its entire corresponding wedge, whereas G5L fills just a demi-wedge. As with PBG1&#8211;8.s-EBt.b-D/Vgall.b cells (above), PBG1&#8211;8.b-EBw.s-D/Vgall.b cells project to either the dorsal or ventral gall in the same glomerulus-specific fashion: even-numbered glomeruli project to the ventral gall whereas odd-numbered glomeruli project to the dorsal gall, as indicated by the D and V designations above the PB.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'To maintain a balanced distribution of information flow, the point of crossover for cells that stay ipsilateral between the first and second substructures shifts by one glomerulus in cells that arborize in G1&#8211;G8 versus those that arborize in G2&#8211;G9 so that four glomeruli per side stay ipsilateral at the first junction and four cross contralaterally at the first junction. More specifically, for cells that arborize in G9&#8211;G2, G9&#8211;G6 stay ipsilateral at the first junction and G5&#8211;G2 cross the midline at the first junction (PB-EB; Fig. 14A), whereas for cells that target G8&#8211;G1, G8&#8211;G5 remain ipsilateral at the first junction and G4&#8211;G1 cross the midline at the first junction (Figs.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'In contrast, there is a 1:1 correspondence between glomeruli and EB wedges: Cells in each glomerulus target separate wedge volumes in the EB and, in some cases, two cells from a single glomerulus occupy separate demi-wedges of the same wedge. In the case of circuits for cell types that arborize in G1&#8211;G8, the EB tile and wedge regions that correspond to specific glomeruli (wedges) or pairs of glomeruli (tiles) are identical. (For example, both G1 tile cells in Fig.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'An MCFO-based \"dissection\" of these connections reveals a surprisingly limited connectome between neuropils. For example, there is a great deal of specificity in central complex regions that provide output to the noduli. As noted above, just one cell type has been identified that connects the PB to NO1 (PBG2&#8211;9.s-EBt.b-NO1.b; Figs.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Of the cells identified to date, a similarly small number of cells connect the PB to NO2 and NO3. Furthermore, there is specificity in the connections between certain layers of the FB and subcompartments of the noduli.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Such highly specific wiring between the PB and the noduli and their subcompartments, as well as between the layers of the intermediate neuropil (the FB) and the subcompartments of the noduli, typifies the specificity of the connections between neuropils of the brain. It also suggests a highly modular division of roles for the subregions of the Drosophila brain.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Another neuron distinguishes G9 from the rest of the glomeruli (see below). Both apparently dendritic and presynaptic contacts exhibit specificity in the subregions of neuropils they target.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Two subclasses comprise each of these cells types, and it is the subclasses that exhibit specificity for glomerular subsets. The subclasses differ in two ways: 1) they arborize in different glomeruli, and 2) they target different regions of the gall.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'Features of the multiglomerular phenotype are not universal; instead, they are characteristic of specific cell types. For example, with rare exceptions the bi-glomerular arborizations of PBG1&#8211;8.b-EBw.s-D/Vgall.b cells are restricted to G1.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The length of these \"dangling arbors\" is consistent within subtypes. There is no correlation between subtype and specific subsets of glomeruli. In other words, it is not the case that, for example, only short arbors correlate with G1&#8211;3, medium from G4/G5, and long from G6/G7.'},\n",
       " {'pmcid': 'PMC4407839',\n",
       "  'pmid': '25380328',\n",
       "  'snippet': 'The only link between the PB and NO1 is via the ellipsoid body, so whereas NO2 and NO3 can be considered to work in conjunction with the FB to elicit a behavior based on output from the PB, NO1 cooperates with the EB to elicit a behavior based on output from the PB. In fact, communication is even more specific: layer 3 of the FB communicates directly only with NO2, layer 2 directly only with the anterior subcompartment of NO3, and layer 1 directly only with NO3M and NO3P. Furthermore, the cells in G1 do not communicate directly with the noduli at all&#8212;neither via the FB nor via the EB.'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'To compare intracellular trafficking of CCR5 internalized by 5P14-RANTES with that of CCR5 internalized by PSC-RANTES, we used an anti-CCR5 antibody together with compartment-specific antibodies (anti-Rab11 for the ERC, anti-TGN38 for the TGN) to perform pulse-chase colocalization immunofluorescence microscopy experiments essentially as described previously [22] (Fig 1A). As expected from previous work [22], CCR5 internalized by PSC-RANTES initially accumulates in a discrete supranuclear Rab11-positive patch that corresponds to the ERC [22, 29], subsequently relocating to a perinuclear region consistent with the TGN [22].'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'We performed arrestin2-CCR5 co-immunoprecipitation experiments in which CHO-CCR5 cells expressing arrestin2 fused to GFP (arrestin2-GFP) were incubated with ligands prior to lysis and immunoprecipitation using a CCR5-specific antibody. Associated levels of arrestin2 were then detected by Western blot using antibodies specific for either arrestin2 or GFP (Fig 4). As expected [18, 32, 33], RANTES/CCL5 induced recruitment of arrestin2-GFP.'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'We performed arrestin2-CCR5 co-immunoprecipitation experiments in which CHO-CCR5 cells expressing arrestin2 fused to GFP (arrestin2-GFP) were incubated with ligands prior to lysis and immunoprecipitation using a CCR5-specific antibody. Associated levels of arrestin2 were then detected by Western blot using antibodies specific for either arrestin2 or GFP (Fig 4).'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'Immunoprecipitates were subjected to Western blot using antibodies against GFP, arrestin2 and CCR5. CP; total cell pellet control, IP; antibody specificity used for immunoprecipitation, IB; antibody specificity used for immunoblot.'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'Phosphorylation of serine and threonine residues in the cytoplasmic C-terminal tail of GPCRs by G protein-coupled receptor kinases (GRKs), is known to facilitate arrestin association [3, 6, 34], and C-terminal phosphorylation of CCR5 has been shown to play a key role in the desensitization process [35]. In order to investigate a potential role for GRK phosphorylation in ligand-directed post-endocytic sorting of CCR5, we used a monoclonal antibody specific for CCR5 phosphorylated at Ser349, one of several C-terminal serine residues previously shown to be targeted for GRK phosphorylation [35], in an immunofluorescence microscopy experiment to assess the extent and localization of phospho-Ser349-CCR5 in CHO-CCR5 cells following ligand treatment at time intervals up to 120 min (Fig 6). As expected from previous work [35], RANTES/CCL5 elicits phosphorylation of CCR5 on Ser 349.'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'CHO-CCR5 cells growing on coverslips were treated with the indicated chemokines (100 nM) for the indicated times, then washed, fixed, permeabilized and labeled with a monoclonal antibody specific for CCR5 phosphorylated on Serine 349 (green) and DAPI nucleic acid stain (blue). Maximal intensity projections are shown.'},\n",
       " {'pmcid': 'PMC4414452',\n",
       "  'pmid': '25923671',\n",
       "  'snippet': 'The extent and the quality of arrestin-GPCR interaction is strongly influenced by the activity of G protein-coupled receptor kinases (GRKs) which recognize and phosphorylate ligand-activated GPCRs [3, 4]. Using an antibody specific for CCR5 phosphorylated on Ser349, one of 4 serine residues known to be targets of GRK, we showed that all three ligands capable of eliciting CCR5 internalization (RANTES/CCL5, PSC-RANTES and 5P14-RANTES) also elicit phosphorylation on Ser349 (Fig 6), with PSC-RANTES, which elicits the highest level of arrestin2-GFP recruitment (Fig 4), generating the strongest signal. While it was interesting to note that for each ligand, the site of accumulation of phosphoSer349-CCR5 is consistent with the site of bulk CCR5 accumulation (TGN for RANTES/CCL5 and PSC-RANTES, ERC for 5P14-RANTES), this experiment did not provide evidence that either the extent or the kinetics of GRK phosphorylation of Ser349 is responsible for modulating the duration of ligand-induced arrestin recruitment to CCR5.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'TRs affect transcription of genes at different levels from influencing the state of chromatin, to interaction with DNA regulatory elements, to the modulation of the general transcription machinery by cell-specific employment of different components. Very frequently, TR genes are subject to differential transcription themselves, thereby serving specific regulatory roles in the regionally restricted hierarchical steps of organ development. In the developing CNS, for example, TRs are expressed in distinct territories such as the prosomeres of the forebrain or the rhombomeres of the hindbrain (Narita and Rijli, 2009; Lauter et al., 2013).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'The large representation of this gene ontology group in the genome underscores the importance of precise control of gene transcription during various processes such as development and body homeostasis. TRs affect transcription of genes at different levels from influencing the state of chromatin, to interaction with DNA regulatory elements, to the modulation of the general transcription machinery by cell-specific employment of different components. Very frequently, TR genes are subject to differential transcription themselves, thereby serving specific regulatory roles in the regionally restricted hierarchical steps of organ development.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'One characteristic of transcription factors (TFs) is that they contain one or more DNA-binding domains (Ptashne and Gann, 1997). Upon binding to a specific DNA sequence TFs activate or repress transcription of target genes. Frequently, multiple TFs act in a combinatorial fashion and the interaction of factors determines the specific regulatory outcome (Pabo and Sauer, 1992; Latchman, 1997; Lee and Young, 2000).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Upon binding to a specific DNA sequence TFs activate or repress transcription of target genes. Frequently, multiple TFs act in a combinatorial fashion and the interaction of factors determines the specific regulatory outcome (Pabo and Sauer, 1992; Latchman, 1997; Lee and Young, 2000). Forced expression of single TFs or TFs in specific combinations can change cell identity from one cell type to another and can induce pluripotency in differentiated cells (Davis et al., 1987; Schafer et al., 1990; St&#252;hmer et al., 2002; Takahashi and Yamanaka, 2006; Young et al., 2007a,b).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Frequently, multiple TFs act in a combinatorial fashion and the interaction of factors determines the specific regulatory outcome (Pabo and Sauer, 1992; Latchman, 1997; Lee and Young, 2000). Forced expression of single TFs or TFs in specific combinations can change cell identity from one cell type to another and can induce pluripotency in differentiated cells (Davis et al., 1987; Schafer et al., 1990; St&#252;hmer et al., 2002; Takahashi and Yamanaka, 2006; Young et al., 2007a,b). TFs interact with factors of the general transcription machinery and other TRs including coactivators and corepressors.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Based on deep sequencing data, 1,202 TR genes were chosen and subjected to a detailed in situ expression analysis in the adult zebrafish telencephalon. This comprehensive expression map provides insights into a potential regulatory role of specific TRs in adult neurogenesis and maintenance of differentiation and function of the telencephalon.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Hierarchical clustering of expression patterns of 769 regionally restricted genes was employed to identify TR genes coexpressed in specific anatomical structures. Ubiquitously expressed genes were excluded from this analysis.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': '1B,C, red). Cluster 2 is composed of the central zone of the dorsal telencephalic area (Dc), the lateral olfactory tract (Lot), the medial olfactory tract (Mot), and the rest of the subpallial parenchyma without specific annotations in the zebrafish brain atlas. We refer to this latter region as Sup in our scoring of expression patterns.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'While this overall clustering approach suggested shared TR gene expression programs in the three clusters of anatomical regions, most TR genes are expressed in multiple ventral and dorsal domains (Fig. 1), suggesting that these factors may be involved in more general functions or act in a combinatorial fashion together with other factors to generate region-specific regulatory control.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': '2; Supplemental Table 1) using our annotation of expression domains to anatomical landmarks (Wullimann et al., 1996). We identified 13 specific patterns (A&#8211;M, Fig. 2) that are shared by at least two TR genes.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'TR genes act frequently in a combinatorial fashion. Coexpression of specific genes may thus reflect a potential functional interaction. Furthermore, synexpression of TR genes in different regions of the telencephalon may indicate common regulatory mechanisms.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'The remaining 640 genes represented unique patterns that were not shared entirely with another gene and were therefore left unclassified. The E-zone tool was installed as a search option in the AGETAZ database (http://cory.itg.kit.edu/agetaz/index.php) for detection of genes coexpressed in a particular region, allowing further rapid evaluation of coexpression of these 640 genes with those classified in specific patterns.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'We identified seven factors exhibiting the pattern E (&#8764;0.6 % of the total investigated genes). Among these genes, we also identified isl1 that was previously reported to be expressed specifically in the subpallium of zebrafish (Mueller and Wullimann, 2009; Ganz et al., 2012).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'We identified six TR genes (almost 0.5% of the investigated genes) expressed specifically at the ventricular zone of the telencephalon, where the different progenitor types reside (Fig. 2M&#8211;M2).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'For further details on the TR genes expressed in particular ventricular regions, consult the AGETAZ database (http://cory.itg.kit.edu/agetaz/index.php) using the search options to group coexpressed genes. Taken together, these clustering results indicate that the ventricular zones, in addition to expressing genes that control cell proliferation and maintenance of stem cells, express neuronal differentiation markers, suggesting that the different neurogenic niches in the telencephalon have region-specific identities.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Hierarchical clustering of TR genes expressed in specific regions of the VZ. A: Clustering of 547 VZ expressed genes focusing on anatomical terms corresponding to ventricular domains (indicated on top).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': '1). The fact that the Va and Vl have expression features similar to pallial structures suggests distinct and specific functions of these brain areas compared to the other subpallial regions. Interestingly, these clusters of anatomical landmarks do not strictly obey the pallial and subpallial divisions, suggesting that regulatory mechanisms are shared between these broad anatomical subdivisions of the telencephalon.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'We scored 13 such expression groups forming patterns A to M. Among the 562 genes, a large proportion was expressed in the entire telencephalon (354 genes, pattern A) or broadly with varying intensities of expression in different parts of the telencephalon (143 genes, pattern B). Other gene patterns comprised more specific areas such as pallial regions or subpallial nuclei, oligodendrocytes, or regions of proliferation such as the ventricular zones including the RMS. We also noted genes that are expressed in varying patterns but whose transcripts were enriched at the perinuclear space subcellularly (10 genes, pattern C).'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'The 640 remaining genes show unique expression patterns and thus could not be categorized into a group. This finding underscores the importance of the free combinatorial action of TR genes that will drive different regulatory outcomes depending on the specific cellular context.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Molecular anatomy and expression patterns of genes involved in the establishment of neurotransmitter systems such as GABAergic neurons have been shown to be highly conserved between zebrafish and rodent models (Mueller and Wullimann, 2009). For instance, the zebrafish orthologs of mammalian Mash1 and dlx2 are known for their involvement in specification of GABAergic neurons (Mueller and Wullimann, 2009, and references therein). Similarly, our data on expression domains of lhx6 in the ventral subdivision of the precommissural dorsal subpallium is in agreement with a previous report published by Mueller et al.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'Patterns I to M are linked to proliferative regions and thus may contain regulators of adult neurogenesis. We found 13 TR genes with a strong and specific ventricular expression, including some genes whose expression was predominantly detected at the ventricular zone, but also in other regions of the telencephalon, such as znf36l2, her6, and sox9a. Only six of the screened genes exhibited an expression restricted exclusively to the ventricular zone.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'In zebrafish, the RMS-like region is composed of type III progenitor cells (M&#228;rz et al., 2010a). We identified 14 genes expressed in the RMS, five transcripts of which are restricted to the posterior RMS (CR774186.3, foxj1a, hif1al, mych, tcf7l2) and five (mycn, sox19b, sox2, pou3f3b, sox4a) are specific to the anterior RMS. Some of these genes are also expressed in the mammalian RMS.'},\n",
       " {'pmcid': 'PMC4418305',\n",
       "  'pmid': '25556858',\n",
       "  'snippet': 'For example, the Vv expresses genes that may control differentiation towards GABAergic neurons, while regions located more dorsally generate neurons of the glutamatergic lineage (Mueller et al., 2008; Mueller and Wullimann, 2009). Interestingly, genes that are expressed in a region-specific manner in the embryonic telencephalon such as nkx2.2, nuclear factor I/X (CCAAT-binding transcription factor), prdm12 (PR domain containing 12) in the embryonic subpallium suggest that mechanisms have been retained from embryo ontogenesis to adult maintenance of the CNS. It is tempting to speculate that a code as employed in the embryonic spinal cord may exist also in the telencephalon that is maintained into adulthood in the zebrafish.'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': 'To determine if binding of R2-C1 to SD2 was specific, a competition assay was performed. Unlabeled R2-C1 effectively competed with biotinylated R2-C1 for binding to immobilized SD2 (Figure&#160;1c), demonstrating that the interaction assayed in the ELISA is due to specific binding of SD2 with R2-C1. The Z&#8217; factor of the ELISA, a measure of assay dynamic range and variation, was 0.74, indicating the suitability of the SD2/R2-C1 ELISA platform for screening.Figure&#160;1Shroom3 and Rho kinase interact with high affinity.'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': 'By varying the concentration of biotinylated R2-C1 with a fixed concentration of SD2, the apparent Kd was calculated to be 4.9&#160;&#177;&#160;0.3&#160;nM (Kd&#160;&#177;&#160;SEM) (Figure&#160;1b). To determine if binding of R2-C1 to SD2 was specific, a competition assay was performed. Unlabeled R2-C1 effectively competed with biotinylated R2-C1 for binding to immobilized SD2 (Figure&#160;1c), demonstrating that the interaction assayed in the ELISA is due to specific binding of SD2 with R2-C1.'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': 'A large fraction of these target protein&#8211;protein interactions with tight affinities (&lt;1&#160;&#956;M) and small surface areas [37]. Consistent with this paradigm, Shroom3 and Rho kinase interact with a relatively tight affinity (4.9&#160;nM for mouse Shroom3/Rho kinase II); however, CCG-17444 interacts with a specific Cys residue, potentially blocking an allosteric change, so small surface area across the protein&#8211;protein interface does not appear to be a major determinant in this case. Thus, small molecule compounds are likely to block protein&#8211;protein interactions through diverse modes of action.'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': 'A significant contributor to disability after CNS injury is the failure of the adult CNS to undergo neural repair. Blocking protein&#8211;protein interactions within specific signaling hubs may provide new avenues for enabling neural repair in the damaged CNS.'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': 'Distinct from strategies that block receptor function, CCG-17444 inhibits a critical protein&#8211;protein interaction within a key intracellular signaling hub that regulates neurite outgrowth inhibition, the interaction between Shroom3 and Rho kinase. Rho kinase regulates a diverse array of biological outcomes [40&#8211;42] so precisely blocking Rho kinase activity in a signaling complex dedicated to regulating neurite outgrowth inhibition, rather than all Rho kinase enzymatic activity, is likely to be more specific and less prone to off-target effects. A significant contributor to disability after CNS injury is the failure of the adult CNS to undergo neural repair.'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': 'For neurite length measurements, the length of the longest neurite per cell (ca. 3&#215; cell body or 50&#160;&#956;m) was measured; GFP positive neurites are positive for the axon marker Tau [16] and the neuron specific tubulin TuJ1 [21, 22]. P19 cells, grown in minimal essential medium-&#945; supplemented with 7.5% calf serum, 2.5% fetal bovine serum, and penicillin&#8211;streptomycin were plated the day before transfection to a density of 2&#160;&#215;&#160;105 cells/well of a 12-well dish and transfected the next day with 2.5&#160;&#181;g of total DNA (0.75&#160;&#181;g of Ngn2, 0.75&#160;&#181;g of GFP, and 1&#160;&#181;g of pUI4/UI5 RNAi expression vectors).'},\n",
       " {'pmcid': 'PMC4467669',\n",
       "  'pmid': '26077244',\n",
       "  'snippet': '72&#160;h after transfection, cells were fixed in 3.7% formaldehyde in PBS and stained for GFP to identify transfected cells [14, 16, 21]. Similar results are obtained when neurite length is determined by staining for GFP or neuron specific markers.'},\n",
       " {'pmcid': 'PMC4468165',\n",
       "  'pmid': '26075743',\n",
       "  'snippet': 'Tissue sections were then blocked for 15 minute with streptavidin and biotin block (Vector Laboratories, 1:50) as well as with anti-rabbit Fab fragment according to manufacturer&#8217;s instructions and BrdU or pHH3 was detected as described above, except that sections were incubated with TRITC (Perkin-Elmer) rather than FITC. Controls in which each primary antibody was omitted were performed and revealed minimal cross-reactivity between stains (data not shown).'},\n",
       " {'pmcid': 'PMC4468165',\n",
       "  'pmid': '26075743',\n",
       "  'snippet': 'Lineage commitment and differentiation of cells occur in a sequential manner and are orchestrated by combinatorial expression of cell type-specific transcription factors and epigenetic modifications. Thus, it is possible that FOXM1 contributes to differentiation of hormone-secreting cell types in the developing pituitary gland rather than or in addition to their proliferation.'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'f Immunofluorescence detection of the SLC10A4 protein was performed with different SLC10A4-directed antibodies (green fluorescence): self-generated polyclonal rabbit anti-Slc10a4 1338 C antibody, rabbit anti-SLC10A4 Sigma Prestige antibody, rabbit anti-Slc10a4 Abnova antibody, and rabbit anti-SLC10A4 Abgent antibody. Membrane protein enriched fractions of the CAD cells were also subjected to Western Blot analysis with the same antibodies and revealed specific bands for the SLC10A4 protein at an apparent molecular weight of 30&#8211;32&#160;kDa.'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'f Immunofluorescence detection of the SLC10A4 protein was performed with different SLC10A4-directed antibodies (green fluorescence): self-generated polyclonal rabbit anti-Slc10a4 1338 C antibody, rabbit anti-SLC10A4 Sigma Prestige antibody, rabbit anti-Slc10a4 Abnova antibody, and rabbit anti-SLC10A4 Abgent antibody. Membrane protein enriched fractions of the CAD cells were also subjected to Western Blot analysis with the same antibodies and revealed specific bands for the SLC10A4 protein at an apparent molecular weight of 30&#8211;32&#160;kDa.'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'The uptake of 5&#160;&#181;M [3H]dopamine, 5&#160;&#181;M [3H]norepinephrine, 5&#160;&#181;M [3H]choline, or 5&#160;&#181;M [3H]serotonin was measured over the given time periods in the presence and absence of Na+ (for CHT1 Na+ was replaced by Li+). Transport via DAT, CHT1, and SERT was blocked by the specific inhibitors nomifensine (10&#160;&#181;M), hemicholinium-3 (HC-3, 1&#160;&#181;M), and citalopram (2&#160;&#181;M), respectively. Values represent mean&#160;&#177;&#160;SD of representative experiments, each with quadruplicate determinations (n&#160;=&#160;4).'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'The uptake of 5&#160;&#181;M [3H]dopamine, 5&#160;&#181;M [3H]norepinephrine, 5&#160;&#181;M [3H]choline, or 5&#160;&#181;M [3H]serotonin was measured over the given time periods in the presence and absence of Na+ (for CHT1 Na+ was replaced by Li+). Transport via DAT, CHT1, and SERT was blocked by the specific inhibitors nomifensine (10&#160;&#181;M), hemicholinium-3 (HC-3, 1&#160;&#181;M), and citalopram (2&#160;&#181;M), respectively. Values represent mean&#160;&#177;&#160;SD of representative experiments, each with quadruplicate determinations (n&#160;=&#160;4).'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'The SLC10A4 protein consists of 437 amino acids in man, rat, and mouse, and shows a close phylogenetic relationship to the sodium-dependent bile acid transporter NTCP. However, whereas NTCP is specifically expressed at the sinusoidal plasma membrane of hepatocytes [2], the SLC10A4 protein was localized in cholinergic and monoaminergic neurons of the central and peripheral nervous system as well as in pheochromocytoma PC12 cells and mast cells [13&#8211;15]. Based on its genetic classification, SLC10A4 first was suggested to be a novel carrier for already established substrates of the SLC10 carrier family, including bile acids and sulfo-conjugated steroid hormones [1, 6, 13].'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'They used the human medulloblastoma cell line TE671 in which they localized the SLC10A4 protein by immunofluorescence and Western Blot analysis with a commercial anti-SLC10A4 antibody and showed that pre-incubation of the cells with 1U/200&#160;&#181;l of thrombin for 3&#160;h increased the cellular uptake of taurocholic acid and lithocholic acid [18]. However, appropriate controls demonstrating specificity of the antibody were not provided and the authors failed to seriously show saturable transport kinetics for this uptake. Furthermore, bioinformatics analysis (Expasy Peptide Cutter, http://www.web.expasy.org/peptide_cutter/) indicates that the mouse and rat SLC10A4 proteins can be cleaved by thrombin at amino acid position 87, but not the human SLC10A4 protein.'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'In contrast to NTCP, ASBT and SOAT, which all are sorted to the plasma membrane, SLC10A4 has a typical intracellular vesicle-like expression pattern in neurons, mast cells and neuronal cell lines. This specific expression pattern of SLC10A4 was also observed in the present study in untreated and differentiated CAD and SH-SY5Y cells. To more closely analyze the sorting domains of SLC10A4, a series of NTCP/SLC10A4 chimeras were generated and analyzed in HEK293 and CAD cells.'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'For permeabilization, cells were incubated for 5&#160;min with PBT buffer (0.2% Triton X-100 and 20&#160;mM Glycine in PBS). The non-specific binding sites were blocked with 1% bovine serum albumin (BSA) plus 4% goat serum (Sigma-Aldrich) in PBS for 30&#160;min at room temperature. Then, the cells were incubated with the primary antibodies, rabbit anti-Slc10a4 (antibody 1338 C, 1:1,000 dilution, see [13, 14] ), Sigma Prestige rabbit anti-SLC10A4 (1:500 dilution) [Sigma-Aldrich Cat# HPA028835 RRID:AB_10603025], Abnova rabbit anti-Slc10a4 (1:500 dilution) [Abnova Corporation Cat# PAB14855 RRID:AB_10696081], or Abgent rabbit anti-SLC10A4 (1:500 dilution) [Abgent Cat# AP10250b RRID:AB_10821224], or with mouse anti-V5 monoclonal antibody (1:5,000 dilution) [Invitrogen Cat# R96025 RRID:AB_159313] in blocking solution overnight at 4&#176;C. The next day, cells were washed with PBS and incubated with the fluorophore-labeled secondary antibody Cy3-conjugated goat anti-rabbit IgG (1:800 dilution) [Jackson ImmunoResearch Cat# 111-165-003 RRID:AB_2338000] or Alexa fluor 488-labelled goat-anti mouse (1:800 dilution) [Molecular Probes (Invitrogen) Cat# A11001 RRID:AB_141367] in blocking solution for 60&#160;min at room temperature.'},\n",
       " {'pmcid': 'PMC4472396',\n",
       "  'pmid': '26084360',\n",
       "  'snippet': 'Samples of 20&#160;&#181;g protein were mixed with Laemmli Sample Buffer (Sigma-Aldrich), separated on a 12% SDS polyacrylamide gel and transferred to a Hybond-ECL nitrocellulose membrane (Amersham Biosciences, Freiburg, Germany). The blotted membranes were blocked with blocking solution containing 5% ECL-blocking agent (Amersham Biosciences) in TBS-T (137&#160;mM NaCl, 10&#160;mM Tris, pH 8.0, 0.05% Tween-20), followed by overnight exposure to antigen-specific primary antibodies at 4&#176;C in the same buffer. After several washing steps in TBS-T, the membranes were probed with the appropriate horseradish-peroxidase-labeled secondary antibodies in TBS-T for 60&#160;min at room temperature.'},\n",
       " {'pmcid': 'PMC4514672',\n",
       "  'pmid': '26207633',\n",
       "  'snippet': 'Furthermore, studies have shown that expression profiles of FFPE tissue are comparable to expression profiles of fresh frozen tissue when using the DASL assay [8]. As such, in this work we assess gene expression patterns of archival chronically inflamed placental tissue selected specifically to include CC, VUE and CD. Using the DASL assay we identify genes differentially expressed between placentae samples with and without chronic inflammation.'},\n",
       " {'pmcid': 'PMC4514672',\n",
       "  'pmid': '26207633',\n",
       "  'snippet': 'The original data files have been deposited in Gene Expression Omnibus (GEO, http://www.ncbi.nlm.nih.gov/geo) and are accessible through GEO Series accession number GSE68474. All quality control and data preprocessing was performed in R, a statistical analysis program, and using microarray-specific packages available through Bioconductor, lumi and LIMMA [15, 16]. Quality control involved assessment of raw expression boxplots, interquartile range, Pearson correlation between samples and density plots; outlier samples were not identified.'},\n",
       " {'pmcid': 'PMC4514672',\n",
       "  'pmid': '26207633',\n",
       "  'snippet': 'Normal tonsil and bone marrow was used as positive controls to demonstrate appropriate immunostaining patterns for all antibodies. Negative controls involved omission of primary antibody and showed negligible background staining.'},\n",
       " {'pmcid': 'PMC4514672',\n",
       "  'pmid': '26207633',\n",
       "  'snippet': 'It is important to recognize that the RIN is a semiquantitative predictor of interexperimental variability for a particular methodology but is not a robust predictor of successful experimental outcomes as this is dependent on the methodology employed [18]. In this respect, the DASL method is specifically designed to amplify short (50 bp) RNA fragments and as such a RIN value &lt; 2 with average RNA fragment lengths of 200 bp suggests suitable RNA integrity for this method [19]. The RNA isolated from all archival samples in this study is of adequate purity and average fragment length to proceed with WG-DASL gene expression analysis.'},\n",
       " {'pmcid': 'PMC4514672',\n",
       "  'pmid': '26207633',\n",
       "  'snippet': 'Biologically these genes have been associated with development of immune tolerance and memory (ICOS [20], CD27 [21]), T cell activation (CXCL13, CXCL9 [22]), NK cell activation (KLRC1 [23]), B cell homing (CXCL13 [24]), immunoglobulin production (IGJ [25]), B cell development (IGJ [26], IGLL1 [27]), decidual regulation of inflammation (ICOS [28], CXCL13 [29]), and suppression of eosinophilic inflammation (SIGLEC8 [30]). Although the 14 oCPI-associated genes are not specifically indicative of a molecular process or abnormality, together they are reminiscent of genes involved in germinal center function with a focus on follicular helper T cells (TFH) cells [31].'},\n",
       " {'pmcid': 'PMC4514672',\n",
       "  'pmid': '26207633',\n",
       "  'snippet': 'Direct comparison of our results to previous findings is hindered by the absence of studies using archival tissue specifically including CC, VUE, and CD. Our study does not stratify samples based on grades of VUE and chronic chorioamnionitis as has been done in other studies focused on these entities.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'A major objective in the skeletogenic field is to understand the sequential mechanisms that direct specification, condensation and overt differentiation during skeletal chondrogenesis. Local signaling from adjacent epithelia specify prechondrogenic fate in neural crest-derived mesenchyme, which then differentiate into chondrocytes several days later [1].'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'Local signaling from adjacent epithelia specify prechondrogenic fate in neural crest-derived mesenchyme, which then differentiate into chondrocytes several days later [1]. The critical intermediate step between specification and overt differentiation is condensation, which has two important features: firstly, mechanical forces control cell shape and organization, setting the characteristic size and shape of skeletal elements and concurrently modulate cell fate choice [2&#8211;5], and secondly, condensation is a prerequisite for overt chondrocyte differentiation [6]. However, the mechanisms that control these morphogenetic processes and the role of molecular signaling pathways in the pharyngeal arches are poorly understood.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'The chick embryo, which becomes increasingly inaccessible beyond HH24 (Hamburger and Hamilton) [8], sinks beyond reach into the yolk and is enveloped in multiple membranes, which complicates in vivo analysis. Each step acts as a prerequisite for the sequence to move to the next phase: specification (epithelial mesenchymal interactions), condensation (cytoskeletal rearrangements), and chondrocytes (overt differentiation). Patterning cartilage in the correct position and of the correct size and shape is also dependent on this sequence.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'The aim of the study was to elucidate mechanisms governing condensation during the multiday period between mesenchyme specification and overt chondrogenesis in the chick embryo (embryonic day E3-8/HH18-34). Local epithelial to mesenchymal interactions specify post-migratory neural crest-derived mesenchyme to a chondrogenic fate by HH18 [1].'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'Cell numbers in the columella mesenchyme increased slightly, but not significantly, from HH24-27 during the period when no cell specific F-actin organization was observed (blue line, Fig 3). This was followed by a precipitous reduction in cell density until HH30 when condensation occurs.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'After HH24, multiple membranes make the embryonic head and pharyngeal arches inaccessible in ovo. To access the pharyngeal arches between specification and overt differentiation we developed an in toto slice culture system. A transverse slice from the chick head, encompassing the brain and the entire second pharyngeal arch region, was extirpated and cultured flat on a porous membrane (see materials and methods).'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'Middle ear mesenchyme specification in the columella occurs at HH18, condensation at HH30 and overt differentiation at HH32-34 [1, 12]. After HH24, multiple membranes make the embryonic head and pharyngeal arches inaccessible in ovo.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'This result is significant for two reasons. Firstly, it demonstrates that actin polymerization and associated cytoskeletal reorganization are sufficient to influence molecular signaling, and secondly, that the effects are not indiscriminate, but pathway specific.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'ROCK function was inhibited to investigate whether Rho Kinase mediated signaling is required for cell shape changes and more specifically, the establishment of the long actin protrusions observed at HH30 in the columella condensation (Fig 6C). Indeed, TGF-&#946; signaling is maintained when ROCK is inhibited by application of 250 &#956;M of Y27632 (68%, n = 5, Fig 6C and 6D).'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'The loss of BMP signaling results in ventral condensation characteristics suggesting that the BMP-independent ventral pattern is the ground state for pharyngeal cartilage. FGF signaling from pharyngeal endoderm and the surface ectoderm of the arches during earlier specification [1] is a potential mediator of this base state. Moreover, ventral condensations are refractory to BMP signaling, supporting the hypothesis that the ground state in pharyngeal condensations is ventral.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'What then, is the significance of BMP induced differential dorsoventral patterning in pharyngeal condensations? BMP acts in an instructive capacity by regulating differential cell cortex tension, which leads to the observed cell shape changes and most likely plays a crucial role in cell fate specification from a prechondrogenic identify toward overt differentiation. Moreover, the stretching of dorsal mesenchyme suggests that they are subject to greater differential cell cortex tension than the more compact, ventral, BMP-independent cells.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'These forces are responsible for the size and shape of the condensation, stimulating the nascent cartilage cells to undergo overt differentiation into chondrocytes, which in turn exhibit enhanced BMP, FGF and TGF-&#946; signaling [61]. Notably, TGF-&#946; activation of Smad2/3 specifically activates RhoA/ROCK mediated cytoskeletal reorganization and chondrocyte specific gene expression in mesenchyme stem cells [36]. This suggests that at the point of condensation TGF-&#946; activity must be suppressed in order to maintain prechondrogenic identity, but thereafter, active TGF-&#946; signaling induces overt differentiation into chondrocytes through Smad2/3 mediated signaling.'},\n",
       " {'pmcid': 'PMC4523177',\n",
       "  'pmid': '26237312',\n",
       "  'snippet': 'A framework of the steps during endochondral ossification in the pharyngeal arch skeletal elements during specification, condensation and overt differentiation.'},\n",
       " {'pmcid': 'PMC4529706',\n",
       "  'pmid': '26253104',\n",
       "  'snippet': 'All antibodies were raised against human antigens except anti-laminin which was raised against basement membrane of Englebreth Holm-Swarm (EHS) mouse sarcoma. After washing in PBS, the slides were incubated in different mixtures of species-specific goat or donkey secondary antibodies conjugated with Dylight488 (1:500; Jackson ImmunoResearch Laboratories, Inc, West Grove, USA), Alexa Fluor 568, Alexa Fluor 488 (1:200; Lifetechnology, Carlsbad, USA) and/or actin-binding TRITC-phalloidin (1&#160;&#956;g/ml; Sigma-Aldrich) for one hour at room temperature in a moist chamber. Slides were then rinsed in PBS before mounted with Dako Fluorescence Medium (DakoCytomation, Glostrup, Denmark) to prevent fading.'},\n",
       " {'pmcid': 'PMC4529706',\n",
       "  'pmid': '26253104',\n",
       "  'snippet': 'Slides were then rinsed in PBS before mounted with Dako Fluorescence Medium (DakoCytomation, Glostrup, Denmark) to prevent fading. Secondary antibody specificity was assessed by omitting the primary antibody.'},\n",
       " {'pmcid': 'PMC4529706',\n",
       "  'pmid': '26253104',\n",
       "  'snippet': 'In the current study, no immunoreactivity in the human urothelium was obtained when the tissue sections were incubated with the secondary antibody alone. Indeed, this control for antibody specificity has been questioned but P2X2 and P2X3 has previously been demonstrated in human urothelium [35] and P2X7 in the rat urothelium [36].'},\n",
       " {'pmcid': 'PMC4529706',\n",
       "  'pmid': '26253104',\n",
       "  'snippet': 'Results from these studies as well as the finding of organized expression of P2X receptors in human bladders from older patients suggest that the purinergic system may be a better target if one wants to treat lower urinary tract symptoms (LUTS). What is promising with our findings is that there are a number of specific agonists against specific P2X receptors [39] which raise the hope of finding efficient drugs with few side effects. The results from our study is highly relevant for the potential treatment of LUTS since the bladders used in our study are from older men and women undergoing cystectomy by various pathologies associated with LUTS.'},\n",
       " {'pmcid': 'PMC4529706',\n",
       "  'pmid': '26253104',\n",
       "  'snippet': 'P2X7 is expressed in smooth muscle, P2X2 in suburothelial cells located closer to the urothelium, P2X3&#160;in suburothelial cells closer to the smooth muscle layer and in ICs surrounding muscle bundles, while all three receptors where expressed in the urothelium. The organized distribution of different P2X receptors in the bladder supports the idea that specific targeting of the different receptors can have different effects on bladder function.'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'Oligos were synthesized by Integrated DNA Technologies (Coralville, IA). The amplicons were designed 80&#8211;120&#160;bp to ensure specificity of the target gene amplification. All samples were run in duplicates and nontemplate controls were used in each run.'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'The qPCR cycling conditions were as follows: 50&#176;C/2&#160;min/1 cycle; 95&#176;C/10&#160;min/1&#160;cycle; 95&#176;C/15&#160;sec, and 60&#176;C/1&#160;min for 40 cycles (Life Technologies, GrandIsland, NY). A melting curve (55&#176;C&#8211;95&#176;C in 0.5&#176;C increments) was performed in each run to assess the specificity of primers and to distinguish between amplicon amplification and primer-dimer formation. Relative quantitation of gene expression in control and EtOH-treated samples from four independent cultures (4 biological replicates; 3&#8211;5 technical replicates per treatment) was calculated from the ratio of the mean quantity of target gene (Gabra4, Gabra1, Gabrd, Gabrg2, or Gabrb2) to that of internal control (Gapdh).'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': '2008), were not altered significantly in this study (Fig.4B). Because our miRNA array data showed that the upregulation of some miRNAs persisted in AW8 neurons after the removal of alcohol, we reasoned that the downregulation in Gabra4 during AW might be mediated by specific miRNAs. We determined changes in miR-155, miR-186, miR-24, and miR-375 expression at various time intervals after onset of AW, and found that these miRNAs were significantly upregulated at AW5&#160;min, AW6, AW8, AW12, and/or AW24&#160;h (Fig.5A&#8211;D).'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'The minimum free energy of hybridization between each of these miRNAs and the 3&#8242;UTR of Gabra4 is presented in Table S4. miRPath analysis predicted that specific biological pathways are induced by the upregulated miRNAs in EtOHc-4D and in AW8 neurons, including the MAPK signaling pathway that has been implicated in the effects of alcohol on the brain (Hansson et&#160;al. 2008), and that multiple target genes could be involved (Table S5).'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'In our model system, the downregulation in Gabra4 expression during AW (AW3-AW24), but not Gabra1, Gabrd, or Gabrb2 at these time points suggests that the effect of AW is time-dependent and subunit specific. Gabra1 expression showed a robust increase at AW0, AW5, and AW3.'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'The expression and function of specific GABAARs subunits are differentially altered in response to acute and chronic alcohol exposure in different areas of the brain (Kumar et&#160;al. 2009).'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'For example, mRNA and protein levels of &#945;4 subunit of GABAARs increased and those of &#945;1 subunit decreased in the cortex of dependent rats and alcohol-withdrawn rats (AW6-8) (Devaud et&#160;al.1995, 1997). Chronic alcohol exposure differentially altered cortical and hippocampal peptide levels of specific GABAAR subunits. For example, it increased &#945;4 peptide levels in the hippocampus, but decreased &#945;1 peptide levels in the cortex after 40&#160;days of EtOH consumption in rats (Matthews et&#160;al.1998).'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'The results of a decade of such studies clearly indicate a pattern of remarkable plasticity of GABAARs subunits in response to alcohol exposure and also indicate that these changes are brain-region specific, multifactorial and therefore probably involve multiple mechanisms that are still not well understood.'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'This may be due to the intrinsic properties of each miRNA such as: 1) its rate of synthesis and degradation or turnover, 2) its protection by the RNA-Induced Silencing Complex (RISC), that is its lifespan, 3) its stability, 4) its activity (Davis and Hata 2009; Kai and Pasquinelli 2010; Ruegger and GroBhans 2012). Another explanation is that alcohol exposure and/or withdrawal may have altered the expression of specific transcription factors that regulate miRNAs expression. We could also speculate that AW may have caused epigenetic changes such as changes in DNA methylation or histone marks of miRNA genes that differentially altered the expression of these miRNAs (Miranda et&#160;al.'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'The downregulation of Gabra4 expression upon AW temporally correlated with the upregulation of a cohort of miRNAs that have been proposed in many studies to have physiological significance related to seizure, stress, alcoholism, epigenetics, immunity, and synaptic functions (Table S6). Our data suggest that miRNAs have specific temporal expression in response to the presence or removal of alcohol. This suggests that miRNAs have different adaptive response to alcohol exposure or withdrawal.'},\n",
       " {'pmcid': 'PMC4559018',\n",
       "  'pmid': '26357588',\n",
       "  'snippet': 'This study shows that AW downregulates Gabra4 expression and upregulates the expression of specific miRNAs during AW. Future studies will be directed toward a possible role for these miRNAs in an in vivo model of AW.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'dkNET was created specifically to address the challenge of connecting researchers in metabolic and digestive diseases to research resources via these types of community databases and web portals. dkNET functions as a &#8220;search engine for data&#8221;, searching across millions of database records contained in hundreds of biomedical databases developed and maintained by independent projects around the world.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'Although the focus of the NIF is neuroscience, the data architecture itself is generic and applicable as a general search engine for data. The SciCrunch infrastructure allows communities like dkNET to create domain-specific data portals on top of a set of shared data resources. A goal of SciCrunch is to ensure that different communities can leverage efforts across biomedicine, through the creation of networked data resources, so as to continually add to the amount of data and resources that are discoverable and usable.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'The community curator decides how the data are to be organized within the portal through the selection of Data Views and the specification of Categories and Subcategories. The initial specification of these categories and subcategories followed the organization established by dkCOIN. The basic navigation scheme used by dkNET is shown in Fig 1.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'Additional relevant sources were selected from those available through the SciCrunch Data Federation. The community curator decides how the data are to be organized within the portal through the selection of Data Views and the specification of Categories and Subcategories. The initial specification of these categories and subcategories followed the organization established by dkCOIN.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'At the top level, dkNET organizes resources under 3 views: Community Resources, More Resources and Literature. Within each view are specific categories, e.g., for Community Resources, the categories comprise Materials, Funding, Protocols, Data and Organism. Each of these, in turn, may be expanded into several subcategories, e.g., Materials is subdivided into Antibodies, Vectors, Embryonic Stem Cell (ESC) lines and PCR primers.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'With hundreds of databases comprising millions of records, it is easy for a user to get overwhelmed with the amount of data available. To accommodate the need to promote awareness and use of dkNET-specific resources while at the same time providing broader search when required, the search portal was divided into 3 major Views, each represented by a separate tab: Community Resources, More Resources and Literature (Fig 2). As each tab is selected, the same search is executed across the different views.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'The Community Resources Data View is the section that provides a given community the opportunity to select certain data sources and organize them according to their needs. In the case of dkNET, Community Resources provides access to a limited number of databases (Table 1) that were specifically developed for NIDDK-supported researchers and ingested via dkNET or were selected from the SciCrunch data federation because they were viewed as specifically valuable to the dkNET community. An example of the latter is the Antibody Registry, a database of over 2 million antibodies for biomedical science developed by the NIF project.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'For certain types of entities, the expansion also includes additional related entities, e.g., specific cell types, or related techniques. These expansions are based on the underlying structure of the ontology where an equivalency statement has been included.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'The majority of these come via the NIF project, e.g., a hippocampal neuron is equivalent to all classes of neurons that are found in the hippocampus [8]. Future versions of dkNET will include similar equivalency classes for entities specifically relevant to the kidney, digestive and genitourinary systems.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'Clicking on the record will take the user to the same record in the NIH Reporter Database. Here, users can explore the data source further, e.g., to find the specific program announcements that funded the research.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'Although this query did not find any opportunities-not completely surprising given the specificities of the terms-information about NIH programs that have previously funded such models can be obtained through NIH Reporter. Switching to table view, the Funding Institute facet can be used to obtain a list of NIH institutes that have funded drosophila models of diabetes.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'We should note, however, that the majority of records for this database do not explicitly mention diabetes within the phenotype description, but were returned because the mice were described in a publication in the journal &#8220;Diabetes&#8221;. If a user is specifically interested in animals that exhibit diabetes, results can be further refined by entering &#8220;diabetes&#8221; as a filter in the &#8220;Description&#8221; column. This action returns a mouse model that is described as &#8220;... protected from high-fat diet-induced diabetes,&#8221;.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'Including this term in the query leads to zero results, because the term doesn&#8217;t appear anywhere in the antibody databases. Antibodies are messy reagents and the issue of validation can be a contentious one and is use-case specific, but there are criteria that people use in determining whether or not an antibody can and should be used in a given application.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'dkNET was designed as a discovery portal for research resources of relevance to the mission of the NIDDK. The design of dkNET balances the need for a focused set of resources specifically developed by and for the NIDDK-funded research community with the broad array and growing number of biomedical databases available. Resources available under Community Resources are largely derived from projects funded by NIDDK to provide materials, protocols and data for researchers.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'A second issue is whether all of the NIDDK community, for example, should be served by a single portal or whether several sub portals serving different NIDDK communities should be designed. SciCrunch was specifically designed to create these types of portals. Setting up a specialized portal using data that is already available can ben done in as little as a few hours.'},\n",
       " {'pmcid': 'PMC4578941',\n",
       "  'pmid': '26393351',\n",
       "  'snippet': 'For a growing resource like dkNET, which continually adds new types of data and new data sources, we believe that the enhanced keyword interface is the most effective way to search dkNET data sources, because it doesn&#8217;t require any a priori knowledge of the contents or structure of the system. But the power of structured data is the ability to take advantage of that structure to create very specific queries that return precise answers. dkNET balances the need for broad keyword based and structured search through the use of categories, facets and filters that allow users to progressively narrow down their search according to what is available.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'These studies have demonstrated the overall importance of miRNAs in spinal cord development or in motor neuron differentiation and function. However, a more specific experimental approach is required to unravel the role of miRNAs in the neural stem cells resident in the developing spinal cord and committed to yield those motor neurons which are affected in SMA.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'MicroRNA regulation is importantly involved in some neurodegenerative states [18,19], and recent studies have shown their function in the control of the timing of many developmental programs, strongly suggesting that these molecules also play an important role in the timely generation of neurons from neural progenitor cells [20]. Recently, few studies have started addressing the role of miRNAs in NSCs, specifically in the developing spinal cord [21,22,23,24] where the lower (&#945;) motor neurons affected by SMA reside. These studies have demonstrated the overall importance of miRNAs in spinal cord development or in motor neuron differentiation and function.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'Over the last decade, microRNAs (miRNAs), short single-stranded RNA molecules (21&#8211;25 nt), were discovered and demonstrated to work as crucial regulators of gene expression at the post-transcriptional level by targeting the complementary sequences in 3&#8242; untranslated regions (3&#8242;UTR) of specific mRNAs [17]. MicroRNA regulation is importantly involved in some neurodegenerative states [18,19], and recent studies have shown their function in the control of the timing of many developmental programs, strongly suggesting that these molecules also play an important role in the timely generation of neurons from neural progenitor cells [20].'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'Self-renewing NSCs homogeneously showed immunoreactivity for Nestin and were negative for GFAP (Glial Fibrillary Acidic Protein), a specific marker for astrocytes (Figure 1C). These results confirm that we have successfully isolated specific NSC populations from embryonic spinal cord of SMN&#916;7 SMA mice, maintaining the expression of the respective molecular markers of their original neural environment.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'Flow cytometry analyses of BrdU labeled cells were performed for a finer characterization of the progression along the cell cycle of NSCs. The distribution in each cell cycle phase of cells labeled with BrdU for 1 h were quantified by using specific gates and the representative data of one of three independent experiments are shown in Figure 2B (right panel). In details, a comparison between SMA NSCs and WT cells highlights a reduced number of cells in G0/1 phase (47.1% &#177; 0.14% vs. 64.5% &#177; 2.59%) and an increased S and G2/M phases (36.6% &#177; 0.6% vs. 29.2% &#177; 0.4% and 15.4% &#177; 1.9% vs. 6.6% &#177; 0.2%), respectively.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'The list of microRNAs distinguishing SMA from WT NSCs is reported in Table 1. Specifically, 13 miRNAs were underexpressed in SMA NSCs compared to the WT, and only three were overexpressed. Among modulated miRNAs, we focused on miR-335-5p and miR-100-5p, which were highly reduced in SMA NSCs with a Log2 FCSMA/WT among the lowest (&#8722;1.69 and &#8722;1.74, respectively).'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'MicroRNAs are potent modulators of gene expression, and even slight variations in their amounts can strongly affect the production of specific proteins and, consequently, cell fate. In order to determine if microRNAs are involved in the SMA state in neural progenitors, we performed a microRNA expression profiling in three SMA NSC samples, extracted from the spinal cords of mice born in three distinct families.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'In conclusion, we have provided a molecular characterization of neural progenitors derived from mouse spinal cord at E13.5, when a peak of neurogenesis takes place, by making comparisons of SMN&#916;7 cells, representing a mouse model of severe SMA. We specifically focus on miRNAs differentially expressed in SMA vs. WT NSCs, and highlight for the first time the underexpression of miR-100-5p and miR-335-5p in SMA cells compared to those which are WT. The differential expression of these miRNAs may be correlated to the increased proliferation activity observed in SMN&#916;7 cells.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'One microgram of total RNA, DNAse I treated for eliminating genomic DNA contamination (Ambion, Life Technologies Corporation, Foster City, CA, USA), was reverse transcribed using High-Capacity cDNA Archive kit (Life Technologies Corporation, Foster City, CA, USA) and amplified using primers reported in Table 2. Specific amplification conditions for each marker were as previously reported [25]. Nestin was used as loading control for testing the efficacy of all the amplification reactions.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'Genomic DNA contamination was eliminated with DNase I-RNase-free (Ambion, Life Technologies Corporation). One microgram of RNA was reverse transcribed with the High-Capacity cDNA Archive kit (Life Technologies Corporation) and analyzed by RT-qPCR, following an amplification protocol specific for each primer pairs. mRNAs of interest were quantified with specific primers (Table 2) using SYBR Green chemistry (Life Technologies Corporation).'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'One microgram of RNA was reverse transcribed with the High-Capacity cDNA Archive kit (Life Technologies Corporation) and analyzed by RT-qPCR, following an amplification protocol specific for each primer pairs. mRNAs of interest were quantified with specific primers (Table 2) using SYBR Green chemistry (Life Technologies Corporation). The comparative &#916;&#916;Ct method was used to quantify relative gene expression levels [29,38] and normalization was performed by simultaneous quantification of Gapdh (Glyceraldehyde 3-phosphate dehydrogenase) mRNA.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'Total RNA from NSC SMA and WT was purified as described above. 5.8S rRNA, used as a reference, and snRNA specific reverse primers (see Table 2) were used to generate cDNA, using High-Capacity cDNA Archive kit (Life Technologies Corporation). The same reverse primers were used in both RT-PCR and qPCR.'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'Specific dilution of primary antibodies were used: Nestin 1:200 (Chemicon International, Inc., Billerica, MA, USA, CAT#AB5922, RRID:AB_91107) and GFAP 1:100 (Sigma, CAT#5-A-5, RRID:AB_2314539). Cells were subsequently incubated for 1 h at room temperature with specific secondary antibodies (1:800 Alexa Four 568, 488 conjugated, Molecular Probes, Life Technologies Corporation), and DAPI (1:1000, Invitrogen, Life Technologies Corporation).'},\n",
       " {'pmcid': 'PMC4581247',\n",
       "  'pmid': '26258776',\n",
       "  'snippet': 'These cells were then plated in tissue culture plates pre-coated with poly-d-lysine (PDL, Sigma-Aldrich, St. Louis, MO, USA) and fixed after 4 h with 4% paraformaldehyde in PBS. Specific dilution of primary antibodies were used: Nestin 1:200 (Chemicon International, Inc., Billerica, MA, USA, CAT#AB5922, RRID:AB_91107) and GFAP 1:100 (Sigma, CAT#5-A-5, RRID:AB_2314539). Cells were subsequently incubated for 1 h at room temperature with specific secondary antibodies (1:800 Alexa Four 568, 488 conjugated, Molecular Probes, Life Technologies Corporation), and DAPI (1:1000, Invitrogen, Life Technologies Corporation).'},\n",
       " {'pmcid': 'PMC4582632',\n",
       "  'pmid': '26408173',\n",
       "  'snippet': 'In contrast, in heterologous expression systems, Nav&#946;4 co-expression shifts the voltage-dependence of activation to negative potentials but no change in inactivation is observed [46, 49, 60]. Modulation by Nav&#946;4 seems to be cell background specific. In the endogenous sodium current recordings we obtained, activation was not studied due to contamination with TTXR currents that cannot be readily subtracted.'},\n",
       " {'pmcid': 'PMC4582632',\n",
       "  'pmid': '26408173',\n",
       "  'snippet': 'The maximum number of evoked action potential was defined as the maximum number of action potentials elicited at a given step depolarization from 0&#160;pA to 1.2&#160;nA for each cell. The number of evoked action potentials was also significantly greater with Nav&#946;4 overexpression when measured specifically at rheobase, and at 2&#215; and 3&#215; rheobase compared to control (Additional file 3: Figure S3). In cells that we examined for evoked activity no significant change was observed in resting membrane potential, input resistance or rheobase (Table&#160;2).Fig.&#160;7Overexpression of Nav&#946;4-WT increases excitability of DRG neurons.'},\n",
       " {'pmcid': 'PMC4582632',\n",
       "  'pmid': '26408173',\n",
       "  'snippet': 'The data was grouped into small diameter neurons (&lt;400&#160;&#956;m2) and medium-large diameter neurons (&gt;400&#160;&#956;m2) and compared using Student&#8217;s t test. It is important to note that due to the limitations of this approach, the Pearson correlation values do not specifically represent colocalization of the proteins but rather describe the population of cells that co-express Nav&#946;4 and Nav1.6. Additionally, images were analyzed using NIS Elements Advance Research (Nikon&#174;) software and mean intensity for Nav&#946;4 and Nav1.6 staining signal was determined for each cell by defining the region of interest.'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'In obese, insulin resistant Zucker rats, Akt2 expression is reduced by more than half, while Akt1 expression remains unaffected in muscle; however, both insulin-stimulated Akt1 and Akt2 activity is significantly diminished in muscle but not adipose [17]. This suggests that Akt&#8217;s involvement in insulin action maybe tissue- and isoform-specific in both expression and activation. Insulin-stimulated human muscle from obese, insulin resistant individuals display a decrease in Akt2 activity but not Akt1 compared to lean, healthy counterparts [18].'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'Recently, Akt substrate 160 (AS160) was identified as an Akt interaction partner using a protein AGC phosphomotif (RXRXXpS/T) antibody and was subsequently characterized by mass spectrometry [23]. Akt (non-isoform specific Akt or pan-Akt) is known to interact with several kinases (e.g., protein kinase C) and phosphatases (e.g., PP2A) [24, 25], many of which play a role in mitogenesis, cell survival, growth, cell differentiation, and glucose uptake [7].'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'The study started with L6 cell culturing, insulin treatment, cell lysis, and supernatant isolation; followed by immunoprecipitation of the &#8220;bait&#8221; protein (Akt2); followed by 1D-SDS-PAGE to resolve Akt2 co-interaction proteins; in-gel trypsin digestion to generate peptide fragments used in HPLC-ESI-MS/MS analysis for identification of co-immunoprecipitating proteins. The supernatant was pre-cleared by immunoprecipitation with non-specific antibodies (NIgG) to remove and identify non-specific proteins, which may contaminate the Atk2 Co-IP and minimizes false positives. Extensive bioinformatics and literature search were used to analyze proteomic data and to identify significantly enriched pathways, in which identified Akt2 interaction partners were involved.'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'Protein concentration of lysate was estimated by Bradford Assay (Bio-Rad) using bovine serum albumin (Sigma, St Louis, MO) as a standard. One mg of supernatant for each sample was precleared with 2 &#956;g of NIgG primary antibody and 25ul of packed protein A agarose beads (Sigma, St Louis, MO) for 4hrs, used to detect non-specific binding. Protein A beads were pelleted and washed three times with 1 ml of lysis buffer.'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'To determine the relative quantities of Akt2 interaction partners in basal and insulin-treated conditions, the PA for each protein identified in a specific sample was normalized against the PA for Akt2 identified in the same sample, which results in Norm:j.'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'From 4 independent biological comparisons (4 basal and 4 insulin stimulated samples, 4 slices/gel lane, 32 HPLC-ESI-MS/MS runs), a total of 835 unique proteins were identified in at least 1 out of the 8 Akt IP samples (Fig 1B). An additional 24 HPLC-ESI-MS/MS runs were performed for the corresponding NIgG samples under identical conditions to filter out non-specific protein binders.'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'Akt2 was detected from all basal and insulin-stimulated cells but was not detected in any of the NIgG immunoprecipitates. Akt1 and Akt3 were not detected, suggesting the antibody we used is specific for Akt2. In total, 49 proteins met the criteria for classification as Akt2 interaction partners (Table 2 ).'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'In the present work, we validated the interaction by immunoprecipitating Akt2 and western blotting for ROCK2. L6 myoblasts were stimulated with or without 100 nM insulin for 15 min and immunoprecipitated with anti-Akt2 antibody followed by immunoblotting with antibodies specific to Akt2 and ROCK2. The results confirmed the insulin-stimulated association of Akt2 and ROCK2 (Fig 4).'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'However, the band was still observed. Since the predicted molecular weight of ROCK2 is &gt;160kD, this band is most likely a non-specific band due to the fact that it is below 150kD and present in all immunoprecipitates, including NIgG (a negative control for non-specific binding).'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'L6 myoblasts have been shown to be a suitable model for insulin-stimulated GLUT4 translocation as well as glucose uptake, of which Akt2 plays a defining role [31]. The strategy of label free identification offers the ability to detect endogenous proteins interactions, without the use of an overexpressed or tagged &#8220;bait&#8221; protein, which may lead to a greater probability of non-specific binding and the discovery of false positives [43].'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'Overall, ROCK2 is suggested to play a negative role in insulin signaling due to its negative feedback on IRS1. Insulin-stimulated Akt2 association with other kinases, such as mTOR, S6K1, and GSK3, also lead to subsequent IRS1 site-specific hyper-serine/threonine phosphorylation [2]. ROCK2 does not contain a putative Akt2 phospho motif (S2 Table), neither does Akt2 contain a putative ROCK2 phospho motif (R/KXS/T; R/KXXS/T), which suggests the interaction of Akt2-ROCK2 is not potentially mediated by phosphorylation.'},\n",
       " {'pmcid': 'PMC4605787',\n",
       "  'pmid': '26465754',\n",
       "  'snippet': 'Please note that preclearing was used not only to reduce the unspecific background, but also to identify non-specific binder.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'Cells were blocked and permeabilized with 3% (wt/vol) bovine serum albumin and 0.1% Triton X-100 in DPBS for 60 min, and then coverslips were incubated with primary antibody overnight at 4&#176;C (anti-Kv2.1 K89/34 IgG1: 1:20 dilution of tissue culture supernatant, Research Resource Identifier [RRID]: AB_10672253; anti-GFP N86/38 IgG2a: 1:5 dilution of tissue culture supernatant, RRID: AB_10671955; both from the UC Davis/National Institutes of Health [NIH] NeuroMab Facility; anti-HA, RRID: AB_2532070). Primary antibodies were detected with appropriate mouse IgG subclass-specific fluorescent secondary antibodies (A21127, A21131, A21121, and A21147; 1:2,000; Thermo Fisher Scientific) and Hoechst 33258 stain (H21491; Thermo Fisher Scientific). Coverslips were washed again in DPBS and mounted using Prolong Gold (P36930; Thermo Fisher Scientific).'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'However, it was notable that V378A channels failed to produce significant current in the majority of patch-clamped cells, suggesting an alteration in protein expression or localization. Expression and localization of wild-type and V378A channels were tested by immunofluorescence labeling of transfected cells with specific anti-Kv2.1 antibodies. CHO-K1 cells were transiently cotransfected with GFP and wild-type Kv2.1 or V378A.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': '3 A) indicated that although the incidence of GFP expression was similar between cultures transfected with wild-type and V378A plasmids, fewer cells expressed V378A protein relative to wild-type Kv2.1. Specifically, although GFP was expressed in &#8764;20% of cells cotransfected with either Kv2.1 wild-type or V378A plasmids, detectable immunolabeling for V378A protein was found in a significantly lower percentage (1.6%) of cells than wild-type Kv2.1 (7.2%; Fig. 3 B).'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'Coexpression revealed a reciprocal effect of wild-type and mutant subunits on subcellular localization, in that each was impacted by the presence of the other. Specifically, a prominent decrease in plasma membrane&#8211;associated immunolabeling for &#916;C237, and an increased intracellular retention, was observed upon its coexpression with V378A. Moreover, coexpression of &#916;C237 with the V378A mutant yielded an increase in V378A plasma membrane&#8211;associated immunolabeling (Fig.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'Notably, the variant we report here is functionally distinct from those whose currents have been reported previously, in that the V378A channel is gated by voltage, whereas the other disease-associated Kv2.1 mutations characterized to date remain constitutively open (Torkamani et al., 2014). V378A channels are voltage dependent and sensitive to the Kv2-specific toxin GxTX, providing compelling evidence that the nonselective cation currents recorded from cells expressing the mutant channels are from bona fide Kv2.1 channels. These findings support the previously proposed pathogenic role for altered ionic selectivity, yet suggest that the abrogation of voltage-activated gating reported for all other Kv2.1 EIEE mutations that have been studied by voltage clamp (Torkamani et al., 2014) is not required for a disease phenotype.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'The patient described here is heterozygous for the de novo KCNB1 mutation that yields V378A Kv2.1. The distinct phenotypes of the V378A mutant, specifically alterations in both ion selectivity and subcellular localization, are somewhat mitigated by coexpression with Kv2.1 isoforms with wild-type properties. This suggests that the Kv2.1 phenotype of the heterozygous patient may not be as drastically altered as it would be in a homozygous phenotype, and that within the tetrameric Kv2.1 channels, the mutant V378A subunits may exert dose-dependent as opposed to dominant effects on wild-type subunits.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'The cell type&#8211;specific differences in V378A trafficking are reminiscent of Kv2.1 expression in different neuronal subpopulations of the mammalian brain. For instance, Kv2.1 clusters colocalize with ryanodine receptor&#8211;rich calcium signaling domains at the plasma membrane to a different extent in striatal and neocortical neurons (Mandikian et al., 2014), and in cultured hippocampal interneurons versus principal neurons (Antonucci et al., 2001).'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'Cell type&#8211;specific differences in the subcellular localization of Kv2.1 channels are also observed in different heterologous cell backgrounds (Scannevin et al., 1996; Mohapatra and Trimmer, 2006). Our results suggest that expression and localization defects may contribute to the pathophysiological impact of disease-causing Kv2.1 mutations in a cell type&#8211;specific fashion. As Kv2.1 channel localization plays a structural role in neurons in contributing to the formation of plasma membrane&#8211;ER junctions and recruiting junctional proteins including those involved in intracellular Ca2+ signaling (Antonucci et al., 2001; Fox et al., 2015), changes in Kv2.1 localization could yield neurological consequences related to nonconducting functions of Kv2.1.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'Additionally, steady-state levels of Kv2.1 phosphorylation also differ between different neuronal subpopulations (Mandikian et al., 2014). Cell type&#8211;specific differences in the subcellular localization of Kv2.1 channels are also observed in different heterologous cell backgrounds (Scannevin et al., 1996; Mohapatra and Trimmer, 2006). Our results suggest that expression and localization defects may contribute to the pathophysiological impact of disease-causing Kv2.1 mutations in a cell type&#8211;specific fashion.'},\n",
       " {'pmcid': 'PMC4621747',\n",
       "  'pmid': '26503721',\n",
       "  'snippet': 'As Kv2.1 channel localization plays a structural role in neurons in contributing to the formation of plasma membrane&#8211;ER junctions and recruiting junctional proteins including those involved in intracellular Ca2+ signaling (Antonucci et al., 2001; Fox et al., 2015), changes in Kv2.1 localization could yield neurological consequences related to nonconducting functions of Kv2.1. The cell type&#8211;specific pharmacological rescue of the V378A variant indicates that effects on subcellular localization should be considered in therapeutic strategies targeting disease-causing Kv2.1 variants, as attempts to pharmacologically inhibit their aberrant depolarizing currents could inadvertently result in greater functional expression of disease-causing channels.'},\n",
       " {'pmcid': 'PMC4643007',\n",
       "  'pmid': '26561856',\n",
       "  'snippet': 'HCV infection begins with the attachment of the viral particle to the cell surface of hepatocytes through attachment factors such as glycosaminoglycans and low density lipoproteins receptor [6,7]. This attachment allows the contact between the viral particle and specific cell entry factors, including the tetraspanin CD81 [8], the scavenger receptor class B type 1 (SRB1) [9] and the tight junction proteins claudin-1 (CLDN1) [10] and occludin (OCLN) [11,12]. Interestingly, two other tight junction proteins, CLDN6 and CLDN9, were described as cofactors that HCV is able to use instead of CLDN1 in certain cell types [13,14].'},\n",
       " {'pmcid': 'PMC4643007',\n",
       "  'pmid': '26561856',\n",
       "  'snippet': 'E1 and E2 envelope glycoproteins are known to play a key role in HCV entry into hepatocytes, the major target of HCV, by interacting with a series of cellulars factors. Indeed, HCV entry is a complex multistep process requiring many specific entry factors. HCV infection begins with the attachment of the viral particle to the cell surface of hepatocytes through attachment factors such as glycosaminoglycans and low density lipoproteins receptor [6,7].'},\n",
       " {'pmcid': 'PMC4643007',\n",
       "  'pmid': '26561856',\n",
       "  'snippet': 'Cells were observed with a Zeiss Axiophot equipped with a 10x magnification, 0.5 numerical aperture objective. Fluorescent signals were collected with a Coolsnap ES camera (Photometrix) using specific fluorescence excitation and emission filters. Images were processed with Image J software.'},\n",
       " {'pmcid': 'PMC4643007',\n",
       "  'pmid': '26561856',\n",
       "  'snippet': 'Two other members of the CLDN family, CLDN6 and CLDN9, are also able to render 293T cells permissive for HCV entry [13,14]. The first extracellular loop of CLDN1, through specific residues, is involved in HCV entry [10,40]. CLDN1 and the tetraspanin CD81 associate to form co-receptor complexes that are critical for HCV entry [37,41,42].'},\n",
       " {'pmcid': 'PMC4643007',\n",
       "  'pmid': '26561856',\n",
       "  'snippet': 'To characterize the functionality of CLDN6 and OCLN variants in HCV infection, we used HCVpp that are retroviral particles pseudotyped with HCV E1E2 envelope proteins from different genotypes (1a, 2a &#8230;) or control envelope protein (VSV-G) [28,46]. HCVpp enable to study specifically the entry step of the HCV lifecycle. We also used HCVcc that are infectious particles produced in cell culture and which allow for the study of the whole lifecycle of HCV [30,47,48].'},\n",
       " {'pmcid': 'PMC4643007',\n",
       "  'pmid': '26561856',\n",
       "  'snippet': 'Only few patients clear HCV spontaneously and up to 80% of infected people will develop a chronic infection. Spontaneous clearance of infection can be predicted by some factors, such as the limited evolution of viral quasispecies, gender, HCV genotype, immune system specificities, as well as a symptomatic or asymptomatic acute infection [61&#8211;63]. In this way, polymorphisms within the interferon lambda gene region are predictive factors for both spontaneous HCV clearance and response to pegylated interferon/ribavirin therapy [64&#8211;68], [69].'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'More recently, UbcM2 was shown to play a major role in the ubiquitylation status of TDP-43, a DNA-binding protein found in aggregates of amyotrophic lateral sclerosis and subtypes of frontotemporal lobar degeneration [17]. These interactions prompted us to carry out a cell type- and structure-specific immunofluorescence analysis of UbcM2 in mouse brain tissue and to determine if expression of the enzyme is altered in aged mice. We report here the first assessment of the localization and abundance of an E2 in the brain.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'In cell lysates derived from mouse cerebellum (CB) and cortex (Ctx), &#945;-UbcM2 detected a single band of the predicted migration for the enzyme (Fig.&#160;1D).Fig.&#160;1Validation of &#945;-UbcM2 antibody. A HeLa cells were treated with 12&#160;pmol of control siCON siRNA (a, b) or UbcM2-specific siRNA (c, d) for 72&#160;h before fixation and immunostaining with &#945;-UbcM2 (b, d). Nuclei were counterstained with DAPI (a, c).'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'To determine the protein expression profile of UbcM2 in mouse brain, a rabbit polyclonal antibody raised against the N-terminal 58 residues of UbcM2 was affinity purified and validated in HeLa cells treated with either control siRNA (siCON) or UbcM2-specific siRNA (siUbcM2). 3&#160;days post-siRNA treatment, cells were fixed and immunostained with &#945;-UbcM2 and nuclei were counterstained with DAPI.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'Validation of &#945;-UbcM2 antibody. A HeLa cells were treated with 12&#160;pmol of control siCON siRNA (a, b) or UbcM2-specific siRNA (c, d) for 72&#160;h before fixation and immunostaining with &#945;-UbcM2 (b, d). Nuclei were counterstained with DAPI (a, c).'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'We next co-immunolabeled sagittal brain sections with &#945;-UbcM2 and antibodies specific for various cell types resident in brain. Co-staining with an antibody against the nuclear neuronal marker NeuN revealed that the overwhelming majority of UbcM2 is expressed in neurons and that essentially all neurons express the enzyme (Fig.&#160;2A).'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'A Nuclear neuronal marker NeuN (green) and UbcM2 (red) with yellow indicating colocalization in the merged images (c, f, i). B&#8211;D Co-staining with cell type-specific marker CNPase for oligodendrocytes (B), Iba1 for microglia (C), or GFAP for astrocytes (D) shown in green with Hoechst (white, a, c, e) to mark nuclei or with UbcM2 (red, b, d, f). White arrowheads indicate oligodendritic (B), microglial (C), or astrocytic (D) nuclei.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'A Nuclear neuronal marker NeuN (green) and UbcM2 (red) with yellow indicating colocalization in the merged images (c, f, i). B&#8211;D Co-staining with cell type-specific marker CNPase for oligodendrocytes (B), Iba1 for microglia (C), or GFAP for astrocytes (D) shown in green with Hoechst (white, a, c, e) to mark nuclei or with UbcM2 (red, b, d, f). White arrowheads indicate oligodendritic (B), microglial (C), or astrocytic (D) nuclei.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'Co-localization of NeuN and UbcM2 is indicated by yellow in the merged images (d, h, l, p). 20&#215;, size bar 30&#160;&#956;m. B Brain substructure-specific UbcM2 expression as measured by fluorescence intensity relative to the area of highest expression (spinal trigeminal nucleus). Error bars indicate 95&#160;% confidence intervals.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'Error bars indicate 95&#160;% confidence intervals. Disjoint confidence interval bars identify regions with significantly different UbcM2 levelsTable&#160;1UbcM2 expression is decreased in the neurons of specific substructures of the mouse brainBrain structureRelative UbcM2 expressionUbcM2 expression % change in aged miceP valueCA10.776&#8722;11.5180.0590CA20.711&#8722;9.7140.1500CA30.801&#8722;7.6140.1800Caudate putamen0.553&#8722;6.4710.4500Cerebellum-granule layer0.951 &#8722;14.530  0.0032 Cingulate ctx0.6791.3170.8900Dentate gyrus0.6301.8060.8100Facial nu0.9551.9570.6900Hypothalamus-lateral0.9196.6370.1800Inferior colliculus0.687&#8722;7.6920.2800Motor ctx-primary0.951 &#8722;11.111  0.0260 Motor ctx-secondary0.984&#8722;5.3720.2600Nucleus accumbens0.634&#8722;7.0510.3500Olfactory nu0.927 &#8722;10.526  0.0410 Pontine nu0.9767.0830.1400Reticular formation-lateral0.976&#8722;0.9170.8500Sensory ctx-primary0.825&#8722;10.8370.0560Spinal trigeminal nu1.000&#8722;8.9430.0600Superior colliculus0.699 &#8722;14.535  0.0320 Thalamus-laterodorsal0.789&#8722;3.2470.5800Thalamus-ventral0.720&#8722;0.8470.9000Visual ctx-secondary0.923 &#8722;11.454  0.0290 The expression levels of UbcM2 in the brains of young and aged mice were measured and compared by average cellular &#945;-UbcM2-derived fluorescence intensity in NeuN-positive nuclei. A 5.5&#160;% global decrease of UbcM2 expression was detected in aged versus their strain-matched, younger counterparts.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'Co-localization of NeuN and UbcM2 is indicated by yellow in the merged images (d, h, l, p). 20&#215;, size bar 30&#160;&#956;m. B Brain substructure-specific UbcM2 expression as measured by fluorescence intensity relative to the area of highest expression (spinal trigeminal nucleus). Error bars indicate 95&#160;% confidence intervals.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'Our previous studies to develop a global UbcM2 knockout mouse strain resulted in a failure to produce homozygous knockouts, consistent with UbcM2 loss causing embryonic lethality (S. Plafker, unpublished results) and knockdown of the enzyme inducing a cell cycle arrest/exit [10]. The expression profiling described in the present study is therefore a requisite first step for directing future efforts at genetically ablating the enzyme in specific anatomical structures of the brain. Such knockouts could provide useful models to determine if and how UPS deficits contribute to neurodegeneration.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'The principal advance of this work is that it represents the first description of the cell- and structure-specific expression and distribution of a UPS E2 in the mouse brain. Our previous studies to develop a global UbcM2 knockout mouse strain resulted in a failure to produce homozygous knockouts, consistent with UbcM2 loss causing embryonic lethality (S. Plafker, unpublished results) and knockdown of the enzyme inducing a cell cycle arrest/exit [10].'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'By immunological staining of paraffin-embedded murine brain sections with a highly specific antibody, we found that UbcM2 is ubiquitously expressed in neurons and oligodendrocytes but is undetectable in the majority of microglia and astrocytes (Fig.&#160;2). These data largely reflect transcriptional profiling data for isolated astrocytes, neurons, and oligodendrocytes showing enrichment of the UbcM2 transcript (referred to by the human gene name, UBE2E3), as well as many other components of the UPS, in neurons and oligodendrocytes relative to astrocytes [27].'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'We show that the class III E2 UbcM2 is ubiquitously expressed in neurons and oligodendrocytes but is not prevalent in microglia or astrocytes. By quantifying relative &#945;-UbcM2-derived fluorescence from immunohistochemically stained mouse brain sections, we have determined the relative structure-specific neuronal expression of the enzyme. Further, we have observed a modest decrease in neuronal UbcM2 expression in particular sub-structures of the aged mouse brain.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'UbcM2-specific siRNA or control siCON siRNA (Dharmacon, Inc.) were combined with INTERFERin (Polyplus transfection) for 10&#160;min at room temperature before being added to 35,000 HeLa or hTERT RPE-1 (cat # CRL-4000; ATCC) cells seeded in 12-well dishes on glass coverslips. A final concentration of 10&#160;nM siRNA was added to the cells in DMEM (cat.'},\n",
       " {'pmcid': 'PMC4644300',\n",
       "  'pmid': '26566974',\n",
       "  'snippet': 'Heat-induced epitope retrieval was performed using a Retriever 2100 (Electron Microscopy Sciences) in the accompanying R-buffer AG, and slides were allowed to cool in the retriever for 4&#8211;20&#160;h. Slides were then washed in PBS and incubated in blocking solution consisting of 10&#160;% goat serum and 3&#160;% BSA in PBS for 1&#8211;3&#160;h at room temperature. Sections were incubated in &#945;-UbcM2 (1:500) or &#945;-Ubiquitin (1:500) and a cell-type specific antibody in blocking solution at 4&#160;&#176;C overnight. The following day, the slides were washed in PBS and incubated in secondary antibody and DAPI at 0.1&#160;&#956;g/mL or Hoechst33342 at 1&#160;&#956;g/mL diluted in 3&#160;% BSA in PBS for 1&#160;h at room temperature in the dark.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'To select the pre-pilot articles, articles were selected by performing a PubMed search filtered for each journal and using the first five publications returned that contained the relevant resource types from approximately January&#8211;March 2013. The following criteria were applied: Resources (primary antibodies, organisms, and non-commercial tools) were considered identifiable if they contained an accurate RRID or by using the same specific resource identification criteria as described in Vasilevsky et al. (2013).'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'Overall, 96% (538/562) of the RRIDs reported by authors were correct (i.e., the RRID pointed to the correct resource). More specifically, 96% of antibodies (413/429), 87% of organisms (48/55) and 99% of tools (77/78) were correctly reported ( Figure 3).'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'Thus, we opted to craft RRIDs as unique, indexable alphanumeric strings based upon authoritative sources that could support use of web search engines to return papers using a particular research resource. We specifically asked authors to assess resources mentioned in the materials and methods section where they would normally provide identifying information, because we wanted to track actual use of the resource and not just mentions.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'Many of these questions were about scope, i.e., whether a particular research resource should be identified, others were for assistance in finding a resource or guidance in adding a resource not yet contained in the community authorities. While this is not a large number, it is also not insignificant, particularly as the project expands, and certainly points out the need for specific help functions.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'All of these features are undesirable in an identifier system. The Antibody Registry, in contrast, was specifically designed to supply useful and stable identifiers for antibodies and not as a commercial source of antibodies. Similarly, the strain nomenclature developed by the Jackson Laboratory, with its superscripts and special characters, is useful for human curators to identify a particular strain, but causes hiccups in most search engines because of all of the special characters.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'As discussed in the next section, our preference is that DOIs be used to identify the particular instance used, e.g., the version of data or software and any supporting workflows, and that the RRID be used to identify the entity or project referenced. Thus, the RRID would be used to identify the Protein Databank, and a PDB identifier or a DOI used to reference the specific data from the PDB. However, we believe that as the RRID system is adopted, each community should set appropriate identifier systems.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': '(2011). Both studies performed specificity tests on a variety of antibodies and, as is common, did not produce concordant results on all of them. Slotta originally generated the antibody now commonly known as MAB3026 (AB_2178887) and provided its provenance: &#8220;It was transferred to Boehringer-Mannheim as Clone 12H11, resold to Roche and finally bought by Chemicon, and it is now sold as MAB3026.&#8221; They then speculate that a mutation may have crept in at some point that altered the specificity of the antibody.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'Both studies performed specificity tests on a variety of antibodies and, as is common, did not produce concordant results on all of them. Slotta originally generated the antibody now commonly known as MAB3026 (AB_2178887) and provided its provenance: &#8220;It was transferred to Boehringer-Mannheim as Clone 12H11, resold to Roche and finally bought by Chemicon, and it is now sold as MAB3026.&#8221; They then speculate that a mutation may have crept in at some point that altered the specificity of the antibody. However, the discrepancies may also be attributed to the additional testing of the antibody in new conditions, revealing problems that had not been apparent during the initial applications.'},\n",
       " {'pmcid': 'PMC4648211',\n",
       "  'pmid': '26594330',\n",
       "  'snippet': 'Even when a resource initially performed well, due to spontaneous mutations in biological resources and interactions between particular software tools and platforms, problems can arise over time. For example, two recent papers have published extensive tests showing that common antibodies for NF-&#954;B show non-specificity under some circumstances ( Listwak et al., 2013; Slotta et al., 2014). Many of these antibodies are extensively used in the literature, but readers of a particular article have no way of knowing that concerns have been raised.'},\n",
       " {'pmcid': 'PMC4660684',\n",
       "  'pmid': '26608466',\n",
       "  'snippet': 'For this, we first investigated the effect of scutellarin on TNC1 astrocytes in vitro. As opposed to BV-2 cells, direct application of scutellarin to TNC1 did not elicit an obvious increase in GFAP expression, a specific marker for astrocytes; moreover, neither TNF-&#945; nor iNOS expression was noticeably altered. It stands to reason therefore that scutellarin does not act directly, if at all, on astrocytes.'},\n",
       " {'pmcid': 'PMC4660684',\n",
       "  'pmid': '26608466',\n",
       "  'snippet': 'Scutellarin acts primarily to amplify the microglia-mediated astrocyte reaction thus highlighting a functional &#8220;cross-talk&#8221; between activated microglia and reactive astrocytes. On the other hand, the specific molecules such as cytokines involved in this are uncertain. Notwithstanding, it is concluded that scutellarin is a potent agent that can facilitate the communication between activated microglia and reactive astrocytes in ischemic injury, an inter-glial cell mechanism that may be crucial to issue repair.'},\n",
       " {'pmcid': 'PMC4660684',\n",
       "  'pmid': '26608466',\n",
       "  'snippet': 'Coronal sections of 7&#160;&#956;m thickness were cut on a microtome (Model: 2165; Leica, Bensheim, Germany). For blocking of nonspecific binding proteins, tissue sections were incubated in 5&#160;% normal goat serum diluted in phosphate-buffered saline (PBS) for 1&#160;h at room temperature (22&#8211;24&#160;&#176;C). The sections were then incubated in a humidified chamber with the following primary antibodies against proinflammatory mediators: TNF-&#945; (rabbit polyclonal IgG1:100) (Chemicon International, Temecula, CA, USA; Cat.'},\n",
       " {'pmcid': 'PMC4660684',\n",
       "  'pmid': '26608466',\n",
       "  'snippet': 'No.sc-25392) diluted in PBS overnight at 4&#160;&#176;C. Following washing in PBS, sections were incubated, with the respective fluorescent secondary antibodies: Cy3-conjugatedsecondary antibody or FITC-conjugated lectin (Lycopersicon esculentum), which labels both microglia and blood vessel endothelial cells for 1&#160;h at room temperature. Brain sections from different groups were also incubated with the primary antibodies against glial fibrillary acidic protein (GFAP), a specific marker for astrocytes (Merck KGaA, Darmstadt, Germany. Cat.'},\n",
       " {'pmcid': 'PMC4681077',\n",
       "  'pmid': '26674138',\n",
       "  'snippet': 'Muscle nociceptors are generally small and contain high levels of the neuropeptide CGRP [15, 16] whereas muscle spindle afferents are generally large-sized, myelinated fibres that express carbonic anhydrase, or the calcium binding proteins calretinin, calbindin, neurocalcin or parvalbumin [19&#8211;25]. Some of these markers, however, are not specific to muscle afferents, as they are also found in skin afferents [26&#8211;28]. Whilst parvalbumin is generally regarded as a reliable marker for proprioceptive afferents [22, 23], genetic studies have revealed that, by itself, it is not a selective marker of muscle proprioceptors nor a marker of muscle afferent nociceptors [26, 29].'},\n",
       " {'pmcid': 'PMC4681077',\n",
       "  'pmid': '26674138',\n",
       "  'snippet': 'Thus, since retrograde tracing from muscles was not used in this study we cannot be sure that all PV-ir cells in this study are muscle proprioceptors. However, only around 1.5&#160;% of L4-5 PV-ir DRG cells arise from skin afferents [22] and a recent anatomical study showed that 27&#160;% of lumbar dorsal root fibres were positive for &#945;3 Na+/K+&#160;ATPase, a specific marker of muscle afferents [13]. The similarity in percentage expression of PV-ir and &#945;3 Na+/K+&#160;ATPase implies that PV-ir afferents should express this ATPase transporter, although this awaits confirmation.'},\n",
       " {'pmcid': 'PMC4681077',\n",
       "  'pmid': '26674138',\n",
       "  'snippet': 'For each animal, and each DRG, a minimum of 500 neuronal profiles (cells with clearly visible nuclei, confirmed by DAPI staining) were counted from 3 to 5 randomly chosen sections and the immunoreactivity (ir) was expressed as a percentage of all cells counted. PV-ir cells were considered positive if the intensity of staining was clearly above background staining seen in small-sized neurons in the same section.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Here, we used the DGRP to perform single- and two-locus associations for size-related developmental traits in Drosophila. We find pervasive trait and sex-specificity of top variants, validate a substantial number of novel growth regulators, and extend our knowledge of the genetic control of size beyond existing growth regulatory networks.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'As the genetic correlation between CS and IOD was moderate (0.46 and 0.51 for females and males, respectively), we expected to map both shared and trait-specific SNPs. To find loci that specifically affect variation in wing size unrelated to the overall organismal size variation we constructed an additional phenotype (rCS) that had the effect of IOD on CS removed via regression. In addition to the effect of the food batch, two cosmopolitan inversions, In(2L)t and In(3R)Mo, were correlated with both CS and IOD and we addressed their effect on size by modeling their presence in the homozygous state (S2 Fig), yielding the inversion-corrected phenotypes CSIC and IODIC.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'This association method uses a linear mixed model to capture confounders such as population structure and cryptic relatedness. As the genetic correlation between CS and IOD was moderate (0.46 and 0.51 for females and males, respectively), we expected to map both shared and trait-specific SNPs. To find loci that specifically affect variation in wing size unrelated to the overall organismal size variation we constructed an additional phenotype (rCS) that had the effect of IOD on CS removed via regression.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'We selected a subset (41% to 69%) of candidates identified by each of our six wing size GWAS (CS, CSIC and rCS in both sexes) for functional validation by tissue-specific RNAi. A total of 64% to 79% of tested genes had significant effects on wing area (p&lt;0.001, Wilcoxon rank sum test, S7 Table, Figs 4A and S9).'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': '(a) Validated genes in females. Bars show the percent change in median wing area compared to CG1315 RNAi upon wing-specific knockdown of candidate genes. Only the lines yielding a significant wing size change (p&lt;0.001, Wilcoxon rank sum test) are depicted.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Knockdown of genes in the whole eye disc often affects eye area, which in turn leads to a bigger or smaller head area between the eyes. As we used exactly this area for determining IOD, we were concerned that effects of knockdown on compound eye development could not be discerned from effects on IOD specifically. For this reason we chose not to perform validation for IOD candidates.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Yet, in contrast to human GWA analyses, which require independent replication, we exploited the fact that our model organism is amenable to direct validation strategies and tested candidates corresponding to a much lower significance threshold of 10&#8722;05 for an involvement in size determination. Using tissue-specific RNAi, we validated 33 novel genes affecting Drosophila wing size. Nominally significant intergenic associations were preferentially located in regions with an enhancer signature and overlapped lincRNA loci.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Our validation of a substantial number of novel genes underscores the complementarity of the GWA approach to classical genetics and highlights the importance of probing natural variants. However, future studies would benefit from utilizing bigger population sizes in order to improve statistical power and from investigating populations with different geographic origins, to address population specificity of associated variants.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'the CRISPR/Cas9 system to be corroborated. Unfortunately we cannot conclude anything about sex-specificity of our variants from the knockdown results. A knockdown is a very different perturbation from the effects of, for example, a regulatory variant.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Interestingly, an intronic SNP in the sex determination gene dsx had the lowest p-value in the female relative wing size GWAS but had a smaller effect size in males. Dsx is a transcription factor with sex-specific isoforms, and has many targets with sex- and tissue specific effects [90]. We also observed sex-specificity for the genome-wide significant two-locus interactions.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Dsx is a transcription factor with sex-specific isoforms, and has many targets with sex- and tissue specific effects [90]. We also observed sex-specificity for the genome-wide significant two-locus interactions.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'We did not observe correlation between Wolbachia infection status and any trait and thus did not include this as a covariate in the model. Specifically, the models used were: CSraw = &#945; + X1&#946;1 + X2&#946;2 + Fu + &#949;, where X1 refers to the sex covariate, X2 refers to the inversion covariate, &#949;~&#925;n(0,&#963;&#949; In2) with n being the number of lines, u~Nk(0, &#963;u2Ik) with k being the number of food batches and Fu an (n,k)-indicator matrix, associating each line to its respective food batch. The GWA analyses were performed using the estimated residual of this model (CS = &#949;).'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'To find loci that specifically affected variation in wing size unrelated to the overall body size variation we constructed an additional phenotype (rCS) that had the effect of IOD on CS removed via regression: CSraw = &#945; + IOD + X1&#946;1 + Fu + &#949;, where X1 and Fu refer again to the sex-effect and the food batch effect. We did not model the inversions because the residuals of this model were not correlated with the inversions.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'We decided to use this line as reference because it was in the same background as most of our tester lines, an essential factor to consider when assessing genes that presumably only have a small effect on size upon knockdown. The CG1315 knockdown had never shown an effect in any setting and it allowed us to evaluate unspecific effects of RNAi knockdown on wing size. Prior to the experiment, driver lines were bred under controlled density to eliminate cross-generational effects of crowding on size.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'Percent change in median wing area compared to CG1315 RNAi upon wing-specific knockdown of the validated candidate genes in males. Only the lines yielding a significant wing size change (p&lt;0.001, Wilcoxon rank sum test) are depicted.'},\n",
       " {'pmcid': 'PMC4709145',\n",
       "  'pmid': '26751788',\n",
       "  'snippet': 'The two p-value distributions differ by a location shift that is not zero (i.e. are not the same); specifically, the&#8211;log10 transformed control p-value distribution (red) is shifted towards the left of the&#8211;log10 transformed candidate p-value distribution (black) (one sided Wilcoxon rank sum test p = 0.02). b: The distribution of candidate effect sizes (percent change in wing size upon knockdown) is shifted towards positive effect sizes (white boxes), whereas the control knockdown effect size distribution (red) is more centered on 0.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Although the mechanisms controlling DNA demethylation are still not well defined, several studies suggest the involvement of two major pathways in this process: the activity of TET methylcytosine dioxygenase proteins [17, 18] and AID/APOBEC cytidine deaminase enzymes [19&#8211;21]. Importantly, active DNA demethylation was reported in human muscle-specific genes in heterokaryons derived from the fusion of human non-muscle cells with mouse muscle cells, indicating a dynamic DNA methylation/demethylation interplay during muscle differentiation [22].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'In the last few years, global analyses of DNA methylation dynamics in pluripotent stem cells and their differentiated progeny have begun to uncover the mechanisms involving coordinated changes occurring in the epigenome that are essential for lineage-specification and maintenance of cellular identity [6&#8211;15]. Very recently, we reported the genome-wide comparison of the DNA methylation profiles of pluripotent embryonic stem cells and muscle stem cells showing, in agreement with other studies, that cellular differentiation is associated with a modest but global increase in DNA methylation, which interestingly is accompanied by lineage-specific demethylation [16]. Although the mechanisms controlling DNA demethylation are still not well defined, several studies suggest the involvement of two major pathways in this process: the activity of TET methylcytosine dioxygenase proteins [17, 18] and AID/APOBEC cytidine deaminase enzymes [19&#8211;21].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'In addition, DNA methylation affects gene expression by regulating promoters and distal regulatory elements, such as enhancers and insulators (reviewed in [5]). In the last few years, global analyses of DNA methylation dynamics in pluripotent stem cells and their differentiated progeny have begun to uncover the mechanisms involving coordinated changes occurring in the epigenome that are essential for lineage-specification and maintenance of cellular identity [6&#8211;15]. Very recently, we reported the genome-wide comparison of the DNA methylation profiles of pluripotent embryonic stem cells and muscle stem cells showing, in agreement with other studies, that cellular differentiation is associated with a modest but global increase in DNA methylation, which interestingly is accompanied by lineage-specific demethylation [16].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Our results showed a common muscle-specific DNA demethylation signature required to acquire and maintain the muscle-cell identity. Notably, downregulation of the muscle-specific cytidine deaminase Apobec2 in ES-derived myogenic precursors reduced myogenin-associated DNA demethylation and dramatically affected the expression of differentiation markers and, ultimately, muscle differentiation.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'In the present study, we addressed for the first time the DNA methylation dynamics of the major genes orchestrating myogenic determination and differentiation by comparing pluripotent ESCs, myogenic precursors from Pax7-inducible ESCs, proliferating muscle stem cells, and their respective myotube derivatives. Our results showed a common muscle-specific DNA demethylation signature required to acquire and maintain the muscle-cell identity. Notably, downregulation of the muscle-specific cytidine deaminase Apobec2 in ES-derived myogenic precursors reduced myogenin-associated DNA demethylation and dramatically affected the expression of differentiation markers and, ultimately, muscle differentiation.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Using bisulphite sequencing analysis, we compared the DNA methylation state of undifferentiated ESCs and muscle stem cells, isolated from adult skeletal muscle tissue, along with respective differentiated myotubes and mature myofibres (Fig.&#160;1a). In addition, to assess whether the DNA methylation events were muscle-lineage specific, we analysed the methylation profiles of the following non-myogenic cell lines: neuronal precursor cells (NPCs), mouse embryonic fibroblasts (MEFs) and cardiomyocytes (HL1). As shown in Additional file 1a, b, all analysed CGIs in Pax3 and Pax7 regulatory regions were completely unmethylated in all the samples.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Since DNA methylation often occurs in non-CGI regions, we investigated whether enhancers and promoters present a cell-specific deposition of this modification. Previous studies of other groups identified two muscle-specific regulatory regions upstream of the MyoD transcription start site (TSS), located at -20&#160;kb and -5&#160;kb, respectively [32&#8211;34].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Since DNA methylation often occurs in non-CGI regions, we investigated whether enhancers and promoters present a cell-specific deposition of this modification. Previous studies of other groups identified two muscle-specific regulatory regions upstream of the MyoD transcription start site (TSS), located at -20&#160;kb and -5&#160;kb, respectively [32&#8211;34]. Importantly, it was shown that the distal MyoD enhancer located at -20&#160;kb of the TSS was modulated by DNA methylation in mouse tissues [31].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Since MyoD is already higher expressed in MBs, this maintenance of active enhancer marks in MTs might be involved in keeping an open chromatin state of the region to maintain gene expression, rather than promoting further transcriptional activation. All together, these results indicate that the unmethylated state of CGIs is conserved across the different cell types, whereas Pax3 and -20&#160;kb MyoD enhancer element show cell type-specific dynamic methylation patterns associated with changes in gene expression.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Strikingly, the terminal differentiation genes Myh1, Myh4 and Myh8 gained the repressive mark H3K27me3 in MB cells after DNA demethylation, suggesting that gene expression was maintained epigenetically repressed by PRC2 complex until the onset of the terminal differentiation stage (Fig.&#160;2b). Interestingly, Mrf4 followed a different DNA methylation pattern, being highly demethylated only in mature myofibres, where it is specifically highly expressed (Fig.&#160;2a).Fig. 2Epigenetic profile of muscle-specific CpG-poor promoter genes during myogenesis.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Interestingly, Mrf4 followed a different DNA methylation pattern, being highly demethylated only in mature myofibres, where it is specifically highly expressed (Fig.&#160;2a).Fig. 2Epigenetic profile of muscle-specific CpG-poor promoter genes during myogenesis. a Scheme of the Myog, Mrf4, Ckm and Myh1, 4 and 8 loci.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Epigenetic profile of muscle-specific CpG-poor promoter genes during myogenesis. a Scheme of the Myog, Mrf4, Ckm and Myh1, 4 and 8 loci.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Interestingly, the promoters of the late differentiation genes Myh8 and Ckm remained fully methylated until the iPax7-MT stage, when they became partially demethylated simultaneously with gene activation (Fig.&#160;3b, c). Importantly, we did not observe any loss of DNA methylation in the absence of Pax7 expression in the control cells (no Dox treatment), indicating that the observed demethylation was skeletal muscle-specific. Finally, the analysis of the promoter regions of the pluripotency genes Pou5f1 and Dppa4, which are progressively silenced, showed a similar gain of DNA methylation during cell differentiation as observed in primary myoblasts (Fig.&#160;3b, c).Fig.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'b Expression profiles of markers of pluripotency (Pou5f1 and Dppa4), muscle lineage commitment (Pax3, Myf5 and MyoD) and muscle differentiation (Myog, Ckm and Myh8) were measured by qRT-PCR at the successive time points of iPax7 ESC-derived myogenic model and normalized by Gapdh expression (n&#8201;=&#8201;3, mean&#8201;&#177;&#8201;SD). c DNA methylation dynamics of Pax3, Myf5 and MyoD enhancers and CpG-poor promoters of muscle-specific genes (Myf5 Myog, Ckm and Myh8) and pluripotency genes (Pou5f1 and Dppa4) during Pax7-induced myogenesis (+Dox shown in red and -Dox shown in black). The results of three biological replicates are shown.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'b Expression profiles of markers of pluripotency (Pou5f1 and Dppa4), muscle lineage commitment (Pax3, Myf5 and MyoD) and muscle differentiation (Myog, Ckm and Myh8) were measured by qRT-PCR at the successive time points of iPax7 ESC-derived myogenic model and normalized by Gapdh expression (n&#8201;=&#8201;3, mean&#8201;&#177;&#8201;SD). c DNA methylation dynamics of Pax3, Myf5 and MyoD enhancers and CpG-poor promoters of muscle-specific genes (Myf5 Myog, Ckm and Myh8) and pluripotency genes (Pou5f1 and Dppa4) during Pax7-induced myogenesis (+Dox shown in red and -Dox shown in black). The results of three biological replicates are shown.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'DNA demethylation at specific myogenic regulatory regions was clearly observed in primary myoblasts and in iPax7 ESC-derived myogenic precursors (Figs.&#160;1b, 2a and 3c). At first, we questioned whether the reported demethylation could result from a passive loss of methylation, in which the lack or the significant reduction of DNA methyltransferases (Dnmts) coupled to the successive cell divisions would lead to the loss of this epigenetic mark.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'All Dnmts were expressed, and, as shown in Additional file 4, the expression levels of Dnmt1 and Dnmt3a did not significantly change during EB differentiation, while Dnmt3b expression decreased during cell differentiation independently of Pax7 expression. These observations would suggest that specific DNA methylation reduction on muscle-related genes might be mediated by an active mechanism rather than by a passive one.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'The expression levels of Tet1, Tet2 and Tet3 did not change in the presence or absence of Pax7 expression during the EB differentiation, as shown in Additional file 4b. However, the expression levels of Apobec2, the skeletal muscle specific member of the AID/APOBEC family, were highly induced upon Pax7-mediated muscle-commitment, being very high at the progenitor stage and derived MTs (Fig.&#160;4a). Importantly, despite the lack of high specificity of the Apobec2 antibody (Additional file 4c) a clear increase of Apobec2 protein was observed by western-blot concomitant with the differentiation progression (Fig.&#160;4a).Fig.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'However, the expression levels of Apobec2, the skeletal muscle specific member of the AID/APOBEC family, were highly induced upon Pax7-mediated muscle-commitment, being very high at the progenitor stage and derived MTs (Fig.&#160;4a). Importantly, despite the lack of high specificity of the Apobec2 antibody (Additional file 4c) a clear increase of Apobec2 protein was observed by western-blot concomitant with the differentiation progression (Fig.&#160;4a).Fig. 4Impaired myogenic progression and differentiation and reduced DNA demethylation in Apobec2 knockdown myogenic precursors.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'In order to test whether Apobec2 is required for the myogenic process, we performed loss of function experiments by down-regulating Apobec2 levels using lentiviral vectors encoding for specific shRNAs. To identify the most appropriate time point for shRNA viral-mediated transduction (before DNA demethylation has taken place) we performed and analysed a daily time course starting from Dox-treated PDGFR&#945;+/FLK1- sorted cells from day 5 EBs until iPax7-MB precursor stage (day 10).'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Tissue-specific stem cells are lineage-restricted cells that have lost pluripotency capacities and gained cell-type identity. Epigenetic regulation, including DNA methylation and the post-translational modifications of histones, is crucial in the establishment and maintenance of cellular identity [1&#8211;3, 42].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'The detailed analysis of CpG-poor regulatory regions for the developmental gene Pax3, MyoD and Myogenin genes as muscle cell-identity transcription factors, and Ckm, Mhy1, Myh4 and Myh8 as myogenic terminal differentiation genes revealed a specific DNA demethylation signature required to acquire and maintain muscle-cell identity during the establishment of the muscle lineage (Figs.&#160;1b, 2a and 3c). These results reinforce observations reported in other cellular models that lineage-specific DNA demethylation, together with pluripotency genes hypermethylation, occur during early cell fate decisions conferring unique DNA methylation patterns which correlate with defined cellular identities [43&#8211;46]. Interestingly, our results showed that the crucial myogenic differentiation factor Myogenin became fully demethylated in muscle-committed cells before gene expression (Fig.&#160;2a).'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'In this study, we analysed for the first time the DNA methylation dynamics of the principal genes orchestrating myogenic determination and differentiation programs by comparing ES cells and their Pax7-induced skeletal myogenic derivatives with muscle stem cells, both in proliferating and differentiating conditions. The detailed analysis of CpG-poor regulatory regions for the developmental gene Pax3, MyoD and Myogenin genes as muscle cell-identity transcription factors, and Ckm, Mhy1, Myh4 and Myh8 as myogenic terminal differentiation genes revealed a specific DNA demethylation signature required to acquire and maintain muscle-cell identity during the establishment of the muscle lineage (Figs.&#160;1b, 2a and 3c). These results reinforce observations reported in other cellular models that lineage-specific DNA demethylation, together with pluripotency genes hypermethylation, occur during early cell fate decisions conferring unique DNA methylation patterns which correlate with defined cellular identities [43&#8211;46].'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'This observation points out an epigenetically accentuated myogenic commitment of adult muscle stem cells compared to myogenic precursors. Importantly, the loss of DNA methylation only took place upon Pax7 expression (+Dox) indicating that Pax7 is required for skeletal muscle-specific demethylation. Notably, the differentially methylated enhancer located at -111&#160;kb of Myf5 TSS is a direct target of Pax7 in satellite cells [56], suggesting a link between Pax7 and DNA methylation.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'In proliferating myoblasts, the differentiation genes would be kept in a silenced/poised transcriptional state by PRC2 and HDACs complexes, whereas in myotubes they would be expressed upon the recruitment of chromatin activating complexes.Fig. 5Model for the epigenetic control of muscle-specific gene expression during myogenesis. Diagram proposing a simplified model where myogenic transcription factors would activate muscle genes by modulating both the recruitment, direct or indirect, of DNA demethylases to reduce DNA methylation levels and chromatin compaction, and histone modifiers/chromatin remodelers to activate transcription of muscle genes.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Model for the epigenetic control of muscle-specific gene expression during myogenesis. Diagram proposing a simplified model where myogenic transcription factors would activate muscle genes by modulating both the recruitment, direct or indirect, of DNA demethylases to reduce DNA methylation levels and chromatin compaction, and histone modifiers/chromatin remodelers to activate transcription of muscle genes.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'This study presents for the first time the DNA methylation dynamics of the principal genes orchestrating the myogenic determination and differentiation programs during myogenesis. Our results show that a common DNA demethylation signature is required to acquire and maintain the muscle-cell identity, and that downregulation of the muscle-specific cytidine deaminase Apobec2 in ES-derived myogenic precursors reduces myogenic-associated DNA demethylation and impairs muscle differentiation. Importantly, we have also identified muscle epigenetic markers that may be useful to ensure the efficient and safe generation of ES- and iPS-derived myogenic progenitors for therapeutic applications.'},\n",
       " {'pmcid': 'PMC4831197',\n",
       "  'pmid': '27075038',\n",
       "  'snippet': 'Treated DNA was amplified as previously described [78], purified and sequenced by Sanger direct sequencing. In Additional file 7 the specific primers for the methylation profiling of the different genomic regions are listed. In Additional file 8 it is shown how the DNA methylation range was quantified and the corresponding validation by pyrosequencing.'},\n",
       " {'pmcid': 'PMC4848777',\n",
       "  'pmid': '27125505',\n",
       "  'snippet': 'We first performed LIX1 loss-of-function experiments using RCAS(A)-ShLIX1 (short-hairpin RNA directed against LIX1) retroviruses. When injected into the presumptive domain of the developing stomach, RCAS(A)-ShLIX1 retroviruses led to a specific decrease in endogenous LIX1 expression, demonstrated by in situ hybridization and RT-qPCR analyses (Fig.&#160;2a, c). In situ hybridization analysis revealed a decrease in the expression of the SMC determination marker SM22 in E6.5 ShLIX1-expressing stomachs compared to controls (Fig.&#160;2b) upon LIX1 silencing.'},\n",
       " {'pmcid': 'PMC4848777',\n",
       "  'pmid': '27125505',\n",
       "  'snippet': 'The complementarity between LIX1 and &#945;SMA expression prompted us to investigate whether LIX1 is required for the process of stomach SMC determination. This was accomplished using the avian replication-competent retroviral (RCAS) transgenesis method that allows in vivo gain- or loss-of-function approaches of specific genes in the stomach mesenchyme (Additional file 2: Figure S2A) [6, 8, 19, 22]. We first performed LIX1 loss-of-function experiments using RCAS(A)-ShLIX1 (short-hairpin RNA directed against LIX1) retroviruses.'},\n",
       " {'pmcid': 'PMC4848777',\n",
       "  'pmid': '27125505',\n",
       "  'snippet': 'In contrast, injection of unrelated RCAS(A)-ShRNA retroviruses, which do not target LIX1, had no effect on &#945;SMA expression (Additional file 3: Figure S3A). Moreover, when RCAS(A)-ShLIX1 retroviruses were co-injected with RCAS(B)-hLIX1 retroviruses, which induce the expression of the human LIX1 protein insensitive to the chick-specific RCAS(A)-ShLIX1 retroviruses, normal expression of &#945;SMA was restored, demonstrating the specificity of the ShLIX1 construct for LIX1 (Additional file 3: Figure S3B). Levels of BARX1 transcripts were comparable in ShLIX1-expressing stomachs compared to controls, indicating that the patterning of the stomach was unaffected by LIX1 silencing (Fig.&#160;2c).'},\n",
       " {'pmcid': 'PMC4848777',\n",
       "  'pmid': '27125505',\n",
       "  'snippet': 'This was demonstrated both by whole-mount in situ hybridization, which showed a larger expression domain of determined SMC markers SM22 and BAPX1 [24] in LIX1-misexpressing stomachs compared to controls (Fig.&#160;3b), and by &#945;SMA immunostaining on sections, showing that sustained LIX1 expression led to a decrease in the size of the submucosa (Fig.&#160;3c, compare white bars). Accordingly, analysis of the enteric nervous system network using in situ hybridization of SOX10 transcripts revealed that enteric nervous system precursors, which normally colonize the SMC domain specifically [8], had migrated into the adjacent tendon territory, further indicating an expanded SMC domain in LIX1-misexpressing stomachs compared to controls (Fig.&#160;3b, white arrowhead). Further analysis by RT-qPCR confirmed that, compared to control stomachs, LIX1-misexpressing stomachs harboured higher levels of SMC determination marker &#945;SMA and BARX1 transcripts at E6, whereas MYOCARDIN and SRF levels were not significantly affected (Fig.&#160;3d).'},\n",
       " {'pmcid': 'PMC4848777',\n",
       "  'pmid': '27125505',\n",
       "  'snippet': 'Isolation of mesodermal and endodermal layers from stage 25 stomachs (referred to as E5) was performed as previously described [8]. The efficiency of dissections was evaluated by monitoring the expression of SHH and BARX1, which are specific markers of the epithelial and mesenchymal layers, respectively.'},\n",
       " {'pmcid': 'PMC4848777',\n",
       "  'pmid': '27125505',\n",
       "  'snippet': 'The chicken DF-1 fibroblast cell line was cultured as previously described [22]. Cell growth in DF-1 cultures was assessed using the Muse Count and Viability reagent following the manufacturer&#8217;s specifications (Muse Cell Analyzer &#8211; Millipore). DF-1 cells were plated on plastic at 2000 cells/cm2 to obtain low-density cultures and 6000 cells/cm2 to obtain high-density cultures.'},\n",
       " {'pmcid': 'PMC4860965',\n",
       "  'pmid': '27462437',\n",
       "  'snippet': 'Together with nitrilase2 and the NitFhit fusion proteins of D. melanogaster and C. elegans, Nit1 is a member of the 10th out of 13 branches of the Nit protein superfamily. Only branch 1 enzymes are known to have specific nitrilase activity performing nitrile hydrolysis, whereas most other branches have apparent or postulated amidase or amide condensation activities [6]. Mammalian Nit1 was reported to cleave several synthetic dipeptide substrates, but has no amidase activity as reported for mammalian nitrilase2 [7, 8].'},\n",
       " {'pmcid': 'PMC4860965',\n",
       "  'pmid': '27462437',\n",
       "  'snippet': 'To prove that this interaction is direct, pull-down assays with purified recombinant fusion proteins were performed. In these experiments, MBP-dNitFhit specifically associated with GST-Armadillo and GST-&#946;-catenin but not with GST alone (Supplementary Figure S8B and C). Taken together, these findings suggested that dNitFhit in forming a complex with Armadillo may modulate Wg signalling in Drosophila as shown for hNit1 above.'},\n",
       " {'pmcid': 'PMC4860965',\n",
       "  'pmid': '27462437',\n",
       "  'snippet': 'Different ways how hNit1 represses &#946;-catenin signalling function can be considered. For example, hNit1 such as p15RS may be involved in the recruitment of specific HDACs [33]. HDACs were reported to form a complex with LEF-1/TCF transcription factors and &#946;-catenin [34, 35], to mediate their repressive function.'},\n",
       " {'pmcid': 'PMC4860965',\n",
       "  'pmid': '27462437',\n",
       "  'snippet': 'Monoclonal anti-FLAG M2, anti-MBP (clone MBP-17) and anti-nitrilase1 (clone1C3) antibodies were purchased from Sigma (Schnelldorf, Germany), anti-&#946;-catenin (clone 14) from BD Biosciences (Heidelberg, Germany), anti-M14M [51] and anti-glyceraldehyde-3-phosphate dehydrogenase (GAPDH) from Merck Millipore (Darmstadt, Germany) and anti-Fhit from Upstate Biotechnology (Millipore Corp., Billerica, MA, USA). A polyclonal rabbit anti-nitrilase1 antibody was generated at Pineda-Antik&#246;rper-Service Berlin by immunization with the nitrilase1-specific peptide CFHERGQDWEQTQKI coupled to KLH. Antibodies were affinity-purified on MBP-hNit1 coupled to Mini-Leak beads (KemEnTec, Taastrup, Denmark).'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'Single-cell-type analysis is a powerful experimental approach, allowing for the capture of information from specific cell types that would normally be lost due to the heterogeneity of cells in a tissue, giving us greater insight into the role of specialized cells. In plants, successful single cell type analysis has been undertaken for only a handful of cell types, including pollen grains, but also, due to ease of isolation, cells of the epidermis, such as root hairs, guard cells and trichomes [1].'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'In order to confirm the changes in abundance of proteins from the quantitative proteomic data we selected nine protein candidates for which peptide specific antibodies were available and performed western blot analysis (Fig.&#160;2a and b). Calculation of the average normalized band density from three blots of distinct biological replicates showed significant and complementary changes in amounts of the majority of detected proteins selected.'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'In order to confirm the presence of photosynthetically active chloroplasts in the EBC, confocal scanning microscopy was used to detect chlorophyll auto fluorescence. Red intrinsic fluorescence by chloroplasts indicates photochemical activity and specifically is a measure of photosystem II activity [23]. As observed in Fig.&#160;3a, red fluorescence from chloroplasts was seen around the periphery of the large EBCs (delineated by green autofluorescence of the cell wall), and these chloroplasts are clearly distinguishable from the chloroplasts in the underlying mesophyll cell layer (Fig.&#160;3b).'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'Additionally, V-ATPase activity would be essential for turgor generation to facilitate rapid cell expansion of the EBC [37]. The role of the V-ATPase in determining cell shape and size through turgor generations has been demonstrated by studying Arabidopsis VHA-C mutants, which showed reduced cell expansion of specific cell types due to reduced turgor [38].Fig. 7Integration of transcript, protein, metabolite and ionome data into EBC metabolic pathways.'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'Of these, four were subunits of the peripheral cytoplasmic V1 sector of the vacuolar H+-ATPase, V-ATPase; VHA-G, VHA-A, VHA-B and VHA-E (Fig.&#160;8). Western blot analysis using subunit specific antibodies confirmed the increase in VHA-B and VHA-E, (Fig.&#160;2) and results were corroborated for VHA-A and VHA-B from our previous EBC transcriptomics study [24]. Transcriptomics data also revealed changes in additional VHA V1 subunits, including VHA-D, VHA-F, VHA-H, but also subunits of the V0 membrane sector, VHA-c and VHA-d (Fig.&#160;8).'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'Multi-omics data integration is challenging for plant-derived pathways and particularly for non-model plants, however, better insight into functional networks can be gained if we incorporate data compiled from different technologies. From information from this study and our previous omics analysis of EBC [24, 28], a picture is emerging of a metabolically active cell, photosynthetically active and undertaking CAM, with salinity treatment resulting in decreased abundance of photosynthetic machinery proteins, increases in enzymes involved in photorespiration, glycolysis and proteins specific for CAM. (Fig.&#160;7).'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': 'The labelling reaction was stopped by the addition of one &#956;l of 10&#160;mM lysine and incubated on ice for a further 10&#160;min. To avoid CyDye specific artifacts resulting from preferential labelling or variable fluorescence characteristics of the gel matrix or glass plates at the different excitation wavelengths used for acquisition, dye swapping between experimental samples was carried out (Additional file 5). Following labelling, equal volumes of rehydration buffer containing 2X DTT and ampholytes was added to each sample (7&#160;M urea, 2&#160;M thiourea, 2&#160;% (w/v) ASB-14, 2&#160;% (w/v) CHAPS, 100&#160;mM DTT, 1&#160;% (v/v) Bio-Lyte 3&#8211;10 ampholytes (Bio-Rad) to give a final concentration of 50&#160;mM DTT and 0.5&#160;% (v/v) Bio-Lyte 3&#8211;10 ampholytes.'},\n",
       " {'pmcid': 'PMC4862212',\n",
       "  'pmid': '27160145',\n",
       "  'snippet': '#sc7455 AB_640163). Custom and in-house made primary antibodies used included; the M. crystallinum phosphoenol pyruvate carboxylase anti-PEPCase CAM isoform (110 kD) [51]; the M. crystallinum aquaporin PIP1;4 peptide specific antibody (41 kD) [52]; M. crystallinum myo-inositol O-methyl transferase anti-IMT antibodies (40 kD) [53, 54]. Dilutions were as follows: VHA-A, VHA-B, VHA-c, were 1/2000; all others were 1/1000.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'The behavioral evidence for distant chemoreception show that the addiction of fish juice to the water causes, in octopus (Wells, 1963) and cuttlefish (Messenger, 1977), active movements. In this context, cephalopods produce the ink that they use as direct deterrent of predators and as an alarm cue for conspecifics (Palumbo et al., 1999; Di Cosmo, 2003; Di Cosmo et al., 2006; Derby, 2014).'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'All the labeled cells are concentrated within the superficial layer of the epithelium. The PCNA immunoreactivity is specifically restricted to the nuclei of type 1 and type 2 OSNs (Fig.&#160;4). Fig.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'We describe the detailed morphology of young male and female octopus olfactory epithelium, and using a combination of classical morphology and 3D reconstruction techniques we propose a new classification for O. vulgaris OSNs. Furthermore using specific markers such as OMP and PCNA we has been able to identify and differentially localize, both mature olfactory sensory neurons (OMP immunoreactivity) and olfactory sensory neurons involved in epithelium turnover (PCNA immunoreactivity).'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'The type 4 cells, classified also as &#8216;ring shaped&#8217; cells (Woodhams and Messenger, 1974), deserve particular attention given that their shape is quite specific. Besides their uncommon size, they possess a large vacuole and are mainly distributed in the deeper layer of the central part of the OE where they appear to give turgor to the OP that characterizes the olfactory organ shape.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'In invertebrates an olfactory sensory neuron-specific protein has been cloned in the mollusk land snail Eobania vermiculata (Mazzatenta et al., 2004).'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'Furthermore the fact that cell type 4 never shows OMP immunoreactivity strongly suggests that the &#8216;ring shaped cells&#8217; do not have an olfactory function. Moreover, both controls of anti-OMP specificity, western and dot blot analyses clearly showed that immunoreactivity is restricted to just the OE protein extract strongly supporting its conserved function. The protein that cross reacts with anti-OMP has a molecular weight of about 100&#8197;kDa, and that homologue and heterologue pre-absorption tests performed with proteins extracts from O. vulgaris OE, as well as recombinant rat OMP, abolished completely OMP immunoreactive band in western and dot blot analysis.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'Anti-OMP specificity has been supported by loss of labeling in controls treated with antibody pre-absorbed with the antigen at 5&#8197;&#181;M final concentration (recombinant rat OMP kindly provided by F. Margolis, University of Maryland School of Medicine, USA). Given that an OMP has not been annotated in O. bimaculoides genome, specificity was tested with a western blot and dot blot assay. Specificity of both secondary antibodies has been tested with omission of primary antibody.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'Given that an OMP has not been annotated in O. bimaculoides genome, specificity was tested with a western blot and dot blot assay. Specificity of both secondary antibodies has been tested with omission of primary antibody.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'Anti-OMP specificity has been supported by loss of labeling in controls treated with antibody pre-absorbed with the antigen at 5&#8197;&#181;M final concentration (recombinant rat OMP kindly provided by F. Margolis, University of Maryland School of Medicine, USA). Given that an OMP has not been annotated in O. bimaculoides genome, specificity was tested with a western blot and dot blot assay.'},\n",
       " {'pmcid': 'PMC4874359',\n",
       "  'pmid': '27069253',\n",
       "  'snippet': 'Anti-PCNA specificity has been determined in a recent study on octopus arm regeneration (Fossati et al., 2013) and supported by loss of labelling in controls with the antibody pre-adsorbed with its antigen. Furthermore, using CLUSTALW2 database, we performed an alignment of mouse PCNA whole protein sequence with PCNA protein sequence annotated in Octopus bimaculoides genome recently published (Albertin et al., 2015).'},\n",
       " {'pmcid': 'PMC4893942',\n",
       "  'pmid': '27335633',\n",
       "  'snippet': 'TAP-L can form homodimers and is located primarily in lysosomes, presumably importing peptides from the cytosol. TAP-L has broad specificity for peptides ranging from a length of 6 to 59 amino acids, with an optimal activity for peptides of 23 residues ( Wolters et al., 2005). TAP-L can transport two peptides at a time ( Herget et al., 2009).'},\n",
       " {'pmcid': 'PMC4893942',\n",
       "  'pmid': '27335633',\n",
       "  'snippet': 'In this article, we describe experimentation designed to specifically detect the ABCB9 protein in bone marrow-derived dendritic cells (BMDCs) by immunoblot. We purchased a rabbit polyclonal antibody generated by Abcam Company using a synthetic peptide as the immunogen, corresponding to a region between residues 475 and 525 of human ABCB9.'},\n",
       " {'pmcid': 'PMC4893942',\n",
       "  'pmid': '27335633',\n",
       "  'snippet': 'It has previously been shown that ABCB9 expression by monocyte-derived human DCs is increased under inflammatory conditions ( Demirel et al., 2007). To validate specificity of antibody staining, we included TAP-L deficient BMDCs as a negative control. TAP-L KO/WT heterozygous mice (ABCB9 tm1 (KOMP) Vlcg), in which the region located between nucleotides 5625 and 33216 of the TAP-L gene has been removed for the insertion of a cassette of 6085bp containing the cDNA conferring resistance to neomycin (Neo), were purchased from The Komp Repository at the University of California at Davis, CA 95616 (see construction of the KO gene; Figure 2A).'},\n",
       " {'pmcid': 'PMC4893942',\n",
       "  'pmid': '27335633',\n",
       "  'snippet': 'It is impossible to know whether it also recognizes TAP-L. Our findings highlight the importance of verifying commercial antibody specificity using knockout cells. If such cells are not available, lentiviruses encoding target-specific shRNA, which are now readily available for an essentially complete range of proteins, can be used to produce cells that provide informative negative controls.'},\n",
       " {'pmcid': 'PMC4893942',\n",
       "  'pmid': '27335633',\n",
       "  'snippet': 'Collectively, these results show that the commercial ABCB9 antibody recognizes a protein with a molecular weight similar to that of TAP-L. It is impossible to know whether it also recognizes TAP-L. Our findings highlight the importance of verifying commercial antibody specificity using knockout cells. If such cells are not available, lentiviruses encoding target-specific shRNA, which are now readily available for an essentially complete range of proteins, can be used to produce cells that provide informative negative controls.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The protein array technology applied here, developed by B&#252;ssow and colleagues 8, is comprised in its current version of a fully annotated set of 7,390 distinct human proteins, that may serve as potential antigens. The aim of this study is to define a cross-reactivity reference list for the two described secondary antibodies, which can then be used to eliminate non-specific binders from ongoing chicken IgY profiling studies. Furthermore, publication of the cross-reactivity reference list provides a valuable resource of potential false-positive binders to researchers using the same antibodies.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The protein array technology applied here, developed by B&#252;ssow and colleagues 8, is comprised in its current version of a fully annotated set of 7,390 distinct human proteins, that may serve as potential antigens. The aim of this study is to define a cross-reactivity reference list for the two described secondary antibodies, which can then be used to eliminate non-specific binders from ongoing chicken IgY profiling studies. Furthermore, publication of the cross-reactivity reference list provides a valuable resource of potential false-positive binders to researchers using the same antibodies.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Other protein array applications include recombinant 6 and hybridoma-derived 7 antibody characterisation studies. This article investigates the cross-reactivity of a rabbit anti-chicken IgY and an alkaline phosphatase-conjugated goat anti-rabbit IgG, which were used for the profiling of IgY antibody responses to human antigens in chickens immunised with human cancer cells. The protein array technology applied here, developed by B&#252;ssow and colleagues 8, is comprised in its current version of a fully annotated set of 7,390 distinct human proteins, that may serve as potential antigens.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The aim of this study is to define a cross-reactivity reference list for the two described secondary antibodies, which can then be used to eliminate non-specific binders from ongoing chicken IgY profiling studies. Furthermore, publication of the cross-reactivity reference list provides a valuable resource of potential false-positive binders to researchers using the same antibodies. may support other researchers using these antibodies in the evaluation of their experiments.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Alkaline phosphatase-conjugated goat anti-rabbit IgG (whole molecule) (Sigma-Aldrich, Product number A3687, Lot code SLBJ6146V) is a polyclonal antibody that targets all rabbit IgGs ( Table 1). The antibody was isolated through immunospecific purification of antisera from a rabbit IgG-immunised goat. Following isolation, the anti-rabbit IgG was conjugated to alkaline phosphatase using glutaraldehyde-based cross-linkage.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The Unipex arrays comprise of 15,300 fully annotated E. coli clones expressing a total of 7,390 distinct in-frame ORF human recombinant proteins. The Unipex proteins are immobilized under denaturing conditions directly on the PVDF membrane surfaces exposing linear sequence epitopes ideally suited for epitope mapping, antibody profiling and antibody cross-reactivity analyses. The details of protein arrays utilised in this study are provided in Table 2.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Antibody cross-reactivity was assessed using Unipex protein arrays. The detailed experimental protocol is provided in Table 3.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Probing protein arrays with antibodies allows the assessment of their specificity and cross-reactivity across a large numbers of potential antigens in parallel 12, 13. Here we investigated the cross-reactivity of a secondary rabbit anti-chicken IgY and a goat anti-rabbit IgG labelled with AP, using a single set of human protein arrays in the absence of chicken serum.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Five of the identified signals were scored as intensity 3, twelve signals were scored as intensity 2 and remainder scored as intensity 1. The original protein array images are shown in Figure S1 and Figure S2 ( Supplementary material) and protein array images with highlighted positive signals, which correspond the cross-reactive proteins listed in Table 4, are shown in Figure S3 and Figure S4 ( Supplementary material).'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Probing protein arrays with antibodies allows the assessment of their specificity and cross-reactivity across a large numbers of potential antigens in parallel 12, 13. Here we investigated the cross-reactivity of a secondary rabbit anti-chicken IgY and a goat anti-rabbit IgG labelled with AP, using a single set of human protein arrays in the absence of chicken serum. We identified a total of 63 binding events, of which 61 corresponded to unique proteins ( Table 4).'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Probing protein arrays with antibodies allows the assessment of their specificity and cross-reactivity across a large numbers of potential antigens in parallel 12, 13. Here we investigated the cross-reactivity of a secondary rabbit anti-chicken IgY and a goat anti-rabbit IgG labelled with AP, using a single set of human protein arrays in the absence of chicken serum.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The 61 identified proteins comprised of a wide range of human proteins, including immunoglobulins, as well as a variety of nuclear, cytoplasmic and cell-membrane proteins with a diverse range of functions ( Table 4). In order to identify shared epitopes that could explain the observed antibody cross-reactivity, and to deduce the origin of the non-specific binding to either of the two tested antibodies, we investigated sequence similarities between the human proteins and the immunogens used to produce the antibodies. We conducted a linear (BLAST) and a segmented (SIM) in silico sequence analysis of chicken IgY and rabbit IgG immunoglobulins against 61 array-identified human proteins as detailed in supplementary Table 1.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The 61 identified proteins comprised of a wide range of human proteins, including immunoglobulins, as well as a variety of nuclear, cytoplasmic and cell-membrane proteins with a diverse range of functions ( Table 4). In order to identify shared epitopes that could explain the observed antibody cross-reactivity, and to deduce the origin of the non-specific binding to either of the two tested antibodies, we investigated sequence similarities between the human proteins and the immunogens used to produce the antibodies. We conducted a linear (BLAST) and a segmented (SIM) in silico sequence analysis of chicken IgY and rabbit IgG immunoglobulins against 61 array-identified human proteins as detailed in supplementary Table 1.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'All 5 proteins that met both threshold criteria belong to the immunoglobulin class of proteins, four being variable regions of Ig heavy chains and one a heavy constant gamma chain. The in silico sequence analysis revealed the highest sequence similarity to Ig heavy variable and constant chains, respectively, of both, the chicken IgY and rabbit IgG in all cases ( Supplementary Table 1) making it impossible with this approach to deduce the origin of the cross-reactivity.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'This work illustrates the cross-reactivity of an antibody-based detection system for IgY binding. The polyclonal anti-IgY rabbit antibody in combination with an anti-rabbit IgG alkaline phosphatase-conjugated antibody was shown to bind to 61 human proteins present on Unipex protein arrays comprising of 7,390 human proteins.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The polyclonal anti-IgY rabbit antibody in combination with an anti-rabbit IgG alkaline phosphatase-conjugated antibody was shown to bind to 61 human proteins present on Unipex protein arrays comprising of 7,390 human proteins. Characterisation of this cross-reactivity provides a &#8216;false-positive&#8217; database for future chicken antisera characterisation on protein array systems not limited to the Unipex protein array used here. These results, in combination with &#8216;false-positives&#8217; from earlier research investigating antibody cross-reactivity by this group 12 and others 13 may provide valuable information for future protein array-based experiments.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Reference lists provided by such experiments would be further strengthened by arrays that include additional portions of the human proteome and/or post-translational modifications. Using antibodies that have been extensively characterised on protein arrays will reduce the risk of identifying irrelevant cross-reactive secondary antibody binding to the array as a host-antigen response.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Characterisation of this cross-reactivity provides a &#8216;false-positive&#8217; database for future chicken antisera characterisation on protein array systems not limited to the Unipex protein array used here. These results, in combination with &#8216;false-positives&#8217; from earlier research investigating antibody cross-reactivity by this group 12 and others 13 may provide valuable information for future protein array-based experiments. Reference lists provided by such experiments would be further strengthened by arrays that include additional portions of the human proteome and/or post-translational modifications.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The presented cross-reactivity reference list may, therefore, show some variation when a different lot of the antibody is used. We have previously shown that conditions applied during affinity chromatography may affect specificity 14. When assessing protein array images, we found a considerable discrepancy in background intensity of array part 1 and 2.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'The secondary antibodies utilized in this study are polyclonal, isolated by immunoaffinity chromatography. The presented cross-reactivity reference list may, therefore, show some variation when a different lot of the antibody is used. We have previously shown that conditions applied during affinity chromatography may affect specificity 14.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'In order to tackle this issue experimentally, a single labelled antibody should be tested on its own in future experiments. Furthermore, if a non-labelled antibody is to be tested, two experiments should be performed, one with a labelled and non-labelled antibody pair such as demonstrated in this study, and one additional experiment with the labelled antibody alone, thereby allowing allocation of exact cross-reactivates by simply subtracting &#8216;false-positives&#8217; from both sets of results.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'We have therefore taken an in silico sequence analysis approach and we found that five of the identified proteins were of the immunoglobulin class of proteins with very high sequence similarities to both, the chicken IgY and the rabbit IgG immunoglobulins. Such cross-reactivity is not surprising considering that the antibodies are polyclonal and the immunogens were immunoglobulins of both hosts. In addition, the data sheet provided with the anti-chicken IgY antibody produced in rabbit (31104, Thermo Fisher) has specified that this antibody may cross-react with immunoglobulins from other species.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Since both antibodies were used as a pair in this study, it was not possible to directly deduce the exact cross-reactivity profile for each individual antibody. We have therefore taken an in silico sequence analysis approach and we found that five of the identified proteins were of the immunoglobulin class of proteins with very high sequence similarities to both, the chicken IgY and the rabbit IgG immunoglobulins.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'In conclusion, the antibodies tested in this study showed cross-reactivity to unrelated human proteins as well as to human immunoglobulin proteins, which are homologous to the original immunogens. Despite the identified non-specific binding, the tested antibodies are suitable for use in protein array experiments as the cross-reactive binding partners can be readily excluded from further analysis. As both antibodies were used as a pair in this study, the possibility to deduce the exact cross-reactivity profile for each individual antibody may be limited.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'Despite the identified non-specific binding, the tested antibodies are suitable for use in protein array experiments as the cross-reactive binding partners can be readily excluded from further analysis. As both antibodies were used as a pair in this study, the possibility to deduce the exact cross-reactivity profile for each individual antibody may be limited. However, the cross-reactivity reference list provided in this paper can be further utilised to validate research using those antibodies in applications other than protein arrays.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'In conclusion, the antibodies tested in this study showed cross-reactivity to unrelated human proteins as well as to human immunoglobulin proteins, which are homologous to the original immunogens. Despite the identified non-specific binding, the tested antibodies are suitable for use in protein array experiments as the cross-reactive binding partners can be readily excluded from further analysis.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'In conclusion, the antibodies tested in this study showed cross-reactivity to unrelated human proteins as well as to human immunoglobulin proteins, which are homologous to the original immunogens. Despite the identified non-specific binding, the tested antibodies are suitable for use in protein array experiments as the cross-reactive binding partners can be readily excluded from further analysis. As both antibodies were used as a pair in this study, the possibility to deduce the exact cross-reactivity profile for each individual antibody may be limited.'},\n",
       " {'pmcid': 'PMC4893991',\n",
       "  'pmid': '27335636',\n",
       "  'snippet': 'As both antibodies were used as a pair in this study, the possibility to deduce the exact cross-reactivity profile for each individual antibody may be limited. However, the cross-reactivity reference list provided in this paper can be further utilised to validate research using those antibodies in applications other than protein arrays.'},\n",
       " {'pmcid': 'PMC4899679',\n",
       "  'pmid': '27294196',\n",
       "  'snippet': 'Through in vitro studies, it is known that Orx neurons are hyperpolarized and inhibited by both GABAA (eg, muscimol) and GABAB (eg, baclofen) receptor agonists and that MCH neurons are inhibited by GABAAR agonists (Eggermann et al., 2003; van den Pol et al., 2004; Xie et al., 2006). We thus investigated whether homeostatic changes in response to state specific prolonged activity or absence thereof would be evident in GABAAR and GABABR immunostaining following SD and SR in the Orx and MCH neurons.'},\n",
       " {'pmcid': 'PMC4899679',\n",
       "  'pmid': '27294196',\n",
       "  'snippet': 'To measure the luminance of the receptors, different approaches were used for the GABAARs concentrated over the plasma membrane versus the GABABRs located in the cytoplasm as well as over the membrane. For membrane GABAARs, a box of 1.5&#8201;&#215;&#8201;0.3 &#956;m2 was placed over the membrane and another over the nucleus to measure and subtract background staining in each cell. For membrane plus cytoplasmic GABABRs, a donut-shaped contour was drawn around the cytoplasm and plasma membrane, and another traced around the nucleus to measure and subtract background staining in each cell.'},\n",
       " {'pmcid': 'PMC4899679',\n",
       "  'pmid': '27294196',\n",
       "  'snippet': 'For membrane GABAARs, a box of 1.5&#8201;&#215;&#8201;0.3 &#956;m2 was placed over the membrane and another over the nucleus to measure and subtract background staining in each cell. For membrane plus cytoplasmic GABABRs, a donut-shaped contour was drawn around the cytoplasm and plasma membrane, and another traced around the nucleus to measure and subtract background staining in each cell.'},\n",
       " {'pmcid': 'PMC4899679',\n",
       "  'pmid': '27294196',\n",
       "  'snippet': 'Deletion of the GABABR also leads to disruption of the sleep&#8211;wake cycle in mice (Vienne et al., 2010). Fragmentation of the cycle also occurred in mice lacking GABABRs specifically on Orx neurons (Matsuki et al., 2009).'},\n",
       " {'pmcid': 'PMC4899679',\n",
       "  'pmid': '27294196',\n",
       "  'snippet': 'These results would also suggest that pharmacological effects upon the GABABR, as upon the GABAAR, can mimic the homeostatic effects of sleep. However, such pharmacological effects are rarely cell specific and thus can affect both wake- and sleep-active cell groups, which as we show here would normally undergo differential homeostatic changes in their GABA receptors depending upon their state selective activity.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'Here, we describe experiments using a co-immunoprecipitation (co-IP) strategy and changes in NMR spectra of the Cx43-CT to confirm the existence of a direct interaction between Cx43-CT and drebrin at areas in the Cx43-CT sequence spatially distinct from tubulin and ZO-1. We specifically identified the N-terminus of drebrin (residues 1&#8211;300) as the fragment that binds to the Cx43-CT and three specific Cx43-CT aminoacidic sequences are affected by this interaction. Of note, mammalian embryonic and adult drebrin isoforms share a highly conserved sequence in this N-terminal region [34].'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'We added the GST-Drb1-300 and GST-Cx43-CT234-382 to the protein A agarose beads as &#8220;prey&#8221; proteins. As negative controls (for detecting non-specific binding), we used protein A agarose beads bound to anti-F-actin antibody mixed with &#8220;prey&#8221; proteins in absence of any &#8220;bait&#8221;. After the pull-down, we analyzed the resulting binding through a Western blot performed by utilizing a rabbit anti-GST primary antibody.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': '(D) Unstained SDS-PAGE gel imaged by UV to detect tubulin-rhodamine after co-IP with Cx43-CT. Lanes 1 and 2 show the absence of non-specific binding of tubulin-rhodamine to protein A and the IF1 antibody respectively, and lane 3 shows the pure tubulin-rhodamine signal.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'The amount of non-specific Drb1-300 bound to the protein A agarose was estimated relatively to the intensity of the protein A band and used to normalize the efficiency of the binding (Fig 4C). After normalization and averaging the results from at least four different repeats, the full length Cx43-CT234-382 and Cx43-CT240-382 &#916;341&#8211;360 constructs consistently showed the strongest binding (Fig 4B, lanes 2 and 7; graph in Fig 4C), while the Cx43-CT316-382 and the Cx43-CT244-382&#916;261&#8211;280 (Fig 4B, lanes 3 and 4; graph in Fig 4C) presented a signal similar to the non-specific binding of Drb1-300 to the protein A agarose beads. These results indicate that the N-terminus portion of the Cx43-CT sequence (residues 240&#8211;316) is involved in the interaction with Drb1-300.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'Results are shown in Fig 4B. The amount of non-specific Drb1-300 bound to the protein A agarose was estimated relatively to the intensity of the protein A band and used to normalize the efficiency of the binding (Fig 4C). After normalization and averaging the results from at least four different repeats, the full length Cx43-CT234-382 and Cx43-CT240-382 &#916;341&#8211;360 constructs consistently showed the strongest binding (Fig 4B, lanes 2 and 7; graph in Fig 4C), while the Cx43-CT316-382 and the Cx43-CT244-382&#916;261&#8211;280 (Fig 4B, lanes 3 and 4; graph in Fig 4C) presented a signal similar to the non-specific binding of Drb1-300 to the protein A agarose beads.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'Drb1-300 co-IP band intensities were normalized to the intensity of the protein A band. The dashed black line indicates the Drb1-300 background level, providing the non-specific binding value. P values obtained through T-test are indicated.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'As we described before (Fig 4B), the amount of Drb1-300 non-specifically bound to the protein A agarose beads was estimated and found to be similar for both antibodies (Fig 5B and 5E). Fig 5C and 5D represent the comparison between the IF1 and the m252-270 antibodies binding the CT234-382 fragment and the CT341-360 deletion.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'The decrease of signal for a subset of residues from each area was fitted according to the nonlinear least squares method, providing a respective KD of 283 &#956;M +/- 90 &#956;M, 279 &#956;M +/- 96 &#956;M, and 344 &#956;M +/- 105 &#956;M. The KD values suggest Area 1 and 2 as primary binding site of interaction with Drb1-300 while the slightly weaker Area 3 could be a secondary site. This observation is in accordance with the co-IP experiments that identified the drebrin-binding domain around residues 261&#8211;280, and more specifically a primary site comprising residues 264&#8211;275. Since the Cx43-CT240-382&#916;281&#8211;300 and &#916;301&#8211;320 constructs do not completely restore the interaction with Drb1-300 (Fig 4), one could expect area 2 and 3 to be secondary binding sites needed for a higher binding affinity.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'Our rat brain analysis showed that Cx43 and drebrin colocalize in astrocytes and particularly in endothelial cells of the blood vessels, which form the blood-brain-barrier [68, 69]. In addition to providing a stable environment for neural function, the blood-brain-barrier also regulates specific channels and transporters to keep the ionic composition optimal for synaptic signaling function [68]. Connexins are involved in the regulation of vascular tone [70] and Cx43 has been detected on the membranes of porcine blood-brain-barrier endothelial cells by Nagasawa et al.'},\n",
       " {'pmcid': 'PMC4900556',\n",
       "  'pmid': '27280719',\n",
       "  'snippet': 'Drebrin is one of the most abundant neuron-specific F-actin-binding proteins found in dendrites and is highly enriched in dendritic spines receiving excitatory inputs [72]. The drebrin-actin complex plays a crucial role in the regulation of dendritic spine morphology as the level of drebrin expression modulates dentritic spine morphology [40].'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'In the hippocampus, for example, excitatory and inhibitory synapses have markedly different properties (Kraushaar and Jonas, 2000; Salin et al., 1996). The underlying molecular mechanisms that control RRP and P are still only partially understood, and it is not known what components of the release machinery account for their synapse specific control.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'ELKS, RIM, Munc13, RIM-binding protein (RIM-BP), Bassoon/Piccolo, and Liprin-&#945; proteins form a protein complex that defines the active zone (Schoch and Gundelfinger, 2006; S&#252;dhof, 2012). This protein complex includes many additional proteins that are not active zone specific (Boyken et al., 2013; Muller et al., 2010; Schoch and Gundelfinger, 2006). ELKS (also called Erc, CAST, and Rab6IP2) was identified as an active zone protein through its interactions with RIM and named for its high content in the amino acids E, L, K, and S (Nakata et al., 1999; Wang et al., 2002; Monier et al., 2002; Ohtsuka et al., 2002).'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(A) Western blot for testing specificity of ELKS2&#945; (1029, top) and ELKS1&#945; (E-1, middle) antibodies against samples of HEK293T cells transfected with ELKS1&#945;B or ELKS2&#945;B cDNAs. The ELKS2&#945; specific antibodies were raised in rabbits to a non-conserved sequence between ELKS1 and ELKS2 (109LSHTDVLSYTDQ120), the E-1 antibody is commercially available. &#946;-actin was used as a loading control.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '&#961;: Spearman rank correlation between ELKS1&#945; and ELKS2&#945;.DOI: http://dx.doi.org/10.7554/eLife.14862.00310.7554/eLife.14862.004Figure 1&#8212;figure supplement 1.ELKS antibody specificity. (A) Western blot for testing specificity of ELKS2&#945; (1029, top) and ELKS1&#945; (E-1, middle) antibodies against samples of HEK293T cells transfected with ELKS1&#945;B or ELKS2&#945;B cDNAs. The ELKS2&#945; specific antibodies were raised in rabbits to a non-conserved sequence between ELKS1 and ELKS2 (109LSHTDVLSYTDQ120), the E-1 antibody is commercially available.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Within a single channel, individual puncta are normalized to the average intensity across all puncta (excitatory synapses: 329 ROIs/30 images/3 independent cultures; inhibitory synapses: 250/30/3). &#961;: Spearman rank correlation between ELKS1&#945; and ELKS2&#945;.DOI: http://dx.doi.org/10.7554/eLife.14862.00310.7554/eLife.14862.004Figure 1&#8212;figure supplement 1.ELKS antibody specificity. (A) Western blot for testing specificity of ELKS2&#945; (1029, top) and ELKS1&#945; (E-1, middle) antibodies against samples of HEK293T cells transfected with ELKS1&#945;B or ELKS2&#945;B cDNAs.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(A) Western blot for testing specificity of ELKS2&#945; (1029, top) and ELKS1&#945; (E-1, middle) antibodies against samples of HEK293T cells transfected with ELKS1&#945;B or ELKS2&#945;B cDNAs. The ELKS2&#945; specific antibodies were raised in rabbits to a non-conserved sequence between ELKS1 and ELKS2 (109LSHTDVLSYTDQ120), the E-1 antibody is commercially available. &#946;-actin was used as a loading control.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(A) Western blot for testing specificity of ELKS2&#945; (1029, top) and ELKS1&#945; (E-1, middle) antibodies against samples of HEK293T cells transfected with ELKS1&#945;B or ELKS2&#945;B cDNAs. The ELKS2&#945; specific antibodies were raised in rabbits to a non-conserved sequence between ELKS1 and ELKS2 (109LSHTDVLSYTDQ120), the E-1 antibody is commercially available.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'In contrast to its roles in enhancing Ca2+ influx and release probability at inhibitory synapses, ELKS1&#945;/2&#945; control the size of the RRP at excitatory hippocampal synapses. These data establish that removal of ELKS has differential, synapse-specific effects on RRP size and P: boosting RRP at excitatory synapses but enhancing P at inhibitory synapses. Using structure-function rescue experiments, we then determine the sequences within ELKS required for RRP enhancement at excitatory synapses.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'The distribution of individual ELKS proteins at excitatory and inhibitory synapses is not known. We generated ELKS2&#945; specific antisera by immunization of rabbits (Figure 1&#8212;figure supplement 1A&#8211;C) and used this, in addition to an available ELKS1&#945; specific antibody, to determine if either or both ELKS proteins are present at excitatory and inhibitory synapses. We employed immunostainings for ELKS1&#945; and ELKS2&#945; in cultured hippocampal neurons and analyzed their distribution using confocal microscopy.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'We thus decided to determine the function of ELKS1&#945;/2&#945; at excitatory synapses in cultured hippocampal neurons employing mice in which the first coding exon of each gene, Erc1 and Erc2, is flanked by loxP sites (Liu et al., 2014; Kaeser et al., 2009). At 3&#8211;5 days in vitro (DIV), we infected the neurons with lentiviruses that express a cre-recombinase tagged with EGFP driven by a neuron-specific synapsin promoter (Liu et al., 2014) to generate ELKS1&#945;/ELKS2&#945; knockout neurons (cDKO). In all experiments, control neurons were genetically identical with the exception that they were infected with lentiviruses that expressed an inactive, truncated cre protein.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(B) In an independent experiment, excitatory neurons were marked with a lentivirus expressing TdTomato (arrowheads) under a CaMKII promoter in addition to a lentivirus that labeled all neurons with nuclear EGFP (merged image). Inhibitory neurons were identified by somatic and dendritic GAD67 staining (arrow, GAD67 antibodies have a nuclear background staining in all neurons). Of the 114 neurons we analyzed, 80 were TdTomato positive (excitatory), 23 were GAD67 positive (inhibitory), and 11 were negative for both markers.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(B) In an independent experiment, excitatory neurons were marked with a lentivirus expressing TdTomato (arrowheads) under a CaMKII promoter in addition to a lentivirus that labeled all neurons with nuclear EGFP (merged image). Inhibitory neurons were identified by somatic and dendritic GAD67 staining (arrow, GAD67 antibodies have a nuclear background staining in all neurons). Of the 114 neurons we analyzed, 80 were TdTomato positive (excitatory), 23 were GAD67 positive (inhibitory), and 11 were negative for both markers.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Thus, in three independent measurements (Figure 5C, Figure 5&#8212;figure supplement 2 and Figure 7F in Liu et al., 2014), no reduction in the inhibitory RRP could be detected. These data confirm our previous experiments and establish synapse-specific roles for ELKS in neurotransmitter release: at excitatory synapses ELKS1&#945;/2&#945; primarily boost the RRP (Figures 1&#8211;5), whereas at inhibitory synapses ELKS1&#945;/2&#945; mainly control action potential induced Ca2+ influx to enhance P (also see Figures 5 and 8&#160;in Liu et al., 2014).10.7554/eLife.14862.011Figure 5.Direct comparison of IPSC and EPSC phenotypes of ELKS1&#945;/2&#945; cDKO. (A) Example traces (top) showing overlayed IPSC responses to pairs of stimuli at 10, 20, 100, and 500 ms interstimulus intervals.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(D) Representative western blot of control, cDKO, and cDKO + rescue neurons used for recording in Figure 7. Samples were blotted using an ELKS1&#945; specific antibody (E-1) and &#946;-actin was used as a loading control.DOI: http://dx.doi.org/10.7554/eLife.14862.016'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': '(D) Representative western blot of control, cDKO, and cDKO + rescue neurons used for recording in Figure 7. Samples were blotted using an ELKS1&#945; specific antibody (E-1) and &#946;-actin was used as a loading control.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'For example, it is widely accepted that Munc13 is required at all synapses to prime vesicles and RIMs anchor and activate Munc13 to support priming (Varoqueaux et al., 2002; Andrews-Zwilling et al., 2006; Deng et al., 2011; Calakos et al., 2004; Augustin et al., 1999). Our findings are a starting point for the dissection of synapse-specific architecture and function of the active zone.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'One possibility to explain synapse-specific roles is that ELKS1&#945; and ELKS2&#945; account for different functions, and that they are localized to specific subsets of synapses. Such differences have been observed for roles of Munc13-1 and bMunc13-2 in short-term plasticity (Rosenmund et al., 2002).'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Our data make this possibility unlikely in the case of ELKS, because there is a strong positive correlation between ELKS1&#945; and ELKS2&#945; levels at all synapses. Furthermore, no ELKS isoform-specific protein interactions are described in the literature, and the interaction sequences are generally well conserved between ELKS1&#945; and ELKS2&#945; (Monier et al., 2002; Wang et al., 2002).'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Interestingly, however, Bassoon and ELKS were found to be enriched in glutamatergic synapses in this study, perhaps supporting a role for these proteins for promoting excitatory transmission in concert with one another. Furthermore, a recent study showed that Bassoon regulates release indirectly via RIM-BP through a specific Ca2+ channel subunit, which could result in a synapse-specific roles for Bassoon (Davydova et al., 2014). In summary, these studies provide support for the hypothesis that ELKS may operate with Bassoon or Liprin-&#945; to promote release in a synapse-specific fashion.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Another possibility is that interaction partners of ELKS that mediate RRP control are distributed in a synapse specific fashion, and ELKS protein interactions at the active zone may be engaged differentially depending on the presence of specific interacting proteins. This model is supported by the observation that the sequence requirements for rescue of the excitatory RRP or the IPSC are different.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Because constructs including CCC but lacking CCA/B localize to synapses but do not to rescue, one possible explanation is that Liprin-&#945; binding is required for ELKS function in controlling the excitatory RRP. Vertebrate genomes contain four genes encoding Liprin-&#945;, but it is not known whether Liprin-&#945; isoforms are differentially distributed to specific active zones. In fact, it is currently not clear for any of the vertebrate Liprin-&#945; isoforms whether there is a tight association with the presynaptic active zone (Zurner et al., 2011; Spangler et al., 2011).'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'Furthermore, a recent study showed that Bassoon regulates release indirectly via RIM-BP through a specific Ca2+ channel subunit, which could result in a synapse-specific roles for Bassoon (Davydova et al., 2014). In summary, these studies provide support for the hypothesis that ELKS may operate with Bassoon or Liprin-&#945; to promote release in a synapse-specific fashion.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'At the D. melanogaster neuromuscular junction, however, Brp is essential for the formation of T-bars. The ELKS homology of Brp is limited to the N-terminal half of the protein (Kittel et al., 2006; Monier et al., 2002), making it possible that such strong scaffolding functions are specific to Brp and absent in ELKS. Alternatively, an ELKS scaffolding function may not be detected in our experiments due to redundant scaffolding activities in other proteins, for example &#946;-ELKS or RIM (Schoch et al., 2002; Kaeser et al., 2009; Liu et al., 2014).'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'In the long-term, it will be important to understand how the synapse-specific molecular control of RRP and P contribute to circuit function. Human genetic experiments reveal that mutations in ERC1/ELKS1 may contribute to autism spectrum disorders (Silva et al., 2014), and it is possible that the pathophysiology arises from synapse-specific misregulation of neurotransmitter release.'},\n",
       " {'pmcid': 'PMC4935463',\n",
       "  'pmid': '27253063',\n",
       "  'snippet': 'ELKS2&#945; specific antibodies were raised in rabbits using an ELKS2 peptide (109LSHTDVLSYTDQ120). Peptides were synthesized and conjugated to keyhole lympet hemocynanin (KLH) via an N-terminal cysteine residue.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Anatomical studies have demonstrated that these subnuclei have distinct neural inputs and downstream targets in the brainstem. Past experiments have generally performed lesions of the entire habenula or its major output tract, the fasciculus retroflexus, and have assigned a wide variety of behavioral functions to the habenula (Lecourtier and Kelly, 2007), but this approach cannot resolve the specific functions of the habenula subnuclei.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Recent experiments targeting the specific input pathways from the septal nuclei to the vMHb and dMHb have implicated these circuits in the regulation of anxiety and fear responses, respectively (Yamaguchi et al., 2013). Nonetheless, it remains to be determined whether the MHb subnuclei are directly involved in the expression of anxiety and fear responses.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'The effects of dMHb ablation in models of depression is significant, but cannot be interpreted simply as &#8220;depressed&#8221; or &#8220;resilient&#8221;. Using a Cre-driver line that is specific for a tachykinin-expressing subpopulation of dMHb neurons, we show that activation of these neurons is sufficient to support self-stimulation reinforcement. Collectively, our data show that the dMHb is involved in the regulation of multiple mood-related behaviors, but also suggest that these behaviors are not mediated by a single neural pathway.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Mice with a tissue-specific null mutation of Pou4f1 in the dMHb and control littermates used in this study were generated and genotyped as previously described (Hsu et al., 2014). The Pou4f1 mutant mice used and their patterns of Pou4f1 transcription factor expression are listed in Figure 1A.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Recently, lesion studies of the specific septal inputs to the dorsal and ventral MHb have shown that the septohabenular pathway is involved in the regulation of anxiety and fear (Yamaguchi et al., 2013). Specifically, lesions of septal inputs to the dMHb increased freezing behavior in the training phase of a conditioned fear paradigm. (Contextual conditioned fear per se was not tested).'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Recently, lesion studies of the specific septal inputs to the dorsal and ventral MHb have shown that the septohabenular pathway is involved in the regulation of anxiety and fear (Yamaguchi et al., 2013). Specifically, lesions of septal inputs to the dMHb increased freezing behavior in the training phase of a conditioned fear paradigm.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'To test for direct involvement of the dMHb in fear conditioning we assessed fear acquisition and the contextual fear response using dMHbCKO mice and controls. The dMHbCKO mouse bears a dMHb-specific deletion of the Pou4f1 coding sequence, and shows a nearly complete developmental loss of dMHb neurons (Fig. 1).'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Experiments to be published elsewhere demonstrate that WRA does not differ significantly between Pou4f1+/- and Pou4f1+/+ mice (Y.-W. Hsu, H. de la Iglesia and E. Turner unpublished observations), and thus that the WRA deficit in dMHbCKO mice is also specifically related to dMHb ablation.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Specific expression of channelrhodopsin in tachykinin-expressing neurons in dMHb-TacChR2 mice. A, B, Comparison of Tac1 and Tac2 mRNA expression in the habenula; the axial level is approximately bregma &#8722;1.6mm.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': '5D). PCR with allele-specific oligos confirmed the correct insertion of the targeted transgene at the Tac2 locus. We note that Allen Institute database in situ hybridization data for Tac2IRESCre crossed with another reporter line, Ai14, shows the same pattern of reporter expression, restricted to the dMHb (Fig.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': '8). We conclude that the reinforcing effect of MHb stimulation is specifically conferred by activation of neurokinin-expressing dMHb neurons, and that ICSS is more effective at detecting this effect than place preference.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Although the dMHb is not part of this amygdala-centered circuitry known to be involved in conditioned fear, a recent report suggests involvement of the septohabenular pathway in fear behavior (Yamaguchi et al., 2013). In this study, immunotoxin-mediated cell targeting was used to ablate neurons in the bed nucleus of the anterior commissure (BAC), part of the septal complex, that project specifically to the dMHb. BAC-ablated mice showed increased freezing response during fear training, but the effect on conditioned fear was not reported.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'BAC-ablated mice showed increased freezing response during fear training, but the effect on conditioned fear was not reported. Here we have used a contextual conditioning protocol, without a specific cue, to reproduce the original study to the extent possible. Our results demonstrate that the dMHb is not essential for the acquisition of acute fear or for the contextual conditioned fear response.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'However, antidepressants work on neurotransmitter systems with broad CNS effects. Because the FST and TST involve different stimuli and different locomotor responses, there is no reason to assume that these tests will always have concordant results in transgenic models that affect specific neural pathways. Supporting this concept, quantitative trait locus analysis of mouse strains that show differential behavior in the TST and FST have mapped both distinct and common gene loci linked to behavior in these tests (Yoshikawa et al., 2002; Tomida et al., 2009).'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'Thus, it is possible that the decreased TST immobility time observed in dMHbCKO mice results from a loss of Tac1 peptide signaling in the habenulopeduncular system; the effect of Tac1 gene deletion on the FST may reside in another pathway. NKB, the Tac2 gene product coexpressed with SP in the dMHb and its terminal fibers in the IP, may also play a role in these effects, but little is known about the specific function of this peptide in the CNS. dMHbCKO mice show marked sensitization to the induction of learned helplessness.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'One strain of genetically altered mice shown to have decreased immobility time (antidepressant effect) in the TST and FST are those with a specific deletion of the Tac1 gene, encoding the neuropeptides SP and neurokinin A (Bilkei-Gorzo et al., 2002). Tac1 mRNA, expression of which distinguishes the dMHb from the adjacent vMHb and LHb, is nearly absent in dMHbCKO mice (Hsu et al., 2014).'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'However, ablation of the dMHb does not simply cause depression-related behaviors, nor does it prevent them in a way that encompasses all of the behavioral constructs used to model mood disorders and fear. As specific tools are used to dissect the underlying brain circuits for each of these mood-related behaviors, these pathways may be shown to impact depression-related behaviors by distinct mechanisms.'},\n",
       " {'pmcid': 'PMC4947983',\n",
       "  'pmid': '27482535',\n",
       "  'snippet': 'In a prior study, we have shown that dMHbCKO mice have markedly reduced voluntary wheel running activity (Hsu et al., 2014). This appears to be a specific deficit in exercise reinforcement, since basal locomotion is not affected. Although not generally used as a model of depression, WRA may interact with affective state and has been shown to produce an antidepressant-like effect in rats and mice (Greenwood et al., 2003; Duman et al., 2008).'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'The N-terminal moiety of LBR, which resides in the nucleoplasm, contains a chromatin-binding TUDOR domain and associates with the nuclear lamina (Makatsori et al., 2004; Pyrpasopoulou et al., 1996; Worman et al., 1988), while the polytopic C-terminal domain anchors LBR in the inner nuclear membrane (INM) and has sequence homology to sterol C14 reductases (Li et al., 2015; Silve et al., 1998; Worman et al., 1990). Sterol C14 reductases are widely conserved throughout evolution and are responsible for the reduction of a specific carbon-carbon double bond as part of the tightly controlled enzymatic cascade which results in the production of cholesterol and cholesterol-related compounds (Benveniste, 2004; Holmer et al., 1998)&#160;(Figure 1).10.7554/eLife.16011.003Figure 1.Cholesterol biosynthesis pathway.Simplified schematic of cholesterol biosynthesis starting from Acetyl-CoA. After 13 enzymatic steps, the intermediate Lanosterol can enter one of two parallel pathways designated Bloch and Kandutsch-Russel pathways, respectively, both of which employ an NADPH-dependent reduction step which can be catalyzed by sterol C14 reductases LBR or TM7SF2 (highlighted in magenta).'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'In order to clarify the cellular function of LBR both in cholesterol metabolism and as a structural component of the nuclear lamina, we used the CRISPR/Cas9 system (Mali et al., 2013) to generate LBR knockout HeLa cell lines (Figure 2&#8212;figure supplement 1A). CRISPR/Cas9 editing of LBR alleles was performed in a recombination-competent HeLa FlpIn cell line (hereafter designated wild type or WT cells), allowing for rapid and efficient introduction of WT rescue and disease-specific LBR alleles into the LBR knockout cell background via site-specific recombination (Turner et al., 2015). CRISPR/Cas9-treated WT cells were screened for the absence of full-length LBR protein by immunoblotting using antibodies against both the N and C termini of the protein (Figure 2&#8212;figure supplement 1B), and via genotyping using PCR primers flanking the CRISPR target site (Figure 2&#8212;figure supplement 1A, arrows).'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'These data, together with our previous observations by immunofluorescence microscopy, indicate that in these cells, LBR does not play a significant role in maintaining NE integrity. We cannot, however, exclude that LBR plays a role in the structural integrity of the nuclear lamina in specific cell types, under conditions of mechanical stress, or developmental stages found only in the context of the living organism.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': '(A&#8211;D) Wild-type or LBR KO HeLa cells were transfected with control siRNA or SMARTpool siRNA targeting to SREBP2 for 48&#160;hr, and cultured in normal medium or cholesterol-restrictive medium for another 48&#160;hr. Total RNAs were then isolated, reversed transcribed into cDNA, and analyzed using real-time PCR with specific primers as indicated. Data were normalized relative to wild-type HeLa cells transfected with control siRNA, cultured in normal conditions.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': '(G and H) HFF, 293T, and HeLa cells were cultured in normal medium or cholesterol-restrictive medium for 2 days, and cells were harvested and split into two aliquots. Total RNAs were extracted from one aliquot, reverse transcribed into cDNA, and analyzed using real-time PCR with gene specific primers as indicated. The data was represented as a relative level to the normal condition (i.e.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': '(G and H) HFF, 293T, and HeLa cells were cultured in normal medium or cholesterol-restrictive medium for 2 days, and cells were harvested and split into two aliquots. Total RNAs were extracted from one aliquot, reverse transcribed into cDNA, and analyzed using real-time PCR with gene specific primers as indicated. The data was represented as a relative level to the normal condition (i.e.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': '(A&#8211;D) Wild-type or LBR KO HeLa cells were transfected with control siRNA or SMARTpool siRNA targeting to SREBP2 for 48&#160;hr, and cultured in normal medium or cholesterol-restrictive medium for another 48&#160;hr. Total RNAs were then isolated, reversed transcribed into cDNA, and analyzed using real-time PCR with specific primers as indicated. Data were normalized relative to wild-type HeLa cells transfected with control siRNA, cultured in normal conditions.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'Given that our study establishes LBR as the major sterol reductase required for cholesterol synthesis in human cells, the question arises why a second, functionally equivalent enzyme is encoded in the human genome. Since cholesterol synthesis can proceed through differential, tissue-specific usage of distinct yet interchangeable segments of the established synthesis pathways (Mitsche et al., 2015), we speculate that TM7SF2 could be required for cholesterol synthesis in a specific physiological context. Interestingly, a recent study suggested that TM7SF2 may participate in healing of burn wounds (Lei et al., 2016), possibly representing a case of a more specialized physiological role.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'The administration of a low cholesterol diet or inhibitors of cholesterol synthesis in animal models with LBR mutations could reveal previously unknown phenotypes and thus provide new insight into disease etiology. Similarly, it would be interesting to explore states in which specific cell types are exposed to conditions of low environmental cholesterol in a physiological context, as for example in the course of embryogenesis, which could help to rationalize the embryonic-lethal phenotypes in Greenberg skeletal dysplasia.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'Total RNAs were isolated from HeLa cells cultured in cholesterol-restrictive medium and reversed transcribed into cDNA using a cDNA amplification kit (SMARTer RACE kit, Clontech). The TM7SF2 cDNA was amplified using a universal forward primer (5&#8217;-CTAATACGACTCACTATAGGGC-3&#8217;) and a gene-specific reverse primer (5&#8217;-TCAGTAGATGTAGGGCATGATGCG-3&#8217;). The resulting PCR product was subjected to Sanger sequencing.'},\n",
       " {'pmcid': 'PMC4951196',\n",
       "  'pmid': '27336722',\n",
       "  'snippet': 'The antibodies used in this study were the following (numbering according to http://antibodyregistry.org): anti-LBR N-terminal domain (AB_775968, Abcam) at 1:2,000, anti-LBR C-terminal domain (AB_10712378, Abcam) at 1:2,000, anti-Tubulin (AB_477583, Sigma) at 1:2000, anti- &#946;-actin (AB_306371, Abcam) at 1:2000, and anti-TM7SF2 (Covance custom antiserum). The specificity of the anti-TM7SF2 antibody was confirmed by inclusion of a TM7SF2 knockout cell lysate as a reference (see Figure 3I).'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The GRO-seq signals (shown as in Figure 1) represent pre-B-ALL patients or cell lines with different cytogenetic subtypes. Notice that GRO-seq signal specifically for a hypodiploid case is not available. (B) The highest recurrence of SV occurs at the convT overlapping region downstream RB1 TSS.DOI: http://dx.doi.org/10.7554/eLife.13087.009'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': '2014) and used to define TADs (Figure 1&#8212;source data 1). This initial analysis led to the identification of specific transcription signal features that occur frequently at breakpoints (convT and Pol2 stalling, Figure 2&#8212;source data 1). In order to characterize the properties of convT and Pol2 stalling sites genome-wide, additional analysis utilized further genomic profiles from B-lineage (B) and ES (E) cells (The ENCODE Project Consortium, 2012; Ginno et al., 2013).'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': '2014) and used to define TADs (Figure 1&#8212;source data 1). This initial analysis led to the identification of specific transcription signal features that occur frequently at breakpoints (convT and Pol2 stalling, Figure 2&#8212;source data 1). In order to characterize the properties of convT and Pol2 stalling sites genome-wide, additional analysis utilized further genomic profiles from B-lineage (B) and ES (E) cells (The ENCODE Project Consortium, 2012; Ginno et al., 2013).'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The GRO-seq signals (shown as in Figure 1) represent pre-B-ALL patients or cell lines with different cytogenetic subtypes. Notice that GRO-seq signal specifically for a hypodiploid case is not available. (B) The highest recurrence of SV occurs at the convT overlapping region downstream RB1 TSS.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The ETV6-RUNX1 data compared to other pre-B-ALL subtypes is displayed. The RSS status annotation is based on motifs identified from ETV6-RUNX1 breakpoints (see Materials&#160;and&#160;methods).DOI: http://dx.doi.org/10.7554/eLife.13087.01410.7554/eLife.13087.015Figure 2&#8212;figure supplement 4.Signal feature span normalized by total transcribed area for TADs sorted by breakpoint frequency.The percentage from total transcribed area that corresponds to convT or Pol2 stalling is shown for TAD quartiles based on the the four different pre-B-ALL subtype SV datasets.DOI: http://dx.doi.org/10.7554/eLife.13087.01510.7554/eLife.13087.016Figure 2&#8212;figure supplement 5.Overlap of TADs with convT in ES cells.The percentage of TAD spanned by convT in ES cells is summarized as boxplots, as in Figure 2B and C. The relative difference to results presented in Figure 2 reflect cell type-specific transcriptional activity. To ease the comparison, the interquartile ranges for the overlap using relevant GRO-seq signals from pre-B/B-lymphoid cells are shown in blue (median plotted in red) for the highest SV frequency bin.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The percentage of TAD spanned by convT in ES cells is summarized as boxplots, as in Figure 2B and C. The relative difference to results presented in Figure 2 reflect cell type-specific transcriptional activity. To ease the comparison, the interquartile ranges for the overlap using relevant GRO-seq signals from pre-B/B-lymphoid cells are shown in blue (median plotted in red) for the highest SV frequency bin.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The profiles peaked near TSS, where the presence of RLFS motifs led to a significant elevation in the median GRO-seq signal level (Figure 3&#8212;figure supplement 1, 2.1-fold increase in B-lineage cells, Wilcoxon rank sum test P&lt;2.2e-16, 95% CI 2.1&#8211;2.3). As a second mechanism, collisions due to convT may halt transcription (Prescott and Proudfoot, 2002) in a dynamic and cell-specific manner. Accordingly, higher antisense signal at convT regions (see Materials&#160;and&#160;methods) increased the overlap with Pol2 stalling sites on the sense strand (Figure 3B), intriguingly exceeding that observed for RLFS motifs (Figure 3A).10.7554/eLife.13087.018Figure 3.Indication of transcription-coupled genetic instability at leukemia SV hotspots lacking RSS motifs.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'Breakpoint recurrence was found to increase the overlap with both Pol2 stalling (Figure 3E)and convT (Figure 3F). The mean overlap observed in 1000-fold random sampling (grey bars) confirmed the specificity of the overlap (note that Pol2 stalling is analyzed from intragenic regions only). The breakpoints in Pol2 stalling sites were concordant with analysis using Pol2 ChIP-seq (by 78%) and they co-localized with both Ser2 and Ser5 phosphorylated forms of Pol2 complex (Figure 3&#8212;figure supplement 2).'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'In addition to DNAse-seq and Pol2 ChIP peaks, the H3K4me3 was found among the top category, confirmed also by ChIP-seq data acquired from REH and Nalm6 cells (Figure 4&#8212;source data 1).10.7554/eLife.13087.024Figure 4.SV with RSS motifs localize to Pol2 stalling regions with broad open chromatin regions. (A) DNA access based on DNAse-seq peak width (GM12878 or H1 ES from ENCODE) is compared between regions with no Pol2 stalling (no color) and overlapping Pol2 stalling (light blue, cell-specific Pol2 stalling coordinates are listed in Figure 2&#8212;source data 1) at TSS, body and end region of transcripts (refer to Figure 3&#8212;source data 2 for statistical analysis based on separate DNAse-seq replicates). (B) The TSS stalling width is compared between TSS harboring R-breakp and TSS with no breakpoints (Wilcoxon rank sum test P is indicated, TSS with R-breakp N&#160;=&#160;38, TSS without breakpoints N&#160;=&#160;11957, 95% CI for size difference 67&#8211;491 bp).'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': '(A) DNA access based on DNAse-seq peak width (GM12878 or H1 ES from ENCODE) is compared between regions with no Pol2 stalling (no color) and overlapping Pol2 stalling (light blue, cell-specific Pol2 stalling coordinates are listed in Figure 2&#8212;source data 1) at TSS, body and end region of transcripts (refer to Figure 3&#8212;source data 2 for statistical analysis based on separate DNAse-seq replicates). (B) The TSS stalling width is compared between TSS harboring R-breakp and TSS with no breakpoints (Wilcoxon rank sum test P is indicated, TSS with R-breakp N&#160;=&#160;38, TSS without breakpoints N&#160;=&#160;11957, 95% CI for size difference 67&#8211;491 bp).'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': \"Among samples with annotation of cytogenetic subtype (N&#160;=&#160;1008), the ETV6-RUNX1 cases (N&#160;=&#160;153) exhibited 10.8-fold higher median level of RAG1 expression relative to other cases with annotated cytogenetic type (Wilcoxon rank sum test P&lt;2.2e-16, 95% CI 8.6&#8211;13.6-fold, Figure 5A) and also high RAG2 expression (Figure 5B). Moreover, AICDA expression was also detected in a specific subset of patients. It was highest in the 'other' group (N&#160;=&#160;267) that does not carry recurrent fusion genes or karyotypic changes (Figure 5C, no statistical evaluation was performed as majority of signal values were below detection level of 4.2 in log2 scale).\"},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The present study represents a systematic investigation of SVs detected in acute pre-B-cell leukemia using WGS in the context of global transcriptional activity in leukemic cells. We identified specific transcriptional features, namely convergence of transcription and Pol2 stalling, as key factors underlying secondary genetic lesions frequently seen in precursor B leukemias.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The cell lines were authenticated by PCR of known fusion genes: ETV6-RUNX1 in REH, MLL-MLLT1 in Kopn8, and the lack of recurrent fusions in Nalm6. In addition, the generated genome-wide results can be used in verification of cell line specific markers. We reasoned that a collection of samples that represent both primary blasts and cell lines of different cytogenetics types (and genetic complexity) would be ideal to capture the patterns of transcriptional activity in the lymphoid lineage and leukemic cells.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The GRO-seq data from lymphoblastoid cells (GSE39878, Wang et al., 2014b; GSE60456, Core et al., 2014), ES cells (GSE41009, Sigova et al., 2013), DRIP-seq data from ES cells (GSE45530, Ginno et al., 2013) and HiC data from human lymphoblastoid GM12878 cells (GSM1551571, GSM1551572, GSM1551574, GSM1551575; Rao et al., 2014) were downloaded from SRA (raw reads) and processed similarly as the new samples: reads were quality controlled and subsequently aligned to the human hg19 reference genome version. Specifically, the quality of raw sequencing reads was confirmed using the FastQC tool (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) and subsequently bases with poor quality scores were trimmed (requiring a minimum 97% of all bases in one read to have a min phred quality score of 10) using the FastX toolkit (http://hannonlab.cshl.edu/fastx_toolkit/). Samples sequenced on multiple lanes were pooled after quality control.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'BedGraph and bigWig files were generated with reads in each sequencing experiment normalized to a total of 107 mapped reads. The bigWig files were further converted to track hubs and visualized as strand-specific, overlaid MultiTracks as a custom Track Hub in the UCSC Genome browser.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': 'The pre-B-ALL breakpoints and annotation data (Andersson et al., 2015; Holmfeldt et al., 2013; Papaemmanuil et al., 2014; Paulsson et al., 2015) were analyzed in context of TADs. Specifically, TADs were overlapped with subtype-specific breakpoints (Figure 1&#8212;source data 1 presents TADs sorted based on the count of breakpoints). Subsequently, TADs with breakpoints were divided into quartiles based on breakpoint frequency per bp to analyze enrichment of feature overlap that exceeds the genomic background level.'},\n",
       " {'pmcid': 'PMC4951197',\n",
       "  'pmid': '27431763',\n",
       "  'snippet': \"The analysis was performed separately for each gene, using Bayesian Information Criterion as a penalty term with the changepoint counted as a parameter (function cpt.mean with parameters penalty = 'BIC1', method ='PELT'). The input dataset representing primary transcription activity at gene loci was generated by overlapping the GRO-seq signal file strand-specifically with transcript coordinates from UCSC and Refseq (see genomic regions used). The analysis only considered regions with annotation match (in minimum 5% of identified transcript covered by annotation; a minimum of 50% overlap with the identified transcript; annotated and detected starts do not differ more than 10 kb).\"},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': \"Expression of target and reference genes were analyzed by real-time polymerase chain reaction (qPCR). The sequence for JAM-C-specific primers were as follows: Forward 5'- aag aac cca ggg aaa cca gat gga-3'; Reverse 5'- tcg ctg cct tga cag gag ttt cta-3&#8217;. The values were normalized to the expression levels of human beta-actin, beta-tubulin, and Glyceraldehyde-3-phosphate dehydrogenase (GAPDH), according to the GeNorm method [32].\"},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'For bead labeling normalization, the % of FITC-positive monocytes was analyzed 24-hrs after injection to determine relative degree of labeling when comparing across groups. Other CD45+ populations were similarly analyzed for FITC to check for non-specific labeling which might obscure results. Flow cytometry was performed using a LSRII analyzer (BD), and analysis was performed using FlowJo X software (Tree Star).'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'Immunostaining of serial sections for CD68+ was used to determine cells that were presumably monocytes and inflammatory cholesterol-loaded macrophage foam cells, and quantification of atherosclerosis through CD68+ content has been extensively used in the past in this regard (Fig 1C, Representative CD68+ images per group). Although neutrophils can also express CD68, there was only background staining for Ly6G (a neutrophil marker) in all groups. Some minor populations of cholesterol-loaded smooth muscle cells can also show CD68 expression, however their speed of migration is relatively low compared to monocytes, and there is no evidence of massive short-term immigration or emigration from plaques (see below) [42, 43].'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'We conducted these studies using a cross-reactive anti-mouse JAM-C antibody [4]. The drawback of using this is that high concentrations of 50 &#956;g/ml are required, as the affinity is relatively low against human JAM-C. We have subsequently developed a higher affinity, human specific anti-JAM-C antibody (225.3) determined by surface plasmon resonance technology (S1A Fig). Treatment of HUVECs with 10&#956;g/ml of this new antibody was sufficient to induce rTEM of primary monocytes (Fig 4B).'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'We have previously shown that down-regulating JAM-C expression on HUVECs using siRNA or JAM-C blockade increased human monocyte rTEM. We conducted these studies using a cross-reactive anti-mouse JAM-C antibody [4]. The drawback of using this is that high concentrations of 50 &#956;g/ml are required, as the affinity is relatively low against human JAM-C. We have subsequently developed a higher affinity, human specific anti-JAM-C antibody (225.3) determined by surface plasmon resonance technology (S1A Fig).'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'Having established an effect of JAM-C blockade on human monocyte rTEM by the in-vitro model, we decided to investigate the effect of blocking JAM-C in-vivo using our inflammatory, atherosclerosis regression model. Specifically, we asked whether the elevated levels of junctional JAM-C in the atherosclerotic plaques would actually promote or inhibit emigration of recruited monocytes, and if JAM-C blockade with a blocking antibody would enhance this process. Using this model, the dynamics of the regressing plaque could be attributed broadly to the reduced recruitment of circulating monocytes to the inflammatory lesion or increased egress of tissue-resident macrophages from the plaque into the circulation/lymphatics [41, 49].'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'Indeed, this is what was found upon analysis of morphometrics and inflammatory macrophage content of aortic plaques. While they did not change in total plaque size compared to recipients, most likely because less extracellular matrix is destroyed by fewer and less inflammatory macrophages [44], anti-JAM-C treated mice specifically demonstrated reduced monocyte-derived cell content relative to other recipients through total and relative CD68+ inflammatory cell content.'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': 'In atherosclerosis this may be particularly effective, given that lipid lowering treatment appears to stimulate some degree of rTEM of monocytes/macrophages [36, 37], so that further enhancement of this process would improve regression of disease and the resolution of plaque inflammation. Through the development of reagents specifically targeting molecules controlling leukocyte homing and accumulation, such as our JAM-C, successful therapy for patients suffering from chronic inflammatory diseases may be possible.'},\n",
       " {'pmcid': 'PMC4956249',\n",
       "  'pmid': '27442505',\n",
       "  'snippet': '(A) Characterization of the novel anti-human JAM-C antibody 225.3. Biacore evaluation of anti-JAM-C Abs on immobilized soluble JAM-C comparing the anti-mouse JAM-C cross-reactive H36 Abs with the anti-human JAM-C 225.3. (B) Validation of JAM-C down-regulation on HUVECs after siRNA delivery by qPCR.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'Decrease of miR-139-5p by knocking down p53. H460 and U2OS cells were transfected with scramble siRNA (SC) or siRNA specific to p53 (p53) and subjected to Western blot and qRT-PCR analysis.DOI: http://dx.doi.org/10.7554/eLife.15978.003'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'Decrease of miR-139-5p by knocking down p53. H460 and U2OS cells were transfected with scramble siRNA (SC) or siRNA specific to p53 (p53) and subjected to Western blot and qRT-PCR analysis.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': '(B) Increased binding of p53 and the endogenous p53RE-containing miR-139 promoter in response to Doxorubicin. H460 or HCT116 p53+/+ cells were treated with 0.5&#956;M Doxorubicin for 18 hr to stimulate the endogenous p53 before chromatin-associated immunoprecipitation assays were conducted with DO-1 p53 antibodies and the p53RE specific primers listed in Supplementary file 1 online. Western blot analysis was performed to confirm induction of p53.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'To test if endogenous p53 binds to this p53RE sequence, we conducted a chromatin-associated immunoprecipitation (ChIP) assay after treating H460 or HCT116 p53+/+ cells with 0.5&#8201;&#956;M Doxorubicin for p53 induction. In both cell lines, the binding of p53 with the p53RE was dramatically increased upon Doxorubicin treatment as compared to non-treatment control, indicated by p53RE sequence pulled down with p53 specific antibody DO-1, but not non-specific immunoglobulin G (Figure 2B). Also, we assessed luciferase expression driven by either a wild type or a mutant-p53RE-motif-containing miR-139 sequence (Figure 2C) in H1299 cells.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': '(B) Increased binding of p53 and the endogenous p53RE-containing miR-139 promoter in response to Doxorubicin. H460 or HCT116 p53+/+ cells were treated with 0.5&#956;M Doxorubicin for 18 hr to stimulate the endogenous p53 before chromatin-associated immunoprecipitation assays were conducted with DO-1 p53 antibodies and the p53RE specific primers listed in Supplementary file 1 online. Western blot analysis was performed to confirm induction of p53.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': '(C) Schematic of the pMIR&#8211;PDE4D luciferase reporter constructs used, which contain either a wild-type or a mutant miR-139-5p target site derived from the PDE4D mRNA. (D) Overexpression of miR-139-5p specifically inhibits luciferase activity from the plasmid harboring a wild-type, but not a mutant, miR-139-5p targeted sequence. H1299 cells were co-transfected with the indicated plasmids and collected 48 hr after transfection for luciferase assay.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'The type-I insulin-like growth factor receptor (IGF-IR), a previously reported miR-139-5p target (Shen et al., 2012), was also decreased by miR-139-5p mimic, indicating the activity of miR-139-5p used here (Figure 3&#8212;figure supplement 1). PDE4D belongs to the family of phosphodiesterases and is a cyclic AMP (cAMP) specific phosphodiesterase with several splice variants (Omori and Kotera, 2007). It is an oncogenic protein that regulates cancer cell proliferation, angiogenesis and apoptosis (Lin et al., 2013; Ogawa et al., 2002; Pullamsetti et al., 2013;&#160;Rahrmann et al., 2009).'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': '(C) Schematic of the pMIR&#8211;PDE4D luciferase reporter constructs used, which contain either a wild-type or a mutant miR-139-5p target site derived from the PDE4D mRNA. (D) Overexpression of miR-139-5p specifically inhibits luciferase activity from the plasmid harboring a wild-type, but not a mutant, miR-139-5p targeted sequence. H1299 cells were co-transfected with the indicated plasmids and collected 48 hr after transfection for luciferase assay.'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'Since PDE4D is a cAMP specific phosphodiesterase, ectopic expression of miR-139-5p in A549 cells led to significant increase of cellular cAMP levels (Figure 4A). Also, consistent with our results shown in Figure 3F and Figure 3G, doxorubicin treatment increased cellular cAMP levels in A549 cells, which were alleviated by the miR-139-5p inhibitor (Figure 4&#8212;figure supplement 1).'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'Also, consistent with our results shown in Figure 3F and Figure 3G, doxorubicin treatment increased cellular cAMP levels in A549 cells, which were alleviated by the miR-139-5p inhibitor (Figure 4&#8212;figure supplement 1). In addition, Nutlin-3, a more specific p53-activating reagent, showed similar effect on cAMP level, which was also reversed in the presence of miR-139-5p inhibitor (Figure 4B). In contrast, in H1299 cells, which are p53-null and express non-detectable PDE4D, neither doxorubicin nor miR-139-5p affected cellular cAMP level (Figure 4&#8212;figure supplement 2).'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'cAMP is an important secondary messenger mediating diverse cellular processes with protein kinase A as its main effector (Task&#233;n and Aandahl, 2004). In particular, lower cAMP levels favor cancer cell survival and proliferation, which can be reversed by inhibition of PDE4D, the cAMP specific phopsphodiesterase (Goldhoff et al., 2008; Lin et al., 2013; Murata et al., 2000; Ogawa et al., 2002). The tumor suppressor role of p53 has been extensively documented over the last two decades, and is highly attributable to its regulation of target genes involved in cell cycle arrest, apoptosis and senescence (Bieging et al., 2014; Levine, 2011).'},\n",
       " {'pmcid': 'PMC4959878',\n",
       "  'pmid': '27383270',\n",
       "  'snippet': 'Cells were harvested and lysed in lysis buffer 48&#8201;hr post transfection. The total protein concentrations were determined using a BioRad protein assay kit and equal amounts of total proteins (50&#160;&#956;g, otherwise indicated specifically) were then subjected to SDS-PAGE, followed by WB with antibodies as indicated in each figure.'},\n",
       " {'pmcid': 'PMC4969039',\n",
       "  'pmid': '27478985',\n",
       "  'snippet': 'Because of their emergence from the abaxial lamina which can be readily imaged, these outgrowths are more amenable to time-lapse imaging than serrations which are often obscured by neighbouring cotyledon tissue and curving of the leaf edge. kan1kan2 outgrowths have elevated intracellular auxin at their tips, and their formation depends on specific patterns of auxin biosynthetic enzyme expression (Wang et al., 2011). They provide a test bed for studying convergence site formation in a starting context which, unlike the apex, is not complicated by the prior patterns of continual primordium initiation.'},\n",
       " {'pmcid': 'PMC4969039',\n",
       "  'pmid': '27478985',\n",
       "  'snippet': 'This comparison of models gives rise to specific predictions that may be used to distinguish between them experimentally. Tandem alignment models require that, at the leaf tip and at the tips of outgrowths (where centres of convergence are located), the epidermis has elevated rates of auxin import and auxin removal (which could be removal into underlying tissue layers).'},\n",
       " {'pmcid': 'PMC4969039',\n",
       "  'pmid': '27478985',\n",
       "  'snippet': 'During this period, there was a constant flow of liquid media (1/4 strength Murashige and Skoog, 0.75% sucrose, 1.1&#8201;&#956;g / ml MES, pH 5.8) at 1&#8201;&#956;l /s through the growth chamber. Seedlings in the chamber were imaged using a Zeiss EXCITER Laser Confocal Microscope every 6 to 24&#8201;hr (times are provided for specific experiments in figure legends). To image GFP and YFP markers, a 488-nm line of an argon ion laser was used.'},\n",
       " {'pmcid': 'PMC4972537',\n",
       "  'pmid': '27383052',\n",
       "  'snippet': 'Here we combine analyses of light-microscopy-based array tomography (AT) data (Micheva and Smith, 2007; Collman et al., 2015) with a publicly hosted EM dataset (Burns et al., 2013; Martone et al., 2002) generated using a custom transmission EM camera array (TEMCA) (Bock et al., 2011), to obtain an integrated view of the myelination of inhibitory neurons in cortex. We find that an unexpectedly large proportion of myelinated axons in cortex arise from a specific subtype of inhibitory neuron: the parvalbumin-positive (PV+) basket cell. These PV+ axons are especially prominent in the middle cortical layers.'},\n",
       " {'pmcid': 'PMC4972537',\n",
       "  'pmid': '27383052',\n",
       "  'snippet': 'Based on cellular morphology and postsynaptic targets, other PV-expressing neuron types were essentially ruled out, and we found minimal evidence for the myelination of non-PV inhibitory axons. Although this cell-type-specificity of axon myelination was unexpected, especially since myelin has generally been thought to ensheath long-range axons of excitatory neurons, it is nevertheless consistent with earlier observations of individual myelinated inhibitory axons, including basket cell axons (Peters and Proskauer, 1980; DeFelipe et al., 1986; Somogyi et al., 1983; Keller and White, 1987).'},\n",
       " {'pmcid': 'PMC4972537',\n",
       "  'pmid': '27383052',\n",
       "  'snippet': 'By combining data from two independent efforts, we show here that a large fraction of cortical myelin is allocated to the axons of inhibitory neurons, specifically to PV basket cells. This work alters our understanding of the distribution of myelin in neocortical gray matter, and opens new avenues in basic neurobiology, systems neuroscience, and neurological disease research.'},\n",
       " {'pmcid': 'PMC4972537',\n",
       "  'pmid': '27383052',\n",
       "  'snippet': 'Antibodies were obtained from commercial sources and their use has been reported in numerous studies (Tomer et al., 2014; Beirowski et al., 2014; Curchoe et al., 2010; Hodgson et al., 1985; Puthussery et al., 2011; Marek et al., 2010; Gonz&#225;lez-Albo et al., 2001; &#193;lvarez-Quil&#243;n et al., 2014; Kamura et al., 2011). We performed AT specific controls for all antibodies. (Supplementary file 1 and Figure 1&#8212;figure supplement 1).'},\n",
       " {'pmcid': 'PMC4972537',\n",
       "  'pmid': '27383052',\n",
       "  'snippet': 'The primary antibodies were applied for 2&#8201;hr at room temperature, or overnight at 4&#176;C. Alexa dye conjugated secondary antibodies, highly cross-adsorbed (Life Technologies,&#160;Carlsbad,&#160;CA), were used at a dilution of 1:150 for 30&#8201;min at room temperature. One difference from the published protocols was the application of sodium borohydride (1% in Tris buffer for 3&#8201;min) as the first step of the immunofluorescent labeling to reduce non-specific staining and autofluorescence. For some samples, after the sections were imaged, the antibodies were eluted using a solution of 0.2&#8201;M NaOH and 0.02% SDS and new antibodies were reapplied.'},\n",
       " {'pmcid': 'PMC4972537',\n",
       "  'pmid': '27383052',\n",
       "  'snippet': 'Three-color imaging was performed by sequential acquisitions with 405 nm, 488 nm, and 561 nm lasers and pinhole parameters set to 1 Airy unit for any given laser. Detection was tuned to each fluorophore&#8217;s specific emission frequency using a 32-channel detector without the use of filters. Laser powers were scaled with increasing acquisition depth by spline interpolation to maintain constant fluorophore visibility.'},\n",
       " {'pmcid': 'PMC4972960',\n",
       "  'pmid': '27488131',\n",
       "  'snippet': 'The human plasminogen (Plg)/plasmin system is a key player in the tightly controlled blood fibrinolytic pathway that results in dissolving small, developing blood clots by rapid, non-specific, proteolytic events [1, 2]. Plg circulates in blood as a proenzyme in two inactive forms: Glu-Plg and Lys-Plg, which are activated to the serine protease plasmin by two physiological Plg activators, tissue-type Plg activator (tPA) and urokinase Plg activator (uPA) [3&#8211;5].'},\n",
       " {'pmcid': 'PMC4972960',\n",
       "  'pmid': '27488131',\n",
       "  'snippet': 'PBP3 is considered as an enzyme non-essential for growth and survival of S. aureus [42]. The specific role for PBP3 is not known. The cell morphology of mutant strains lacking the pbpC gene that encodes PBP3 do not differ significantly from cells expressing PBP3, but the mutant cells exhibit increased methicillin resistance [29, 42].'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'In contrast, the p75NTR receptor is highly expressed in the EGL, but not the IGL (Carter et al., 2003). The p75NTR can mediate many different functions depending on the cell context by binding to distinct co-receptors and recruiting specific intracellular binding proteins (Barker, 2004; Charalampopoulos et al., 2012). The most well-characterized function for p75NTR is in promoting neuronal apoptosis following brain injury (Friedman, 2010; Ib&#225;&#241;ez and Simi, 2012).'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'The lack of p75NTR was sufficient to increase the size of the cerebellum, a difference that persisted into adulthood, compromising the normal motor/balance function of these animals. In addition, we investigated the p75NTR ligands expressed in the cerebellum that elicited cell cycle arrest of GCPs, and demonstrate a specific role for proNT3 in blocking sonic hedgehog-induced proliferation.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'No significant difference was observed when cultured neurons were incubated with or without proNT-3 (Figure 5 &#8212;figure supplement 1), suggesting that the difference in BrdU incorporation was due to cell cycle arrest and not apoptosis of GCPs. Neither proNGF nor proBDNF had any effect on BrdU incorporation induced by Shh (Figure 5a), suggesting that this was a specific effect of proNT-3. Both proNGF and proBDNF, as well as proNT3, induced death of cultured hippocampal neurons at the doses tested, confirming that they were functional (Figure 5&#8212;figure supplement 2).10.7554/eLife.16654.008Figure 4.Mature neurotrophins have no effect on proliferation of cultured P7 rat GCPs in the absence or presence of Shh.GCPs from P7 rat were cultured with BrdU for 48&#8201;hr in the absence or presence of Shh with or without the different mature neurotrophins.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'ProNT3 was detected in the media with or with KCl treatment, indicating that cerebellar neurons can secrete proNT3, and the release was not dependent on activity (Figure 7b). To validate the specificity of the anti-proNT3 antibody, immunoprecipitation and immunostaining were performed on brain tissue from newborn Ntf3-/- mice and compared to tissue from Ntf3+/+ and Ntf3+/- mice (Figure 7 &#8212;figure supplement 1).'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'p75NTR is expressed in many brain regions during development, and even in the cerebellum this receptor is expressed in Purkinje cells as well as the GCPs in the EGL (Figure 1). To determine the specific role of p75NTR in the EGL, we generated mice that lack p75NTR expression in the EGL by mating floxed Ngfr mice (Ngfr fl/fl) (Bogenmann et al., 2011) with Atoh1(Math1)-CRE mice (Jackson labs). These mice retained p75NTR expression in Purkinje cells and the meninges but lacked p75NTR in the EGL (Figure 9a).'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'In addition to Ki67 expression, we also analyzed EdU incorporation and similarly observed higher level of incorporation at P7, P10, and P14 in the Ngfrfl/fl:Atoh1-Cre mice compared to floxed mice without Cre recombinase, which were identical to WT (Figure 9c).10.7554/eLife.16654.020Figure 9.Specific deletion of p75NTR from the EGL elicits increased GCP proliferation. (a) Ngfrfl/fl mice crossed with the Atoh1-Cre show lack of p75NTR specifically in the EGL while retaining p75NTR expression in Purkinje cells and meninges. Size bar indicates 20 &#181;m. (b) Both Ngfr-/- and Ngfrfl/fl-Atoh1-Cre mice show increased Ki67 labeling in the EGL at P7, P10 and P14.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Analysis of Ki67 expression in these animals demonstrated that the absence of p75NTR specifically from the GCP population was sufficient to increase the number of proliferating GCPs at P7, P10, and P14, indicating a delay in cell cycle withdrawal in the EGL, similar to what we observed in the global Ngfr-/- mice (Figure 9b). In addition to Ki67 expression, we also analyzed EdU incorporation and similarly observed higher level of incorporation at P7, P10, and P14 in the Ngfrfl/fl:Atoh1-Cre mice compared to floxed mice without Cre recombinase, which were identical to WT (Figure 9c).10.7554/eLife.16654.020Figure 9.Specific deletion of p75NTR from the EGL elicits increased GCP proliferation. (a) Ngfrfl/fl mice crossed with the Atoh1-Cre show lack of p75NTR specifically in the EGL while retaining p75NTR expression in Purkinje cells and meninges.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'These mice retained p75NTR expression in Purkinje cells and the meninges but lacked p75NTR in the EGL (Figure 9a). Analysis of Ki67 expression in these animals demonstrated that the absence of p75NTR specifically from the GCP population was sufficient to increase the number of proliferating GCPs at P7, P10, and P14, indicating a delay in cell cycle withdrawal in the EGL, similar to what we observed in the global Ngfr-/- mice (Figure 9b). In addition to Ki67 expression, we also analyzed EdU incorporation and similarly observed higher level of incorporation at P7, P10, and P14 in the Ngfrfl/fl:Atoh1-Cre mice compared to floxed mice without Cre recombinase, which were identical to WT (Figure 9c).10.7554/eLife.16654.020Figure 9.Specific deletion of p75NTR from the EGL elicits increased GCP proliferation.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': '(a) Ngfrfl/fl mice crossed with the Atoh1-Cre show lack of p75NTR specifically in the EGL while retaining p75NTR expression in Purkinje cells and meninges. Size bar indicates 20 &#181;m. (b) Both Ngfr-/- and Ngfrfl/fl-Atoh1-Cre mice show increased Ki67 labeling in the EGL at P7, P10 and P14.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Since we observed continued proliferation of GCPs during the development of the cerebellum when p75NTR was removed either globally or specifically from the EGL, with an increase in cerebellar size (Figure 3g), we sought to determine whether there was any persistent consequence for cerebellar size and function in adult animals. The increased cerebellar size in both the global and EGL-specific (Ngfrfl/fl:Atoh1-Cre) p75NTR-/-mice persisted into adulthood (Figure 10a).'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'The latency to fall was significantly faster compared to the WT mice. However, the Ngfrfl/fl:Atoh1-Cre mice that specifically lacked p75NTR in the developing EGL also showed significant motor deficits compared to WT mice beginning at 20&#8201;rpm, indicating that the absence of this receptor specifically from the EGL during development caused a persistent deficit in motor function in the adult.10.7554/eLife.16654.022Figure 10.The absence of p75NTR during EGL development has persistent effects in the adult. (a) Cerebellar size of both Ngfr-/- and Ngfrfl/fl-Atoh1-Cre was increased in the adult compared to WT mice.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Since we observed continued proliferation of GCPs during the development of the cerebellum when p75NTR was removed either globally or specifically from the EGL, with an increase in cerebellar size (Figure 3g), we sought to determine whether there was any persistent consequence for cerebellar size and function in adult animals. The increased cerebellar size in both the global and EGL-specific (Ngfrfl/fl:Atoh1-Cre) p75NTR-/-mice persisted into adulthood (Figure 10a). To further elucidate a possible functional consequence of these developmental changes, we analyzed the motor/balance coordination in these animals using the rotarod test.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'However, whether proneurotrophins play a role in cerebellar development has not been extensively investigated. p75NTR is abundantly expressed in granule cells only in the EGL, specifically in the outer proliferative layer, which gets progressively smaller with development as the progenitors cease proliferating and migrate to the IGL. The p75NTR can bind many different ligands, associate with different co-receptors, and recruit a panoply of intracellular binding proteins to activate specific signaling pathways (Charalampopoulos et al., 2012; Roux and Barker, 2002), and therefore can influence multiple functions depending on the cellular context.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'p75NTR is abundantly expressed in granule cells only in the EGL, specifically in the outer proliferative layer, which gets progressively smaller with development as the progenitors cease proliferating and migrate to the IGL. The p75NTR can bind many different ligands, associate with different co-receptors, and recruit a panoply of intracellular binding proteins to activate specific signaling pathways (Charalampopoulos et al., 2012; Roux and Barker, 2002), and therefore can influence multiple functions depending on the cellular context. The most well characterized effect mediated by p75NTR is cell death (Frade et al., 1996; Friedman, 2000), especially after injury such as that induced by seizures (Roux et al., 1999; Troy et al., 2002; Unsain et al., 2008).'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Several factors have been identified that promote cell cycle exit, including BMP2 and 4 (Rios et al., 2004), Wnt3 (Anne et al., 2013), and PACAP (Nicot et al., 2002), each acting by a distinct mechanism to block Shh activation of the Gli transcription factors. However, despite the presence of these other factors, the specific absence of p75NTR caused the GCPs to continue proliferating beyond the usual developmental period, indicating that this receptor is important for the timing of cell cycle withdrawal. A question that remains to be addressed is whether the different anti-proliferative factors have redundant effects to ensure normal development of the cerebellum, or if each factor is necessary for maturation of different subpopulations of granule cells, all of them necessary for proper function.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'In WT mice the number of proliferating GCPs in the EGL began to decrease at P7, and continued to decrease until few proliferating cells remained in the EGL at P14. In contrast, in both the global Ngfr-/- mice and the mice specifically lacking p75NTR in the EGL (Ngfrfl/fl:Atoh1-Cre mice), the number of proliferating, Ki67-positive cells remained elevated at P7, P10, and P14. However, the granule cells eventually ceased proliferating, and no tumors formed in these mice.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Shh is expressed by Purkinje cells and regulates the proliferation of the GCPs (Dahmane et al., 1999). Shh signals by regulating the Gli transcription factors, specifically activating Gli2, which then targets Gli1 for transcriptional induction (Blaess et al., 2006; Hui and Angers, 2011). Gli1 and Gli2 are acetylated proteins, and their transcriptional activation requires deacetylation by HDAC1 (Canettieri et al., 2010), which is highly expressed in proliferating GCPs (Yoo et al., 2013).'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Thus, proNT-3 proved to be the only neurotrophin to promote cell cycle exit in this neuronal population, suggesting that there is specificity among the different p75NTR ligands for their ability to influence distinct cellular functions. The basis for this specificity is unknown, however p75NTR has been shown to bind many different ligands to trigger distinct responses. Even the same ligand binding to p75NTR on the same cells may trigger different responses depending on the state of the cell.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Although both the Val66 and Met66 pro domains bound the SorCS2 receptor, only the Val66 pro domain induced growth cone collapse, the Met66 pro domain did not (Anastasia et al., 2013), indicating that slight differences in the pro domain can lead to differences in functional outcome. Our current study also identified SorCS2 as the co-receptor for proNT3 actions on the GCP population, suggesting that the distinct pro domains of proneurotrophins differentially bind to the SorCS2 co-receptor and influence how p75NTR interacts with specific signaling pathways to regulate cellular responses, which may account for the specificity of proNT3 in regulating GCP cell cycle exit.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Surprisingly, proNGF and proBDNF had no effect on GCP proliferation, although they effectively promoted death of hippocampal neurons. Thus, proNT-3 proved to be the only neurotrophin to promote cell cycle exit in this neuronal population, suggesting that there is specificity among the different p75NTR ligands for their ability to influence distinct cellular functions. The basis for this specificity is unknown, however p75NTR has been shown to bind many different ligands to trigger distinct responses.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Not surprisingly, therefore, the global Ngfr-/- mice performed poorly on the rotarod test. To ascertain whether the changes in the granule cell population resulting from the specific lack of p75NTR in the EGL had any consequence for motor function, floxed Ngfr mice were mated with the Atoh1-Cre mice to eliminate p75NTR from the EGL while retaining the receptor in Purkinje cells and other neuronal populations. These mice also demonstrated a significant delay in GCP cell cycle exit, a larger cerebellum, and a deficit in motor/balance function on the rotarod test.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'These mice also demonstrated a significant delay in GCP cell cycle exit, a larger cerebellum, and a deficit in motor/balance function on the rotarod test. Although the deficit was not as severe as that seen with the global Ngfr knockout mice, these data indicate that the EGL-specific lack of p75NTR during development was sufficient to cause persistent loss of function into adulthood. Thus, the delayed withdrawal from the cell cycle resulting in expanded proliferation of GCPs are likely to have altered the ratio of granule cells to other neuronal populations, impacting the development of appropriate circuitry for motor function.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'In summary, we demonstrate a novel function for p75NTR in regulating the timing of cell cycle withdrawal in granule neuron progenitors in the developing cerebellum. ProNT3 specifically antagonized Shh-induced proliferation of GCPs, decreasing the level of HDAC1 and induction of Gli1 mRNA, indicating a potential mechanism for interfering with Shh signaling and facilitating exit of these progenitors from the cell cycle prior to migrating and differentiating. Since precise regulation of these events is critical for normal development, the continued proliferation of GCPs in the absence of p75NTR led to increased cerebellar size that persisted into adulthood, with deficits in motor behavior.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Cells were then fixed with 4% paraformaldehyde (PFA)/PBS for 15&#8201;min and permeabilized with 0.5% Triton &#215;&#160;100 for 20&#8201;min. TUNEL assay was performed following the manufacturers specification (Promega G3250). 10 pictures per coverslip were taken with a Nikon Eclipse TE200 microscope.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Controls for immunostaining included incubation with secondary antibodies in the absence of primary antibodies. The proNT3 antibody was tested for specificity on sections from Ntf3-/- mice compared to WT from P0 pups, since the Ntf3 -/- mice die after birth.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Cells were washed with PBS 2 &#215; 10&#8201;min and permeabilized with 0.5% Triton &#215;&#160;100 in PBS for 15&#8201;min, and then blocked with 1% BSA, 5% donkey serum in PBS for 1&#160;hr at RT. Cells were incubated with primary antibody overnight at 4&#176;C. Anti-BrdU (Millipore 05&#8211;633, RRID:AB_309861) was diluted 1:50 according to the manufacturers specification. The plates were washed with PBS 3 &#215; 15&#8201;min each and incubated with Licor secondary antibody diluted 1:5,000 for 1&#8201;hr at RT.'},\n",
       " {'pmcid': 'PMC4975574',\n",
       "  'pmid': '27434667',\n",
       "  'snippet': 'Experimental groups were compared using either student&#8217;s t-test or ANOVA followed by Tukey&#8217;s posthoc analysis, as appropriate, p&lt;0.05 was considered significant. The specific statistical analysis is indicated in each figure legend.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'In the retina, high levels of type 1 cannabinoid receptors (CB1R), as well as the eCB production and degradation machinery, have been found in all vertebrate models tested, and within nearly all retinal cell types throughout development and adulthood (Straiker et al., 1999; Yazulla, 2008; Zabouri et al., 2011). Although eCB modulation of voltage-dependent Ca2+ and K+ conductances, typically resulting in the downregulation of neurotransmitter release, has been demonstrated at multiple stages of retinal processing from photoreceptors to retinal ganglion cells (RGCs) (Fan and Yazulla, 2003; Lalonde et al., 2006; Middleton and Protti, 2011; Yazulla et al., 2000), these observations have not been directly linked to specific perceptual outcomes. On the other hand, cannabis consumption in humans has been reported to positively modulate contrast discrimination (Russo et al., 2004; West, 1991).'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': '(a) Schematic representation of the imaging configuration and retinal circuitry showing bipolar and amacrine cell type locations. (b) GCaMP6s expressed in bipolar cells, with ROIs exemplifying ON- and OFF-specific bipolar axon terminals. (c) Average calcium responses of individual bipolar axon terminal to 10 light-ON (top) or 10 light-OFF (bottom) flashes before (black) and after (red) WIN 55,212-2 perfusion.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': '(e) A significant change in &#9651;F/F peak amplitude of bipolar axon terminals (n = 10 terminals from four animals) was observed to light-OFF following WIN 55,212-2 perfusion. (f) GCaMP6s expressed in amacrine cells, with ROI exemplifying ON- and OFF-specific terminals. (g) Average calcium responses of amacrine cell terminals to 10 light-ON (top) or 10 light-OFF (bottom) flashes before (black) and after (red) WIN 55,212-2 infusion.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': '(a) Schematic representation of the imaging configuration and retinal circuitry showing bipolar and amacrine cell type locations. (b) GCaMP6s expressed in bipolar cells, with ROIs exemplifying ON- and OFF-specific bipolar axon terminals. (c) Average calcium responses of individual bipolar axon terminal to 10 light-ON (top) or 10 light-OFF (bottom) flashes before (black) and after (red) WIN 55,212-2 perfusion.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': '(e) A significant change in &#9651;F/F peak amplitude of bipolar axon terminals (n = 10 terminals from four animals) was observed to light-OFF following WIN 55,212-2 perfusion. (f) GCaMP6s expressed in amacrine cells, with ROI exemplifying ON- and OFF-specific terminals. (g) Average calcium responses of amacrine cell terminals to 10 light-ON (top) or 10 light-OFF (bottom) flashes before (black) and after (red) WIN 55,212-2 infusion.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'The enhancement produced by elevating endogenous AEA using URB597 (Figure 2e) was also prevented by AM-251 (Figure 4c, n = 10, p=0.500). The selective GABA-A receptor blocker gabazine (6 &#181;M) had no specific effect on visually-evoked spike rates and did not prevent the WIN 55,212-2-induced increase in RGC firing in response to light (Figure 4d, n = 10, p=0.012). However, strychnine (60&#160;&#181;M), a potent and selective antagonist of glycine receptors (GlyRs) prevented (Figure 4e, n = 10, p=0.423) and reversed (Figure 4f, n = 10, p=0.044) the enhancement caused by WIN 55,212-2 application.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'We therefore wondered whether AMPK activation might be suppressing NKCC1 function. Because no phospho-specific antibody against the AMPK site S77 on NKCC1 is available, we instead tested this hypothesis functionally by using the selective AMPK inhibitor dorsomorphin. We found that dorsomorphin (10 &#181;M) reversed the effects of WIN 55,212-2, reducing RGCs spiking rate in response to light-OFF flashes (before: 37.5 &#177; 6.3&#8201;Hz, after: 22.9 &#177; 2.6&#8201;Hz, Figure 4l, n = 10, p=0.0095, paired t-test).'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'With synaptic block, WIN 55,212-2 application still significantly decreased the cyan:yellow ratio in RGCs (Figure 5g, n = 14, p&lt;0.001). These data indicate that retinal CB1R activation rapidly decreases [Cl&#8722;]i specifically in RGCs through a NKCC1-dependent and neurotransmission-independent mechanism.10.7554/eLife.15932.009Figure 5.WIN 55,212-2 decreases intracellular Cl&#8722; in RGCs. (a) Cerulean(CFP)-Topaz(YFP) modified version of the FRET-based Cl&#8722; indicator clomeleon provides a ratiometric readout of changes in intracellular Cl&#8722; concentration due to the quenching of YFP fluorescence by Cl&#8722;.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'Only cells with a stable baseline CFP/YFP ratio (&lt;10% drift) were analyzed. (c) Somatic measurements of CFP/YFP from the cells in panel b, reveal an RGC-specific change in the CFP/YFP fluorescence ratio in response to WIN 55,212-2. (d) Measurements of WIN 55,212-2-induced effects on CFP/YFP fluorescence ratios in all 15 RGCs imaged in seven tadpoles.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'Pretreatment with AM-251 (n = 8) or bumetanide (n = 9) prevented any changes in [Cl&#8722;]i upon addition of WIN 55,212-2 (Figure 5f, p=0.996 for both). These data suggest that CB1R activation induces a RGC-specific decrease in [Cl&#8722;]i by regulating NKCC1 activity. To determine if synaptic activity might contribute to the changes in [Cl&#8722;]i, we blocked synaptic transmission with a cocktail of DNQX (20 &#181;M), AP4 (50 &#181;M), picrotoxin (100 &#181;M), strychnine (60 &#181;M) and TTX (1 &#181;M).'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'Only cells with a stable baseline CFP/YFP ratio (&lt;10% drift) were analyzed. (c) Somatic measurements of CFP/YFP from the cells in panel b, reveal an RGC-specific change in the CFP/YFP fluorescence ratio in response to WIN 55,212-2. (d) Measurements of WIN 55,212-2-induced effects on CFP/YFP fluorescence ratios in all 15 RGCs imaged in seven tadpoles.'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'While a reduction in synaptic inhibition could potentially contribute to the cannabinoid-dependent enhancement of RGC firing that we detected, it is unlikely to fully account for this phenomenon for three reasons. First, the primary effect that we observed was a specific decrease in RGC [Cl]i leading to membrane hyperpolarization. This should enhance inhibitory driving force rather than decrease it and therefore is more consistent with our model that tonic pre-hyperpolarization helps prime the cell to fire more robustly to subsequent depolarization, which we demonstrated can occur in RGCs (Figure 6j,k).'},\n",
       " {'pmcid': 'PMC4987138',\n",
       "  'pmid': '27501334',\n",
       "  'snippet': 'For all immunolabeling, control experiments omitting the primary antibody were performed. We also controlled for the specificity of the primary antibody by co-application of a blocking peptide provided by Dr. Ken Mackie. Slides were then rinsed in phosphate buffer and coverslipped in Vectashield Mounting Medium with DAPI nuclear stain (Vector Laboratories, Burlingame, CA).'},\n",
       " {'pmcid': 'PMC4987413',\n",
       "  'pmid': '27570824',\n",
       "  'snippet': 'A variety of treatments that alter the excitatory/inhibitory balance of the visual cortex has been reported to control ocular dominance plasticity (Takesian and Hensch, 2013). Specific pharmacogenetic inhibition of PV+ neurons in visual cortex extends the critical period for ocular dominace plasticity (Kuhlman et al., 2013). Reducing PV+ neuron activity may affect ocular dominance plasticity by shifting the ability for synapses to undergo synaptic plasticity (Kirkwood and Bear, 1994; Harauzov et al., 2010; Kuhlman et al., 2010).'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'It was shown that PC somatic synapses facing PV+ axon terminals contain mainly &#945;1 subunit-containing GABAARs (Klausberger et al., 2002) whereas somatic synapses opposite to CCK+ boutons and AIS synapses predominantly contain GABAARs incorporating the &#945;2 subunit (Nusser et al., 1996; Fritschy et al., 1998; Nyiri et al., 2001). Such a presynaptic input-specific distribution of distinct postsynaptic receptor types held a great potential for the selective pharmacological manipulation of cortical networks in various psychiatric disorders (e.g. schizophrenia or anxiety; reviewed by Lewis et al., 2005; Rudolph and Mohler, 2014).'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Carbon-platinum-carbon coating of the surfaces stabilizes the phospholipids and transmembrane proteins in the replica, allowing their visualization with specific antibodies. If an antibody recognizes intracellular epitope(s), specific immunolabeling will be detected in the P-face of the membrane, whereas extracellular epitopes will be visualized on the E-face. The most widely used application of SDS-FRL is to perform quantitative comparisons between distinct subcellular compartments.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'The basic principle of SDS-FRL is that following random fracturing of frozen tissue, the fracturing plane often crosses the lipid bilayer of plasma membranes with transmembrane proteins remaining either in the protoplasmic (P-face) or extracellular (E-face) half of the membrane (Fujimoto, 1995; Rash et al., 1998; Masugi-Tokita and Shigemoto, 2007). Carbon-platinum-carbon coating of the surfaces stabilizes the phospholipids and transmembrane proteins in the replica, allowing their visualization with specific antibodies. If an antibody recognizes intracellular epitope(s), specific immunolabeling will be detected in the P-face of the membrane, whereas extracellular epitopes will be visualized on the E-face.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Thus, throughout the study, we used immunoreactions for Nav1.6 or Kv1.1 subunits and defined a membrane as an AIS if it was either Nav1.6 or Kv1.1 immunopositive (Lorincz and Nusser, 2010; Kirizs et al., 2014). To avoid complications arising from potential competition between different antibodies on the same replica, the mirror replica method was used for the co-localization of the &#947;2 subunit and neuroligin-2 (NL-2), a GABAergic synapse-specific cell adhesion molecule (Figures 2 and 3). Immunogold particles for NL-2 were clustered over small areas of somatic and AIS P-face membranes that contained a high density of IMPs, characteristic of GABAergic synapses (Kasugai et al., 2010).'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'In contrast, we found that normalized &#947;2 subunit densities were significantly higher in CA3 and CA1 AIS synapses than in somatic synapses of both regions (Figure 3K). These results demonstrate that all GABAergic synapses on the soma and AIS of both CA1 and CA3 PCs contain the &#947;2 subunit and NL-2, enabling their use as GABAergic synapse-specific markers.10.7554/eLife.18426.003Figure 2.Identification of axon initial segment (AIS) and somatic GABAergic synapses in CA3 PCs using replica immunolabeling. (A) Mirror replica image of an AIS identified by immungold labeling for the Nav1.6 subunit (15 nm gold) on the P-face.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Next, we asked whether the two most abundant &#945; subunits expressed by hippocampal PCs, the &#945;1 and &#945;2 subunits, show input-specific distributions. First, we carried out face-matched mirror replica labeling for NL-2 and the &#945;1 subunit (Figure 4).'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'If distinct perisomatic synapses have GABAARs with different subunit compositions, it is likely to be reflected in differential distributions of the &#946; subunit isoforms as well. We obtained antibodies that provided specific immunolabeling against all &#946; subunit isoforms (&#946;1, &#946;2, &#946;3) and co-localized them with the &#947;2 subunit in face-matched mirror replicas. Strong clustering of immunogold particles labeling the &#946;1 (Figure 8), &#946;2 (Figure 9) or &#946;3 (Figure 10) subunits was found over IMP clusters characteristic of postsynaptic membranes on the P-face in both PC AISs and somata.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Antibodies used in this study recognized either intracellular or extracellular epitopes on their target proteins and consequently were visualized by gold particles labeling on the protoplasmic face (P-face) or the exoplasmic face (E-face), respectively. The nonspecific background labeling was measured on either E-face or P-face structures of the replicas, depending on the specific labeling of the target proteins. Background labeling was very low for NL-2 and the GABAAR subunits, and therefore was not subtracted from the synaptic density values.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'To identify the AIS of hippocampal PCs, immunogold labeling for Nav1.6 or Kv1.1 subunits was carried out (Lorincz and Nusser, 2010; Kirizs et al., 2014). A GABAergic synapse-specific adhesion molecule (Varoqueaux et al., 2004), NL-2, was used to identify AIS and somatic synapses on the P-face of the plasma membrane, where GABAergic synapses appear as small membrane patches rich in IMPs (Kasugai et al., 2010). To quantitatively analyze the density of gold particles labeling different GABAAR subunits in perisomatic synapses, the face-matched mirror replica technique was used (Hagiwara et al., 2005; Kerti-Szigeti et al., 2014).'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'cerebellar Purkinje cells, and medial habenula neurons), and observe very few gold particles labeling for &#946;1 or &#946;2 subunits in GABAergic synapses (zero or 1&#8211;3 gold particles / synapse; data not shown). These results indicate that our immunogold labeling for the &#946;1 (&#946;1(Gp; aa342-430)) and &#946;2 (&#946;2(Gp; aa343-430)) subunits in hippocampal perisomatic synapses is probably due to specific antibody-protein interactions. We observed a similar labeling pattern with our guinea-pig anti-&#946;3 antibody (&#946;3(Gp; aa344-429)) to that published by Kasugai et al.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Our &#947;2 labeling on the E-face, obtained with a rabbit antibody (&#947;2(Rb, aa39-67)), was very similar to the immunogold labeling seen on the P-face with a rabbit anti-&#947;2 antibody (&#947;2(Rb, aa319-366)), recognizing an intracellular epitope (Figure 1A&#8211;D). We could not purchase anti-&#946; subunit antibodies raised against different epitopes, and therefore we could not test the specificity of the labeling using two antibodies. However, we performed SDS-FRL immunogold labeling for the &#946;1 and &#946;2 subunits in brain areas and nerve cells where the genes of these subunits are not expressed (e.g.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Specificity of immunogold labeling for the &#945;1, &#945;2 and &#947;2 subunits was verified by using two antibodies directed against different epitopes of the same protein. Similar labeling patterns were obtained with a rabbit anti-&#945;1 (&#945;1(Rb; aa1-9)) and a mouse anti-&#945;1 (&#945;1(Mo; aa28-43)) antibody, indicating the specificity of the reaction.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Similar labeling patterns were obtained with a rabbit anti-&#945;1 (&#945;1(Rb; aa1-9)) and a mouse anti-&#945;1 (&#945;1(Mo; aa28-43)) antibody, indicating the specificity of the reaction. Mirror replica labeling was used to assess the specificity of our &#945;2 and &#947;2 labeling, by using antibodies directed against an extracellular and an intracellular epitope. We observe similar gold particle labeling patterns with a rabbit anti-&#945;2 antibody (&#945;2(Rb; aa322-357)) on the P-face compared to that obtained with a guinea-pig anti-&#945;2 antibody (&#945;2(Gp; aa1-9)) on the E-face.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'Specificity of immunogold labeling for the &#945;1, &#945;2 and &#947;2 subunits was verified by using two antibodies directed against different epitopes of the same protein. Similar labeling patterns were obtained with a rabbit anti-&#945;1 (&#945;1(Rb; aa1-9)) and a mouse anti-&#945;1 (&#945;1(Mo; aa28-43)) antibody, indicating the specificity of the reaction. Mirror replica labeling was used to assess the specificity of our &#945;2 and &#947;2 labeling, by using antibodies directed against an extracellular and an intracellular epitope.'},\n",
       " {'pmcid': 'PMC4990423',\n",
       "  'pmid': '27537197',\n",
       "  'snippet': 'We observed a similar labeling pattern with our guinea-pig anti-&#946;3 antibody (&#946;3(Gp; aa344-429)) to that published by Kasugai et al. (2010) with a different antibody against the &#946;3 subunit, the specificity of which was proven in &#946;3-/- mice.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Similarly, the infection of HeLa cells with Sendai virus led to a detection of the active form of TBK1 at the Golgi apparatus (Additional file 2A&#8211;E). TBK1&#8211;/&#8211; MEFs confirmed the specificity of the used antibody raised against p-TBK1S172 (Fig.&#160;1f) and, at later time points post-infection, we observed that the active form of TBK1 accumulates very close to the Golgi apparatus, at the centrosome (Additional file 1D and E). Importantly, RLR activation promoted the K63-linked polyubiquitination of TBK1, as previously reported (Additional file 2F) [13].Fig.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '(Ub)n, polyubiquitin. * Indicates non-specific bands. b&#8211;e MEFs were either left unstimulated (control) or infected with SeV for 6&#160;h (+ SeV).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '(Ub)n, polyubiquitin. * Indicates non-specific bands. b&#8211;e MEFs were either left unstimulated (control) or infected with SeV for 6&#160;h (+ SeV).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '(Ub)n, polyubiquitin. * Indicates non-specific bands. b, c HEK293T cells stably expressing HA-TLR3 were either left untreated (control) or stimulated with poly(I:C) (10&#160;&#956;g/mL) for 1&#160;h (+ Poly(I:C)).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '(Ub)n, polyubiquitin. * Indicates non-specific bands. e Increased concentrations of mitochondria- (P5) or Golgi (P25)-enriched fractions of unstimulated (Unst) or poly(I:C)-treated HEK293T cells stably expressing HA-TLR3 were incubated with recombinant GST-IRF3 in the presence of ATP.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '(Ub)n, polyubiquitin. * Indicates non-specific bands. b, c HEK293T cells stably expressing HA-TLR3 were either left untreated (control) or stimulated with poly(I:C) (10&#160;&#956;g/mL) for 1&#160;h (+ Poly(I:C)).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '(Ub)n, polyubiquitin. * Indicates non-specific bands. e Increased concentrations of mitochondria- (P5) or Golgi (P25)-enriched fractions of unstimulated (Unst) or poly(I:C)-treated HEK293T cells stably expressing HA-TLR3 were incubated with recombinant GST-IRF3 in the presence of ATP.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Interestingly, transfection with low- or high-molecular weight poly(I:C) for 2&#160;hours triggered a TBK1 aggregation at the Golgi apparatus (Fig.&#160;3a). TBK1 staining was lost in TBK1&#8211;/&#8211; MEFs (Fig.&#160;3b) confirming the specificity of the anti-TBK1 antibody and this TBK1 aggregation was also detected with another antibody raised against the kinase (Additional file 3A). TBK1 aggregation was associated with the nuclear translocation of IRF3 (Fig.&#160;3c) and these aggregates do not co-localize with the autophagosomes (data not shown).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'e&#8211;g MEFs were either left untreated (MOCK) or transfected with HMW poly(I:C) (5&#160;&#956;g/mL) for 2 or 4&#160;h (trPoly(I:C)). The indicated proteins were analyzed by immunofluorescence analysis with specific antibodies. The Golgi apparatus was stained with an antibody against GM130.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'c MEFs were either left untreated (MOCK) or transfected with HMW poly(I:C) (5&#160;&#956;g/mL) for 2&#160;h (trPoly(I:C)). The indicated proteins were analyzed by immunofluorescence analysis with specific antibodies. Scale bars, 10&#160;&#956;m. d WT, MAVS&#8211;/&#8211;, or STING&#8211;/&#8211; MEFs were either left untreated (MOCK) or transfected with HMW poly(I:C) (5&#160;&#956;g/mL) for 4&#160;h (trPoly(I:C)).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'e&#8211;g MEFs were either left untreated (MOCK) or transfected with HMW poly(I:C) (5&#160;&#956;g/mL) for 2 or 4&#160;h (trPoly(I:C)). The indicated proteins were analyzed by immunofluorescence analysis with specific antibodies. The Golgi apparatus was stained with an antibody against GM130.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'c MEFs were either left untreated (MOCK) or transfected with HMW poly(I:C) (5&#160;&#956;g/mL) for 2&#160;h (trPoly(I:C)). The indicated proteins were analyzed by immunofluorescence analysis with specific antibodies. Scale bars, 10&#160;&#956;m. d WT, MAVS&#8211;/&#8211;, or STING&#8211;/&#8211; MEFs were either left untreated (MOCK) or transfected with HMW poly(I:C) (5&#160;&#956;g/mL) for 4&#160;h (trPoly(I:C)).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Cell lysates were then analyzed by immunoblotting with antibodies against the indicated proteins. * Indicates a non-specific band. f&#8211;i WT or OPTN KO HeLa cells were infected with SeV for the indicated times.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Then, 72&#160;h later, cells were either left unstimulated or infected with SeV for 6 or 8&#160;h. Cell lysates were analyzed by immunoblotting with antibodies against the indicated proteins. * Indicates non-specific bands. d MEFs were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN 1 and OPTN 2).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '* Indicates non-specific bands. d MEFs were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN 1 and OPTN 2). Then, 72&#160;h later, cells were either left untreated (MOCK) or transfected with high molecular weight poly(I:C) (5&#160;&#956;g/mL) for 4&#160;h (trPoly(I:C)).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '&#176; Corresponds to TBK1 detection from a previous immunoblot. k HEK293-TLR3 cells were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN A and OPTN B) or a NEMO-specific siRNA. The cells were then transfected, 48&#160;h later, with an IFN&#946; promoter reporter or with a NF-&#954;B reporter and the Renilla luciferase gene as an internal control.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Samples were then analyzed by immunoblotting with antibodies against the indicated proteins. b HEK293T cells were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN A and OPTN B) or a NEMO-specific siRNA. The cells were then transfected, 48&#160;h later, with an IFN&#946; promoter reporter or with a NF-&#954;B reporter and the Renilla luciferase gene as an internal control.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'ns, not significant. c MEFs were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN 1 and OPTN 2) or a NEMO-specific siRNA. Then, 72&#160;h later, cells were either left unstimulated or infected with SeV for 6 or 8&#160;h. Cell lysates were analyzed by immunoblotting with antibodies against the indicated proteins.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Cell lysates were then analyzed by immunoblotting with antibodies against the indicated proteins. * Indicates a non-specific band. f&#8211;i WT or OPTN KO HeLa cells were infected with SeV for the indicated times.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Then, 72&#160;h later, cells were either left unstimulated or infected with SeV for 6 or 8&#160;h. Cell lysates were analyzed by immunoblotting with antibodies against the indicated proteins. * Indicates non-specific bands. d MEFs were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN 1 and OPTN 2).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '* Indicates non-specific bands. d MEFs were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN 1 and OPTN 2). Then, 72&#160;h later, cells were either left untreated (MOCK) or transfected with high molecular weight poly(I:C) (5&#160;&#956;g/mL) for 4&#160;h (trPoly(I:C)).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': '&#176; Corresponds to TBK1 detection from a previous immunoblot. k HEK293-TLR3 cells were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN A and OPTN B) or a NEMO-specific siRNA. The cells were then transfected, 48&#160;h later, with an IFN&#946; promoter reporter or with a NF-&#954;B reporter and the Renilla luciferase gene as an internal control.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Samples were then analyzed by immunoblotting with antibodies against the indicated proteins. b HEK293T cells were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN A and OPTN B) or a NEMO-specific siRNA. The cells were then transfected, 48&#160;h later, with an IFN&#946; promoter reporter or with a NF-&#954;B reporter and the Renilla luciferase gene as an internal control.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'ns, not significant. c MEFs were transfected with a control non-specific siRNA (NS) or with two individual OPTN-specific siRNAs (OPTN 1 and OPTN 2) or a NEMO-specific siRNA. Then, 72&#160;h later, cells were either left unstimulated or infected with SeV for 6 or 8&#160;h. Cell lysates were analyzed by immunoblotting with antibodies against the indicated proteins.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Cell lysates were then analyzed by immunoblotting with antibodies against the indicated proteins. * Indicates a non-specific band. b, c Primary MEFs isolated from WT or OPTN470T mice were infected with SeV for the indicated times.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Cell lysates were then analyzed by immunoblotting with antibodies against the indicated proteins. * Indicates a non-specific band. b, c Primary MEFs isolated from WT or OPTN470T mice were infected with SeV for the indicated times.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Non-structural protein 3 (NS3) of the Bluetongue virus, a dsRNA virus, has been localized to the Golgi apparatus and shown to specifically modulate the type I IFN signaling pathway [31, 32]. We confirmed that NS3 expression led to the detection of this protein at the Golgi apparatus (Fig.&#160;7a) and that, in luciferase assays, NS3 affected the stimulation of the IFN&#946; promoter but not NF-&#954;B activation after RLR stimulation (Fig.&#160;7b).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Surprisingly, after the stimulation of cells with IL-1&#946; or TNF&#945;, after mitophagy induction or in cancer dependent on KRAS signaling, TBK1 is phosphorylated whereas IRF3 is not [25, 26, 33, 34]. It has been therefore suggested that TBK1 autoactivation and substrate specificity are both dependent on the subcellular distribution of TBK1, with various adaptor proteins each directing TBK1 to discrete signaling complexes for different cellular responses [12, 15, 16]. Consistent with this hypothesis, we observed that the active form of TBK1 is present at the Golgi apparatus after the stimulation of RLRs or TLR3, and that its substrate, IRF3, is phosphorylated.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'They were fixed by incubation in 4&#160;% paraformaldehyde in phosphate buffered saline (PBS) for 10&#160;minutes, and then permeabilized by incubation with 0.15&#160;% Triton X-100 in PBS for 15&#160;minutes. Non-specific binding sites were blocked by incubating cells in a solution of 2&#160;% BSA in PBS for 1&#160;hour. The cells were then incubated for 1&#160;hour at room temperature or overnight at 4&#160;&#176;C with the primary antibodies.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'The cells were then incubated for 1&#160;hour at room temperature or overnight at 4&#160;&#176;C with the primary antibodies. They were washed three times, for five minutes each, in PBS and were then incubated for 1&#160;hour with the specific Alexa Fluor-conjugated secondary antibodies (Invitrogen, Life Technologies, Grand Island, NY, USA). Nuclei were stained with DAPI (Sigma) and cells were again washed three times with PBS.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'The cells were lysed and their ubiquitin conjugate content was analyzed at room temperature in denaturing conditions (8&#160;M urea, 0.1&#160;M NaH2PO4, 10&#160;mM Tris-HCl pH&#160;8, 1&#160;% Triton X-100, 1&#160;% NP-40, 20&#160;mM imidazole). Immunoblot analysis was performed with specific antibodies and the antigen&#8211;antibody complexes were visualized by chemiluminescence (Immobilon Western, Merck Millipore, Billerica, MA, USA).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'The primary antibodies used for immunoblotting were rabbit monoclonal anti-TBK1 (Abcam, Cambridge, UK, Abcam Cat# ab40676 RRID:AB_776632, 1:5000 dilution), rabbit monoclonal anti-phosphorylated TBK1 (phospho S172) (Abcam Cat# ab109272 RRID:AB_10862438, 1:2000), rabbit monoclonal anti-phosphorylated TBK1 (phospho S172) (Cell Signaling Technology, Danvers, MA, USA, Cell Signaling Technology Cat# 5483P RRID:AB_10693472, 1:2000), rabbit monoclonal anti-IRF3 (Cell Signaling Technology Cat# 4302S RRID:AB_1904036, 1:5000), rabbit monoclonal anti-phosphorylated IRF3 (phospho S386) (Abcam Cat# ab76493 RRID:AB_1523836, 1:5000), rabbit monoclonal anti-phosphorylated IRF3 (phospho S396) (Cell Signaling Technology Cat# 4947S RRID:AB_823547, 1:2000), mouse monoclonal anti-MAVS (Enzo Life Sciences, Farmingdale, NY, USA, Enzo Life Sciences Cat# ALX-804-847 RRID:AB_10539976, 1:2000), rabbit polyclonal anti-TRAF3 (Santa Cruz Biotechnology, Santa Cruz, CA, USA, Santa Cruz Biotechnology Cat# sc-1828 RRID:AB_2209427, 1:1000), rabbit polyclonal anti-HA (Sigma-Aldrich Cat# H6908 RRID:AB_260070, 1:2000), rabbit polyclonal anti-TRIF (Cell Signaling Technology Cat# 4596S RRID:AB_2256555, 1:2000), rabbit polyclonal anti-GAPDH (Sigma-Aldrich Cat# G9545 RRID:AB_796208, 1:20000), rabbit polyclonal anti-KTN (Santa Cruz Biotechnology Cat# sc-33562 RRID:AB_2133047, 1:10000), mouse monoclonal anti-LAMP2 (Santa Cruz Biotechnology Cat# sc-18822 RRID:AB_626858, 1:10000), mouse monoclonal anti-EEA1 (BD Biosciences, East Rutherford, NJ, USA, BD Biosciences Cat# 610456 RRID:AB_397829, 1:10000), rabbit monoclonal anti-Syntaxin6 (Cell Signaling Technology Cat# 2869S RRID:AB_2196500, 1:5000), rabbit polyclonal anti-VDAC (Millipore Cat# AB10527 RRID:AB_10806766, 1:10000), mouse anti-glutathione-S-transferase (BD Biosciences Cat# 554805 RRID:AB_395536, 1:2000), mouse monoclonal anti-Lys-63 specific (Millipore Cat# 05-1313 RRID:AB_1587585, 1:500), mouse monoclonal anti-actin (Sigma-Aldrich Cat# A3853 RRID:AB_262137, 1:10000), rabbit polyclonal anti-optineurin (Abcam Cat# ab23666 RRID:AB_447598, 1:5000), rabbit polyclonal anti-optineurin (Cayman Chemical, Ann Arbor, MI, USA, Cayman Chemical Cat# 100000 RRID:AB_10078198, 1:4000), mouse monoclonal anti-optineurin (Santa Cruz Biotechnology Cat# sc-166576 RRID:AB_2156554, 1:2000), mouse anti-IKK&#947; (NEMO) (BD Biosciences Cat# 611306 RRID:AB_398832, 1:2000), rabbit polyclonal anti-ERK1/2 (Cell Signaling Technology Cat# 9102 RRID:AB_330744, 1:5000), mouse monoclonal anti-phosphorylated ERK1/2 (Cell Signaling Technology Cat# 9106 RRID:AB_331768, 1:2000), rabbit polyclonal anti-I&#954;B&#945; (Cell Signaling Technology Cat# 9242&#160;L RRID:AB_823540, 1:5000), mouse monoclonal anti-phosphorylated-I&#954;B&#945; (Cell Signaling Technology Cat# 9246&#160;L RRID:AB_226714, 1:2000), mouse monoclonal anti-GFP (Roche Molecular Biochemicals, Basel, Switzerland, Roche Cat# 11814460001 RRID:AB_390913, 1:2000), mouse monoclonal anti-GM130 (BD Biosciences Cat# 610822 RRID:AB_398141, 1:4000), and mouse monoclonal anti-giantin (Abcam Cat# ab37266 RRID:AB_880195, 1:2000).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'siRNAs were used at a final concentration of 20 nM. Control non-specific siRNAs and the specific siRNAs were purchased from Sigma-Aldrich. The siRNAs used were: OPTN 1 (5&#8242; GCUUCAAGAGGCACACACAdTdT 3&#8242;), OPTN 2 (GUUUGAGAUGCAAAGCAAAdTdT), mNEMO (GCGAGUUCAACAAGCUGAAdTdT), OPTN A (GAGUCAUGAGAAUGAGAAAdTdT), OPTN B (CCAUGAAAGGGAGAUUUGAdTdT), OPTN C (CCAAAGAAAGAGUUUCAGAdTdT), OPTN D (GAAAGCAUGCUAUCAGAAAdTdT), OPTN E (CCAAGAAUUACUUCGAACAdTdT), and hNEMO (GAGUCGUUGGAGGCUGCCACUAAGdTdT).'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'Results are expressed as 2&#8211;&#916;Cp, where Cp is the cycle threshold number. Dissociation curves were analyzed after each run, for each amplicon, to assess the specificity of quantification with SYBR Green.'},\n",
       " {'pmcid': 'PMC4991008',\n",
       "  'pmid': '27538435',\n",
       "  'snippet': 'cDNA was then amplified by PCR with a LightCycler 480 (Roche Diagnostics, Basel, Switzerland), using SYBR Green (Fast Start DNA Master SYBR- Green I; Roche Applied Science, Roche Diagnostics). Specific primers were used for IL-6, IFNB1, or GAPDH (as the housekeeping gene), under the following conditions: 95&#160;&#176;C for 15&#160;minutes, 40&#160;cycles of 94&#160;&#176;C for 15&#160;s, and at 55&#160;&#176;C for 30&#160;s, then 72&#160;&#176;C for 30&#160;s. Specific primers were designed with the Universal Probe Library System (Roche Applied Science). The sequences of the primers used were as follows: hIL-6 (Forward: GAAAGTGGCTATGCAGTTTGAA, reverse: GAGGTAAGCCTACACTTTCCAAGA), hIFNB1 (CGACACTGTTCGTGTTGTCA, GAAGCACAACAGGAGAGCAA), hGAPDH (AGCCACATC GCTCAGACAC, AATACGACCAAATCCGTTGACT), mIL-6 (GCTACCAAACTGGATATAATCAGGA, CCAGGTAGCTATGGTACTCCAGAA), mIFNB1 (CACAGCCCTCTCCATCAACTA, CATTTCCGAATGTTCGTCCT), and mGAPDH (AGCTTGTCATCAACGGGAAG, TTTGAT GTTAGTGGGGTCTCG).'},\n",
       " {'pmcid': 'PMC4993585',\n",
       "  'pmid': '27472897',\n",
       "  'snippet': 'Transcription activator-like effectors (TALEs) are important effectors of plant pathogenic bacteria of the genus Xanthomonas (Boch et al., 2009). The bacteria inject TALEs via their Type III secretion system (T3SS) into host cells, where they translocate to the nucleus and bind host gene promoters in a sequence-specific manner. The DNA binding domain consists of variable repeats that together account for a predictable DNA recognition code (Boch et al., 2009; Moscou and Bogdanove, 2009).'},\n",
       " {'pmcid': 'PMC4993585',\n",
       "  'pmid': '27472897',\n",
       "  'snippet': 'The rice accession IRBB5 carrying TFIIA&#947;5V39E is indistinguishable from the near-isogenic line IR24 in plant morphology and agronomic performance, including heading date, flag leaf length, number of panicles per plant, panicle length, grains per panicle, 1000-grain weight, yield per plant, seed setting rate, grain length, width, and thickness, with only slightly reduced plant height (Supplementary file 1). Here, we have shown that not only the specific point mutant TFIIA&#947;5V39E has increased Xoo resistance, but also that this can also be achieved by RNAi mediated knockdown of TFIIA&#947;5. In addition, we have shown that TALEs from other Xanthomonas pathogens, such as Xoc, exploit TFIIA&#947;5.'},\n",
       " {'pmcid': 'PMC4993585',\n",
       "  'pmid': '27472897',\n",
       "  'snippet': 'The 2-cm leaf segments next to the bacterial infection sites in the rice plants were collected for RNA isolation. Quantitative reverse transcription-PCR (qRT-PCR) was conducted using gene-specific primers (Supplementary file 3) as described previously (Qiu et al., 2007). The expression level of the rice actin gene was used to normalize the measurement of the expression.'},\n",
       " {'pmcid': 'PMC4993585',\n",
       "  'pmid': '27472897',\n",
       "  'snippet': 'CoIP assays were performed to study the interaction between TALE proteins and plant proteins in planta. The 9&#215;myc DNA fragment was amplified from pN-TAPa vector (Rubio et al., 2005) by using myc-specific primers (Supplementary file 2) and inserted into the SmaI- and BamHI-digested pU1301 vector (Cao et al., 2007), resulting in a vector that we named pU1301-9myc. The DNA fragments of full-length, truncated, or TFB region of TALEs were ligated into the pU1031-9myc vector.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'However, the specific identity and circuitry of neurons responsible for sensing and responding to nutritional cues is not completely understood. This is in part due to the complexity of the vertebrate brain, in which monitoring activity in specific neuronal subtypes is challenging. A less-complex nervous system, consisting of approximately 10,000 neurons (Scott et al., 2001), compared with 70 million neurons in the mouse brain (Economo et al., 2016), makes Drosophila larvae a powerful system to elucidate central brain circuitry underlying systemic responses to nutrient deprivation (Bjordal et al., 2014).'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Nuclei of the solitary tract integrate energy status signals from relevant inputs such as blood-borne endocrine signals and synaptic signals from the gastrointestinal tract and peripheral neurons, to modulate appetite and feeding (Grill and Hayes, 2012). However, the specific identity and circuitry of neurons responsible for sensing and responding to nutritional cues is not completely understood. This is in part due to the complexity of the vertebrate brain, in which monitoring activity in specific neuronal subtypes is challenging.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'IP3R1 is expressed in different classes of neurons where it regulates processes ranging from synaptic plasticity (Nishiyama et al., 2000) to axonal guidance (Xiang et al., 2002). Due to the broad expression of most components of metazoan intracellular signaling, including the IP3R family, identifying cell-specific function in vivo can be challenging. Drosophila genetics provides the tools for such cell-specific analysis.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Due to the broad expression of most components of metazoan intracellular signaling, including the IP3R family, identifying cell-specific function in vivo can be challenging. Drosophila genetics provides the tools for such cell-specific analysis.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Among the tested subsets, the strongest pupariation deficit was observed upon IP3R knockdown in glutamatergic neurons (Figure 2&#8212;figure supplement 1A). Peptidergic neurons also showed a significant pupariation deficit, but this effect was not diet specific (Figure 2&#8212;figure supplement 1A and Table 2). Interestingly, when pan-neuronal knockdown of the IP3R was restricted to the brain lobes by introducing tsh-GAL80, pupariation was restored from less than 25% (Figure 1E) to more than 75% (Figure 2&#8212;figure supplement 1B).'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'The hits are summarized in Table 1&#160;N&#160;&#8805;&#160;3 batches with 25 larvae each.DOI: http://dx.doi.org/10.7554/eLife.17495.01010.7554/eLife.17495.011Figure 3&#8212;figure supplement 2.Cholinergic neurons are important for responding to nutrient stress.A and B Traces represent time series of mean normalized GCaMP6m responses (&#177; SEM) from the glutamatergic cells in the mVG. (A) Stimulation with either saline or carbachol after incubation with a mAChR specific antagonist, atropine. (B) GCaMP6m and RFP channels upon CCh stimulation.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Upon stimulation with CCh, calcium transients were observed in multiple VGN6341 marked cells from larvae fed on a normal diet (Figure 3C,D and Figure 3&#8212;figure supplement 2C). The specificity of this response was ascertained by pre-incubation with atropine, an established and specific antagonist of the mAChR (Fryer and Maclagan, 1984), which abolished the response (Figure 3&#8212;figure supplement 2A and B). Not surprisingly, the response to CCh was significantly attenuated in itpr mutants on a normal diet (Figure 3C,D and Figure 3&#8212;figure supplement 2C).'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'They do not arise from peripheral mVG neurons that appear to project solely to the midline neuropil of the VG. Similar connectivity was observed upon marking the mVG glutamatergic neurons and specifically the mNSCs (Figure 4C).10.7554/eLife.17495.016Figure 4.Glutamatergic neurons in the larval ventral ganglion project to peptidergic neurons in the mNSC.A and B Selected confocal stacks showing the neurites marked by VGN6341-GAL4 driven UAS-eGFP (green) and their merged patterns with dimm-LexA-driven expression of LexAop-mCherry (red). The boxed area in A is shown in B as a high-magnification image.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'In contrast, normal pupariation was observed by restricting knockdown of IP3R to VGN6341-marked glutamatergic neurons of the central brain (Figure 2B). Taken together these data support the innervation of central brain peptidergic neurons, specifically the mNSCs, by glutamatergic neurons of the ventral ganglion. This innervation appears relevant for pupariation on PDD.10.7554/eLife.17495.018Figure 5.Glutamatergic neurons in the larval ventral ganglion convey signals to peptidergic neurons of the mNSC.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Although somatic peptide release is known (De-Miguel and Nicholls, 2015; Trueta and De-Miguel, 2012), we also tested if peptide release was affected more specifically in varicosities of the mNSCs that project to the ring gland. In the itpr mutant, ANF::GFP release was significantly reduced upon stimulation with 50&#160;&#181;m carbachol (Figure 7A,C and E).'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Expression levels of genes encoding enzymes of the ecdysteroid biosynthetic pathway have been studied (McBrayer et al., 2007; Shimada-Niwa and Niwa, 2014), in larvae on a normal diet. Specifically, transcripts of shadow (sad), spookier (spok), phantom (phm), neverland (nvd) and disembodied (dib) are up-regulated dramatically before pupariation (McBrayer et al., 2007; Shimada-Niwa and Niwa, 2014; Warren et al., 2006). It is therefore likely that the neural circuit identified for pupariation on PDD affects ecdysone synthesis by regulating ecdysteroid biosynthesis.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': 'Technical duplicates were performed for each qPCR reaction. A melt analysis was performed at the end of the reaction to ensure the specificity of the product. The fold change of gene expression in any experimental condition relative to wild-type was calculated as 2&#8722;&#9651;&#9651;Ct where &#9651;&#9651;Ct = (Ct (target gene) &#8211;Ct (rp49)) Expt.'},\n",
       " {'pmcid': 'PMC4993588',\n",
       "  'pmid': '27494275',\n",
       "  'snippet': '5&#8242; and 3&#8242; homology arms comprising about 200 bases each of NLS::LexA and the HSP70 terminator were amplified from pBPnlsLexA::p65Uw, a gift of Gerald Rubin (Pfeiffer et al., 2010) and inserted into pSK+-rpsL-kana (Wang et al., 2009) to create a selectable generic landing cassette. Primers carrying gene-specific homology arms to target the cassette to the first coding exon of dimmed were used to PCR-amplify this cassette, and the resulting dimmed-flanked marker cassette was recombined into P[acman] BAC clone CH321-46B06 (Venken et al., 2009) (obtained from Children&#8217;s Hospital Oakland Research Institute, Oakland, CA), replacing the coding portion of the first dimmed coding exon while leaving intact the endogenous 5&#8242; UTR as well as the following introns and exons (although these are presumably no longer transcribed because of the inserted terminator sequences). The landing-site cassette was then replaced via a second recombination with full-length LexA::p65-HSP70, also amplified from pBPnlsLexA::p65Uw.'},\n",
       " {'pmcid': 'PMC4996649',\n",
       "  'pmid': '27474796',\n",
       "  'snippet': 'Specification of pigmented immunocyte precursors begins in cleavage stages when micromeres express Delta, which initiates Notch-mediated specification of a non-skeletogenic mesoderm domain in an adjacent ring of veg2-derived blastomeres (Sherwood and McClay, 1999; Sweet et al., 2002). This domain is first demarcated by expression of the transcription factor glial cells missing (GCM), which is critical to the specification of pigmented immunocytes in the dorsal half of the domain (Ransick and Davidson, 2006, 2012; Ruffins and Ettensohn, 1996). During gastrulation, the pigmented immunocytes transition to mesenchyme and migrate to the ectoderm (Gibson and Burke, 1985, 1987; Takata and Kominami, 2004).'},\n",
       " {'pmcid': 'PMC4996649',\n",
       "  'pmid': '27474796',\n",
       "  'snippet': 'Pigmented immunocytes are a well-studied cell lineage in urchin embryos because of their distinctive reddish granules and their early specification within the nested rings of endomesodermal cells in the vegetal plate. Specification of pigmented immunocyte precursors begins in cleavage stages when micromeres express Delta, which initiates Notch-mediated specification of a non-skeletogenic mesoderm domain in an adjacent ring of veg2-derived blastomeres (Sherwood and McClay, 1999; Sweet et al., 2002).'},\n",
       " {'pmcid': 'PMC4996649',\n",
       "  'pmid': '27474796',\n",
       "  'snippet': 'Pigmented immunocytes are a well-studied cell lineage in urchin embryos because of their distinctive reddish granules and their early specification within the nested rings of endomesodermal cells in the vegetal plate. Specification of pigmented immunocyte precursors begins in cleavage stages when micromeres express Delta, which initiates Notch-mediated specification of a non-skeletogenic mesoderm domain in an adjacent ring of veg2-derived blastomeres (Sherwood and McClay, 1999; Sweet et al., 2002). This domain is first demarcated by expression of the transcription factor glial cells missing (GCM), which is critical to the specification of pigmented immunocytes in the dorsal half of the domain (Ransick and Davidson, 2006, 2012; Ruffins and Ettensohn, 1996).'},\n",
       " {'pmcid': 'PMC4996649',\n",
       "  'pmid': '27474796',\n",
       "  'snippet': 'In hatched blastulae, cells in the vegetal plate that are immunoreactive with the immunocyte-specific antibody Sp1 (Gibson and Burke, 1985), are also immunoreactive with anti-Sp-Eph (Krupke and Burke, 2014) (Figure 3A&#8211;C). Sp-Efn is present at the same time on the basal surfaces of ectodermal cells (Figure 3D).'},\n",
       " {'pmcid': 'PMC4996649',\n",
       "  'pmid': '27474796',\n",
       "  'snippet': 'Morpholino antisense oligonucleotides were obtained from GeneTools. To control for morpholino specificity, we used a control, irrelevant morpholino, which would have no effect on phenotype, as well as two experimental morpholinos against different target sequences that produced the same aberrant phenotype for each gene. Antibodies were used to confirm loss of immunoreactivity and in the case of the Sp-Efn MO rescue experiments are reported in which eggs were injected with an Sp-Efn morpholino and one cell of a 2-cell embryo injected with Sp-Efn RNA.'},\n",
       " {'pmcid': 'PMC4996649',\n",
       "  'pmid': '27474796',\n",
       "  'snippet': 'Antibodies employed have all been described previously, or are commercially developed. Data on antigens and specificity are available in the cited references.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'Bearing in mind these observations, we have designed and generated two active cell-penetrating peptides, based on protein-protein interaction sequences found in Ras-ERK components, able to attenuate the activation of this signalling cascade in vivo. The first peptide, hereafter referred to as RB1 (Figure 1A), was designed around the 59&#8211;73 portion, the KIM sequence, of the ERK1/2 specific phosphatase MKP3 (Liu et al., 2006). The KIM sequence is responsible for the interaction with the common docking (CD) domain of ERK1/2.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'The KIM sequence is responsible for the interaction with the common docking (CD) domain of ERK1/2. The second peptide, hereafter referred to as RB3 (Figure 1B), was designed with the support of molecular graphics tools, taking advantage of the published crystal structure of a ternary Ras:SOS:Ras*GDP complex (Sondermann et al., 2004), compared to the crystal structure of the CDC25 domain of Ras-GRF1, a neuronal specific Ras guanine nucleotide exchange factor (Ras-GEF). The peptide sequence (1173&#8211;1203 portion of the CDC25 domain) includes an &#945;-helix crucial for the GTP exchange activity on RAS proteins.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'Doses are reported in a&#160;logarithmic scale (Log10) on the X-axis. The IC50&#160;was calculated for each specific response using GraphPad Prism software.DOI: http://dx.doi.org/10.7554/eLife.17111.00310.7554/eLife.17111.004Figure 2&#8212;figure supplement 1.RB1 and RB3 show a dose-dependent inhibition of ERK activation and its substrates H3 and S6.The highest doses of RB1 and RB3 prevented glutamate-induced ERK activation (A,B, scale bar 100&#160;&#181;m), H3 activation (C,D scale bar 30&#160;&#181;m) and S6 activation (E,F scale bar 30&#160;&#181;m), whereas the lowest doses of RB1 and RB3 failed to inhibit ERK, H3 and S6.DOI: http://dx.doi.org/10.7554/eLife.17111.004'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'Doses are reported in a&#160;logarithmic scale (Log10) on the X-axis. The IC50&#160;was calculated for each specific response using GraphPad Prism software.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'We then investigated whether RB1 and RB3 may also affect the phosphorylation of (Ser10)-acetylated (Lys14) histone H3 (pAc-H3) and S6 ribosomal protein, (pS6, Ser235/236 specific site), two well-characterised ERK substrates (Santini et al., 2007, 2009; Darmopil et al., 2009; Orellana et al., 2012). RB1 was effective in reducing pAc-H3 with an IC50 of 1.12&#160;&#181;M (Figure 2C, representative pictures in Figure 2&#8212;figure supplement 1C).'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'Doses are reported in logarithmic scale (Log10) on the X-axis. The IC50was calculated for the specific response using GraphPad Prism software. As shown in the graph, PD325901 is effective in reducing glutamate-induced ERK activation with an IC50 of 1.15 nM.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'Doses are reported in logarithmic scale (Log10) on the X-axis. The IC50was calculated for the specific response using GraphPad Prism software. As shown in the graph, PD325901 is effective in reducing glutamate-induced ERK activation with an IC50 of 1.15 nM.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'The potent inhibition of ERK phosphorylation resulting from PD325901 prompted us to test its potential benefits in the treatment of cocaine-mediated processes in the CPP paradigm. In this classical Pavlovian conditioning procedure, the drug serves as an unconditioned stimulus (US) that is repeatedly paired with a distinct environment (conditioned stimulus, CS) within the testing chamber and the animal learns to associate this specific context with the rewarding effects of the drug (Tzschentke, 2007).'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'Furthermore, we investigated whether the inhibition of retrieval of cocaine-related memories in CPP was specifically due to MEK inhibition in the brain. Thus, we selected one compound from our previous experiment (see Figure 6), Trametinib, as a negative control for&#160;use in cocaine CPP.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'On day 2, PD325901 treatment altered the memory performance (Two-way ANOVA, treatment effect: F2,32 = 9.536, p&lt;0.001). Specifically, only 25 mg/kg of PD325901 significantly compromised memory retrieval when compared with VEH (Bonferroni&#8217;s post-hoc, PD325901 10 mg vs VEH: p=1.000; PD325901 25 mg vs VEH: p&lt;0.001) and a dose-dependent effect was found for PD325901 (Bonferroni&#8217;s post-hoc, PD325901 25 mg vs PD325901 10 mg: p&lt;0.01). Mice treated with 10 mg/kg of PD325901 and vehicle treated mice both showed significantly increased latency 24 hr after training (Paired-samples t-test, PD325901 10 mg: t10 = &#8722;7.493, p&lt;0.0001; VEH: t13 = &#8722;4.804, p&lt;0.0001), while mice injected with 25 mg/kg of PD325901 did not (Paired-samples t-test, PD325901 25 mg: t9 = &#8722;1.418, p=0.190), demonstrating a significantly impaired memory.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'On day 2, PD325901 treatment altered the memory performance (Two-way ANOVA, treatment effect: F2,32 = 9.536, p&lt;0.001). Specifically, only 25 mg/kg of PD325901 significantly compromised memory retrieval when compared with VEH (Bonferroni&#8217;s post-hoc, PD325901 10 mg vs VEH: p=1.000; PD325901 25 mg vs VEH: p&lt;0.001) and a dose-dependent effect was found for PD325901 (Bonferroni&#8217;s post-hoc, PD325901 25 mg vs PD325901 10 mg: p&lt;0.01). Mice treated with 10 mg/kg of PD325901 and vehicle treated mice both showed significantly increased latency 24 hr after training (Paired-samples t-test, PD325901 10 mg: t10 = &#8722;7.493, p&lt;0.0001; VEH: t13 = &#8722;4.804, p&lt;0.0001), while mice injected with 25 mg/kg of PD325901 did not (Paired-samples t-test, PD325901 25 mg: t9 = &#8722;1.418, p=0.190), demonstrating a significantly impaired memory.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'RB1 was designed around the 59&#8211;73 portion, the KIM sequence (Liu et al., 2006), of the ERK1/2 specific phosphatase MKP3. RB3 was designed, using the MOE software package, (Molecular Operating Environment, version 10.10, Chemical Computing group, Montreal Canada, http://www.chemcomp.com) by aligning and superposing the CDC25 domain of Ras-GRF1 (Freedman et al., 2006) the published crystal structure of a ternary Ras:SOS:Ras*GDP complex (Sondermann et al., 2004) using the default settings.'},\n",
       " {'pmcid': 'PMC4996650',\n",
       "  'pmid': '27557444',\n",
       "  'snippet': 'The IC50&#160;was calculated for the specific response using GraphPad Prism software.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'The activity of cofilin is negatively regulated by phosphorylation. Specifically, phosphorylation of serine 3 of cofilin suppresses its depolymerizing and F-actin severing activity (Bamburg, 1999). Importantly, increased cofilin activity can lead to the depolymerization and severing of F-actin, which in turn results in the shrinkage and loss of spines (Rust, 2015; Zhou et al., 2004; Davis et al., 2011; Shankar et al., 2007).'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Subtype-specific apical/basal spine analyses of the Golgi impregnated neurons revealed a significant decrease for all spine subtypes in sleep-deprived mice (Figure 1F, for all spine types, Student&#8217;s t-tests p&lt;0.005, for separate apical and basal spine analyses see Supplementary Figure 1C,D). Sleep deprivation causes the greatest reduction in apical/basal spine density between 60 &#181;m and 150 &#181;m from the soma (Figure 1G, for separate apical and basal spine analyses see Figure 1&#8212;figure supplement 1E,F).'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Indeed, 5h of sleep deprivation reduced cofilin Ser-3 phosphorylation, suggesting an increase in cofilin activity in the hippocampus (NSD: 100.0 &#177; 6.9%; SD: 67.7 &#177; 9.2%; Student t-test p=0.0090; Figure 3A). A similar effect was not evident in the prefrontal cortex (NSD, n&#160;=&#160;5: 100.0 &#177; 1.84%; SD, n&#160;=&#160;5: 101.92 &#177; 2.41%; Student t-test p=0.54; Figure 3&#8212;figure supplement 1), indicating sleep deprivation affects cofilin phosphorylation in a brain region-specific fashion.10.7554/eLife.13424.008Figure 3.Increased cofilin activity in the hippocampus mediates the spine loss associated with sleep deprivation. (A) Five hours of sleep deprivation leads to a reduction in cofilin phosphorylation at serine 3 in the hippocampus.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Moreover, using a behaviorally na&#239;ve cohort of mice, we found that cofilinS3D expression did not alter short-term object-place memory in the same task (Figure 4&#8212;figure supplement 1D). Together, these findings demonstrate that cofilinS3D expression specifically prevents the cognitive deficits caused by sleep deprivation. Although we can not rule out the possibility of off-target effects of the cofilinS3D mutant, we think that these are unlikely as expression of this mutant form of cofilin reversed the effects of sleep deprivation, restoring spine loss and memory to non-sleep deprived levels while not having an effect in non-sleep deprived mice.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'In contrast, spaced 4-train LTP was unaffected by sleep deprivation in hippocampal slices from mice expressing the inactive cofilinS3D (Figure 4C). The expression of cofilinS3D or sleep deprivation did not alter basal synaptic properties or paired-pulse facilitation (Figure 4&#8212;figure supplement 1E-H) suggesting that the spine loss caused by sleep deprivation specifically impairs long-lasting forms of synaptic plasticity.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'To test this hypothesis, we engineered a catalytically inactive form of PDE4A5 (referred to as PDE4A5catnull) in which an aspartate group located deep within the cAMP binding pocket of PDE4A5 (PDE4A5D577A), that is critical for catalytic activity, is replaced with an alanine group (Baillie et al., 2003; McCahill et al., 2005). Expression of PDE4A5catnull outcompetes the low levels of active, endogenous PDE4A5 from PDE4A5-containing signalosome complexes that specifically sequester it (Houslay, 2010), thereby preventing the breakdown of cAMP in the vicinity of those complexes. We used the viral approach (Havekes et al., 2014) to express PDE4A5catnull selectively in hippocampal neurons (Figure 5A,B).'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': '(G) PDE4A5catnull&#916;4 protein levels in the hippocampus 4 weeks after viral injection. A sample blot probed with an isoform-nonspecific PDE4A antibody revealed the presence of both wild-type PDE4A5 protein and truncated PDE4A5catnull&#916;4 protein. Probing the blot with an antibody for the HA-tag confirmed that the truncated protein is indeed the N-terminal lacking catalytically inactive PDE4A5catnull&#916;4.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': '(G) PDE4A5catnull&#916;4 protein levels in the hippocampus 4 weeks after viral injection. A sample blot probed with an isoform-nonspecific PDE4A antibody revealed the presence of both wild-type PDE4A5 protein and truncated PDE4A5catnull&#916;4 protein. Probing the blot with an antibody for the HA-tag confirmed that the truncated protein is indeed the N-terminal lacking catalytically inactive PDE4A5catnull&#916;4.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Although the catalytic unit of the 25 distinct PDE4 isoforms is highly conserved, each has a unique N-terminal localization sequence that directs isoform targeting to a specific and unique set of protein complexes (signalosomes) (Houslay, 2010). This allows for a highly orchestrated sequestering of cAMP signaling in specific intracellular domains rather than a general, global degradation of cAMP throughout the cell (Houslay, 2010). We therefore aimed to determine whether the rescue of memory impairments by expression of PDE4A5catnull requires the unique N-terminal domain of PDE4A5.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Although the catalytic unit of the 25 distinct PDE4 isoforms is highly conserved, each has a unique N-terminal localization sequence that directs isoform targeting to a specific and unique set of protein complexes (signalosomes) (Houslay, 2010). This allows for a highly orchestrated sequestering of cAMP signaling in specific intracellular domains rather than a general, global degradation of cAMP throughout the cell (Houslay, 2010).'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'The inability of PDE4A5catnull&#916;4 to prevent the memory deficit caused by sleep deprivation was not a consequence of altered exploration levels during training (Figure 5&#8212;figure supplement 1H). This finding indicates that the memory rescue in the previous experiment was a result of the full length PDE4A5catnull being sequestered to specific signalosomes through the isoform-unique N-terminal region rather than a consequence of PDE4A5catnull being unable totarget the functionally relevant complexes sequestering full length PDE4A5. It also indicates that displacing sequestered, active endogenous PDE4A5 in hippocampal excitatory neurons is sufficient to prevent memory deficits induced by 5&#8201;hr of sleep deprivation.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Consistent with the&#160;notion that a key functional role of the isoform-unique N-terminal region of PDE4 isoforms is the targeting to signalosomes so as to exert functionally distinct actions (Houslay, 2010), the hippocampal expression of a catalytically in active version of PDE4A5 lacking the isoform unique N-terminal domain fails to rescue the cognitive deficits associated with sleep loss. The latter observation suggests that the isoform-specific N-terminal domain of PDE4A5 targets this specific PDE isoform to signalosomes that degrade cAMP in the vicinity of complexes that are particularly sensitive to sleep deprivation such as the complexes that contain LIMK and cofilin. Consistent with this, no such dominant negative phenotype is evident in a catalytically inactive PDE4A construct engineered to lack such an N-terminal targeting region.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Our work reveals that PDE4A5 is a critical mediator of the impact of sleep deprivation on memory consolidation. Indeed, one reason why hippocampal area CA1 is specifically vulnerable to sleep deprivation may be the high level of PDE4A5 expression in this region (McPhee et al., 2001). Specific PDE4 isoforms are sequestered by distinct signalosome complexes that regulate localized cAMP signaling and impart functionally distinct roles (Houslay, 2010).'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Indeed, one reason why hippocampal area CA1 is specifically vulnerable to sleep deprivation may be the high level of PDE4A5 expression in this region (McPhee et al., 2001). Specific PDE4 isoforms are sequestered by distinct signalosome complexes that regulate localized cAMP signaling and impart functionally distinct roles (Houslay, 2010). Impairing the function of PDE4A5 signalosomes through expression of a full length catalytically inactive form of PDE4A5 exerts a dominant negative action, phenotypically identified here as preventing the alterations in LIMK and cofilin signaling caused by sleep deprivation.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'Purified plasmid DNA was produced using Qiagen QIAprep kits and stored at 4&#176;C. The pAAV9-CaMKII&#945;0.4-PDE4A5catnull-VSV, pAAV9-CaMKII&#945;0.4-PDE4A5catnull&#916;4-HA, pAAV9-CaMKII&#945;0.4-CofilinS3D-HA, pAAV9-CaMKII&#945;0.4-CofilinS3A-HA and pAAV9-CaMKII&#945;0.4-eGFP were constructed by standard methods and packaged by the University of Pennsylvania viral core. Transduced cofilinS3D may compete with endogenous cofilin for binding to the cofilin-specific phosphatase slingshot, thereby leading to inactivation of the endogenous protein (Sarmiere and Bamburg, 2004; Konakahara et al., 2004). Titers ranged from 2.4 &#215; 1012 to 4.91 &#215; 1013 genome copy numbers.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'The cAMP-specific PDE activity assays and western blots to assess sleep deprivation-induced changes in PDE4A5 levels were conducted as described (Vecsey et al., 2009). Hippocampal tissue was lysed using a tissue ruptor (Qiagen,&#160;Germany) in lysis buffer (Tris 50&#8201;mM, pH: 9, sodium deoxycholate 1%, Sodium fluoride 50&#8201;mM, activated sodium vanadate 20 &#181;M, EDTA 20 &#181;M, and beta-glycerophosphate 40 &#181;M. Additional phosphatase inhibitor cocktail (Thermo scientific) and protease inhibitors (Roche,&#160;Switzerland) were added to the freshly prepared buffer just prior to tissue lysis.'},\n",
       " {'pmcid': 'PMC4996653',\n",
       "  'pmid': '27549340',\n",
       "  'snippet': 'On average 155 spines were analyzed per an animal. Importantly, for the comparison of the spine numbers using golgi and diolistic staining methods we focused specifically on the second and third branch of the apical dendrites as the diolistic staining technique is suboptimal to label and analyze branches farther away from the soma.'},\n",
       " {'pmcid': 'PMC4999312',\n",
       "  'pmid': '27504807',\n",
       "  'snippet': 'The images are representative of 7~8 mice analyzed per group. (B&#8211;E)&#160;Taqman gene expression analysis quantitating the mRNA abundance of the endothelial cell specific markers Cdh5 (B) and Pecam1 (C), the hypoxia responsive gene Slc2a1 (Glut1) (D), and the macrophage marker Emr1 (F4/80) (E) on day 4 post FAL in the adductor and calf muscles (mean &#177; SEM; n&#160;=&#160;7~8 mice per group). Source data are included as Figure 1&#8212;figure supplement 8&#8212;source data 1.DOI: http://dx.doi.org/10.7554/eLife.18414.01910.7554/eLife.18414.020Figure 1&#8212;figure supplement 8&#8212;source data 1.Source data for Figure 1&#8212;figure supplement 8.This file contains raw source data used to make the graphs presented in Figure 1&#8212;figure supplement 8.DOI: http://dx.doi.org/10.7554/eLife.18414.020'},\n",
       " {'pmcid': 'PMC4999312',\n",
       "  'pmid': '27504807',\n",
       "  'snippet': 'The images are representative of 7~8 mice analyzed per group. (B&#8211;E)&#160;Taqman gene expression analysis quantitating the mRNA abundance of the endothelial cell specific markers Cdh5 (B) and Pecam1 (C), the hypoxia responsive gene Slc2a1 (Glut1) (D), and the macrophage marker Emr1 (F4/80) (E) on day 4 post FAL in the adductor and calf muscles (mean &#177; SEM; n&#160;=&#160;7~8 mice per group). Source data are included as Figure 1&#8212;figure supplement 8&#8212;source data 1.'},\n",
       " {'pmcid': 'PMC4999312',\n",
       "  'pmid': '27504807',\n",
       "  'snippet': 'These genes included components of the Notch signaling pathway; for example, Dll4, Hey1, Hes1, and Lfng (Figure 7A,B). This may be significant because the Notch pathway plays a major role during developmental angiogenesis, including tip/stalk cell specification and endothelial cell sprouting (Roca and Adams, 2007; Phng and Gerhardt, 2009). Indeed, the hypersprouting defects observed in E2KO, E3KO and Map3k10-/- Map3k11-/- mice resemble those previously reported for mice with reduced Dll4/Notch signaling (Hellstr&#246;m et al., 2007; Suchting et al., 2007; Benedito et al., 2009), including Lfng-/- mice (Benedito et al., 2009).'},\n",
       " {'pmcid': 'PMC4999312',\n",
       "  'pmid': '27504807',\n",
       "  'snippet': 'These interactions between VEGF and Notch signaling underly the specification of endothelial cell phenotypes during angiogenesis, including highly motile tip cells that extend numerous filopodia and trailing stalk cells with low motility that form the lumen of nascent tubules (Roca and Adams, 2007; Phng and Gerhardt, 2009; Eilken and Adams, 2010; Jakobsson et al., 2010). The proper specification and interplay of tip and stalk cells is essential for the orchestration of sprouting angiogenesis that mediates expansion of vascular networks. Moreover, VEGF and Notch signaling are also critical for formation of the native collateral circulation (Cristofaro et al., 2013).'},\n",
       " {'pmcid': 'PMC4999312',\n",
       "  'pmid': '27504807',\n",
       "  'snippet': 'VEGF signaling promotes endothelial cell survival, proliferation and motility while Dll4&#8211;Notch signaling suppresses VEGF signaling, in part, by regulating VEGFR expression. These interactions between VEGF and Notch signaling underly the specification of endothelial cell phenotypes during angiogenesis, including highly motile tip cells that extend numerous filopodia and trailing stalk cells with low motility that form the lumen of nascent tubules (Roca and Adams, 2007; Phng and Gerhardt, 2009; Eilken and Adams, 2010; Jakobsson et al., 2010). The proper specification and interplay of tip and stalk cells is essential for the orchestration of sprouting angiogenesis that mediates expansion of vascular networks.'},\n",
       " {'pmcid': 'PMC4999312',\n",
       "  'pmid': '27504807',\n",
       "  'snippet': 'All mice used in this study were backcrossed (&#8805; ten generations) to the C57BL/6J strain. The mice were housed in a specific pathogen-free facility accredited by the American Association for Laboratory Animal Care. The animal studies were approved by the Institutional Animal Care and Use Committees of the University of Massachusetts Medical School, Tufts University School of Medicine, and Brigham and Women&#8217;s Hospital.'},\n",
       " {'pmcid': 'PMC5000811',\n",
       "  'pmid': '27570829',\n",
       "  'snippet': 'Individuals living in nontropical climates show seasonal changes in physiology and behavior primarily controlled by day length (photoperiod). Specifically, white-footed mice (Peromyscus leucopus) display reduced hippocampal function, neurogenesis, and cognitive capacity in response to short days. It is unknown, however, whether these changes are preceded or accompanied by alterations in cerebral blood flow.'},\n",
       " {'pmcid': 'PMC5000811',\n",
       "  'pmid': '27570829',\n",
       "  'snippet': 'Upon sedation, mice were transcardially perfused with 250 &#956;l of FITC-tagged tomato (Lycopersicon esculentum) lectin over the course of 1 min (0.5 mg/ml; Vector Laboratories). This lectin binds to complex-type N-glycans glycoproteins found on the luminal surface of capillary endothelial cells, enabling morphological visualization and quantification specifically of capillary bed features (Robertson et al., 2015). This technique specifically allows for the visualization of patent (open) vessels and leaves inactive vessels unstained (Inai et al., 2004).'},\n",
       " {'pmcid': 'PMC5000811',\n",
       "  'pmid': '27570829',\n",
       "  'snippet': 'This lectin binds to complex-type N-glycans glycoproteins found on the luminal surface of capillary endothelial cells, enabling morphological visualization and quantification specifically of capillary bed features (Robertson et al., 2015). This technique specifically allows for the visualization of patent (open) vessels and leaves inactive vessels unstained (Inai et al., 2004). Importantly, cerebral capillary density is recognized as a key variable in the study of brain angioplasticity in response to metabolic acclimatization (Boero et al., 1999; Benderro et al., 2012; Benderro and LaManna, 2014).'},\n",
       " {'pmcid': 'PMC5000811',\n",
       "  'pmid': '27570829',\n",
       "  'snippet': 'Our data suggest that this phenomenon is accompanied by capillary remodeling in the hippocampus and cortex; thus, short-day reductions in blood flow may precede declines in brain function. It is important to note that due to limitations in the penetration depth of the laser speckle instrument, we were unable to reliably measure hippocampal-specific blood flow using this method. However, changes in blood flow within the hippocampus are evident as FITC-lectin binds only active capillary beds when administered via transcardial perfusion, leaving inactive blood vessels unstained.'},\n",
       " {'pmcid': 'PMC5000811',\n",
       "  'pmid': '27570829',\n",
       "  'snippet': 'It also plays a large role in endometrial remodeling during menses and the regulation of vascularization in response to immune activation (Freitas et al., 1999; Parks et al., 2004). MMP2 is specifically implicated in seasonal changes in reproductive function in Siberian hamsters (Phodopus sungorus), another seasonally breeding small rodent (Salverson et al., 2008; Shahed et al., 2015a,b). In this species, MMP2 facilitates photostimulated ovarian recrudescence after extended short-day exposure.'},\n",
       " {'pmcid': 'PMC5004264',\n",
       "  'pmid': '27576487',\n",
       "  'snippet': 'The known structures of delta-endotoxins from B. thuringiensis (Cyt family; 3-domain Cry proteins; the aegerolysin-like Cry34 and Cry37; the Toxin_10 family Cry35; and the Etx/Mtx2 family Cry45, Cry46, and Cry51) are structurally distinct from the Cry6 fold (Fig.&#160;7), which represents a new structural family among insecticidal toxins and a new paradigm for invertebrate pathogenic toxins. Based on the present study, we can now further elucidate key features of the Cry6Aa toxin that are responsible for its specificity and mode of action.Fig. 7Structural families of B. thuringiensis delta-endotoxins.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Because stereocilia were present at extremely low concentrations in the extract, however, there was a large excess of antibody-coupled beads over stereocilia membranes. As a consequence, substantial amounts of non-stereocilia protein bound nonspecifically to the beads. While phalloidin staining showed that many of the D10 beads were decorated with stereocilia (Figure 1B), nuclei and other contaminants were also associated with the beads, presumably nonspecifically.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'As a consequence, substantial amounts of non-stereocilia protein bound nonspecifically to the beads. While phalloidin staining showed that many of the D10 beads were decorated with stereocilia (Figure 1B), nuclei and other contaminants were also associated with the beads, presumably nonspecifically. Membranes were eluted using sonication (fraction S4); they were then washed several times prior to detergent extraction, yielding enriched stereocilia membranes (fraction P6).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Other proteins were known stereocilia proteins (ESPN, ANKRD24, MYO1H). Although ATP2B2 is part of a protein group also containing ATP2B1 and ATP2B4, which are not located in stereocilia (Dumont et al., 2001), we estimated ATP2B2&#8217;s contribution to the ATP2B1/2/4 group riBAQ value for each fraction; we divided the intensities from peptides only found in each isoform by the sum of all isoform-specific peptides, generating the fractional contribution for each isoform. After making this adjustment, ATP2B2 was also one of the most enriched proteins (Figure 2A).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Each aliquot of beads was washed thoroughly, then eluted with SDS; eluted proteins were then analyzed by shotgun and targeted mass spectrometry. Passing the extracts over control beads allowed us to identify proteins that nonspecifically bound to the immunoaffinity reagents, many of which were abundant; these proteins were then computationally subtracted from the proteins identified in the 138-1 eluates. We carried out three independent immunoprecipitation experiments, each with ~500 ear-equivalents; one to three technical replicates from each eluate were subjected to shotgun mass spectrometry, and the results from the six shotgun runs were analyzed together.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Because MYO7A and its interacting proteins should be highly enriched by the 138-1 antibody from the starting material, we analyzed immunoaffinity purification experiments using shotgun mass-spectrometry, comparing the relative abundance of each protein in the 138-1 eluate and in the stereocilia RIPA extract. This ratio&#8212;the immunoaffinity enrichment&#8212;will be high for MYO7A itself and for proteins that specifically coprecipitated with MYO7A. Figure 3B shows all proteins detected in the experiment, with those detected only in the 138-1 immunoprecipitate or in S7 segregated from those detected in both.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Containing five ankyrin domains, ANKRD24 is enriched in chick and mouse hair bundles (Shin et al., 2013; Wilmarth et al., 2015; Krey et al., 2015); little else is known about ANKRD24, although it is a hair-cell-specific protein in the chick ear (Ku et al., 2014). By shotgun mass spectrometry analysis, most of the ANKRD24 in the S7 extract was present in MYO7A complexes (Figure 3E).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Although a smaller fraction of MYO6 in the S7 extract was precipitated as compared to MYO7A (right panels), MYO6 was &gt;10-fold more abundant in the extract than MYO7A, suggesting that MYO6 and MYO7A could be present in a nearly equimolar complex. GIPC3 and ANKRD24 were also specifically precipitated using the 138-1 antibody (Figure 4F,G).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'To provide stronger evidence that MYO7A forms a protein complex with PDZD7 and ADGRV1, we used targeted proteomics to determine how much of each protein was precipitated by 138-1 anti-MYO7A or control immunoaffinity beads. PDZD7 was present at significantly greater levels in 138-1 precipitations than in control precipitations (Figure 4H), confirming its specific precipitation with MYO7A. By contrast, only one peptide for ADGRV1 showed significant enrichment in 138-1 immunoprecipitates compared to controls, and the fraction precipitated was very small (Figure 4I).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'The MYO7A-PDZD7 stoichiometry was high, as the majority of the PDZD7 was immunoprecipitated. ANKRD24 also specifically co-immunoprecipitated with MYO7A (Figure 5B), although a much lower fraction of the total ANKRD24 was recovered. By contrast, PDZD7 tagged with Strep-tag II did not co-immunoprecipitate with ANKRD24-GFP, supporting the specificity of the interactions of PDZD7 and ANKRD24 with MYO7A (Figure 5C).10.7554/eLife.18312.016Figure 5.Interaction of mouse MYO7A and partners in HEK293 cells.PDZD7, ANKRD24, and GFP-MYO7A proteins are mouse; HA-MYO7A is human.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'To confirm that PDZD7 interacts directly with MYO7A and extend the results to mammals, we co-expressed mouse or human proteins in HEK293 cells and immunoprecipitated GFP-MYO7A with the 138-1 monoclonal antibody. When detected using an anti-PDZD7 antibody, PDZD7 fused to mCherry was specifically immunoprecipitated by 138-1 along with GFP-MYO7A (Figure 5A). The MYO7A-PDZD7 stoichiometry was high, as the majority of the PDZD7 was immunoprecipitated.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'ANKRD24 also specifically co-immunoprecipitated with MYO7A (Figure 5B), although a much lower fraction of the total ANKRD24 was recovered. By contrast, PDZD7 tagged with Strep-tag II did not co-immunoprecipitate with ANKRD24-GFP, supporting the specificity of the interactions of PDZD7 and ANKRD24 with MYO7A (Figure 5C).10.7554/eLife.18312.016Figure 5.Interaction of mouse MYO7A and partners in HEK293 cells.PDZD7, ANKRD24, and GFP-MYO7A proteins are mouse; HA-MYO7A is human. (A&#8211;C) Immunoprecipitation and protein immunoblotting.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': \"To visualize the PDZD7-MYO7A interaction in tissue-culture cells, we used the nanoscale pull-down method (Bird et al., 2016), which relies on a fusion between a binding partner and a MYO10 'heavy meromyosin-like' construct, which contains the motor, light-chain-binding region, and single-alpha-helix domain. This fusion protein&#8212;and any co-associating proteins&#8212;specifically concentrate at the tips of COS7 filopodia, where they are easily visualized (Berg and Cheney, 2002; Kerber and Cheney, 2011). The human MYO7A tail (residues 967&#8211;2175, including the two tandem MyTH4-FERM domains and the SH3 domain) was fused to a construct containing GFP and bovine MYO10 heavy meromyosin (MYO10HMM; motor and light-chain binding domains).\"},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Cells with the highest levels of mCherry signal had very disorganized hair bundles (Figure 8&#8212;figure supplement 1), a phenotype we rarely see in cells injectoporated with control plasmids (Xiong et al., 2012; Zhao et al., 2014; Xiong et al., 2014). By contrast, cells with the lowest levels of signal had mCherry-PDZD7 located specifically in hair bundles, largely at stereocilia bases (Figure 8H&#8211;I). We noted that many cells with intermediate levels of mCherry-PDZD7 had some signal located specifically at stereocilia tips (Figure 8&#8212;figure supplement 1).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'By contrast, cells with the lowest levels of signal had mCherry-PDZD7 located specifically in hair bundles, largely at stereocilia bases (Figure 8H&#8211;I). We noted that many cells with intermediate levels of mCherry-PDZD7 had some signal located specifically at stereocilia tips (Figure 8&#8212;figure supplement 1).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'By scaling up to hundreds to thousands of chicken ears per preparation and by exploiting monoclonal antibodies specific for stereocilia proteins, we were able to use biochemical methods to characterize protein complexes from stereocilia membranes of hair cells. We demonstrated the utility of the D10 enrichment method by examining protein complexes that include MYO7A.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Finally, MYO7A and MYO6 may be in a multimolecular, bidirectional motor complex; like with other bidirectional motor complexes (Hancock, 2014), reciprocal regulation of the two motors could steer the complex to the top or bottom of the stereocilia. Indeed, MYO6 was the most abundant protein co-precipitated with MYO7A, and we are presently investigating the specificity of this interaction. Regardless of the mechanism, the data here show that MYO7A binds PDZD7 and is necessary for at least one step in its localization at the base of the hair bundle.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Several other proteins are specifically co-precipitated with MYO7A; those with high enrichment and relatively high stoichiometry include MYO6, MYL4, COLEC12, ANKRD24, GIPC3, and MYL1. Although each of these will require in-depth characterization such as we have done here for PDZD7, we can speculate to their possible roles in hair cells.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'All of these proteins interact with the actin cytoskeleton, and it is likely that some of them co-precipitate because of their association with substoichiometric levels of actin filaments also bound to MYO7A. Future experiments will be needed to discern which of these proteins interacts specifically with MYO7A complexes and which co-precipitate on associated actin.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'The presence of MYO6 is quite interesting, as it could mediate pointed-end-directed motility in a complex that also contains a barbed-end-direct motor. More specific investigation of how MYO6 and MYO7A interact will be required; our preliminary co-expression experiments indicated only a low-stoichiometry interaction when both were expressed together in HEK cells (Figure 5&#8212;source data 1). A relatively large fraction of ANKRD24 present in the stereocilia-membrane extract was co-precipitated, yet ANKRD24 did not appear to bind directly to MYO7A (Figures 5,6).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'To accurately measure the stoichiometry of the detected complexes, additional purification will be required. We can identify co-precipitating proteins using enrichment analysis, but despite a nearly 10,000-fold purification of MYO7A, the immunoaffinity eluates have many proteins that have bound nonspecifically. This result is entirely expected; the proteins that interact with MYO7A are rare within stereocilia, so MYO7A protein complexes are present at extremely small levels in the stereocilia detergent extract.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'In experiments that used targeted proteomics to determine whether a protein was specifically immunoprecipitated by the 138-1 antibody as compare to a control antibody, we used the two-tailed Student&#8217;s t-test to compare a peptide&#8217;s signal in the 138-1 eluates with that in the control eluates. In each case, 4&#8211;6 replicates from 138-1 and control eluates were compared.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Immunoaffinity purification was carried out serially; the RIPA extract was first incubated with beads with control antibody, then the unbound material was then incubated with beads coupled with the specific antibody (e.g., 138-1 anti-MYO7A). The RIPA extract (1.5&#8201;ml; 500 ear-equivalents) or flow-through material was added to 50 &#181;l antibody-coupled beads; the beads and extract were rotated for 1&#8201;hr at room temperature.'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Proteins were transferred to PVDF (Immobilon-P; EMD Millipore, Billerica MA), and were visualized with India Ink (1:5000) in PBS/0.05% Tween-20. Membranes were blocked with Prime Blocking Agent (GE Healthcare Life Sciences, Pittsburgh PA; #RPN418), and probed with specific primary antibodies which were detected with species-specific HRP-coupled secondary antibodies (Jackson ImmunoResearch #111-035-144 and #115-035-003) and ECL Prime (GE Healthcare Life Sciences #RPN2232).'},\n",
       " {'pmcid': 'PMC5005036',\n",
       "  'pmid': '27525485',\n",
       "  'snippet': 'Fragment ion peaks for each peptide were chosen according to the following criteria: 1) three or more co-eluting fragment ions contributed to the peak signal, 2) at least two or more data points were collected across the peak, and 3) one or more spectra within the peak were matched to correct peptide sequence within the spectral database. If spectra within a specific sample were not identified then a) the retention time of the chosen peak must be within 2 min of the retention time of an identified peak for that peptide from another sample and b) the type of daughter ions contributing to the peak must match the identified peptide peak from another sample. If no peak matching these criteria was found in a particular sample the peak area was counted as zero.'},\n",
       " {'pmcid': 'PMC5011436',\n",
       "  'pmid': '27474797',\n",
       "  'snippet': 'In addition, we determined that loss of sts5 leads also to increased levels of Myc-Efc25 proteins as compared to the control sts5+ cells (Figure 4&#8212;figure supplement 2B,C). Taken together, these findings indicate a role for Sts5 in regulating the cellular abundance of specific mRNAs, affecting cell morphology in particular during cell stress.'},\n",
       " {'pmcid': 'PMC5011436',\n",
       "  'pmid': '27474797',\n",
       "  'snippet': 'To test the role of Orb6 kinase in preventing the degradation of Sts5-regulated mRNAs, qPCR analysis was performed to probe the levels of specific transcripts following Orb6-as2 kinase inhibition with 1-NA-PP1. We found that mRNA levels of ssp1, efc25 and psu1 declined upon Orb6 kinase inhibition (Figure 6D).'},\n",
       " {'pmcid': 'PMC5011436',\n",
       "  'pmid': '27474797',\n",
       "  'snippet': 'Together, these findings support the idea that Orb6 kinase prevents Sts5 recruitment to Dcp1-containing granules and attenuates P-body formation, in a manner that is at least in part Sts5-dependent. Additionally, the function of Orb6 kinase activity has the effect of decreasing the degradation of specific mRNAs.'},\n",
       " {'pmcid': 'PMC5011436',\n",
       "  'pmid': '27474797',\n",
       "  'snippet': 'In summary, our results support a role for NDR kinases in the spatial control of polarized cell growth, during cell proliferation and in response to the nutritional environment by mediating the translational availability of specific mRNAs. Future research will seek to identify nutrient-sensitive signaling pathways upstream of Orb6 and define the specific roles of Orb6 kinase and Sts5 in the control of P-body assembly.'},\n",
       " {'pmcid': 'PMC5011436',\n",
       "  'pmid': '27474797',\n",
       "  'snippet': 'In summary, our results support a role for NDR kinases in the spatial control of polarized cell growth, during cell proliferation and in response to the nutritional environment by mediating the translational availability of specific mRNAs. Future research will seek to identify nutrient-sensitive signaling pathways upstream of Orb6 and define the specific roles of Orb6 kinase and Sts5 in the control of P-body assembly. Due to the conservation of these factors, this work has the potential to open new avenues of research linking nutrient-sensitive signaling and P-body regulation with implications for studies of cancer and neurodegenerative diseases.'},\n",
       " {'pmcid': 'PMC5011436',\n",
       "  'pmid': '27474797',\n",
       "  'snippet': 'For threshold selection, we adapted a method similar to one previously described for image threshold selection of RNA FISH images (Raj et al., 2008). Specifically, we applied a Laplacian of Gaussian filter to reduce noise and highlight areas of rapid change in each cell and chose the threshold where the histogram reached a plateau, indicating a region where above-background pixels can be clearly detected.'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'Electrical recordings from the CM9 muscle group on the adult fly proboscis previously revealed a decrease in the amount of neurotransmitter released from the CM9 NMJ in flies raised on a high-calorie diet compared to flies raised on a low-calorie diet (Rawson et al., 2012). The CM9 NMJ is ideal for this research since it combines motor neuron-specific genetic manipulations with a robust synaptic recording preparation allowing us to interrogate the cell autonomous effects of diet (Figure 1A). Using this system, we have extended our initial observations by first determining that changing only the yeast component of the diet is sufficient to alter neurotransmission.'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'All RNAi knock-downs utilize the Gal4:UAS binary expression system by combining the transgenic UAS construct (i.e. UAS-4eBPRNAi) with the CM9 motor neuron-specific E49-Gal4 driver. *p&lt;0.05 versus 1X controls determined using ANOVA.'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'Analysis of neurotransmission in these animals found that knockdown of 4eBP in the CM9 motor neuron reduced the presynaptic release of neurotransmitter in animals raised on a 1X diet compared to controls (Figure 2F&#8211;I; see Table 1 for E49-Gal4/+ control values). We also observe no difference between neurotransmitter release in 4eBP knockdown animals (4eBPRNAi) raised on 1X, 2X, or 1-2X diet conditions consistent with the effects of 4eBP knockdown being specific to diet regulation of neurotransmission and not basal release (Figure 2F&#8211;I). The effectiveness and specificity of all RNAi constructs were determined using quantitative RT-PCR and finds that the reductions in 4eBP mRNA levels are approximately 60% in control experiments (data not shown).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'To determine what signaling systems within the motor neuron are responsible for the effects of diet on neurotransmission, we used motor-neuron-specific RNAi to screen important nutrient-sensing pathways using viability in a diet sensitive glued mutant fly background (Rawson et al., 2012) followed by analysis of promising candidates using the proboscis extension reflex (PER) (Figure 2A) (Gordon and Scott, 2009; Kimura et al., 1986). This motor reflex requires the CM9 motor neuron and provides a simple assay for investigating CM9 motor neuron function by analyzing the velocity of proboscis extension using particle-tracking software to track bristle paths during the PER (Figure 2A, panels i-iv) (Rawson et al., 2012).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'This motor reflex requires the CM9 motor neuron and provides a simple assay for investigating CM9 motor neuron function by analyzing the velocity of proboscis extension using particle-tracking software to track bristle paths during the PER (Figure 2A, panels i-iv) (Rawson et al., 2012). For these analyses, we combine the E49-Gal4 driver, which is expressed in a few number of neurons in the adult including the CM9 motor neuron, with gene-specific RNAi allowing us to focus on the cell autonomous effects of insulin signaling without grossly altering whole animal insulin signaling (Gordon and Scott, 2009; Rawson et al., 2012). This approach identified thor, the Drosophila homologue of eukaryotic initiation factor 4e binding protein (4eBP), as a critical presynaptic mediator of the positive effects of the 1X diet on motor function (Figure 2B and C).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'We also observe no difference between neurotransmitter release in 4eBP knockdown animals (4eBPRNAi) raised on 1X, 2X, or 1-2X diet conditions consistent with the effects of 4eBP knockdown being specific to diet regulation of neurotransmission and not basal release (Figure 2F&#8211;I). The effectiveness and specificity of all RNAi constructs were determined using quantitative RT-PCR and finds that the reductions in 4eBP mRNA levels are approximately 60% in control experiments (data not shown). All values for the electrophysiological analyses are listed in Table 1.10.7554/eLife.16807.006Figure 2.Insulin/DILP signaling negatively regulates presynaptic release at the CM9 NMJ.'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'All RNAi knock-downs utilize the Gal4:UAS binary expression system by combining the transgenic UAS construct (i.e. UAS-4eBPRNAi) with the CM9 motor neuron-specific E49-Gal4 driver. *p&lt;0.05 versus 1X controls determined using ANOVA.'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'This epistasis between chico and 4eBP is similar to what has been recently observed for Drosophila lifespan (Bai et al., 2015). The regulation of neurotransmission by insulin signaling appears to be specific to the adult life stage since we do not observe the same effects of diet or presynaptic knockdown of 4eBP or chico on SV release from larval NMJs (Figure 2&#8212;figure supplement 1).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'This analysis found that both the 2X diet and the 1-2X diet shift conditions resulted in increased phosphorylation of InR and Akt compared to the 1X diet condition supporting that our 2X and 1-2X diet conditions result in increased insulin signaling in our flies. Consistent with insulin signaling in the CM9 motor neuron being responsible for the effects of diet on SV release, we observe that CM9-specific knock-down of the Drosophila IRS-1 homologue chico (chicoRNAi) in flies raised on the 2X and 1-2X shift diets resulted in a significant increase SV release compared to the 2X controls (Figure 2J&#8211;L). This effect of chicoRNAion SV release in flies raised on the 2X diet was phenocopied by the overexpression of a dominant negative insulin receptor (InRDN) in the CM9 motor neuron (Figure 2K and L) (Peru Y Col&#243;n de Portugal et al., 2012).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'This supports that dFOXO binding to the 4eBP gene is increased under our 1X diet condition compared to the 2X diet condition consistent with dFOXO driving the expression of 4eBP under 1X diet conditions. Importantly, these molecular data suggest that the regulation of 4eBP mRNA levels by diet is conserved among all motor neurons and not specific to CM9 MNs. Electrophysiological recordings from CM9 NMJs in dFOXO mutants raised on 1X, 2X or 1-2X shift diets found that neurotransmission is reduced in dFOXO mutants compared to 1X controls but are not different than the 2X controls, similar to what we observed in the 4eBPRNAi flies (Figure 3E&#8211;G, Table 1).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'Surprisingly, we also observed that 4eBP mRNA was also highly enriched even though 4eBP mRNAs do not contain a predicted STS (Figure 5E). The RIP of complexin and 4eBP mRNA was specific since a number of other neuronal mRNAs were not enriched in these RIP experiments including mRNAs for tubulin, syntaxin and the cacophony voltage-gated calcium channel (Figure 5E).10.7554/eLife.16807.011Figure 5.The role of Staufen during the regulation of neurotransmission by diet. (A) Representative traces of EPSPs from CM9 NMJs from 21 day old staufenRNAi raised on a 1X diet or staufenRNAi and control flies subjected to a diet switch from 1X to 2X diet on day 20 and recorded on day 21.'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'Importantly, we observe that our rapamycin treatment condition does result in the predicted change in the phosphorylation state of 4eBP demonstrating that TOR can regulate 4eBP in adult Drosophila motor neurons. Because most of the data on the effects of TOR on synapse function suggest a post-synaptic role for this complex, these data suggest that the regulation of 4eBP within the CM9 motor neuron is compartmentalized with the presynaptic pool regulated specifically by FOXO and the post-synaptic pool regulated by TOR. The localization of the TOR complex in neurons is unknown, but it presumably is localized within the cytoplasm and lysosomes (Betz and Hall, 2013).'},\n",
       " {'pmcid': 'PMC5012858',\n",
       "  'pmid': '27525480',\n",
       "  'snippet': 'For motor neuron-specific analysis, thoracic ganglion from flies expressing GFP in all motor neurons (D42-Gal4, UAS-10X-GFP) were removed and dissociated using 1 mg/ml collagenase (Sigma, St. Louis, MO) and sorted from non-fluorescent cells on a Beckman Dickinson Aria FACS unit using a 70&#160;&#181;m tip. Total RNA was extracted by FACS sorting ~50,000 Drosophila neurons into RTL buffer from an RNAEasy kit, and subsequently extracted via the same kit using the standard protocol (Qiagen, Hilden, Germany).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Despite many years of study, the molecular, cellular, and physiological mechanisms driving both the protective and deleterious effects of T21 are poorly understood. A few chr21-encoded genes have been implicated in the development of specific comorbidities, such as APP in Alzheimer&#8217;s disease (Wiseman et al., 2015), and DYRK1A and ERG in hematopoietic malignancies (Stankiewicz and Crispino, 2013; Malinge et al., 2012). Therefore, research in this area could inform a wide range of medical conditions affecting not only those with DS, but also the typical population.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Collectively, our understanding of the mechanisms driving such inter-individual variation in the population with DS is minimal. More specifically, it is unclear what gene expression changes are consistently caused by T21, versus those that are context-dependent. Integrated analyses of a large body of studies have indicated that the changes in gene expression caused by T21 involve various signaling pathways (Scarpato et al., 2014), however, these studies vary widely in cell type, number of samples, and even analysis platform, among other variables (Volk et al., 2013; Costa et al., 2011).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'T21 was confirmed by PCR analysis of the chr21-encoded RCAN1 gene (Figure 1&#8212;figure supplement 1D). We included samples from different genetic backgrounds, ages, and genders, in order to avoid identifying differences that are specific to a particular pair of isogenic or genetically related cell lines and which would not withstand the effects of inter-individual variation. To illustrate this point, comparison of one pair of disomic male individuals of similar age yielded thousands of differentially expressed genes (DEGs), with similar numbers of upregulated and downregulated DEGs (Figure 1A&#8211;B, Male 1 vs.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'In order to characterize the mechanism driving the consistent changes caused by T21, we examined the regulatory differences between DEGs encoded on chr21 and those not encoded on chr21. Several lines of evidence indicate that, while chr21 DEGs are regulated mostly by increased gene dosage, non-chr21 DEGs may be driven by specific pathways that are subject to signal amplification, with a bias toward upregulation, and greatly affected by inter-individual variation. First, violin plots display the relatively small number of chr21 DEGs, showing mostly upregulation clustered around 1.5 fold, versus a much larger number of non-chr21 DEGs, showing both up- and downregulation with no obvious clustering of fold changes (Figure 1D, Figure 1&#8212;figure supplement 2B).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'IRF3, IRF5, IRF7, STAT1) (Figure 2A). Importantly, most of these signals are derived from non-chr21 DEGs, and would be missed by analyses focused specifically on chr21-encoded genes (Figure 2A). This analysis also identified two known repressors of IFN signaling, MAPK1 and TRIM24, as upstream regulators inactivated in T21 cells, consistent with activation of the IFN pathway (Huang et al., 2008; Tisserand et al., 2011).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': '(C) Western blots as in A for IFN-&#947; treatment. * indicates non-specific bands.DOI: http://dx.doi.org/10.7554/eLife.16220.009'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Similar results were observed for the other Type I ligand, IFN-&#946;. However, ligand-specific differences were also observed. For example, IDO1 was more strongly induced by IFN-&#945; and -&#946; in the T21 cell line in pair 1, but this was not the case when using IFN-&#947; (Figure 3A&#8211;C).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'We next investigated whether T21 cells produce a stronger response to specific IFN ligands than their D21 counterparts. To test this, we treated three pairs of fibroblasts &#8211;roughly matched by age and sex&#8211; with various doses of the Type I ligands IFN-&#945; or -&#946;, or with the Type II ligand IFN-&#947;, and monitored the&#160;expression of key ISGs via western blot.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'We also monitored phosphorylation of STAT1. Overall, these efforts revealed that trisomy 21 cells show stronger induction of ISGs upon treatment with all three ligands, albeit with variation across specific cell lines and ligands (Figure 3). For example, stimulation with IFN-&#945; led to stronger induction in the T21 cell line for MX1 in pairs 1 and 2, stronger induction of IDO1 in pairs 1 and 3, and stronger induction of ISG15 in pairs 1 and 2 (Figure 3A).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': '(C) Western blots as in A for IFN-&#947; treatment. * indicates non-specific bands.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'proliferation and/or survival) of T21 cells relative to D21 cells. We hypothesized that core gene expression changes in T21 cells lead to a rewiring of signaling cascades, creating differential requirements for specific kinases during cell survival and proliferation. In order to identify such kinases, we introduced a library of 3,075 shRNAs targeting 654 kinases into each of the 12 fibroblast cell lines we subjected to transcriptome analysis.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'A comparison of the upstream regulator analyses of the four cell types included in this study revealed both conserved and cell type-specific features. The upstream regulator analysis shows that IFN activation is conserved, as is predicted inactivation of the IFN repressors MAPK1 and TRIM24 (Figure 6C).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'A well-established aspect of the IFN response is the selective control of protein translation, purportedly to prevent the synthesis of viral proteins during the course of infection (Johnson et al., 1968). Mechanistically, it has been shown that IFN signaling impairs processing of rRNAs and controls the activity and/or expression of specific translation factors (Walsh et al., 2013; Maroun, 1978). On the other hand, the MYC family of transcription factors are known drivers of ribosome biogenesis, protein synthesis and cell growth (van Riggelen et al., 2010; Boon et al., 2001; Kim et al., 2000; Arabi et al., 2005).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'This is in stark contrast to the notion that T21 affects gene expression either stochastically or through large rearrangements of chromatin domains. In fact, Manhattan plots of the DEGs in monocytes and T cells derived from the same individuals not only confirm the absence of large domains of chromatin deregulation, but also highlight the high degree of cell type-specific changes caused by the trisomy (Figure 8G).10.7554/eLife.16220.023Figure 8.Trisomy 21 activates the IFN gene expression program in a cell type-specific manner. (A) Principal component analysis (PCA) of all RNA-seq samples from this study colored by cell type.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Expectedly, lymphoblastoids and T cells showed a greater degree of overlap than other pairwise comparisons. Overall, these results indicate that while T21 operates within, and is modulated by, cell type-specific regulatory landscapes, it nonetheless activates the IFN transcriptional response consistently by inducing different gene sets within this program. This is in stark contrast to the notion that T21 affects gene expression either stochastically or through large rearrangements of chromatin domains.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'For example, APP is relatively more highly expressed in fibroblasts, U2AF1 more highly expressed in lymphoblastoids, ETS2 more highly expressed in monocytes, and DYRK1A more highly expressed in T cells (Figure 8C, Supplementary file 1F). The IFN receptors on chr21 also showed some degree of cell type-specific expression (e.g. IFNAR2 lowly expressed in fibroblasts, IFNGR2 lowly expressed in T cells, Figure 8D).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Within this background, the global impact of the trisomy on the transcriptome is secondary to the effects of the cell type of origin (Figure 8B). Next, we asked to what degree genes encoded on chr21 could be affected by these cell type-specific regulatory landscapes. Indeed, it was easy to identify many chr21 genes displaying obvious differences in relative expression among cell types.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': '(D,E) Box and whisker plots as in (C) for chr21-encoded IFN receptors and representative ISGs. (F) Venn diagram showing the cell type-specificity of the Interferon alpha gene expression programs identified by IPA for each cell type. (G) Manhattan plots for chromosomes 19 and 21 comparing the DEGs from monocytes and T cells derived from the same individuals.DOI: http://dx.doi.org/10.7554/eLife.16220.023'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'To address this, we compared the DEGs comprising the T21-induced Interferon alpha signature identified by IPA in each cell type (Figure 6C). Remarkably, this exercise revealed a large degree of cell type-specificity, with most IFN-related genes being differentially expressed in only one cell type (Figure 8F). In fact, the only common genes among all four signatures are three IFN-&#945;-related genes encoded on chr21: IFNAR1, IFNAR2, and MX1.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'IFNAR2 lowly expressed in fibroblasts, IFNGR2 lowly expressed in T cells, Figure 8D). Furthermore, relative differences in cell type-specific expression is also evident for canonical ISGs (Figure 8E). These observations led us to ask to what degree the IFN transcriptional response elicited by T21 is conserved across cell types.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Having performed transcriptome analysis of cell types of different origins, we investigated to what degree the gene expression changes caused by T21 are affected by cell type-specific regulatory landscapes. A principal component analysis (PCA) shows the fibroblast transcriptomes segregating strongly (PC1 80.5%) from those of the cell types of hematopoietic origin (Figure 8A).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': '(D,E) Box and whisker plots as in (C) for chr21-encoded IFN receptors and representative ISGs. (F) Venn diagram showing the cell type-specificity of the Interferon alpha gene expression programs identified by IPA for each cell type. (G) Manhattan plots for chromosomes 19 and 21 comparing the DEGs from monocytes and T cells derived from the same individuals.'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'Additional research will also be required to elucidate the interplay between hyperactive IFN signaling in DS with other important factors encoded on chr21 (e.g. DYRK1A, APP) (Malinge et al., 2012; Wiseman et al., 2015) or elsewhere in the genome, that have been involved in the&#160;development of the specific comorbidities. For example, the Sonic Hedgehog (SHH) pathway has been implicated in the etiology of structural and cognitive defects in a mouse model of DS, including cerebellar atrophy (Das et al., 2013).'},\n",
       " {'pmcid': 'PMC5012864',\n",
       "  'pmid': '27472900',\n",
       "  'snippet': 'We utilized a multiplexing strategy consisting of four different barcodes with each nucleotide represented at each position of the barcode, allowing us to sequence four samples in each lane on a HiSeq 2000 Illumina instrument. To accomplish this, each sample was quantified and mixed together at a final concentration of 10 ng/&#956;L and using Illumina-specific oligonucleotides and qPCR, we determined the cluster formation efficiency (i.e. effective concentration) of our library to be slightly greater than that of a known library.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'Following import the mitochondria were re-isolated, lysed in digitonin-containing buffer and subjected to blue native electrophoresis and autoradiography. Asterisk (*) indicates non-specific band. (D) Mitochondria isolated from control and hTim10b expressing cells were solubilised in digitonin-containing buffer.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'Following import the mitochondria were re-isolated, lysed in digitonin-containing buffer and subjected to blue native electrophoresis and autoradiography. Asterisk (*) indicates non-specific band. (D) Mitochondria isolated from control and hTim10b expressing cells were solubilised in digitonin-containing buffer.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'Eluate fractions were then analysed by label-free quantitative mass spectrometry to reveal the interacting partners of hTim10b. Among proteins that co-purified specifically with hTim10b3XFLAG were known interacting partners hTim10a, hTim9 and hTim22 (Figure 1E and Figure 1&#8212;source data 1), which suggested the purification of functional TIM22 complexes. The uncharacterised protein, C19orf52, also displayed significant enrichment in the hTim10b3XFLAG sample (Figure 1E and Figure 1&#8212;source data 1).'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'We affinity-purified hTim10b3XFLAG and associated proteins using anti-FLAG resin (Figure 1D) and a stable cell line containing an empty vector was used as a negative control (since we often observed leaky expression of hTim10b in un-induced cells). A single protein of the expected molecular weight was detected in the eluate fraction by western blotting with FLAG-specific antibodies (Figure 1D, lane 7, upper panel). Eluate fractions were then analysed by label-free quantitative mass spectrometry to reveal the interacting partners of hTim10b.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'As can be seen (Figure 3B, lanes 2, and 8), the FLAG antibodies shifted [35S]-hTim22 (hTIM22 complex) in both hTim10b3XFLAG and C19orf523XFLAG containing mitochondria, suggesting the presence of all three proteins within the same complex. In an alternative approach C19orf523XFLAG was shown to specifically immunoprecipitate hTim22 (Figure 3C), while C19orf52 could be co-immunoprecipitated with hTim10b3XFLAG (Figure 3D). As a final source of confirmation immunoprecipitation of C19orf523XFLAG and subsequent quantitative mass spectrometry-based proteomics analysis revealed the presence of all known TIM22 complex subunits, including hTim22, hTim10b, hTim10a and hTim9 (Figure 3&#8212;source data 1).'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'Mitochondria from hTim10b3XFLAG and C19orf523XFLAG expressing cells were solubilised in digitonin-containing buffer prior to analysis by BN-PAGE and immunodecoration (Figure 3A). C19orf52 specific antibodies revealed the presence of a high molecular weight complex of approximately 450-kDa, reminiscent of the TIM22 complex in both control (Figure 3A, lane 1) and hTim10b3XFLAG mitochondria (Figure 3A, lane 2). We observed a slight delay in the migration of the C19orf52-containing complex in mitochondria isolated from hTim10b3XFLAG expressing cells (Figure 3A, lane 2 and 4).'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'As a final source of confirmation immunoprecipitation of C19orf523XFLAG and subsequent quantitative mass spectrometry-based proteomics analysis revealed the presence of all known TIM22 complex subunits, including hTim22, hTim10b, hTim10a and hTim9 (Figure 3&#8212;source data 1). These findings confirm that C19orf52 is a bona fide subunit of the human TIM22 complex, and represents a novel metazoan specific subunit of the translocation machine. Given that the molecular weight of the protein is 29&#8201;kDa and the nomenclature of the mitochondrial protein import machinery (Pfanner et al., 1996), we henceforth rename this protein as Tim29 and its gene as TIMM29.10.7554/eLife.17463.008Figure 3.C19orf52 is a subunit of the human TIM22 complex.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'As can be seen, Tim29 (Figure 4A) and hTim22 (Figure 4B) were depleted from mitochondria while the control protein Mfn2 remained unaffected (Figure 4A and B, bottom panels). Specific depletion of TIM22 complex substrates, including hTim23, ANT3 and the glutamate carrier was observed in both Tim29 and hTim22 depleted mitochondria (quantifications shown in Figure 4A and B, lower panels). The depletion of Tim29 also led to a significant reduction in the levels of hTim22 (Figure 4A), while the lack of hTim22 had no obvious effect on the levels of Tim29, indicating Tim29 is imported in a TIM22-independent manner (Figure 4B).'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'To explore the relationship between TOM and TIM22 further, we performed chemical cross-linking with dithiobis (succinimidylpropionate) (DSP), an amino-group-specific homobifunctional crosslinker that is cleaved by reducing agents. Mitochondria isolated from control and Tim293XFLAG expressing cells were incubated with DSP, solubilised in SDS to dissociate non-covalent interactions and then incubated with anti-FLAG resin for purification of Tim293XFLAG-containing species.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'We have captured an interaction between the TOM and TIM22 complexes in organello using both Tim29 and hTom22 as bait. Furthermore, using chemical crosslinking we narrowed down specific interactions between: (i) Tim29 and hTim9; (ii) Tim29 and hTim10b; hTim10b and hTim9; and Tim29 and hTom40. We were puzzled to see an interaction between Tim29 and hTom40, since in yeast TIM23 interacts with TOM via Tom22, which exposes a domain into the intermembrane space (Waegemann et al., 2015; Chacinska et al., 2010).'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'For crosslinking experiments, mitochondria (0.5&#8211;1 mg) were resuspended in import buffer (20&#8201;mM HEPES-KOH pH 7.4, 250&#8201;mM sucrose, 80&#8201;mM KOAc, 5&#8201;mM MgOAc, and 5&#8201;mM ATP) and incubated with the amino-group-specific homobifunctional and cleavable crosslinker, DSP; dithiobis&#160;(succinimidylpropionate) (ThermoFisher) at a final concentration of 0.2&#8201;mM for 1&#8201;hr at 4&#176;C. Crosslinking reactions were quenched using 100&#8201;mM Tris-Cl, pH 7.4 for 30&#8201;min at 4&#176;C. Mitochondria were reisolated and solubilised in lysis buffer (20&#8201;mM Tris-Cl, pH 7.4, 1&#8201;mM EDTA, 1% (w/v) SDS) and boiled at 95&#176;C for 5&#8201;min. Clarified samples were diluted with 1% Triton X-100-containing buffer (1% (v/v) Triton X-100, 20&#8201;mM Tris-Cl pH 7.4, 150&#8201;mM NaCl, 1X complete protease inhibitor) and incubated with pre-equilibrated anti-FLAG resin for 1&#8201;hr at 4&#176;C. Following washing specifically bound proteins were eluted in 2X Laemmli buffer.'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'Open reading frames (ORF) encoding mitochondrial precursor proteins were cloned into pGEM4Z (Promega). The desired ORFs were amplified by PCR using vector specific primers (M13_FORWARD and M13_REVERSE) and applied to in vitro&#160;transcription reactions using the mMessage SP6 transcription kit (Ambion). mRNA was isolated by LiCl precipitation according to the manufacturer&#8217;s instructions and applied to in vitro translation reactions using rabbit reticulocyte lysate (Promega) in the presence of [35S]-methionine/cysteine (Perkin-Elmer).'},\n",
       " {'pmcid': 'PMC5016092',\n",
       "  'pmid': '27554484',\n",
       "  'snippet': 'The intensity of relevant bands was determined by subtracting the background signal to give the true relative amount of protein (assembled complexes for BN-PAGE analysis and imported proteins for SDS-PAGE analysis). Each specific time point was normalised against the value in control mitochondria with the longest import time point and were computed as a&#160;% of this control. Differences in the import of proteins between control (scrambled siRNA) and Tim29 or hTim22 knock-down mitochondria were examined using a paired two-tailed Student&#8217;s t-test on 2 d.f.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'The vertebrate retina provides an advantageous model to study how specific neuronal cell types organize themselves during development to form functional circuits. The ~100 neuronal cell types of the retina vertically organize into layers, with light-transducing photoreceptors in the outermost cellular layer (ONL, or outer nuclear layer) and retinal ganglion cells (RGCs) &#8211; responsible for the sole output from the retina &#8211; in the innermost layer (RGL, or retinal ganglion cell layer).'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Between these, both functionally and physically, are the interneuron cell types (horizontal, amacrine, and bipolar cells) in the inner nuclear layer (INL), which process and relay visual information from photoreceptors to RGCs. These three cellular layers are separated by two synaptic plexiform layers, where the neurites from these cell types mingle and form synaptic connections with specific partners in specific substrata. Additionally, many cell types are horizontally spaced in a mosaic pattern, such that there is a low probability of finding two neurons of the same subtype (i.e., homotypic) in close proximity.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'In chick, Dscams label-specific sublaminae of the IPL and can instruct neurite targeting to these layers (Yamagata and Sanes, 2008). DSCAM protein localization is punctate throughout the IPL in mouse, and is not confined to specific sublaminae (de Andrade et al., 2014). Despite this, some neuronal types do have disorganized neurite stratification in Dscam mutants, although the disorganization varies with genetic background (Fuerst et al., 2010).'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Dscams also contribute to the vertical organization of the retina. In chick, Dscams label-specific sublaminae of the IPL and can instruct neurite targeting to these layers (Yamagata and Sanes, 2008). DSCAM protein localization is punctate throughout the IPL in mouse, and is not confined to specific sublaminae (de Andrade et al., 2014).'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Both DSCAM and DSCAML1 also have canonical PDZ-interacting motifs at their C-termini by which they interact with scaffolding proteins in the MAGI (membrane-associated guanylate kinase with inverted orientation) and PSD95 families (Yamagata and Sanes, 2010). Because this motif is common to both Dscams, we chose to test the functional significance of these C-terminal interactions, with the initial hypothesis that this interaction would be required for a specific subset of Dscams&#8217; functions.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'We engineered mouse mutations in which the sequences encoding the final 10 amino acids of DSCAM and DSCAML1, which interact with PDZ domain-containing proteins, were replaced with epitope tags. Contrary to our initial hypothesis, we found that rather than distinguishing phenotypes based on specific molecular mechanisms, these mutations distinguished cell types. Some cell types were essentially like controls in cell number, spacing, and stratification, whereas others were nearly as disrupted as null mutants.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': '(C) Western blots of DSCAM protein immunoprecipitated from neonatal brains showed no change in the size or amount of DSCAM in Dscam&#8710;C/&#8710;C mutants. The antibody specificity is confirmed by the lack of signal from Dscam-/- brains. (D&#8211;F) Immunofluorescent labeling for DSCAM in vertically sectioned retinas from 3-week old mice demonstrated that the protein is found in a normal, punctate localization in the synaptic plexiform layers in Dscam+/+ (D) and Dscam&#8710;C/&#8710;C (E) mice, consistent with earlier reports for wild type DSCAM (de Andrade et al., 2014).'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': '(D&#8211;F) Immunofluorescent labeling for DSCAM in vertically sectioned retinas from 3-week old mice demonstrated that the protein is found in a normal, punctate localization in the synaptic plexiform layers in Dscam+/+ (D) and Dscam&#8710;C/&#8710;C (E) mice, consistent with earlier reports for wild type DSCAM (de Andrade et al., 2014). No staining above background was found in Dscam-/- retinas (F), demonstrating the specificity of the DSCAM antibody. (G&#8211;I) Hematoxylin and eosin staining of adult retinas shows that, compared to controls (G), Dscam-/- retinas (I) are severely expanded and disorganized.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': '(C) Western blots of DSCAM protein immunoprecipitated from neonatal brains showed no change in the size or amount of DSCAM in Dscam&#8710;C/&#8710;C mutants. The antibody specificity is confirmed by the lack of signal from Dscam-/- brains. (D&#8211;F) Immunofluorescent labeling for DSCAM in vertically sectioned retinas from 3-week old mice demonstrated that the protein is found in a normal, punctate localization in the synaptic plexiform layers in Dscam+/+ (D) and Dscam&#8710;C/&#8710;C (E) mice, consistent with earlier reports for wild type DSCAM (de Andrade et al., 2014).'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': '(D&#8211;F) Immunofluorescent labeling for DSCAM in vertically sectioned retinas from 3-week old mice demonstrated that the protein is found in a normal, punctate localization in the synaptic plexiform layers in Dscam+/+ (D) and Dscam&#8710;C/&#8710;C (E) mice, consistent with earlier reports for wild type DSCAM (de Andrade et al., 2014). No staining above background was found in Dscam-/- retinas (F), demonstrating the specificity of the DSCAM antibody. (G&#8211;I) Hematoxylin and eosin staining of adult retinas shows that, compared to controls (G), Dscam-/- retinas (I) are severely expanded and disorganized.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'We used two methods to assess selective instability. First, we quantified the colocalization between DSCAM and melanopsin to see if there was a reduction in DSCAM specifically in ipRGCs. We did not find any difference in this colocalization between Dscam+/+&#160;and Dscam&#8710;C/&#8710;C retinas (Figure 1&#8212;figure supplement 2).'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Thus, neither immunofluorescence nor immunoprecipitation (Figure 1B) provide any evidence supporting protein instability. It remains possible, however, that a subtle difference in cell-type specific protein stability or localization beyond the resolution of these experiments could contribute to the range of phenotypes we have observed. Similarly, the tag could be interfering with function in some cell types; however, we do not believe this to be the case as we performed similar experiments with DSCAML1 using a different tag and obtained similar results, as described below.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'In Dscam&#8710;C/&#8710;C mutants, DA cells are severely fasciculated (Figure 6M), while M1 ipRGCs more mildly so (Figure 6O). Interestingly, there is a&#160;clear co-fasciculation between these cell types in Dscam&#8710;C/&#8710;C mutants (Figure 7E), suggesting that the M1 fasciculation may be influenced by the DA cells.10.7554/eLife.16144.016Figure 7.Laminar specificity in &#8710;C mutants.Neurite stratification in the IPL was analyzed in immunolabeled cryosections. (A,B) bNOS-positive amacrine cells stratified normally in Dscam&#8710;C/&#8710;C retinas, as did ipRGCs and DA cells (C,D), which co-stratify in the OFF region adjacent to the INL.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'We tested the functional significance of the Dscams&#8217; PDZ-interacting domains in vivoby replacing the C-terminal ten amino acids with epitope tags (Dscam&#8710;C and Dscaml1&#8710;C). The Dscams have similar functions in the different cell types in which they are expressed: (1) They promote developmental cell death, (2) they promote self-avoidance at the cell type level both between cell bodies and neurites, preventing cell body clustering and fasciculation of processes, and (3) they can promote the laminar specificity of neurite stratification and synapse maturation (Fuerst et al., 2009, 2008). We hypothesized that interactions with PDZ domain-containing proteins via the C-termini would mediate a subset of these functions.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'It could have resulted in an unstable or mis-trafficked protein, causing a complete loss of function. Our data do not support this, as protein is found at normal levels by both immunoprecipitation and immunohistochemistry, is on the cell surface by live cell staining and surface biotinylation, and is in the correct topological orientation, although more subtle or specific defects may still have been missed by these analyses. The PDZ-interacting C-termini of DSCAM and DSCAML1 could also have been totally superfluous for its function.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Phrased genetically, the Dscam-/- and Dscaml1-/- alleles represent a&#160;complete loss of function. The functions we have defined, promoting developmental cell death, promoting self-avoidance in both processes and cell bodies to enable mosaic spacing and coverage factor, and promoting laminar specificity and synapse maturation in at least some cell types and genetic backgrounds, are based on the loss of function mutant phenotype. The deletion of the PDZ-binding C-termini of DSCAM and DSCAML1 had several possible outcomes in this context.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Similarly, the vertebrate gamma protocadherin cluster (Pcdhg) can also produce diverse recognition units. There are only 22 Pcdhg isoforms, specified by alternative promoter choice within the gene cluster instead of alternative splicing, but the resulting proteins form cis multimers with trans homophilic binding specificity at the multimer level (Schreiner and Weiner, 2010; Thu et al., 2014; Wu and Maniatis, 1999). This incorporation of isoforms into multimers can generate thousands of distinctly homophilic interactors.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Without extensive isoform diversity, vertebrate DSCAMs are not able to provide this fine level of self-recognition. Our conclusion that DSCAMs function through at least two molecular mechanisms, one that requires a C-terminal PDZ-interacting motif, and one that does not, fits a model in which DSCAMs interact with cell-type-specific adhesion mechanisms to serve their function. This is most easily discussed for self-avoidance, where the deletion of DSCAM or DSCAML1 results in the clumping and fasciculation of homotypic cells &#8211; phenotypes that could be described as excessive adhesion.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'This is most easily discussed for self-avoidance, where the deletion of DSCAM or DSCAML1 results in the clumping and fasciculation of homotypic cells &#8211; phenotypes that could be described as excessive adhesion. Under this model, the DSCAMs serve a generic role, and the specificity and complexity of the system is conferred by the cell-type-specific adhesion mechanisms. Thus, in cell types in which the PDZ-interacting C-terminus is required, the predominant cell adhesion system may also interact with PDZ-scaffolded complexes.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Interestingly, there was increased co-localization during development, indicating that these interactions are dynamic and transient (de Andrade et al., 2014). None of the analyzed PDZ proteins were clearly localized to specific laminae, but were punctate throughout the IPL. Thus, DSCAM and DSCAML1 could interact with different PDZ proteins in different cell types throughout development.'},\n",
       " {'pmcid': 'PMC5026468',\n",
       "  'pmid': '27637097',\n",
       "  'snippet': 'Dscaml1-/- mice were genotyped with a previously unreported primer set. A common forward primer (ATGCCACTGTGCCTGGCTGTT) was used with a reverse primer specific to either wild type sequence (CCCAGCAGTTGAGTGCCCTGG) or mutant sequence (TATCCACAACCAACGCACCCAAGC).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'We report that the consumption of a high fat diet causes differential exon inclusion/exclusion in the transcriptome. Bioinformatics analysis implicated a role for NOVA pre-mRNA splicing factors and this was confirmed by studies of mice with adipocyte-specific NOVA deficiency. Functional studies demonstrated that NOVA acts to suppress adipose tissue thermogenesis.'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'This SVF contamination of the adipocyte fraction may be mediated by lipid-loaded macrophages (Xu et al., 2013). Control studies were performed using a gene (Fabp4) that is expressed by both adipocytes and SVF.DOI: http://dx.doi.org/10.7554/eLife.17672.00810.7554/eLife.17672.009Figure 1&#8212;figure supplement 4.Biological pathway enrichment analysis of differential gene expression and alternative pre-mRNA splicing caused by feeding a HFD.Genes that are differentially expressed and/or alternatively spliced in response to the consumption of a HFD (Figure 1B) were examined by pathway analysis to detect significant enrichment for specific biological processes.DOI: http://dx.doi.org/10.7554/eLife.17672.00910.7554/eLife.17672.010Figure 1&#8212;figure supplement 5.NOVA expression in human subcutaneous&#160;adipose tissue.NOVA1 and NOVA2 mRNA expression in abdominal subcutaneous adipose of obese (BMI&gt;30&#160;kg/m2) and non-obese (BMI&lt;30&#160;kg/m2) matched humans was examined (mean &#177; SEM; n&#160;=&#160;30 obese, n&#160;=&#160;26 non-obese; *p&lt;0.05).DOI: http://dx.doi.org/10.7554/eLife.17672.01010.7554/eLife.17672.011Table 1.Summary of RNA-seq data.DOI: http://dx.doi.org/10.7554/eLife.17672.011GEO Accession Subseries/ SuperseriesBiological groupsSample numberPlatformMean read number / sample (after trimming, if applicable)Read length (after trimming, if applicable)Mean read alignment rateGSE76294/ GSE76134FWT (3) F&#8710;N1 (3) F&#8710;N2 (3)9Illumina HiSeq 2000/ 2 x 100&#160;bp135,500,000100&#160;bp89.4%GSE76133/ GSE76134CD (3) HFD (3)6Illumina NextSeq 500/ 2 x 150&#160;bp406,200,00090&#160;bp74.9%GSE76317/ GSE76134FWT (4) F&#8710;N1,2 (4)8Illumina NextSeq 500/ 2 x 150&#160;bp319,700,000130&#160;bp92.5%'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'Genes that are differentially expressed and/or alternatively spliced in response to the consumption of a HFD (Figure 1B) were examined by pathway analysis to detect significant enrichment for specific biological processes.'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'Reduced expression of Nova1 or Nova2 in mice causes developmental defects and neonatal lethality (Jensen et al., 2000; Ruggiu et al., 2009). We therefore established Nova1LoxP/LoxP&#160;and Nova2LoxP/LoxP mice to study the role of NOVA proteins in adult mice with tissue-specific NOVA deficiency (Figure 2&#8212;figure supplement 1). Adipoq-Cre&#8722;/+ mice were used to selectively ablate Nova1 and Nova2 genes in mature adipocytes.'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'Analysis of white adipocytes with single gene ablations of Nova1 or Nova2 identified fewer changes in alternative pre-mRNA splicing, including 10 &amp; 7 skipped exons, 1 &amp; 4 mutually exclusive exon inclusions, 12 &amp; 16 intron retentions, 1 &amp; 0 alternative 5&#8217; splice sites, and 2 &amp; 4 alternative 3&#8217; splice sites, respectively (FDR&lt;0.05; absolute &#8710;Inc level &gt;0.1). These data indicate that&#160;NOVA1 and NOVA2 can cause isoform-specific changes in adipocyte pre-mRNA alternative splicing. However, the effect of compound NOVA1 plus NOVA2 deficiency to cause widespread changes in alternative pre-mRNA splicing (Figure 2) indicates that these NOVA proteins exhibit some functional redundancy in adipocytes.10.7554/eLife.17672.012Figure 2.NOVA proteins contribute to alternative pre-mRNA splicing associated with diet-induced obesity.'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'The amount of phosphorylated cJun was examined by Phosphorimager analysis. (B) In vitro kinase assays were performed using Flag-tagged JNK2&#945;2 and JNK2&#946;2.DOI: http://dx.doi.org/10.7554/eLife.17672.01910.7554/eLife.17672.020Figure 3&#8212;figure supplement 3.Effect of adipocyte-specific JNK-deficiency on thermogenic gene expression. (A) Core body temperature of FWT and F&#8710;J1,2 mice fed a CD or n HFD (16 wk) was measured by telemetry using an implanted probe (mean &#177; SEM; n&#160;=&#160;7~10).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'This analysis demonstrated that adipocytes and hepatocytes express different alternatively spliced JNK isoforms (Figure 3&#8212;figure supplement 1C). These tissue-specific differences in Mapk8/9 pre-mRNA splicing may be influenced by the selective expression of NOVA proteins in adipocytes, but not hepatocytes (Figures 1F and 3A), although NOVA-independent mechanisms likely also contribute to the observed cell type-specific pattern of JNK isoform expression.10.7554/eLife.17672.014Figure 3.NOVA promotes signal transduction by JNK in adipose tissue. (A) The expression of Nova mRNA in adipocytes and hepatocytes was measured by quantitative RT-PCR (mean &#177; SEM; n=7~8; **p&lt;0.01; ***p&lt;0.001).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'We found that hepatic expression of NOVA2 caused a decrease in the Mapk8&#945;/&#946; ratio and an increase in the Mapk9&#945;/&#946; ratio, indicating that NOVA can promote the expression of the Mapk8&#946; and Mapk9&#945; alternatively spliced isoforms (Figure 3B). In contrast, adipocyte-specific deficiency of NOVA1 plus NOVA2 caused an increase in the Mapk8&#945;/&#946; ratio and a decrease in the Mapk9&#945;/&#946; ratio, indicating that NOVA-deficiency promotes the expression of the Mapk8&#945; and Mapk9&#946; alternatively spliced isoforms (Figure 3C). These observations are consistent with the finding that HFD consumption causes both decreased NOVA expression (Figure 1H) and significant changes in the mutually exclusive inclusion of Mapk8 exons 7a and 7b (FDR=0.0066).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'To examine the consequences of reduced JNK signaling in adipocytes, we established F&#8710;J1,2 mice with adipocyte-specific JNK-deficiency (Adipoq-Cre&#8722;/+ Mapk8LoxP/LoxP Mapk9LoxP/LoxP). These mice exhibited increased core body temperature and increased expression of genes that mediate sympathetic activation (Adr3b) and thermogenesis (Ucp1) in adipocytes (Figure 3&#8212;figure supplement 3).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'The relative area of islets is presented (mean &#177; SEM; n=6 mice). Statistically significant differences between FWT and F&#8710;N1,2 mice are indicated: **p&lt;0.01; ***p&lt;0.001.DOI: http://dx.doi.org/10.7554/eLife.17672.02710.7554/eLife.17672.028Figure 4&#8212;figure supplement 3.Effect of adipose tissue-specific ablation of the Nova1 or Nova2 genes. (A) The change in body mass of FWT, F&#8710;N1 and F&#8710;N2 mice fed a CD or ais presented (mean &#177; SEM; n=8~21).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': '(E) Energy expenditure (EE) by HFD-fed (4 wks) FWT mice (n&#160;=&#160;9) and F&#8710;N1,2 mice (n&#160;=&#160;8) was examined using metabolic cages over 3 days (12&#160;hr light; 12 hr dark). The&#160;source data are included as Figure 4&#8212;source data 4.DOI: http://dx.doi.org/10.7554/eLife.17672.02110.7554/eLife.17672.022Figure 4&#8212;source data 1.Source data for Figure 4B.DOI: http://dx.doi.org/10.7554/eLife.17672.02210.7554/eLife.17672.023Figure 4&#8212;source data 2.Source data for Figure 4C.DOI: http://dx.doi.org/10.7554/eLife.17672.02310.7554/eLife.17672.024Figure 4&#8212;source data 3.Source data for Figure 4D.DOI: http://dx.doi.org/10.7554/eLife.17672.02410.7554/eLife.17672.025Figure 4&#8212;source data 4.Source data for Figure 4E.DOI: http://dx.doi.org/10.7554/eLife.17672.02510.7554/eLife.17672.026Figure 4&#8212;figure supplement 1.Effect of adipose tissue-specific NOVA1/2-deficiency on organ mass. (A) The change in% body mass of FWT and F&#8710;N1,2 mice fed a CD or ais presented (mean &#177; SEM; n=8~30; *p&lt;0.05).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'Moreover, HFD-fed FWT mice were found to be more glucose intolerant than F&#8710;N1,2 mice (Figure 4D) and the HFD-induced hyperglycemia observed in FWT mice was suppressed in F&#8710;N1,2 mice (Figure 4C). Similar (although smaller) phenotypes were detected in mice with adipocyte-specific single gene ablation of Nova1 or Nova2 (Figure 4&#8212;figure supplement 3).'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'The increased adipose tissue browning detected in mice with adipocyte-specific NOVA deficiency was observed using floxed Nova alleles and a Cre driver (Adipoq-Cre) that is expressed in mature adipocytes. It is therefore likely that the increased white adipose tissue browning caused by NOVA deficiency does not reflect a change in adipocyte differentiation from specialized progenitor cells, but rather a change in mature adipocyte function.'},\n",
       " {'pmcid': 'PMC5026472',\n",
       "  'pmid': '27635635',\n",
       "  'snippet': 'The mice were housed at 21&#176;C (alternatively at 4&#176;C or 30&#176;C, where indicated) in a specific pathogen-free facility accredited by the American Association for Laboratory Animal Care (AALAC). The Institutional Animal Care and Use Committee (IACUC) of the University of Massachusetts Medical School approved all studies using animals.'},\n",
       " {'pmcid': 'PMC5026473',\n",
       "  'pmid': '27552055',\n",
       "  'snippet': 'These trends are clearly evident from a plot of the top 25 Tom1-dependent modification sites within all proteins (Figure 3&#8212;figure supplement 2C) or just ribosomal proteins (Figure 3E). To address whether the strong effects on ribosomal protein ubiquitination seen in tom1&#8710; cells were due specifically to loss of Tom1&#8217;s E3 activity, the GlyGly SILAC analysis was repeated with WT and tom1CA cells. Quantitative analysis of the data confirmed a disproportionate loss of ribosomal protein ubiquitination (Figure 3&#8212;figure supplement 2D).10.7554/eLife.19105.008Figure 3.Diminished ubiquitination and accumulation of insoluble ribosomal proteins in tom1 cells.'},\n",
       " {'pmcid': 'PMC5026473',\n",
       "  'pmid': '27552055',\n",
       "  'snippet': 'Thus, we reasoned that if Tom1 plays a broad role in PQC of unassembled ribosomal proteins as suggested by the experiments shown in Figure 3&#8212;figure supplement 1A and B, perhaps it accounts for the frequent recovery of ribosomal proteins in prior global ubiquitin conjugate profiling efforts. To address this possibility, we performed quantitative GlyGly profiling of tom1&#8710; and TOM1 cells using SILAC (Figure 3&#8212;figure supplement 1C) to identify changes in the level of ubiquitination of specific lysines that occur upon loss of Tom1. Analysis of three biological replicates (Figure 3A and Figure 3&#8212;figure supplement 2A and Supplementary file 3B) revealed 1980 unique ubiquitination sites in 920 distinct proteins, of which 972 unique sites in 532 proteins were quantified.'},\n",
       " {'pmcid': 'PMC5026473',\n",
       "  'pmid': '27552055',\n",
       "  'snippet': 'A phenotype of tom1CA cells that we describe here that is particularly notable is that they are exquisitely sensitive to perturbations that alter the balance between production of individual ribosomal proteins or total ribosomal proteins and rRNAs. Specifically, tom1CA mutants are unable to sustain growth upon overexpression of RPL26, are sensitive to loss of the Rpl4-specific chaperone Acl4 (Stelter et al., 2015), are sensitive to reduction in expression (through deletion of one of two alleles) of a single ribosomal protein, and are hypersensitive to reduction of function in three different proteins involved in transcription by RNA polymerase I. Moreover, tom1 mutants accumulate high levels of insoluble ribosomal proteins (which is consistent with their inability to degrade excess ribosomal proteins that may exist at any given point in time), and combining tom1 with a mutation that causes an imbalance in ribosome production (e.g.'},\n",
       " {'pmcid': 'PMC5026473',\n",
       "  'pmid': '27552055',\n",
       "  'snippet': 'Cleared lysate of Acl4-FLAGRpl4, Acl4-FLAGRpl4&#8710;ext expression was applied to a Ni-NTA column equilibrated in buffer containing 20 mM Tris-Base pH 8.0, 500 mM NaCl, and 5 mM &#946;-mercaptoethanol and eluted via an imidazole gradient. Pooled fractions were cleaved with ubiquitin-like-specific protease 1 (Ulp1) for 12 hr. Cleaved proteins were bound to a HiTrapQ HP (GE Healthcare) column equilibrated in buffer containing 20 mM Tris-Base pH 8.0, 100 mM NaCl, and 5 mM DTT and eluted via a linear NaCl gradient, concentrated, and injected onto a HiLoad 16/60 Superdex 200 column equilibrated in 20 mM Tris-Base pH 8.0, 100 mM NaCl, and 5 mM DTT.'},\n",
       " {'pmcid': 'PMC5026473',\n",
       "  'pmid': '27552055',\n",
       "  'snippet': 'In short, one aliquot of the K-&#949;-GlyGly peptide specific antibody (PTMScan Ubiquitin Remnant Motif (K-&#949;-GlyGly) Kit #5562, Limited Use License, Cell Signaling Technology) was washed with 3&#8211;1 ml aliquots of 100 mM sodium borate (pH 9.0) and cross-linked in the presence of 1 ml of DMP cross-linking solution (100 mM sodium borate, pH 8.0, 20 mM dimethyl pimelimidate, DMP) for 30 min at room temperature with gentle rotation. Antibody bound beads were pelleted after each wash by centrifugation at 2000 &#215;g for 30 s and kept on ice whenever possible.'},\n",
       " {'pmcid': 'PMC5028188',\n",
       "  'pmid': '27642710',\n",
       "  'snippet': 'The G2-Gata4 enhancer directed LacZ reporter expression in both PPF and PHMP at E10.5 and E11.5 in the mouse embryo (Figure 1A). However, no beta galactosidase activity was observed in the posterior and medial part of the septum transversum, where the liver is connected with the digestive tract (asterisk in Figure 1A), indicating that the G2 enhancer is not active in this specific domain.10.7554/eLife.16009.002Figure 1.Cell lineages in the septum transversum (ST), posthepatic mesenchymal plate (PHMP) and pleuroperitoneal folds (PPF). (A) G2-LacZ embryo at the stage E11.5.'},\n",
       " {'pmcid': 'PMC5028188',\n",
       "  'pmid': '27642710',\n",
       "  'snippet': 'The G2-Gata4LacZ and G2-Gata4Cre transgenic lines have been previously described (Rojas et al., 2005; Delgado et al., 2014). The Tg(Wt1-cre)#Jbeb (Wt1Cre; MGI:5308608) mouse has been used in previous studies to trace or delete specific genes in WT1-expressing cells (del Monte et al., 2011; Wessels et al., 2012; Cano et al., 2013; Carmona et al., 2013; Casanova et al., 2013; Cano et al., 2016). For lineage tracing studies, heterozygote G2-Gata4Cre+/- mice and homozygote Wt1Cre+/+ were crossed with Rosa26R (B6.129S4-Gt(ROSA)26Sortm1Sor/J, RRID:IMSR_JAX:003474) and Rosa26EYFP (B6.129X1-Gt(ROSA)26Sortm1(EYFP)&#160;Cos/J, RRID:IMSR_JAX:006148) mice to generate permanent reporter expression in G2-Gata4 and Wt1-expressing cells.'},\n",
       " {'pmcid': 'PMC5028190',\n",
       "  'pmid': '27642785',\n",
       "  'snippet': 'In C. elegans these foraging behaviours are comparatively simple, consisting of a series of forward or backward movements, specific turns, or changes in direction&#160;(Gray et al., 2005; Piggott et al., 2011; Chen et al., 2013; Hendricks, 2015). Food availability has been demonstrated to affect various aspects of these key elements in C. elegans locomotion&#160;(Sawin et al., 2000; Gray et al., 2005;&#160;Chalasani et al., 2007; Flavell et al., 2013).'},\n",
       " {'pmcid': 'PMC5028190',\n",
       "  'pmid': '27642785',\n",
       "  'snippet': 'Unlike most other organisms studied to date, C. elegans mutants that lack all AMPK signalling are viable, but show clear phenotypes when subjected to energy stress (Narbonne and Roy, 2006; 2009). Therefore, using calcium imaging, cell type&#8211;specific optogenetic techniques, and classic genetic analysis, we identified and characterized the neural circuit in which AMPK functions as a molecular switch. This circuit includes the AIB and AIY interneurons; two neurons that form one of the primary circuits that dictate appropriate food- and odour-evoked behaviours (Gray et al., 2005; Chalasani et al., 2007).'},\n",
       " {'pmcid': 'PMC5028190',\n",
       "  'pmid': '27642785',\n",
       "  'snippet': 'We discovered that AMPK modulates AIB and AIY activity through two distinct mechanisms to ultimately ensure that adaptive foraging behaviours are appropriately triggered during periods of starvation. In the AIB interneurons AMPK regulates the abundance of the &#945;-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptor GLR-1 in the postsynaptic elements, presumably by phosphorylation of serine 907 and 924 resulting in changes in synaptic strength and specific behavioural outputs. In addition, we also demonstrate that AMPK modulates a key metabotropic glutamate receptor called MGL-1 in the AIY interneuron at both mRNA and protein levels leading to an increase in AIY neuronal activity in starved animals.'},\n",
       " {'pmcid': 'PMC5028190',\n",
       "  'pmid': '27642785',\n",
       "  'snippet': 'We and others found that removal of aak-2, the more prominent of two catalytic subunits of AMPK present in C. elegans, disrupts most cellular AMPK signalling and affects various aspects of C. elegans foraging behaviour (Lee et al., 2008;&#160;Cunningham et al., 2014). More specifically, aak-2 mutants exhibit a behavioural profile in response to acute starvation that is more typical of satiated animals, where animals tend to explore locally rather than foraging in more distant locations. This is reflected by the frequency of body bends; which is considered as a representative readout for locomotory movement away from a nutrient-depleted environment in search of new resources, and reversals; a behaviour most often associated with local exploration (Gray et al., 2005).'},\n",
       " {'pmcid': 'PMC5028190',\n",
       "  'pmid': '27642785',\n",
       "  'snippet': 'To discern whether aak-2 is specifically required in neurons, muscle, or both tissues to modulate distal exploratory behaviours we used tissue-specific promoters to drive aak-2 expression and test its ability to correct the behavioural defects typical of AMPK mutants. While an aak-2 cDNA (Narbonne and Roy, 2009) expressed under the control of a pan neuronal promoter [Punc-119::aak-2] rescued the distal exploratory defect of aak-2 mutants, aak-2 mutants continued to exhibit prolonged local exploration when the same aak-2 cDNA was reconstituted in body wall muscle&#160;using&#160;[Punc-54::aak-2] transgene (Figure 1C).'},\n",
       " ...]"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "outputs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "11342"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(outputs)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Step 3**  \n",
    "Write outputs to file ```.tsv```  \n",
    "The pattern is ```PMID\\tPMCID\\tSnippet\\tAntibody related?\\tSpecificity?\\n```    \n",
    "In which antibody related? and specificity? are empty."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "file = open('train_ex_antibody.tsv', 'a')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "writing to file : 100%|██████████| 11342/11342 [00:00<00:00, 327936.61it/s]\n"
     ]
    }
   ],
   "source": [
    "for article_index in trange(len(outputs), desc='writing to file '):\n",
    "    file.write('%s\\t%s\\t%s\\t\\t\\t\\n' % (outputs[article_index]['pmid'], \n",
    "                                       outputs[article_index]['pmcid'], \n",
    "                                       outputs[article_index]['snippet']))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "file.close()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
